{{short description|Organic compounds containing amine and carboxylic groups}}
{{About|the class of chemicals|the structures and properties of the standard proteinogenic amino acids|Proteinogenic amino acid}}
{{Pp-move-indef}}
{{Use dmy dates|date=October 2020}} 
[[File:L-amino acid structure.svg|thumb|Structure of a typical <small>L</small>-alpha-amino acid in the "neutral" form]]

'''Amino acids''' are [[organic compound]]s that contain both [[amino]] and [[carboxylic acid]] [[functional group]]s.<ref>{{Lehninger4th | name-list-style = vanc }}</ref> Although over 500 amino acids exist in nature, by far the most important are the [[Proteinogenic amino acid|22 α-amino acids]] incorporated into [[protein]]s.<ref>{{cite journal | title = Norine: update of the nonribosomal peptide resource |first1 =Areski |last1 = Flissi |first2 = Emma |last2= Ricart |first3 = Clémentine |last3= Campart |first4 = Mickael |last4= Chevalier |first5 = Yoann |last5= Dufresne |first6 = Juraj |last6= Michalik |first7 = Philippe |last7= Jacques |first8 = Christophe |last8= Flahaut |first9 = Frédérique |last9= Lisacek |first10 = Valérie |last10= Leclère |first11 = Maude |last11= Pupin | journal =Nucleic Acids Research | volume = 48| issue = D1 | year = 2020 | pages= D465–D469 |doi = 10.1093/nar/gkz1000| pmid =31691799 |pmc = 7145658 }}</ref><!--<ref>{{cite journal |title = New Naturally Occurring Amino Acids|vauthors = Wagner I, Musso H|doi = 10.1002/anie.198308161|journal = [[Angewandte Chemie International Edition in English]]|volume = 22|issue = 11|pages = 816–828|date = November 1983}}{{Closed access}}</ref>--> Only these 22 appear in the [[genetic code]] of life.<ref>{{Cite web |year=2009 |editor=Richard Cammack |title=Newsletter 2009 |url=http://www.chem.qmul.ac.uk/iubmb/newsletter/2009.html#item35 |url-status=dead |archive-url=https://web.archive.org/web/20170912194130/http://www.chem.qmul.ac.uk/iubmb/newsletter/2009.html#item35 |archive-date=2017-09-12 |access-date=2012-04-16 |publisher=Biochemical Nomenclature Committee of IUPAC and NC-IUBMB |at=Pyrrolysine}}</ref><ref name="pmid20847933">{{Cite journal |last1=Rother |first1=Michael |last2=Krzycki |first2=Joseph A. |date=2010-01-01 |title=Selenocysteine, Pyrrolysine, and the Unique Energy Metabolism of Methanogenic Archaea |journal=Archaea |volume=2010 |pages=1–14 |doi=10.1155/2010/453642 |issn=1472-3646 |pmc=2933860 |pmid=20847933 |doi-access=free}}</ref>

Amino acids can be classified according to the locations of the core structural functional groups ([[Alpha and beta carbon|alpha- <span style="white-space: nowrap">(α-)</span>, beta- <span style="white-space: nowrap">(β-)</span>, gamma- <span style="white-space: nowrap">(γ-)</span>]] amino acids, etc.), other categories relate to [[Chemical polarity|polarity]], [[ionization]], and side chain group type ([[aliphatic]], [[Open-chain compound|acyclic]], [[aromatic]], [[Chemical polarity|polar]], etc.). In the form of proteins, amino acid ''[[Residue (chemistry)#Biochemistry|residues]]'' form the second-largest component ([[water]] being the largest) of human [[muscle]]s and other [[tissue (biology)|tissues]].<ref>{{cite book|title = Human nutrition in the developing world|last = Latham|first = Michael C.|name-list-style = vanc|publisher = Food and Agriculture Organization of the United Nations|year = 1997|location = Rome|chapter = Chapter 8. Body composition, the functions of food, metabolism and energy|chapter-url = http://www.fao.org/docrep/W0073E/w0073e04.htm#P1625_217364|series = Food and Nutrition Series – No. 29|access-date = 9 September 2012|archive-date = 8 October 2012|archive-url = https://web.archive.org/web/20121008212843/http://www.fao.org/docrep/W0073e/w0073e04.htm#P1625_217364|url-status = dead}}</ref> Beyond their role as residues in proteins, amino acids participate in a number of processes such as [[neurotransmitter]] transport and [[biosynthesis]]. It is thought that they played a key role in [[abiogenesis|enabling life on Earth and its emergence]].

Amino acids are formally named by the [[International Union of Pure and Applied Chemistry|IUPAC]]-[[International Union of Biochemistry and Molecular Biology|IUBMB]] Joint Commission on Biochemical Nomenclature in terms of the fictitious "neutral" structure shown in the illustration. For example, the systematic name of alanine is 2-aminopropanoic acid, based on the formula {{chem2|CH3\sCH(NH2)\sCOOH}}. The Commission justified this approach as follows:<ref name = iupaciub>{{cite web | url = http://www.chem.qmul.ac.uk/iupac/AminoAcid/AA1n2.html | title = Nomenclature and Symbolism for Amino Acids and Peptides | publisher = IUPAC-IUB Joint Commission on Biochemical Nomenclature | year = 1983 | access-date = 17 November 2008 | archive-url = https://web.archive.org/web/20081009023202/http://www.chem.qmul.ac.uk/iupac/AminoAcid/AA1n2.html | archive-date = 9 October 2008 | url-status=dead}}</ref>

<blockquote>The systematic names and formulas given refer to hypothetical forms in which amino groups are unprotonated and carboxyl groups are undissociated. This convention is useful to avoid various nomenclatural problems but should not be taken to imply that these structures represent an appreciable fraction of the amino-acid molecules.</blockquote>

==History==

The first few amino acids were discovered in the early 1800s.<ref>{{cite journal | vauthors = Vickery HB, Schmidt CL | year = 1931 | title = The history of the discovery of the amino acids | journal = Chem. Rev. | volume = 9 | issue = 2| pages = 169–318 | doi=10.1021/cr60033a001}}</ref><ref>{{cite web |last=Hansen |first=Sabine |name-list-style=vanc |title=Die Entdeckung der proteinogenen Aminosäuren von 1805 in Paris bis 1935 in Illinois |location=Berlin |date=May 2015 |url=https://www.arginium.de/wp-content/uploads/2015/12/Entdeckung-der-Aminos%C3%A4uren.pdf |archive-url=https://web.archive.org/web/20171201232937/https://www.arginium.de/wp-content/uploads/2015/12/Entdeckung-der-Aminos%C3%A4uren.pdf |archive-date=1 December 2017 |language=de}}</ref> In 1806, French chemists [[Louis-Nicolas Vauquelin]] and [[Pierre Jean Robiquet]] isolated a compound from [[asparagus]] that was subsequently named [[asparagine]], the first amino acid to be discovered.<ref>{{Cite journal|title=The discovery of a new plant principle in Asparagus sativus |vauthors=Vauquelin LN, Robiquet PJ |journal=Annales de Chimie |year=1806 |volume=57 |pages=88–93}}</ref><ref name=Anfinsen>{{Cite book |title=Advances in Protein Chemistry |vauthors=Anfinsen CB, Edsall JT, Richards FM |year=1972 |pages=[https://archive.org/details/advancesinprotei26anfi/page/99 99, 103] |publisher=Academic Press |location=New York |isbn=978-0-12-034226-6 |url=https://archive.org/details/advancesinprotei26anfi/page/99 }}</ref> [[Cystine]] was discovered in 1810,<ref>{{Cite journal|title=On cystic oxide, a new species of urinary calculus | vauthors = Wollaston WH |s2cid=110151163 |journal=Philosophical Transactions of the Royal Society |year=1810 |volume=100|pages=223–230 |doi=10.1098/rstl.1810.0015}}</ref> although its monomer, [[cysteine]], remained undiscovered until 1884.<ref>{{Cite journal |title=Über cystin und cystein | vauthors = Baumann E |journal=Z Physiol Chem |year=1884 |volume=8 |issue=4 |pages=299–305 |url=http://vlp.mpiwg-berlin.mpg.de/library/data/lit16533 |access-date=28 March 2011 |archive-url=https://web.archive.org/web/20110314075450/http://vlp.mpiwg-berlin.mpg.de/library/data/lit16533 |archive-date=14 March 2011 |url-status=dead }}</ref><ref name=Anfinsen/>{{efn|The late discovery is explained by the fact that cysteine becomes oxidized to cystine in air.}} [[Glycine]] and [[leucine]] were discovered in 1820.<ref>{{Cite journal|title=Sur la conversion des matières animales en nouvelles substances par le moyen de l'acide sulfurique | vauthors = Braconnot HM |journal=Annales de Chimie et de Physique |series=2nd Series |year=1820 |volume=13 |pages=113–125}}</ref> The last of the 20 common amino acids to be discovered was [[threonine]] in 1935 by [[William Cumming Rose]], who also determined the [[essential amino acid]]s and established the minimum daily requirements of all amino acids for optimal growth.<ref>{{cite journal|vauthors = Simoni RD, Hill RL, Vaughan M|title = The discovery of the amino acid threonine: the work of William C. Rose [classical article]|journal = The Journal of Biological Chemistry|volume = 277|issue = 37|pages = E25|date = September 2002|doi = 10.1016/S0021-9258(20)74369-3|pmid = 12218068|url = http://www.jbc.org/content/277/37/e25|doi-access = free|access-date = 4 July 2015|archive-date = 10 June 2019|archive-url = https://web.archive.org/web/20190610150715/http://www.jbc.org/content/277/37/e25|url-status = live}}</ref><ref>{{cite journal|title=Feeding Experiments with Mixtures of Highly Purified Amino Acids. VIII. Isolation and Identification of a New Essential Amino Acid|vauthors = McCoy RH, Meyer CE, Rose WC|year = 1935|journal = Journal of Biological Chemistry|volume = 112|pages = 283–302|doi = 10.1016/S0021-9258(18)74986-7|doi-access = free}}</ref>

The unity of the chemical category was recognized by [[Charles Adolphe Wurtz|Wurtz]] in 1865, but he gave no particular name to it.<ref>Menten, P. ''Dictionnaire de chimie: Une approche étymologique et historique''. De Boeck, Bruxelles. [https://books.google.com/books?id=NKTKDgAAQBAJ link] {{Webarchive|url=https://web.archive.org/web/20191228193229/https://books.google.com/books?id=NKTKDgAAQBAJ |date=28 December 2019 }}.</ref> The first use of the term "amino acid" in the English language dates from 1898,<ref>{{cite web |url=https://www.etymonline.com/word/amino- |last=Harper |first=Douglas |name-list-style=vanc |work=Online Etymology Dictionary |title=amino- |access-date=19 July 2010 |archive-date=2 December 2017 |archive-url=https://web.archive.org/web/20171202102757/https://www.etymonline.com/word/amino- |url-status=live }}</ref> while the German term, {{lang|de|Aminosäure}}, was used earlier.<ref>{{cite journal | vauthors = Paal C | year = 1894 | title = Ueber die Einwirkung von Phenyl-i-cyanat auf organische Aminosäuren | journal = Berichte der Deutschen Chemischen Gesellschaft | volume = 27 | pages = 974–979 | doi = 10.1002/cber.189402701205 | url = https://zenodo.org/record/1425732 | archive-url = https://web.archive.org/web/20200725075835/https://zenodo.org/record/1425732 | url-status = dead | archive-date = 2020-07-25 }}</ref> [[Protein]]s were found to yield amino acids after enzymatic digestion or acid [[hydrolysis]]. In 1902, [[Hermann Emil Fischer|Emil Fischer]] and [[Franz Hofmeister]] independently proposed that proteins are formed from many amino acids, whereby bonds are formed between the amino group of one amino acid with the carboxyl group of another, resulting in a linear structure that Fischer termed "[[peptide]]".<ref>{{cite book | first = Joseph S. | last = Fruton | name-list-style = vanc | title = Contrasts in Scientific Style: Research Groups in the Chemical and Biochemical Sciences |volume=191 |year=1990 |chapter=Chapter 5- Emil Fischer and Franz Hofmeister |chapter-url=https://books.google.com/books?id=tRlC9NyNNN8C&pg=PA163 |pages=163–165 |publisher=American Philosophical Society |isbn=978-0-87169-191-0 }}</ref>

==General structure==
[[File:ProteinogenicAminoAcids.svg|thumb|600x600px|The 21 [[Proteinogenic amino acid|proteinogenic α-amino acids]] found in [[Eukaryote|eukaryotes]], grouped according to their side chains' [[PKa|p''K''<sub>a</sub>]] values and charges carried at [[PH#Living systems|physiological pH (7.4)]]]]
'''2-''', '''alpha-''', or '''α-amino acids'''<ref>{{cite web|url=http://www.merriam-webster.com/medical/alpha-amino%20acid|title=Alpha amino acid|work=Merriam-Webster Medical |access-date=3 January 2015|archive-date=3 January 2015|archive-url=https://web.archive.org/web/20150103191856/http://www.merriam-webster.com/medical/alpha-amino%20acid|url-status=live}}.</ref> have the generic [[Chemical formula|formula]] {{chem2|H2NCHRCOOH}} in most cases,{{efn|[[Proline]] and other cyclic amino acids are an exception to this general formula. Cyclization of the α-amino acid creates the corresponding secondary amine. These are occasionally referred to as [[imino acid]]s.}} where R is an [[organic chemistry|organic]] [[substituent]] known as a "[[Substituent|side chain]]".<ref>{{Cite web|title = An introduction to amino acids|url = http://www.chemguide.co.uk/organicprops/aminoacids/background.html|website = chemguide|access-date = 4 July 2015|date = August 2007|last = Clark|first = Jim|archive-date = 30 April 2015|archive-url = https://web.archive.org/web/20150430051143/http://www.chemguide.co.uk/organicprops/aminoacids/background.html|url-status = live}}</ref> 

Of the many hundreds of described amino acids, 22 are [[Proteinogenic amino acid|proteinogenic]] ("protein-building").<ref>{{cite encyclopedia|year = 2008|title = Amino acids|encyclopedia = Peptides from A to Z: A Concise Encyclopedia|url = https://books.google.com/books?id=doe9NwgJTAsC&pg=PA20|publisher = Wiley-VCH|location = Germany|isbn = 9783527621170|via = Google Books|page = 20|last1 = Jakubke|first1 = Hans-Dieter|last2 = Sewald|first2 = Norbert|name-list-style = vanc|access-date = 5 January 2016|archive-date = 17 May 2016|archive-url = https://web.archive.org/web/20160517144350/https://books.google.com/books?id=doe9NwgJTAsC&pg=PA20|url-status = live}}</ref><ref>{{cite book |editor1-first = Loredano|editor1-last = Pollegioni|editor2-first = Stefano|editor2-last = Servi | name-list-style = vanc | title = Unnatural Amino Acids: Methods and Protocols|year = 2012|publisher = Humana Press|isbn = 978-1-61779-331-8|page = v|oclc = 756512314|series = Methods in Molecular Biology |volume=794|doi = 10.1007/978-1-61779-331-8|s2cid = 3705304 }}</ref><ref>{{cite journal | vauthors = Hertweck C | title = Biosynthesis and Charging of Pyrrolysine, the 22nd Genetically Encoded Amino Acid| journal = [[Angewandte Chemie International Edition]] | volume = 50 | issue = 41 | pages = 9540–9541 | date = October 2011 | pmid = 21796749 | doi = 10.1002/anie.201103769 | s2cid = 5359077}}{{Closed access}}
</ref> It is these 22 compounds that combine to give a vast array of peptides and proteins assembled by [[Ribosome|ribosomes]].<ref name="NIGMS">{{cite web|url = https://publications.nigms.nih.gov/structlife/chapter1.html|title = Chapter 1: Proteins are the Body's Worker Molecules|publisher = National Institute of General Medical Sciences|access-date = 20 May 2008|date = 27 October 2011|website = The Structures of Life|archive-date = 7 June 2014|archive-url = https://web.archive.org/web/20140607084902/https://publications.nigms.nih.gov/structlife/chapter1.html|url-status = dead}}</ref> Non-proteinogenic or modified amino acids may arise from [[post-translational modification]] or during [[nonribosomal peptide]] synthesis.

===Chirality===
The [[carbon]] atom next to the [[carboxyl group]] is called the [[alpha carbon|α–carbon]]. In proteinogenic amino acids, it bears the amine and the R group or [[Substituent|side chain]] specific to each amino acid. With four distinct substituents, the α–carbon is [[stereogenic]] in all α-amino acids except glycine. All chiral proteogenic amino acids have the <small>L</small> configuration. They are "left-handed" [[enantiomer]]s, which refers to the [[stereoisomers]] of the alpha carbon.

A few <small>D</small>-amino acids ("right-handed") have been found in nature, e.g., in [[bacterial envelope]]s, as a [[Neuromodulation|neuromodulator]] (<small>D</small>-[[serine]]), and in some [[antibiotic]]s.<ref>{{cite book | title = Biochemical Pathways: An Atlas of Biochemistry and Molecular Biology | publisher = Wiley-Blackwell | year = 2012 | isbn = 978-0-470-14684-2 | location = Oxford | editor-last = Michal | editor-first = Gerhard | editor2-last = Schomburg | editor2-first = Dietmar | name-list-style = vanc | page = 5 | edition = 2nd }}</ref><ref name="Creighton">{{Cite book |last=Creighton |first=Thomas H. |name-list-style=vanc |title=Proteins: structures and molecular properties |publisher=W. H. Freeman |location=San Francisco |year=1993 |chapter=Chapter 1 |isbn=978-0-7167-7030-5 |chapter-url-access=registration |chapter-url=https://archive.org/details/proteinsstructur0000crei }}</ref> Rarely, [[D-Amino acid|<small>D</small>-amino acid residues]] are found in proteins, and are converted from the <small>L</small>-amino acid as a [[post-translational modification]].<ref>{{Cite journal |last=Genchi |first=Giuseppe |date=2017-09-01 |title=An overview on d-amino acids |url=https://doi.org/10.1007/s00726-017-2459-5 |journal=Amino Acids |language=en |volume=49 |issue=9 |pages=1521–1533 |doi=10.1007/s00726-017-2459-5 |pmid=28681245 |s2cid=254088816 |issn=1438-2199}}</ref>{{efn|The <small>L</small> and <small>D</small> convention for amino acid configuration refers not to the optical activity of the amino acid itself but rather to the optical activity of the isomer of [[glyceraldehyde]] from which that amino acid can, in theory, be synthesized (<small>D</small>-glyceraldehyde is dextrorotatory; <small>L</small>-glyceraldehyde is levorotatory).

An alternative convention is to use the [[Cahn–Ingold–Prelog priority rules|(''S'') and (''R'') designators]] to specify the ''absolute configuration''.<ref name=Cahn>{{Cite journal | author = Cahn, R.S. | author-link = Robert Sidney Cahn | author2 = Ingold, C.K. | author2-link = Christopher Kelk Ingold | author3 = Prelog, V. | author3-link = Vladimir Prelog | title = Specification of Molecular Chirality | journal = [[Angewandte Chemie International Edition]] | volume = 5 | issue = 4 | pages = 385–415 | year = 1966 | doi = 10.1002/anie.196603851}}</ref> Almost all of the amino acids in proteins are (''S'') at the α carbon, with [[cysteine]] being (''R'') and glycine non-[[Chirality (chemistry)|chiral]].<ref>{{cite web | last = Hatem | first = Salama Mohamed Ali |name-list-style=vanc | year = 2006 | url = http://geb.uni-giessen.de/geb/volltexte/2006/3038/index.html | title = Gas chromatographic determination of Amino Acid Enantiomers in tobacco and bottled wines | publisher = University of Giessen | access-date = 17 November 2008 | archive-url = https://web.archive.org/web/20090122104055/http://geb.uni-giessen.de/geb/volltexte/2006/3038/index.html | archive-date = 22 January 2009 | url-status = dead }}</ref> Cysteine has its side chain in the same geometric location as the other amino acids, but the ''R''/''S'' terminology is reversed because [[sulfur]] has higher atomic number compared to the carboxyl oxygen which gives the side chain a higher priority by the [[Cahn–Ingold–Prelog priority rules|Cahn-Ingold-Prelog sequence rules]].}}

===Side chains===

==== Polar charged side chains ====

Five amino acids possess a charge at neutral pH. Often these side chains appear at the surfaces on proteins to enable their solubility in water, and side chains with opposite charges form important electrostatic contacts called [[Salt bridge (protein and supramolecular)|salt bridges]] that maintain structures within a single protein or between interfacing proteins.<ref name="Garrett-2010">{{Cite book |last1=Garrett |first1=Reginald H. |url=https://www.worldcat.org/oclc/297392560 |title=Biochemistry |last2=Grisham |first2=Charles M. |date=2010 |publisher=Brooks/Cole, Cengage Learning |isbn=978-0-495-10935-8 |edition=4th |location=Belmont, CA |pages=74,134-176,430-442 |oclc=297392560}}</ref> Many proteins bind metal into their structures specifically, and these interactions are commonly mediated by charged side chains such as [[aspartate]], [[glutamate]] and [[histidine]]. Under certain conditions, each ion-forming group can be charged, forming double salts.<ref>{{Cite journal |last1=Novikov |first1=Anton P. |last2=Safonov |first2=Alexey V. |last3=German |first3=Konstantin E. |last4=Grigoriev |first4=Mikhail S. |date=2023-12-01 |title=What kind of interactions we may get moving from zwitter to "dritter" ions: C–O⋯Re(O4) and Re–O⋯Re(O4) anion⋯anion interactions make structural difference between L-histidinium perrhenate and pertechnetate |url=https://pubs.rsc.org/en/content/articlelanding/2024/ce/d3ce01164j |journal=CrystEngComm |volume=26 |pages=61–69 |language=en |doi=10.1039/D3CE01164J |s2cid=265572280 |issn=1466-8033}}</ref>

The two negatively charged amino acids at neutral pH are [[Aspartic acid|aspartate]] (Asp, D) and [[Glutamic acid|glutamate]] (Glu, E). The anionic carboxylate groups behave as [[Brønsted–Lowry acid–base theory|Brønsted bases]] in most circumstances.<ref name="Garrett-2010" /> Enzymes in very low pH environments, like the aspartic protease [[pepsin]] in mammalian stomachs, may have catalytic aspartate or glutamate residues that act as Brønsted acids.

[[File:Histidine lysine arginine sidechains.png|thumb|450x450px|Functional groups found in histidine (left), lysine (middle) and arginine (right)]]

There are three amino acids with side chains that are cations at neutral pH: [[arginine]] (Arg, R), [[lysine]] (Lys, K) and [[histidine]] (His, H). Arginine has a charged [[Guanidine|guanidino]] group and lysine a charged alkyl amino group, and are fully protonated at pH 7. Histidine's imidazole group has a pK<sub>a</sub> of 6.0, and is only around 10 % protonated at neutral pH. Because histidine is easily found in its basic and conjugate acid forms it often participates in catalytic proton transfers in enzyme reactions.<ref name="Garrett-2010" />

==== Polar uncharged side chains ====

The polar, uncharged amino acids [[serine]] (Ser, S), [[threonine]] (Thr, T), [[asparagine]] (Asn, N) and [[glutamine]] (Gln, Q) readily form hydrogen bonds with water and other amino acids.<ref name="Garrett-2010" /> They do not ionize in normal conditions, a prominent exception being the catalytic serine in [[Serine protease#Catalytic mechanism|serine proteases]]. This is an example of severe perturbation, and is not characteristic of serine residues in general. Threonine has two chiral centers, not only the <small>L</small> (2''S'') chiral center at the α-carbon shared by all amino acids apart from achiral glycine, but also (3''R'') at the β-carbon. The full stereochemical specification is (2''S'',3''R'')-<small>L</small>-threonine.

==== Hydrophobic side chains ====

Nonpolar amino acid interactions are the primary driving force behind the processes that [[Protein folding|fold proteins]] into their functional three dimensional structures.<ref name="Garrett-2010" /> None of these amino acids' side chains ionize easily, and therefore do not have pK<sub>a</sub>s, with the exception of [[tyrosine]] (Tyr, Y). The hydroxyl of tyrosine can deprotonate at high pH forming the negatively charged phenolate. Because of this one could place tyrosine into the polar, uncharged amino acid category, but its very low solubility in water matches the characteristics of hydrophobic amino acids well.

==== Special case side chains ====

Several side chains are not described well by the charged, polar and hydrophobic categories. [[Glycine]] (Gly, G) could be considered a polar amino acid since its small size means that its solubility is largely determined by the amino and carboxylate groups. However, the lack of any side chain provides glycine with a unique flexibility among amino acids with large ramifications to protein folding.<ref name="Garrett-2010" /> [[Cysteine]] (Cys, C) can also form hydrogen bonds readily, which would place it in the polar amino acid category, though it can often be found in protein structures forming covalent bonds, called [[disulphide bonds]], with other cysteines. These bonds influence the folding and stability of proteins, and are essential in the formation of [[Antibody#Structure|antibodies]]. [[Proline]] (Pro, P) has an alkyl side chain and could be considered hydrophobic, but because the side chain joins back onto the alpha amino group it becomes particularly inflexible when incorporated into proteins. Similar to glycine this influences protein structure in a way unique among amino acids. [[Selenocysteine]] (Sec, U) is a rare amino acid not directly encoded by DNA, but is incorporated into proteins via the ribosome. Selenocysteine has a lower redox potential compared to the similar cysteine, and participates in several unique enzymatic reactions.<ref>{{Cite journal |last1=Papp |first1=Laura Vanda |last2=Lu |first2=Jun |last3=Holmgren |first3=Arne |last4=Khanna |first4=Kum Kum |date=2007-07-01 |title=From Selenium to Selenoproteins: Synthesis, Identity, and Their Role in Human Health |url=https://www.liebertpub.com/doi/10.1089/ars.2007.1528 |journal=Antioxidants & Redox Signaling |volume=9 |issue=7 |pages=775–806 |doi=10.1089/ars.2007.1528 |pmid=17508906 |issn=1523-0864}}</ref> [[Pyrrolysine]] (Pyl, O) is another amino acid not encoded in DNA, but synthesized into protein by ribosomes.<ref>{{Cite journal |last1=Hao |first1=Bing |last2=Gong |first2=Weimin |last3=Ferguson |first3=Tsuneo K. |last4=James |first4=Carey M. |last5=Krzycki |first5=Joseph A. |last6=Chan |first6=Michael K. |date=2002-05-24 |title=A New UAG-Encoded Residue in the Structure of a Methanogen Methyltransferase |url=https://www.science.org/doi/10.1126/science.1069556 |journal=Science |language=en |volume=296 |issue=5572 |pages=1462–1466 |doi=10.1126/science.1069556 |pmid=12029132 |bibcode=2002Sci...296.1462H |s2cid=35519996 |issn=0036-8075}}</ref> It is found in archaeal species where it participates in the catalytic activity of several methyltransferases.

==== β- and γ-amino acids ====
Amino acids with the structure {{chem2|NH3+\sCXY\sCXY\sCO2-}}, such as [[β-alanine]], a component of [[carnosine]] and a few other peptides, are β-amino acids. Ones with the structure {{chem2|NH3+\sCXY\sCXY\sCXY\sCO2-}} are γ-amino acids, and so on, where X and Y are two substituents (one of which is normally H).<ref name = iupaciub />

===Zwitterions===
<!--[[File:Amino acid zwitterions.svg|thumb|right|An amino acid in its (1) molecular and (2) zwitterionic forms]]-->
{{main|Zwitterion}}

[[File:Bronsted_character_of_ionizing_groups_in_proteins.png|thumb|upright=1.8|Ionization and Brønsted character of N-terminal amino, C-terminal carboxylate, and side chains of amino acid residues]]
The common natural forms of amino acids have a [[zwitterionic]] structure, with {{chem2|\sNH3+}} ({{chem2|\sNH2+\s}} in the case of proline) and {{chem2|\sCO2-}} functional groups attached to the same C atom, and are thus α-amino acids, and are the only ones found in proteins during translation in the ribosome.
In aqueous solution at pH close to neutrality, amino acids exist as [[zwitterion]]s, i.e. as dipolar ions with both {{chem2|NH3+}} and {{chem2|CO2-}} in charged states, so the overall structure is {{chem2|NH3+\sCHR\sCO2-}}. At [[Acid–base homeostasis|physiological pH]] the so-called "neutral forms" {{chem2|\sNH2\sCHR\sCO2H}} are not present to any measurable degree.<ref>{{cite book | last1 = Steinhardt | first1 = J. | first2 = J. A. | last2 = Reynolds | title = Multiple equilibria in proteins | publisher = Academic Press | place= New York | isbn = 978-0126654509| pages = 176–21|date = 1969
}}</ref> Although the two charges in the zwitterion structure add up to zero it is misleading to call a species with a net charge of zero "uncharged".

In strongly acidic conditions (pH below 3), the carboxylate group becomes protonated and the structure becomes an ammonio carboxylic acid, {{chem2|NH3+\sCHR\sCO2H}}. This is relevant for enzymes like pepsin that are active in acidic environments such as the mammalian stomach and [[lysosomes]], but does not significantly apply to intracellular enzymes. In highly basic conditions (pH greater than 10, not normally seen in physiological conditions), the ammonio group is deprotonated to give {{chem2|NH2\sCHR\sCO2-}}.

Although various definitions of acids and bases are used in chemistry, the only one that is useful for chemistry in aqueous solution is [[Brønsted–Lowry acid–base theory|that of Brønsted]]:<ref>{{cite journal | last1 = Brønsted | first1= J. N. | journal = Recueil des Travaux Chimiques des Pays-Bas | volume = 42 | pages = 718–728 |year= 1923| title = Einige Bemerkungen über den Begriff der Säuren und Basen| issue= 8 | doi= 10.1002/recl.19230420815 |trans-title = Remarks on the concept of acids and bases}}</ref><ref name="Vollhardt-2007" /> an acid is a species that can donate a proton to another species, and a base is one that can accept a proton. This criterion is used to label the groups in the above illustration. The carboxylate side chains of aspartate and glutamate residues are the principal Brønsted bases in proteins. Likewise, lysine, tyrosine and cysteine will typically act as a Brønsted acid. Histidine under these conditions can act both as a Brønsted acid and a base.

===Isoelectric point===
[[File:Titration Curves of 20 Amino Acids Organized by Side Chain.png|thumb|right|360px|Composite of [[titration curve]]s of twenty proteinogenic amino acids grouped by side chain category]]

For amino acids with uncharged side-chains the zwitterion predominates at pH values between the two p''K''<sub>a</sub> values, but coexists in [[Chemical equilibrium|equilibrium]] with small amounts of net negative and net positive ions. At the midpoint between the two p''K''<sub>a</sub> values, the trace amount of net negative and trace of net positive ions balance, so that average net charge of all forms present is zero.<ref>{{cite book | vauthors = Fennema OR |title=Food Chemistry 3rd Ed |publisher=CRC Press |pages=327–328 |isbn=978-0-8247-9691-4 |date=1996-06-19 }}</ref> This pH is known as the [[isoelectric point]] p''I'', so p''I'' = {{sfrac|1|2}}(p''K''<sub>a1</sub> + p''K''<sub>a2</sub>).

For amino acids with charged side chains, the p''K''<sub>a</sub> of the side chain is involved. Thus for aspartate or glutamate with negative side chains, the terminal amino group is essentially entirely in the charged form {{chem2|\sNH3+}}, but this positive charge needs to be balanced by the state with just one C-terminal carboxylate group is negatively charged. This occurs halfway between the two carboxylate p''K''<sub>a</sub> values: p''I'' = {{sfrac|1|2}}(p''K''<sub>a1</sub> + p''K''<sub>a(R)</sub>), where p''K''<sub>a(R)</sub> is the side chain p''K''<sub>a</sub>.<ref name="Vollhardt-2007">{{Cite book |last=Vollhardt |first=K. Peter C. |url=https://www.worldcat.org/oclc/61448218 |title=Organic chemistry : structure and function |date=2007 |publisher=W.H. Freeman |others=Neil Eric Schore |isbn=978-0-7167-9949-8 |edition=5th |location=New York |pages=58–66 |oclc=61448218}}</ref>

Similar considerations apply to other amino acids with ionizable side-chains, including not only glutamate (similar to aspartate), but also cysteine, histidine, lysine, tyrosine and arginine with positive side chains.

Amino acids have zero mobility in [[electrophoresis]] at their isoelectric point, although this behaviour is more usually exploited for peptides and proteins than single amino acids. Zwitterions have minimum solubility at their isoelectric point, and some amino acids (in particular, with nonpolar side chains) can be isolated by precipitation from water by adjusting the pH to the required isoelectric point.

==Physicochemical properties==
The 20 canonical amino acids can be classified according to their properties. Important factors are charge, [[hydrophilicity]] or [[hydrophobicity]], size, and functional groups.<ref name="Creighton" /> These properties influence [[protein structure]] and [[protein–protein interaction]]s. The water-soluble proteins tend to have their hydrophobic residues ([[leucine|Leu]], [[isoleucine|Ile]], [[valine|Val]], [[phenylalanine|Phe]], and [[tryptophan|Trp]]) buried in the middle of the protein, whereas hydrophilic side chains are exposed to the aqueous solvent. (In [[biochemistry]], a residue refers to a specific [[monomer]] ''within'' the [[polymer]]ic chain of a [[polysaccharide]], protein or [[nucleic acid]].) The [[integral membrane protein]]s tend to have outer rings of exposed [[hydrophobic]] amino acids that anchor them in the [[lipid bilayer]]. Some [[peripheral membrane protein]]s have a patch of hydrophobic amino acids on their surface that sticks to the membrane. In a similar fashion, proteins that have to bind to positively charged molecules have surfaces rich in negatively charged amino acids such as [[glutamate]] and [[aspartate]], while proteins binding to negatively charged molecules have surfaces rich in positively charged amino acids like [[lysine]] and [[arginine]]. For example, lysine and arginine are present in large amounts in the [[Low complexity regions in proteins|low-complexity regions]] of nucleic-acid binding proteins.<ref name="Ntountoumi-2019">{{cite journal | vauthors = Ntountoumi C, Vlastaridis P, Mossialos D, Stathopoulos C, Iliopoulos I, Promponas V, Oliver SG, Amoutzias GD | display-authors = 6 | title = Low complexity regions in the proteins of prokaryotes perform important functional roles and are highly conserved | journal = Nucleic Acids Research | volume = 47 | issue = 19 | pages = 9998–10009 | date = November 2019 | pmid = 31504783 | pmc = 6821194 | doi = 10.1093/nar/gkz730 }}</ref> There are various [[hydrophobicity scale]]s of amino acid residues.<ref>{{cite journal| vauthors = Urry DW | title = The change in Gibbs free energy for hydrophobic association: Derivation and evaluation by means of inverse temperature transitions | journal = Chemical Physics Letters | volume = 399 | issue = 1–3 | pages = 177–183 | year = 2004 | doi = 10.1016/S0009-2614(04)01565-9 | bibcode = 2004CPL...399..177U }}</ref>

Some amino acids have special properties. Cysteine can form covalent [[disulfide bond]]s to other cysteine residues. [[Proline]] forms [[cyclic compound|a cycle]] to the polypeptide backbone, and glycine is more flexible than other amino acids.

Glycine and proline are strongly present within low complexity regions of both eukaryotic and prokaryotic proteins, whereas the opposite is the case with cysteine, phenylalanine, tryptophan, methionine, valine, leucine, isoleucine, which are highly reactive, or complex, or hydrophobic.<ref name="Ntountoumi-2019" /><ref>{{cite journal | vauthors = Marcotte EM, Pellegrini M, Yeates TO, Eisenberg D | title = A census of protein repeats | journal = Journal of Molecular Biology | volume = 293 | issue = 1 | pages = 151–60 | date = October 1999 | pmid = 10512723 | doi = 10.1006/jmbi.1999.3136 }}</ref><ref>{{cite journal | vauthors = Haerty W, Golding GB | title = Low-complexity sequences and single amino acid repeats: not just "junk" peptide sequences | journal = Genome | volume = 53 | issue = 10 | pages = 753–62 | date = October 2010 | pmid = 20962881 | doi = 10.1139/G10-063 | veditors = Bonen L }}</ref>

Many proteins undergo a range of [[posttranslational modification]]s, whereby additional chemical groups are attached to the amino acid residue side chains sometimes producing [[lipoprotein]]s (that are hydrophobic),<ref>{{cite journal | vauthors = Magee T, Seabra MC | title = Fatty acylation and prenylation of proteins: what's hot in fat | journal = Current Opinion in Cell Biology | volume = 17 | issue = 2 | pages = 190–196 | date = April 2005 | pmid = 15780596 | doi = 10.1016/j.ceb.2005.02.003 }}</ref> or [[glycoprotein]]s (that are hydrophilic)<ref>{{cite journal | vauthors = Pilobello KT, Mahal LK | title = Deciphering the glycocode: the complexity and analytical challenge of glycomics | journal = Current Opinion in Chemical Biology | volume = 11 | issue = 3 | pages = 300–305 | date = June 2007 | pmid = 17500024 | doi = 10.1016/j.cbpa.2007.05.002 }}</ref> allowing the protein to attach temporarily to a membrane. For example, a signaling protein can attach and then detach from a cell membrane, because it contains cysteine residues that can have the fatty acid [[palmitic acid]] added to them and subsequently removed.<ref>{{cite journal | vauthors = Smotrys JE, Linder ME | title = Palmitoylation of intracellular signaling proteins: regulation and function | journal = Annual Review of Biochemistry | volume = 73 | issue = 1 | pages = 559–587 | year = 2004 | pmid = 15189153 | doi = 10.1146/annurev.biochem.73.011303.073954 }}</ref>

===Table of standard amino acid abbreviations and properties===
{{Redirect|Amino acid code|base-pair encoding of amino acids|Genetic code#Codons}}
{{Main|Proteinogenic amino acid}}

Although one-letter symbols are included in the table, IUPAC–IUBMB recommend<ref name = iupaciub/> that "Use of the one-letter symbols should be restricted to the comparison of long sequences".

The one-letter notation was chosen by IUPAC-IUB based on the following rules:<ref name=":0">{{Cite journal |date=10 July 1968 |title=IUPAC-IUB Commission on Biochemical Nomenclature A One-Letter Notation for Amino Acid Sequences |url=https://www.jbc.org/article/S0021-9258(19)34176-6/pdf |journal=Journal of Biological Chemistry |language=en |volume=243 |issue=13 |pages=3557–3559 |doi=10.1016/S0021-9258(19)34176-6|doi-access=free }}</ref>

* Initial letters are used where there is no ambuiguity: C cysteine, H histidine, I isoleucine, M methionine, S serine, V valine,<ref name=":0" />
* Where arbitrary assignment is needed, the structurally simpler amino acids are given precedence: A Alanine, G glycine, L leucine, P proline, T threonine,<ref name=":0" />
* F ''PH''enylalanine and R a''R''ginine are assigned by being phonetically suggestive,<ref name=":0" />
* W tryptophane is assigned based on the double ring being visually suggestive to the bulky letter W,<ref name=":0" />
* K lysine and Y tyrosine are assigned as alphabetically nearest to their initials L and T (note that U was avoided for its similarity with V, while X was reserved for undetermined or atypical amino acids); for tyrosine the mnemonic t''Y''rosine was also proposed,<ref name=":1">{{Cite journal |last=Saffran |first=M. |date=April 1998 |title=Amino acid names and parlor games: from trivial names to a one-letter code, amino acid names have strained students' memories. Is a more rational nomenclature possible? |url=http://linkinghub.elsevier.com/retrieve/pii/S0307441297001672 |journal=Biochemical Education |language=en |volume=26 |issue=2 |pages=116–118 |doi=10.1016/S0307-4412(97)00167-2}}</ref>
* D aspartate was assigned arbitrarily, with the proposed mnemonic aspar''D''ic acid;<ref name=":2">{{Cite journal |last=Adoga |first=Godwin I |last2=Nicholson |first2=Bh |date=January 1988 |title=Letters to the editor |url=https://onlinelibrary.wiley.com/doi/pdf/10.1016/0307-4412%2888%2990026-X |journal=Biochemical Education |language=en |volume=16 |issue=1 |pages=49 |doi=10.1016/0307-4412(88)90026-X}}</ref> E glutamate was assigned in alphabetical sequence being larger by merely one [[Methylene group|methylene]] –CH2– group,<ref name=":1" />
* N asparagine was assigned arbitrarily, with the proposed mnemonic asparagi''N''e;<ref name=":2" /> Q glutamine was assigned in alphabetical sequence of those still available (note again that O was avoided due to similarity with D), with the proposed mnemonic ''Q''lutamine.<ref name=":2" />

{| class="wikitable sortable" style="text-align:center;"
! rowspan=2 | Amino acid
! colspan=2 | 3- and 1-letter symbols
! colspan=3 | Side chain
! rowspan=2 | [[Hydropathy index|Hydropathy <br/>index]]<ref>{{cite journal | vauthors = Kyte J, Doolittle RF | title = A simple method for displaying the hydropathic character of a protein | journal = Journal of Molecular Biology | volume = 157 | issue = 1 | pages = 105–132 | date = May 1982 | pmid = 7108955 | doi = 10.1016/0022-2836(82)90515-0 | citeseerx = 10.1.1.458.454 }}</ref>
! colspan=2 | [[Molar absorptivity]]<ref name="Freifelder">{{Cite book| title=Physical Biochemistry| edition=2nd|vauthors = Freifelder D | publisher=W. H. Freeman and Company| isbn=978-0-7167-1315-9 | year=1983}}{{Page needed|date=September 2010}}</ref>
! rowspan=2 | [[Molecular mass]]
! rowspan=2 | Abundance in<br/>proteins (%)<ref>{{cite journal | vauthors = Kozlowski LP | title = Proteome-p''I'': proteome isoelectric point database | journal = Nucleic Acids Research | volume = 45 | issue = D1 | pages = D1112–D1116 | date = January 2017 | pmid = 27789699 | pmc = 5210655 | doi = 10.1093/nar/gkw978 }}</ref>
! rowspan=2 | Standard genetic coding,<br/>[[Nucleic acid notation#IUPAC notation|IUPAC notation]]
|-
! 3
! 1
! Class
! [[Chemical polarity]]<ref name="Hausman">{{cite book | last1 = Hausman | first1 = Robert E. | last2 = Cooper | first2 = Geoffrey M. | name-list-style = vanc |title=The cell: a molecular approach |publisher=ASM Press |location=Washington, D.C |year=2004 |page=51 |isbn=978-0-87893-214-6}}</ref>
! Net charge<br/>at pH&nbsp;7.4<ref name="Hausman" />
! Wavelength,<br/>''λ''<sub>max</sub> (nm)
! Coefficient ''ε''<br/>(mM<sup>−1</sup>·cm<sup>−1</sup>)
|-
| [[Alanine]]
| Ala
| A
| Aliphatic
| Nonpolar
| Neutral
| 1.8
| 
|
| 89.094
| 8.76
| GCN
|-
| [[Arginine]]
| Arg
| R
| Fixed cation
| Basic polar
| Positive
| −4.5
| 
|
| 174.203
| 5.78
| MGR, CGY<ref>Codons can also be expressed by: CGN, AGR</ref>
|-
| [[Asparagine]]
| Asn
| N
| Amide
| Polar
| Neutral
| −3.5
| 
|
| 132.119
| 3.93
| AAY
|-
| [[Aspartate]]
| Asp
| D
| Anion
| [[Brønsted base]]
| Negative
| −3.5
| 
|
| 133.104
| 5.49
| GAY
|-
| [[Cysteine]]
| Cys
| C
| Thiol
| Brønsted acid
| Neutral
| 2.5
| 250
| 0.3
| 121.154
| 1.38
| UGY
|-
| [[Glutamine]]
| Gln
| Q
| Amide
| Polar
| Neutral
| −3.5
|
|
| 146.146
| 3.9
| CAR
|-
| [[Glutamate]]
| Glu
| E
| Anion
| Brønsted base
| Negative
| −3.5
|
|
| 147.131
| 6.32
| GAR
|-
| [[Glycine]]
| Gly
| G
| Aliphatic
| Nonpolar
| Neutral
| −0.4
|
|
| 75.067
| 7.03
| GGN
|-
| [[Histidine]]
| His
| H
| Cationic
| Brønsted acid and base
| Positive, 10%<br/>Neutral, 90%
| −3.2
| 211
| 5.9
| 155.156
| 2.26
| CAY
|-
| [[Isoleucine]]
| Ile
| I
| Aliphatic
| Nonpolar
| Neutral
| 4.5
|
|
| 131.175
| 5.49
| AUH
|-
| [[Leucine]]
| Leu
| L
| Aliphatic
| Nonpolar
| Neutral
| 3.8
|
|
| 131.175
| 9.68
| YUR, CUY<ref>codons can also be expressed by: CUN, UUR</ref>
|-
| [[Lysine]]
| Lys
| K
| Cation
| Brønsted acid
| Positive
| −3.9
|
|
| 146.189
| 5.19
| AAR
|-
| [[Methionine]]
| Met
| M
| Thioether
| Nonpolar
| Neutral
| 1.9
|
|
| 149.208
| 2.32
| AUG
|-
| [[Phenylalanine]]
| Phe
| F
| Aromatic
| Nonpolar
| Neutral
| 2.8
| 257, 206, 188
| 0.2, 9.3, 60.0
| 165.192
| 3.87
| UUY
|-
| [[Proline]]
| Pro
| P
| Cyclic
| Nonpolar
| Neutral
| −1.6
|
|
| 115.132
| 5.02
| CCN
|-
| [[Serine]]
| Ser
| S
| Hydroxylic
| Polar
| Neutral
| −0.8
|
|
| 105.093
| 7.14
| UCN, AGY
|-
| [[Threonine]]
| Thr
| T
| Hydroxylic
| Polar
| Neutral
| −0.7
|
|
| 119.119
| 5.53
| ACN
|-
| [[Tryptophan]]
| Trp
| W
| Aromatic
| Nonpolar
| Neutral
| −0.9
| 280, 219
| 5.6, 47.0
| 204.228
| 1.25
| UGG
|-
| [[Tyrosine]]
| Tyr
| Y
| Aromatic
| Brønsted acid
| Neutral
| −1.3
| 274, 222, 193
| 1.4, 8.0, 48.0
| 181.191
| 2.91
| UAY
|-
| [[Valine]]
| Val
| V
| Aliphatic
| Nonpolar
| Neutral
| 4.2
|
|
| 117.148
| 6.73
| GUN
|}

Two additional amino acids are in some species coded for by [[codons]] that are usually interpreted as [[stop codon]]s:

{| class="wikitable" style="text-align:center;"
|-
! 21st and 22nd amino acids
! 3-letter
! 1-letter
! [[Molecular mass]]
|-
| [[Selenocysteine]]
| Sec
| U
| 168.064
|-
| [[Pyrrolysine]]
| Pyl
| O
| 255.313
|}

In addition to the specific amino acid codes, placeholders are used in cases where [[Protein sequencing|chemical]] or [[X-ray crystallography|crystallographic]] analysis of a peptide or protein cannot conclusively determine the identity of a residue. They are also used to summarize [[Conserved sequence|conserved protein sequence]] motifs. The use of single letters to indicate sets of similar residues is similar to the use of [[Nucleic acid notation|abbreviation codes for degenerate bases]].<ref>{{cite journal | vauthors = Aasland R, Abrams C, Ampe C, Ball LJ, Bedford MT, Cesareni G, Gimona M, Hurley JH, Jarchau T, Lehto VP, Lemmon MA, Linding R, Mayer BJ, Nagai M, Sudol M, Walter U, Winder SJ | title = Normalization of nomenclature for peptide motifs as ligands of modular protein domains | journal = FEBS Letters | volume = 513 | issue = 1 | pages = 141–144 | date = February 2002 | pmid = 11911894 | doi = 10.1111/j.1432-1033.1968.tb00350.x }}</ref><ref>{{cite journal | author = IUPAC–IUB Commission on Biochemical Nomenclature | title = A one-letter notation for amino acid sequences | journal = Pure and Applied Chemistry | volume = 31 | issue = 4 | pages = 641–645 | pmid = 5080161 | year = 1972 | doi = 10.1351/pac197231040639| doi-access = free }}</ref>

{| class="wikitable" style="text-align:center;"
|-
! Ambiguous amino acids
! 3-letter
! 1-letter
! Amino acids included
! Codons included
|-
| Any / unknown
| Xaa
| X
| All
| NNN
|-
| [[Asparagine]] or [[aspartate]]
| Asx
| B
| D, N
| RAY
|-
| [[Glutamine]] or [[glutamate]]
| Glx
| Z
| E, Q
| SAR
|-
| [[Leucine]] or isoleucine
| Xle
| J
| I, L
| YTR, ATH, CTY<ref>Codons can also be expressed by: CTN, ATH, TTR; MTY, YTR, ATA; MTY, HTA, YTG</ref>
|-
| [[Hydrophobic]]
|
| Φ
| V, I, L, F, W, Y, M
| NTN, TAY, TGG
|-
| [[Aromatic]]
|
| Ω
| F, W, Y, H
| YWY, TTY, TGG<ref>Codons can also be expressed by: TWY, CAY, TGG</ref>
|-
| [[Aliphatic]] (non-aromatic)
|
| Ψ
| V, I, L, M
| VTN, TTR<ref>Codons can also be expressed by: NTR, VTY</ref>
|-
| Small
|
| π
| P, G, A, S
| BCN, RGY, GGR
|-
| [[Hydrophilic]]
|
| ζ
| S, T, H, N, Q, E, D, K, R
| VAN, WCN, CGN, AGY<ref>Codons can also be expressed by: VAN, WCN, MGY, CGP</ref>
|-
| [[Cation|Positively-charged]]
|
| +
| K, R, H
| ARR, CRY, CGR
|-
| [[Anion|Negatively-charged]]
|
| −
| D, E
| GAN
|}

'''Unk''' is sometimes used instead of '''Xaa''', but is less standard.

'''Ter''' or '''*''' (from termination) is used in notation for mutations in proteins when a stop codon occurs. It corresponds to no amino acid at all.<ref name="HGSV_recommendations">{{Cite web |url=http://varnomen.hgvs.org/recommendations/protein/variant/substitution/ |title=HGVS: Sequence Variant Nomenclature, Protein Recommendations |access-date=23 September 2021 |archive-date=24 September 2021 |archive-url=https://web.archive.org/web/20210924091505/http://varnomen.hgvs.org/recommendations/protein/variant/substitution/ |url-status=live }}</ref>

In addition, many [[Non-proteinogenic amino acids|nonstandard amino acids]] have a specific code. For example, several peptide drugs, such as [[Bortezomib]] and [[MG132]], are [[peptide synthesis|artificially synthesized]] and retain their [[protecting group]]s, which have specific codes. Bortezomib is [[pyrazinoic acid|Pyz]]–Phe–boroLeu, and MG132 is [[Carboxybenzyl|Z]]–Leu–Leu–Leu–al. To aid in the analysis of protein structure, [[photo-reactive amino acid analog]]s are available. These include [[photoleucine]] ('''pLeu''') and [[photomethionine]] ('''pMet''').<ref>{{cite journal | vauthors = Suchanek M, Radzikowska A, Thiele C | title = Photo-leucine and photo-methionine allow identification of protein–protein interactions in living cells | journal = Nature Methods | volume = 2 | issue = 4 | pages = 261–267 | date = April 2005 | pmid = 15782218 | doi = 10.1038/nmeth752 | doi-access = free }}</ref>

==Occurrence and functions in biochemistry==
{{multiple image
<!-- Layout parameters -->| align             = right
| direction         = vertical
| total_width       = 340
<!-- Header -->| header_align      = <!-- center (default), left, right -->
| header            = <!--image 5-->
| image5            = Protein primary structure.svg
| alt5              = A protein depicted as a long unbranched string of linked circles each representing amino acids
| width5            = 
| height5           = 
| caption5          = A [[polypeptide]] is an unbranched chain of amino acids.

<!--image 6-->| image6            = Beta alanine comparison.svg
| alt6              = Diagrammatic comparison of the structures of β-alanine and α-alanine
| width6            = 
| height6           = 
| caption6          = β-Alanine and its α-alanine isomer

<!--image 7-->| image7            = Selenocysteine skeletal 3D.svg
| alt7              = A diagram showing the structure of selenocysteine
| width7            = 
| height7           = 
| caption7          = The amino acid [[selenocysteine]]
| image1            = 
}}

===Proteinogenic amino acids===
{{main|Proteinogenic amino acid}} {{See also|Protein primary structure|Posttranslational modification}}

Amino acids are the precursors to proteins.<ref name="NIGMS"/> They join by condensation reactions to form short polymer chains called peptides or longer chains called either polypeptides or proteins. These chains are linear and unbranched, with each amino acid residue within the chain attached to two neighboring amino acids. In nature, the process of making proteins encoded by RNA genetic material is called ''[[translation (biology)|translation]]'' and involves the step-by-step addition of amino acids to a growing protein chain by a [[ribozyme]] that is called a [[ribosome]].<ref>{{cite journal | vauthors = Rodnina MV, Beringer M, Wintermeyer W | title = How ribosomes make peptide bonds | journal = Trends in Biochemical Sciences | volume = 32 | issue = 1 | pages = 20–26 | date = January 2007 | pmid = 17157507 | doi = 10.1016/j.tibs.2006.11.007 }}</ref> The order in which the amino acids are added is read through the [[genetic code]] from an [[Messenger RNA|mRNA]] template, which is an [[RNA]] derived from one of the organism's [[gene]]s.

Twenty-two amino acids are naturally incorporated into polypeptides and are called [[proteinogenic]] or natural amino acids.<ref name="Creighton" /> Of these, 20 are encoded by the universal genetic code. The remaining 2, [[selenocysteine]] and [[pyrrolysine]], are incorporated into proteins by unique synthetic mechanisms. Selenocysteine is incorporated when the mRNA being translated includes a [[SECIS element]], which causes the UGA codon to encode selenocysteine instead of a stop codon.<ref>{{cite journal | vauthors = Driscoll DM, Copeland PR | title = Mechanism and regulation of selenoprotein synthesis | journal = Annual Review of Nutrition | volume = 23 | issue = 1 | pages = 17–40 | year = 2003 | pmid = 12524431 | doi = 10.1146/annurev.nutr.23.011702.073318 }}</ref> [[Pyrrolysine]] is used by some [[methanogen]]ic [[archaea]] in enzymes that they use to produce [[methane]]. It is coded for with the codon UAG, which is normally a stop codon in other organisms.<ref>{{cite journal | vauthors = Krzycki JA | title = The direct genetic encoding of pyrrolysine | journal = Current Opinion in Microbiology | volume = 8 | issue = 6 | pages = 706–712 | date = December 2005 | pmid = 16256420 | doi = 10.1016/j.mib.2005.10.009 }}</ref>

Several independent evolutionary studies have suggested that Gly, Ala, Asp, Val, Ser, Pro, Glu, Leu, Thr may belong to a group of amino acids that constituted the early genetic code, whereas Cys, Met, Tyr, Trp, His, Phe may belong to a group of amino acids that constituted later additions of the genetic code.<ref>{{cite journal | 
doi=10.1073/pnas.72.5.1909| 
title=A Co-Evolution Theory of the Genetic Code| 
year=1975| 
last1=Wong| 
first1=J. T.-F.| 
journal=Proceedings of the National Academy of Sciences| 
volume=72| 
issue=5| 
pages=1909–1912| 
pmid=1057181| 
pmc=432657| 
bibcode=1975PNAS...72.1909T| 
doi-access=free}}</ref><ref>{{cite journal| vauthors = Trifonov EN |date=December 2000|title=Consensus temporal order of amino acids and evolution of the triplet code |journal=Gene |volume=261|issue=1|pages=139–151|doi=10.1016/S0378-1119(00)00476-5|pmid=11164045}}</ref><ref>{{cite journal | vauthors = Higgs PG, Pudritz RE | title = A thermodynamic basis for prebiotic amino acid synthesis and the nature of the first genetic code | journal = Astrobiology | volume = 9 | issue = 5 | pages = 483–90 | date = June 2009 | pmid = 19566427 | doi = 10.1089/ast.2008.0280 | arxiv = 0904.0402 | bibcode = 2009AsBio...9..483H | s2cid = 9039622 }}</ref>

===Standard vs nonstandard amino acids===

The 20 amino acids that are encoded directly by the codons of the universal genetic code are called ''standard'' or ''canonical'' amino acids. A modified form of methionine ([[N-Formylmethionine|''N''-formylmethionine]]) is often incorporated in place of methionine as the initial amino acid of proteins in bacteria, mitochondria and [[plastid]]s (including chloroplasts). Other amino acids are called ''nonstandard'' or ''non-canonical''. Most of the nonstandard amino acids are also non-proteinogenic (i.e. they cannot be incorporated into proteins during translation), but two of them are proteinogenic, as they can be incorporated translationally into proteins by exploiting information not encoded in the universal genetic code.

The two nonstandard proteinogenic amino acids are selenocysteine (present in many non-eukaryotes as well as most eukaryotes, but not coded directly by DNA) and [[pyrrolysine]] (found only in some [[archaea]] and at least one [[bacterium]]). The incorporation of these nonstandard amino acids is rare. For example, 25 human proteins include selenocysteine in their primary structure,<ref>{{cite journal | vauthors = Kryukov GV, Castellano S, Novoselov SV, Lobanov AV, Zehtab O, Guigó R, Gladyshev VN | title = Characterization of mammalian selenoproteomes | journal = Science | volume = 300 | issue = 5624 | pages = 1439–1443 | date = May 2003 | pmid = 12775843 | doi = 10.1126/science.1083516 | bibcode = 2003Sci...300.1439K | s2cid = 10363908 | url = http://digitalcommons.unl.edu/cgi/viewcontent.cgi?article=1072&context=biochemgladyshev | access-date = 12 June 2019 | archive-date = 23 July 2018 | archive-url = https://web.archive.org/web/20180723073438/http://digitalcommons.unl.edu/cgi/viewcontent.cgi?article=1072&context=biochemgladyshev | url-status = live }}</ref> and the structurally characterized enzymes (selenoenzymes) employ selenocysteine as the catalytic [[moiety (chemistry)|moiety]] in their active sites.<ref>{{cite journal | vauthors = Gromer S, Urig S, Becker K | title = The thioredoxin system—from science to clinic | journal = Medicinal Research Reviews | volume = 24 | issue = 1 | pages = 40–89 | date = January 2004 | pmid = 14595672 | doi = 10.1002/med.10051 | s2cid = 1944741 }}</ref> Pyrrolysine and selenocysteine are encoded via variant codons. For example, selenocysteine is encoded by stop codon and [[SECIS element]].<ref name="Tjong">{{cite thesis|url=https://diginole.lib.fsu.edu/islandora/object/fsu%3A175939|last=Tjong|first=Harianto|name-list-style=vanc|title=Modeling Electrostatic Contributions to Protein Folding and Binding|date=2008|publisher=Florida State University|type=PhD thesis|page=1 footnote|access-date=28 January 2020|archive-date=28 January 2020|archive-url=https://web.archive.org/web/20200128234717/https://diginole.lib.fsu.edu/islandora/object/fsu:175939|url-status=live}}</ref><ref name="VoJw6fIISSkC p.299">{{cite journal|last1=Stewart|first1=L.|last2=Burgin|first2=A. B.|journal=Frontiers in Drug Design & Discovery |name-list-style=vanc|date=2005|title=Whole Gene Synthesis: A Gene-O-Matic Future|url=https://books.google.com/books?id=VoJw6fIISSkC&pg=PA299|publisher=[[Bentham Science Publishers]]|volume=1|page=299|doi=10.2174/1574088054583318|isbn=978-1-60805-199-1|issn=1574-0889|access-date=5 January 2016|archive-date=14 April 2021|archive-url=https://web.archive.org/web/20210414224011/https://books.google.com/books?id=VoJw6fIISSkC&pg=PA299|url-status=live}}</ref><ref name="url_The_Genetic_Codes_NCBI">{{cite web|date=7 April 2008|title=The Genetic Codes|url=https://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi?mode=c|access-date=10 March 2010|publisher=National Center for Biotechnology Information (NCBI)|vauthors=Elzanowski A, Ostell J|archive-date=20 August 2016|archive-url=https://web.archive.org/web/20160820125755/http://130.14.29.110/Taxonomy/Utils/wprintgc.cgi?mode=c|url-status=live}}</ref>

[[N-Formylmethionine|''N''-formylmethionine]] (which is often the initial amino acid of proteins in bacteria, [[Mitochondrion|mitochondria]], and [[chloroplast]]s) is generally considered as a form of [[methionine]] rather than as a separate proteinogenic amino acid. Codon–[[transfer RNA|tRNA]] combinations not found in nature can also be used to [[Expanded genetic code|"expand" the genetic code]] and form novel proteins known as [[alloprotein]]s incorporating [[non-proteinogenic amino acid]]s.<ref name="pmid16260173">{{cite journal | vauthors = Xie J, Schultz PG | title = Adding amino acids to the genetic repertoire | journal = Current Opinion in Chemical Biology | volume = 9 | issue = 6 | pages = 548–554 | date = December 2005 | pmid = 16260173 | doi = 10.1016/j.cbpa.2005.10.011 }}</ref><ref name="pmid19318213">{{cite journal | vauthors = Wang Q, Parrish AR, Wang L | title = Expanding the genetic code for biological studies | journal = Chemistry & Biology | volume = 16 | issue = 3 | pages = 323–336 | date = March 2009 | pmid = 19318213 | pmc = 2696486 | doi = 10.1016/j.chembiol.2009.03.001 }}</ref><ref name="isbn0-387-22046-1">{{cite book | vauthors = Simon M | title = Emergent computation: emphasizing bioinformatics | url = https://archive.org/details/emergentcomputat00simo_754 | url-access = limited | publisher = AIP Press/Springer Science+Business Media | location = New York | year = 2005 | pages = [https://archive.org/details/emergentcomputat00simo_754/page/n116 105–106] | isbn = 978-0-387-22046-8 }}</ref>

===Non-proteinogenic amino acids===
{{main|Non-proteinogenic amino acids}}

Aside from the 22 [[proteinogenic amino acid]]s, many ''non-proteinogenic'' amino acids are known. Those either are not found in proteins (for example [[carnitine]], [[Gamma-aminobutyric acid|GABA]], [[levothyroxine]]) or are not produced directly and in isolation by standard cellular machinery. For example, [[hydroxyproline]] , is synthesised from [[proline]]. Another example is [[selenomethionine]]).

Non-proteinogenic amino acids that are found in proteins are formed by [[post-translational modification]]. Such modifications can also determine the localization of the protein, e.g., the addition of long hydrophobic groups can cause a protein to bind to a [[phospholipid]] membrane.<ref>{{cite journal | vauthors = Blenis J, Resh MD | title = Subcellular localization specified by protein acylation and phosphorylation | journal = Current Opinion in Cell Biology | volume = 5 | issue = 6 | pages = 984–989 | date = December 1993 | pmid = 8129952 | doi = 10.1016/0955-0674(93)90081-Z }}</ref> Examples:
*the [[carboxylation]] of [[glutamate]] allows for better binding of [[calcium in biology|calcium cations]],<ref>{{cite journal | vauthors = Vermeer C | title = Gamma-carboxyglutamate-containing proteins and the vitamin K-dependent carboxylase | journal = The Biochemical Journal | volume = 266 | issue = 3 | pages = 625–636 | date = March 1990 | pmid = 2183788 | pmc = 1131186 | doi = 10.1042/bj2660625 }}</ref> 
*[[Hydroxyproline]], generated by [[hydroxylation]] of [[proline]], is a major component of the [[connective tissue]] [[collagen]].<ref>{{cite journal | vauthors = Bhattacharjee A, Bansal M | title = Collagen Structure: the Madras triple helix and the current scenario | journal = IUBMB Life | volume = 57 | issue = 3 | pages = 161–172 | date = March 2005 | pmid = 16036578 | doi = 10.1080/15216540500090710 | s2cid = 7211864 }}</ref>
* [[Hypusine]] in the [[Eukaryotic initiation factor|translation initiation factor]] [[EIF5A]], contains a modification of lysine.<ref>{{cite journal | vauthors = Park MH | title = The post-translational synthesis of a polyamine-derived amino acid, hypusine, in the eukaryotic translation initiation factor 5A (eIF5A) | journal = Journal of Biochemistry | volume = 139 | issue = 2 | pages = 161–169 | date = February 2006 | pmid = 16452303 | pmc = 2494880 | doi = 10.1093/jb/mvj034 }}</ref> 

Some non-proteinogenic amino acids are not found in proteins. Examples include [[2-aminoisobutyric acid]] and the neurotransmitter [[gamma-aminobutyric acid]]. Non-proteinogenic amino acids often occur as intermediates in the [[metabolic pathway]]s for standard amino acids&nbsp;– for example, [[ornithine]] and [[citrulline]] occur in the [[urea cycle]], part of amino acid [[catabolism]] (see below).<ref>{{cite journal | vauthors = Curis E, Nicolis I, Moinard C, Osowska S, Zerrouk N, Bénazeth S, Cynober L | s2cid = 23877884 | title = Almost all about citrulline in mammals | journal = Amino Acids | volume = 29 | issue = 3 | pages = 177–205 | date = November 2005 | pmid = 16082501 | doi = 10.1007/s00726-005-0235-4 }}</ref> A rare exception to the dominance of α-amino acids in biology is the β-amino acid [[beta alanine]] (3-aminopropanoic acid), which is used in plants and microorganisms in the synthesis of [[pantothenic acid]] (vitamin B<sub>5</sub>), a component of [[coenzyme A]].<ref>{{cite journal | vauthors = Coxon KM, Chakauya E, Ottenhof HH, Whitney HM, Blundell TL, Abell C, Smith AG | title = Pantothenate biosynthesis in higher plants | journal = Biochemical Society Transactions | volume = 33 | issue = Pt 4 | pages = 743–746 | date = August 2005 | pmid = 16042590 | doi = 10.1042/BST0330743 }}</ref>

===In mammalian nutrition===
[[File:Amino acids in food and blood.png|thumb|right|380px|Share of amino acid in various human diets and the resulting mix of amino acids in human blood serum. Glutamate and glutamine are the most frequent in food at over 10%, while alanine, glutamine, and glycine are the most common in blood.|alt=Diagram showing the relative occurrence of amino acids in blood serum as obtained from diverse diets.]]
{{Main|Essential amino acid}}
{{further|Protein (nutrient)|Amino acid synthesis}}

Amino acids are not typical component of food: animals eat proteins. The protein is broken down into amino acids in the process of digestion. They are then used to synthesize new proteins, other biomolecules, or are oxidized to [[urea]] and [[carbon dioxide]] as a source of energy.<ref>{{cite journal | vauthors = Sakami W, Harrington H | title = Amino acid metabolism | journal = Annual Review of Biochemistry | volume = 32 | issue = 1 | pages = 355–398 | year = 1963 | pmid = 14144484 | doi = 10.1146/annurev.bi.32.070163.002035 }}</ref> The oxidation pathway starts with the removal of the amino group by a [[transaminase]]; the amino group is then fed into the [[urea cycle]]. The other product of transamidation is a [[keto acid]] that enters the [[citric acid cycle]].<ref>{{cite journal | vauthors = Brosnan JT | title = Glutamate, at the interface between amino acid and carbohydrate metabolism | journal = The Journal of Nutrition | volume = 130 | issue = 4S Suppl | pages = 988S–990S | date = April 2000 | pmid = 10736367 | doi = 10.1093/jn/130.4.988S | doi-access = free }}</ref> [[Glucogenic amino acid]]s can also be converted into glucose, through [[gluconeogenesis]].<ref>{{cite journal | vauthors = Young VR, Ajami AM | title = Glutamine: the emperor or his clothes? | journal = The Journal of Nutrition | volume = 131 | issue = 9 Suppl | pages = 2449S–2459S, 2486S–2487S | date = September 2001 | pmid = 11533293 | doi = 10.1093/jn/131.9.2449S | doi-access = free }}</ref>  

Of the 20 standard amino acids, nine ([[Histidine|His]], [[Isoleucine|Ile]], [[Leucine|Leu]], [[Lysine|Lys]], [[Methionine|Met]], [[Phenylalanine|Phe]], [[Threonine|Thr]], [[Tryptophan|Trp]] and [[Valine|Val]]) are called [[essential amino acid]]s because the [[human body]] cannot [[biosynthesis|synthesize]] them from other compounds at the level needed for normal growth, so they must be obtained from food.<ref>{{cite journal | vauthors = Young VR | title = Adult amino acid requirements: the case for a major revision in current recommendations | journal = The Journal of Nutrition | volume = 124 | issue = 8 Suppl | pages = 1517S–1523S | date = August 1994 | pmid = 8064412 | doi = 10.1093/jn/124.suppl_8.1517S | doi-access = free }}</ref><ref>{{cite journal | vauthors = Fürst P, Stehle P | title = What are the essential elements needed for the determination of amino acid requirements in humans? | journal = The Journal of Nutrition | volume = 134 | issue = 6 Suppl | pages = 1558S–1565S | date = June 2004 | pmid = 15173430 | doi = 10.1093/jn/134.6.1558S | doi-access = free }}</ref><ref>{{cite journal | vauthors = Reeds PJ | title = Dispensable and indispensable amino acids for humans | journal = The Journal of Nutrition | volume = 130 | issue = 7 | pages = 1835S–1840S | date = July 2000 | pmid = 10867060 | doi = 10.1093/jn/130.7.1835S | doi-access = free }}</ref> 

====Semi-essential and conditionally essential amino acids, and juvenile requirements====
In addition, cysteine, [[tyrosine]], and [[arginine]] are considered semiessential amino acids, and taurine a semi-essential aminosulfonic acid in children. Some amino acids are [[Essential amino acid#Essentiality in humans|conditionally essential]] for certain ages or medical conditions. Essential amino acids may also vary from [[species]] to species.{{efn|For example, [[ruminant]]s such as cows obtain a number of amino acids via [[microbe]]s in the [[reticulorumen|first two stomach chambers]].}} The metabolic pathways that synthesize these monomers are not fully developed.<ref>{{cite journal | vauthors = Imura K, Okada A | title = Amino acid metabolism in pediatric patients | journal = Nutrition | volume = 14 | issue = 1 | pages = 143–148 | date = January 1998 | pmid = 9437700 | doi = 10.1016/S0899-9007(97)00230-X }}</ref><ref>{{cite journal | vauthors = Lourenço R, Camilo ME | title = Taurine: a conditionally essential amino acid in humans? An overview in health and disease | journal = Nutricion Hospitalaria | volume = 17 | issue = 6 | pages = 262–270 | year = 2002 | pmid = 12514918 }}</ref>

===Non-protein functions===
{{Catecholamine and trace amine biosynthesis|align=right|caption=[[Catecholamine]]s and [[trace amine]]s are synthesized from phenylalanine and tyrosine in humans.}}
{{Further|Amino acid neurotransmitter}}

Many proteinogenic and non-proteinogenic amino acids have biological functions beyond being precursors to proteins and peptides.In humans, amino acids also have important roles in diverse biosynthetic pathways. [[Plant defense against herbivory|Defenses against herbivores]] in plants sometimes employ amino acids.<ref name="Hylin1969">{{Cite journal|last=Hylin |first=John W. | name-list-style = vanc |year=1969 |title=Toxic peptides and amino acids in foods and feeds |journal=Journal of Agricultural and Food Chemistry |volume=17 |issue=3 |pages=492–496 |doi=10.1021/jf60163a003}}</ref> Examples:
====Standard amino acids====
* [[Tryptophan]] is a precursor of the neurotransmitter [[serotonin]].<ref>{{cite journal | vauthors = Savelieva KV, Zhao S, Pogorelov VM, Rajan I, Yang Q, Cullinan E, Lanthorn TH | title = Genetic disruption of both tryptophan hydroxylase genes dramatically reduces serotonin and affects behavior in models sensitive to antidepressants | journal = PLOS ONE| volume = 3 | issue = 10 | pages = e3301 | year = 2008 | pmid = 18923670 | pmc = 2565062 | doi = 10.1371/journal.pone.0003301 | veditors = Bartolomucci A | bibcode = 2008PLoSO...3.3301S | doi-access = free }}</ref>
* [[Tyrosine]] (and its precursor phenylalanine) are precursors of the [[catecholamine]] [[neurotransmitter]]s [[dopamine]], [[epinephrine]] and [[norepinephrine]] and various [[trace amine]]s.
* [[Phenylalanine]] is a precursor of [[phenethylamine]] and tyrosine in humans. In plants, it is a precursor of various [[phenylpropanoid]]s, which are important in plant metabolism.
* [[Glycine]] is a precursor of [[porphyrin]]s such as [[heme]].<ref>{{cite journal | vauthors = Shemin D, Rittenberg D | title = The biological utilization of glycine for the synthesis of the protoporphyrin of hemoglobin | journal = The Journal of Biological Chemistry | volume = 166 | issue = 2 | pages = 621–625 | date = December 1946 | doi = 10.1016/S0021-9258(17)35200-6 | pmid = 20276176 | url = http://www.jbc.org/cgi/reprint/166/2/621 | doi-access = free | access-date = 3 November 2008 | archive-date = 7 May 2022 | archive-url = https://web.archive.org/web/20220507191150/https://www.jbc.org/cgi/reprint/166/2/621 | url-status = live }}</ref>
* [[Arginine]] is a precursor of [[nitric oxide]].<ref>{{cite journal | vauthors = Tejero J, Biswas A, Wang ZQ, Page RC, Haque MM, Hemann C, Zweier JL, Misra S, Stuehr DJ | title = Stabilization and characterization of a heme-oxy reaction intermediate in inducible nitric-oxide synthase | journal = The Journal of Biological Chemistry | volume = 283 | issue = 48 | pages = 33498–33507 | date = November 2008 | pmid = 18815130 | pmc = 2586280 | doi = 10.1074/jbc.M806122200 | doi-access = free }}</ref>
* [[Ornithine]] and [[S-Adenosyl methionine|''S''-adenosylmethionine]] are precursors of [[polyamine]]s.<ref>{{cite journal | vauthors = Rodríguez-Caso C, Montañez R, Cascante M, Sánchez-Jiménez F, Medina MA | title = Mathematical modeling of polyamine metabolism in mammals | journal = The Journal of Biological Chemistry | volume = 281 | issue = 31 | pages = 21799–21812 | date = August 2006 | pmid = 16709566 | doi = 10.1074/jbc.M602756200 | doi-access = free }}</ref>
* [[Aspartate]], [[glycine]], and [[glutamine]] are precursors of [[nucleotide]]s.<ref name = "Stryer_2002">{{cite book | last1 = Stryer | first1 = Lubert | last2 = Berg | first2 = Jeremy M. | last3 = Tymoczko | first3 = John L. | name-list-style = vanc | title = Biochemistry | url = https://archive.org/details/biochemistry200100jere | url-access = registration | date = 2002 | publisher = W.H. Freeman | location = New York | isbn = 978-0-7167-4684-3 | edition = 5th | pages = [https://archive.org/details/biochemistry200100jere/page/693 693–698] }}</ref> However, not all of the functions of other abundant nonstandard amino acids are known.

====Roles for nonstandard amino acids====
*[[Carnitine]] is used in [[lipid|lipid transport]]. 
*[[gamma-aminobutyric acid]] is a neurotransmitter.<ref name="pmid12467378">{{cite journal | vauthors = Petroff OA | s2cid = 84891972 | title = GABA and glutamate in the human brain | journal = The Neuroscientist | volume = 8 | issue = 6 | pages = 562–573 | date = December 2002 | pmid = 12467378 | doi = 10.1177/1073858402238515 }}</ref> 
*[[5-HTP]] (5-hydroxytryptophan) is used for experimental treatment of depression.<ref>{{cite journal | vauthors = Turner EH, Loftis JM, Blackwell AD | title = Serotonin a la carte: supplementation with the serotonin precursor 5-hydroxytryptophan | journal = Pharmacology & Therapeutics | volume = 109 | issue = 3 | pages = 325–338 | date = March 2006 | pmid = 16023217 | doi = 10.1016/j.pharmthera.2005.06.004 | s2cid = 2563606 | url = https://escholarship.org/uc/item/58h866d5 | access-date = 12 June 2019 | archive-date = 13 April 2020 | archive-url = https://web.archive.org/web/20200413044234/https://escholarship.org/uc/item/58h866d5 | url-status = live }}</ref> 
*[[L-DOPA|<small>L</small>-DOPA]] (<small>L</small>-dihydroxyphenylalanine) for [[Parkinson's]] treatment,<ref>{{cite journal | vauthors = Kostrzewa RM, Nowak P, Kostrzewa JP, Kostrzewa RA, Brus R | s2cid = 33603501 | title = Peculiarities of L-DOPA treatment of Parkinson's disease | journal = Amino Acids | volume = 28 | issue = 2 | pages = 157–164 | date = March 2005 | pmid = 15750845 | doi = 10.1007/s00726-005-0162-4 }}</ref> 
*[[Eflornithine]] inhibits [[ornithine decarboxylase]] and used in the treatment of [[African trypanosomiasis|sleeping sickness]].<ref>{{cite journal | vauthors = Heby O, Persson L, Rentala M | s2cid = 26273053 | title = Targeting the polyamine biosynthetic enzymes: a promising approach to therapy of African sleeping sickness, Chagas' disease, and leishmaniasis | journal = Amino Acids | volume = 33 | issue = 2 | pages = 359–366 | date = August 2007 | pmid = 17610127 | doi = 10.1007/s00726-007-0537-9 }}</ref>
*[[Canavanine]], an analogue of [[arginine]] found in many [[legume]]s is an [[antifeedant]], protecting the plant from predators.<ref>{{cite journal | vauthors = Rosenthal GA | s2cid = 3144019 | title = L-Canavanine: a higher plant insecticidal allelochemical | journal = Amino Acids | volume = 21 | issue = 3 | pages = 319–330 | year = 2001 | pmid = 11764412 | doi = 10.1007/s007260170017 }}</ref>
*[[Mimosine]] found in some legumes, is another possible [[antifeedant]].<ref>{{cite journal |last=Hammond |first=Andrew C. |date=1 May 1995 |title=Leucaena toxicosis and its control in ruminants |url=https://academic.oup.com/jas/article-abstract/73/5/1487/4718854?redirectedFrom=fulltext |journal=[[Journal of Animal Science]] |volume=73 |issue=5 |pages=1487–1492 |doi=10.2527/1995.7351487x |pmid=7665380 |access-date=7 May 2022 |archive-date=7 May 2022 |archive-url=https://web.archive.org/web/20220507185253/https://academic.oup.com/jas/article-abstract/73/5/1487/4718854?redirectedFrom=fulltext |url-status=live }}</ref> This compound is an analogue of [[tyrosine]] and can poison animals that graze on these plants.

==Uses in industry==
===Animal feed===
Amino acids are sometimes added to [[Compound feed|animal feed]] because some of the components of these feeds, such as [[soybean]]s, have low levels of some of the [[essential amino acid]]s, especially of lysine, methionine, threonine, and tryptophan.<ref name=Leuchtenberger2005>{{cite journal | vauthors = Leuchtenberger W, Huthmacher K, Drauz K | s2cid = 24161808 | title = Biotechnological production of amino acids and derivatives: current status and prospects | journal = Applied Microbiology and Biotechnology | volume = 69 | issue = 1 | pages = 1–8 | date = November 2005 | pmid = 16195792 | doi = 10.1007/s00253-005-0155-y }}</ref> Likewise amino acids are used to chelate metal cations in order to improve the absorption of minerals from feed supplements.<ref>{{cite book|last=Ashmead|first=H. DeWayne | name-list-style = vanc |title=The Role of Amino Acid Chelates in Animal Nutrition|year=1993|publisher=Noyes Publications|location=Westwood}}</ref>

===Food===
The [[food industry]] is a major consumer of amino acids, especially [[glutamic acid]], which is used as a [[flavor enhancer]],<ref name=Garattini>{{cite journal | vauthors = Garattini S | title = Glutamic acid, twenty years later | journal = The Journal of Nutrition | volume = 130 | issue = 4S Suppl | pages = 901S–909S | date = April 2000 | pmid = 10736350 | doi = 10.1093/jn/130.4.901S | doi-access = free }}</ref> and [[aspartame]] (aspartylphenylalanine 1-methyl ester), which is used as an [[artificial sweetener]].<ref>{{cite journal | vauthors = Stegink LD | title = The aspartame story: a model for the clinical testing of a food additive | journal = The American Journal of Clinical Nutrition | volume = 46 | issue = 1 Suppl | pages = 204–215 | date = July 1987 | pmid = 3300262 | doi = 10.1093/ajcn/46.1.204 }}</ref> Amino acids are sometimes added to food by manufacturers to alleviate symptoms of mineral deficiencies, such as anemia, by improving mineral absorption and reducing negative side effects from inorganic mineral supplementation.<ref name=Ullmann/>

===Chemical building blocks===
{{further|Asymmetric synthesis}}

Amino acids are low-cost [[feedstock]]s used in [[chiral pool synthesis]] as [[enantiomer|enantiomerically pure]] building blocks.<ref name=Hanessian1993>{{cite journal | vauthors = Hanessian S | year =1993 | title = Reflections on the total synthesis of natural products: Art, craft, logic, and the chiron approach |journal=Pure and Applied Chemistry | volume = 65 | issue = 6 | pages = 1189–1204 | doi = 10.1351/pac199365061189 | s2cid =43992655 | doi-access = free }}</ref><ref name=Blaser1992>{{cite journal | last = Blaser | first = Hans Ulrich | name-list-style = vanc | year = 1992 | title = The chiral pool as a source of enantioselective catalysts and auxiliaries |journal=Chemical Reviews |volume=92 |issue=5 |pages=935–952 |doi=10.1021/cr00013a009}}</ref>

Amino acids are used in the synthesis of some [[cosmetics]].<ref name=Leuchtenberger2005/>

==Aspirational uses==
===Fertilizer===
The [[Chelation|chelating]] ability of amino acids is sometimes used in fertilizers to facilitate the delivery of minerals to plants in order to correct mineral deficiencies, such as iron chlorosis. These fertilizers are also used to prevent deficiencies from occurring and to improve the overall health of the plants.<ref>{{cite book|last=Ashmead|first=H. DeWayne | name-list-style = vanc |title=Foliar Feeding of Plants with Amino Acid Chelates|year=1986|publisher=Noyes Publications|location=Park Ridge}}</ref> 

===Biodegradable plastics===
{{further|Biodegradable plastic|Biopolymer}}
Amino acids have been considered as components of biodegradable polymers, which have applications as [[environmentally friendly]] packaging and in medicine in [[drug delivery]] and the construction of [[prosthetic implant]]s.<ref name=Sanda1999>{{cite journal | vauthors = Sanda F, Endo T | year = 1999 | title = Syntheses and functions of polymers based on amino acids | journal = Macromolecular Chemistry and Physics | volume = 200 | issue = 12 | pages = 2651–2661 | doi = 10.1002/(SICI)1521-3935(19991201)200:12<2651::AID-MACP2651>3.0.CO;2-P | doi-access = free }}</ref> An interesting example of such materials is [[polyaspartate]], a water-soluble biodegradable polymer that may have applications in disposable [[diaper]]s and agriculture.<ref name=Gross2002>{{cite journal | vauthors = Gross RA, Kalra B | title = Biodegradable polymers for the environment | journal = Science | volume = 297 | issue = 5582 | pages = 803–807 | date = August 2002 | pmid = 12161646 | doi = 10.1126/science.297.5582.803 | bibcode = 2002Sci...297..803G | url = https://zenodo.org/record/1231185 | access-date = 12 June 2019 | archive-date = 25 July 2020 | archive-url = https://web.archive.org/web/20200725075829/https://zenodo.org/record/1231185 | url-status = live }}</ref> Due to its solubility and ability to [[chelate]] metal ions, polyaspartate is also being used as a biodegradable anti[[Fouling|scaling]] agent and a [[corrosion inhibitor]].<ref>{{Cite book|title= Commercial poly(aspartic acid) and Its Uses | vauthors = Low KC, Wheeler AP, Koskan LP |series= Advances in Chemistry Series |volume= 248 |publisher= [[American Chemical Society]] |location= Washington, D.C. |year= 1996}}</ref><ref name=Thombre2005>{{cite journal| vauthors = Thombre SM, Sarwade BD | year = 2005 | title = Synthesis and Biodegradability of Polyaspartic Acid: A Critical Review | journal = Journal of Macromolecular Science, Part A | volume = 42 | issue = 9 | pages = 1299–1315 | doi = 10.1080/10601320500189604| s2cid = 94818855 }}</ref>

==Synthesis==
{{Main|Amino acid synthesis}}
[[File:Strecker amino acid synthesis scheme.svg|thumb|400px|right|The Strecker amino acid synthesis|alt=For the steps in the reaction, see the text.]]

===Chemical synthesis===
The commercial production of amino acids usually relies on mutant bacteria that overproduce individual amino acids using glucose as a carbon source. Some amino acids are produced by enzymatic conversions of synthetic intermediates. [[2-Aminothiazoline-4-carboxylic acid]] is an intermediate in one industrial synthesis of [[cysteine|<small>L</small>-cysteine]] for example. [[Aspartic acid]] is produced by the addition of ammonia to [[fumarate]] using a lyase.<ref name=Ullmann>{{Ullmann|first1=Karlheinz |last1=Drauz |first2=Ian |last2=Grayson |first3=Axel |last3=Kleemann |first4=Hans-Peter |last4=Krimmer |first5=Wolfgang |last5=Leuchtenberger |first6=Christoph |last6=Weckbecker |name-list-style=vanc |year=2007| doi=10.1002/14356007.a02_057.pub2|title=Amino Acids}}</ref>

===Biosynthesis===
In plants, nitrogen is first assimilated into organic compounds in the form of [[glutamate]], formed from alpha-ketoglutarate and ammonia in the mitochondrion. For other amino acids, plants use [[transaminase]]s to move the amino group from glutamate to another alpha-keto acid. For example, aspartate aminotransferase converts glutamate and oxaloacetate to alpha-ketoglutarate and aspartate.<ref>{{Cite book | last1 = Jones | first1 = Russell Celyn | last2 = Buchanan | first2 = Bob B. | last3 = Gruissem | first3 = Wilhelm | name-list-style = vanc | title = Biochemistry & molecular biology of plants | publisher = American Society of Plant Physiologists | location = Rockville, Md | year = 2000 | pages = [https://archive.org/details/biochemistrymole00buch/page/371 371–372] | isbn = 978-0-943088-39-6 | url = https://archive.org/details/biochemistrymole00buch/page/371 }}</ref> Other organisms use transaminases for amino acid synthesis, too.

Nonstandard amino acids are usually formed through modifications to standard amino acids. For example, [[homocysteine]] is formed through the [[transsulfuration pathway]] or by the demethylation of methionine via the intermediate metabolite [[S-adenosylmethionine|''S''-adenosylmethionine]],<ref name="Brosnan">{{cite journal | vauthors = Brosnan JT, Brosnan ME | title = The sulfur-containing amino acids: an overview | journal = The Journal of Nutrition | volume = 136 | issue = 6 Suppl | pages = 1636S–1640S | date = June 2006 | pmid = 16702333 | doi = 10.1093/jn/136.6.1636S | doi-access = free }}</ref> while [[hydroxyproline]] is made by a [[post translational modification]] of [[proline]].<ref>{{cite book | vauthors = Kivirikko KI, Pihlajaniemi T | chapter = Collagen Hydroxylases and the Protein Disulfide Isomerase Subunit of Prolyl 4-Hydroxylases | title = Advances in Enzymology and Related Areas of Molecular Biology | volume = 72 | pages = 325–398 | year = 1998 | pmid = 9559057 | doi = 10.1002/9780470123188.ch9 | isbn = 9780470123188 | series = Advances in Enzymology – and Related Areas of Molecular Biology }}</ref>

[[Microorganism]]s and plants synthesize many uncommon amino acids. For example, some microbes make [[2-aminoisobutyric acid]] and [[lanthionine]], which is a sulfide-bridged derivative of alanine. Both of these amino acids are found in peptidic [[lantibiotics]] such as [[alamethicin]].<ref>{{cite journal | vauthors = Whitmore L, Wallace BA | s2cid = 24638475 | title = Analysis of peptaibol sequence composition: implications for in vivo synthesis and channel formation | journal = European Biophysics Journal | volume = 33 | issue = 3 | pages = 233–237 | date = May 2004 | pmid = 14534753 | doi = 10.1007/s00249-003-0348-1 }}</ref> However, in plants, [[1-aminocyclopropane-1-carboxylic acid]] is a small disubstituted cyclic amino acid that is an intermediate in the production of the plant hormone [[ethylene#Ethylene as a plant hormone|ethylene]].<ref>{{cite journal | vauthors = Alexander L, Grierson D | title = Ethylene biosynthesis and action in tomato: a model for climacteric fruit ripening | journal = Journal of Experimental Botany | volume = 53 | issue = 377 | pages = 2039–2055 | date = October 2002 | pmid = 12324528 | doi = 10.1093/jxb/erf072 | doi-access = free }}</ref>

===Primordial synthesis===
The formation of amino acids and peptides are assumed to precede and perhaps induce the [[abiogenesis|emergence of life on earth]]. Amino acids can form from simple precursors under various conditions.<ref name="10.1016/j.gsf.2017.07.007"/> Surface-based chemical metabolism of amino acids and very small compounds may have led to the build-up of amino acids, coenzymes and phosphate-based small carbon molecules.<ref>{{cite journal |last1=Danchin |first1=Antoine |title=From chemical metabolism to life: the origin of the genetic coding process |journal=Beilstein Journal of Organic Chemistry |date=12 June 2017 |volume=13 |issue=1 |pages=1119–1135 |doi=10.3762/bjoc.13.111 |pmid=28684991 |pmc=5480338 |language=en |issn=1860-5397}}</ref>{{additional citation needed|date=September 2022}} Amino acids and similar building blocks could have been elaborated into proto-[[peptide]]s, with peptides being considered key players in the origin of life.<ref name="10.1021/acs.chemrev.9b00664">{{cite journal |last1=Frenkel-Pinter |first1=Moran |last2=Samanta |first2=Mousumi |last3=Ashkenasy |first3=Gonen |last4=Leman |first4=Luke J. |title=Prebiotic Peptides: Molecular Hubs in the Origin of Life |journal=Chemical Reviews |date=10 June 2020 |volume=120 |issue=11 |pages=4707–4765 |doi=10.1021/acs.chemrev.9b00664 |pmid=32101414 |s2cid=211536416 |url=https://pubs.acs.org/doi/10.1021/acs.chemrev.9b00664 |language=en |issn=0009-2665|url-access=subscription}}</ref>

In the famous [[Urey-Miller experiment]], the passage of an electric arc through a mixture of methane, hydrogen, and ammonia produces a large number of amino acids. Since then, scientists have discovered a range of ways and components by which the potentially prebiotic formation and chemical evolution of peptides may have occurred, such as condensing agents, the design of self-replicating peptides and a number of non-enzymatic mechanisms by which amino acids could have emerged and elaborated into peptides.<ref name="10.1021/acs.chemrev.9b00664"/> Several hypotheses invoke the [[Strecker synthesis]] whereby hydrogen cyanide, simple aldehydes, ammonia, and water produce amino acids.<ref name="10.1016/j.gsf.2017.07.007">{{cite journal |doi=10.1016/j.gsf.2017.07.007|title=Origins of building blocks of life: A review |year=2018 |last1=Kitadai |first1=Norio |last2=Maruyama |first2=Shigenori |journal=Geoscience Frontiers |volume=9 |issue=4 |pages=1117–1153 |bibcode=2018GeoFr...9.1117K |s2cid=102659869 |doi-access=free }}</ref>

According to a review, amino acids, and even peptides, "turn up fairly regularly in the [[primordial soup|various experimental broths]] that have been allowed to be cooked from simple chemicals. This is because [[nucleotide]]s are far more difficult to synthesize chemically than amino acids." For a chronological order, it suggests that there must have been a 'protein world' or at least a 'polypeptide world', possibly later followed by the '[[RNA world]]' and the '[[DNA world]]'.<ref>{{cite journal |last1=Milner-White |first1=E. James |title=Protein three-dimensional structures at the origin of life |journal=Interface Focus |date=6 December 2019 |volume=9 |issue=6 |pages=20190057 |doi=10.1098/rsfs.2019.0057|pmid=31641431 |pmc=6802138 }}</ref> [[Codon]]–amino acids mappings may be the [[biology|biological]] information system at the primordial origin of life on Earth.<ref>{{cite journal |last1=Chatterjee |first1=Sankar |last2=Yadav |first2=Surya |title=The Coevolution of Biomolecules and Prebiotic Information Systems in the Origin of Life: A Visualization Model for Assembling the First Gene |journal=Life |date=June 2022 |volume=12 |issue=6 |pages=834 |doi=10.3390/life12060834 |pmid=35743865 |pmc=9225589 |bibcode=2022Life...12..834C |language=en |issn=2075-1729|doi-access=free }}</ref> While amino acids and consequently simple peptides must have formed under different experimentally probed geochemical scenarios, the transition from an abiotic world to the first life forms is to a large extent still unresolved.<ref>{{cite journal |last1=Kirschning |first1=Andreas |title=The coenzyme/protein pair and the molecular evolution of life |journal=Natural Product Reports |date=26 May 2021 |volume=38 |issue=5 |pages=993–1010 |doi=10.1039/D0NP00037J |pmid=33206101 |s2cid=227037164 |language=en |issn=1460-4752|doi-access=free }}</ref>

==Reactions==

Amino acids undergo the reactions expected of the constituent functional groups.<ref>{{cite book | vauthors = Elmore DT, Barrett GC | title = Amino acids and peptides | url = https://archive.org/details/aminoacidspeptid00barr_040 | url-access = limited |publisher=Cambridge University Press |location=Cambridge, UK |year=1998 |pages=[https://archive.org/details/aminoacidspeptid00barr_040/page/n64 48]–60 |isbn=978-0-521-46827-5}}</ref><ref>{{cite journal | vauthors = Gutteridge A, Thornton JM | title = Understanding nature's catalytic toolkit | journal = Trends in Biochemical Sciences | volume = 30 | issue = 11 | pages = 622–629 | date = November 2005 | pmid = 16214343 | doi = 10.1016/j.tibs.2005.09.006 }}</ref>

===Peptide bond formation===
{{see also|Peptide synthesis|Peptide bond}}

[[File:Peptidformationball.svg|thumbnail|right|400px|The condensation of two amino acids to form a [[dipeptide]]. The two amino acid ''residues'' are linked through a ''[[peptide bond]]''.|alt=Two amino acids are shown next to each other. One loses a hydrogen and oxygen from its carboxyl group (COOH) and the other loses a hydrogen from its amino group (NH2). This reaction produces a molecule of water (H2O) and two amino acids joined by a peptide bond (–CO–NH–). The two joined amino acids are called a dipeptide.]]

As both the amine and carboxylic acid groups of amino acids can react to form amide bonds, one amino acid molecule can react with another and become joined through an amide linkage. This [[polymerization]] of amino acids is what creates proteins. This [[condensation reaction]] yields the newly formed peptide bond and a molecule of water. In cells, this reaction does not occur directly; instead, the amino acid is first activated by attachment to a [[transfer RNA]] molecule through an [[ester]] bond. This aminoacyl-tRNA is produced in an [[Adenosine triphosphate|ATP]]-dependent reaction carried out by an [[aminoacyl tRNA synthetase]].<ref>{{cite journal | vauthors = Ibba M, Söll D | title = The renaissance of aminoacyl-tRNA synthesis | journal = EMBO Reports | volume = 2 | issue = 5 | pages = 382–387 | date = May 2001 | pmid = 11375928 | pmc = 1083889 | doi = 10.1093/embo-reports/kve095 }}</ref> This aminoacyl-tRNA is then a substrate for the ribosome, which catalyzes the attack of the amino group of the elongating protein chain on the ester bond.<ref>{{cite journal | vauthors = Lengyel P, Söll D | title = Mechanism of protein biosynthesis | journal = Bacteriological Reviews | volume = 33 | issue = 2 | pages = 264–301 | date = June 1969 | pmid = 4896351 | pmc = 378322 | doi = 10.1128/MMBR.33.2.264-301.1969 }}</ref> As a result of this mechanism, all proteins made by ribosomes are synthesized starting at their ''N''-terminus and moving toward their ''C''-terminus.

However, not all peptide bonds are formed in this way. In a few cases, peptides are synthesized by specific enzymes. For example, the tripeptide [[glutathione]] is an essential part of the defenses of cells against oxidative stress. This peptide is synthesized in two steps from free amino acids.<ref>{{cite journal | vauthors = Wu G, Fang YZ, Yang S, Lupton JR, Turner ND | title = Glutathione metabolism and its implications for health | journal = The Journal of Nutrition | volume = 134 | issue = 3 | pages = 489–492 | date = March 2004 | pmid = 14988435 | doi = 10.1093/jn/134.3.489 | doi-access = free }}</ref> In the first step, [[gamma-glutamylcysteine synthetase]] condenses cysteine and [[glutamate]] through a peptide bond formed between the side chain carboxyl of the glutamate (the gamma carbon of this side chain) and the amino group of the cysteine. This dipeptide is then condensed with glycine by [[glutathione synthetase]] to form glutathione.<ref>{{cite journal | vauthors = Meister A | title = Glutathione metabolism and its selective modification | journal = The Journal of Biological Chemistry | volume = 263 | issue = 33 | pages = 17205–17208 | date = November 1988 | doi = 10.1016/S0021-9258(19)77815-6 | pmid = 3053703 | doi-access = free }}</ref>

In chemistry, peptides are synthesized by a variety of reactions. One of the most-used in [[peptide synthesis|solid-phase peptide synthesis]] uses the aromatic oxime derivatives of amino acids as activated units. These are added in sequence onto the growing peptide chain, which is attached to a solid resin support.<ref>{{cite journal | first1 = Louis A. | last1 = Carpino | name-list-style = vanc | year = 1992 | title = 1-Hydroxy-7-azabenzotriazole. An efficient peptide coupling additive |journal=Journal of the American Chemical Society |volume=115 |pages=4397–4398 |doi=10.1021/ja00063a082 |issue=10}}</ref> Libraries of peptides are used in drug discovery through [[high-throughput screening]].<ref>{{cite journal | vauthors = Marasco D, Perretta G, Sabatella M, Ruvo M | title = Past and future perspectives of synthetic peptide libraries | journal = Current Protein & Peptide Science | volume = 9 | issue = 5 | pages = 447–467 | date = October 2008 | pmid = 18855697 | doi = 10.2174/138920308785915209 }}</ref>

The combination of functional groups allow amino acids to be effective polydentate ligands for metal–amino acid chelates.<ref>{{cite journal | vauthors = Konara S, Gagnona K, Clearfield A, Thompson C, Hartle J, Ericson C, Nelson C | title = Structural determination and characterization of copper and zinc bis-glycinates with X-ray crystallography and mass spectrometry | journal = Journal of Coordination Chemistry | year = 2010 | volume = 63 | issue = 19 | doi = 10.1080/00958972.2010.514336 | pages = 3335–3347 | s2cid = 94822047 }}</ref>
The multiple side chains of amino acids can also undergo chemical reactions.

===Catabolism===
[[File:Amino acid catabolism revised.png|thumb|300px|Catabolism of proteinogenic amino acids. Amino acids can be classified according to the properties of their main degradation products:<ref>{{cite book | vauthors = Stipanuk MH | date = 2006 | title = Biochemical, physiological, & molecular aspects of human nutrition | edition = 2nd | publisher = Saunders Elsevier }}</ref>
<br/>* ''Glucogenic'', with the products having the ability to form [[glucose]] by [[gluconeogenesis]]
<br/>* ''Ketogenic'', with the products not having the ability to form glucose. These products may still be used for [[ketogenesis]] or [[lipid synthesis]].
<br/>* Amino acids catabolized into both glucogenic and ketogenic products.]]

Degradation of an amino acid often involves [[deamination]] by moving its amino group to α-ketoglutarate, forming [[glutamate]]. This process involves transaminases, often the same as those used in amination during synthesis. In many vertebrates, the amino group is then removed through the [[urea cycle]] and is excreted in the form of [[urea]]. However, amino acid degradation can produce [[uric acid]] or ammonia instead. For example, [[serine dehydratase]] converts serine to pyruvate and ammonia.<ref name = "Stryer_2002" /> After removal of one or more amino groups, the remainder of the molecule can sometimes be used to synthesize new amino acids, or it can be used for energy by entering [[glycolysis]] or the [[citric acid cycle]], as detailed in image at right.

===Complexation===

Amino acids are bidentate ligands, forming [[transition metal amino acid complexes]].<ref>{{cite journal | vauthors = Dghaym RD, Dhawan R, Arndtsen BA | title = The Use of Carbon Monoxide and Imines as Peptide Derivative Synthons: A Facile Palladium-Catalyzed Synthesis of α-Amino Acid Derived Imidazolines | journal = Angewandte Chemie | volume = 40 | issue = 17 | pages = 3228–3230 | date = September 2001 | pmid = 29712039 | doi = 10.1002/(SICI)1521-3773(19980703)37:12<1634::AID-ANIE1634>3.0.CO;2-C }}</ref>

[[File:AAcomplexation.png|420 px]]

==Chemical analysis==

The total nitrogen content of organic matter is mainly formed by the amino groups in proteins. The Total Kjeldahl Nitrogen ([[TKN]]) is a measure of nitrogen widely used in the analysis of (waste) water, soil, food, feed and organic matter in general. As the name suggests, the [[Kjeldahl method]] is applied. More sensitive methods are available.<ref name="pmid23959242">{{cite journal | vauthors = Muñoz-Huerta RF, Guevara-Gonzalez RG, Contreras-Medina LM, Torres-Pacheco I, Prado-Olivarez J, Ocampo-Velazquez RV | title = A review of methods for sensing the nitrogen status in plants: advantages, disadvantages and recent advances | journal = Sensors | location = Basel, Switzerland | volume = 13 | issue = 8 | pages = 10823–43 | date = August 2013 | pmid = 23959242 | pmc = 3812630 | doi = 10.3390/s130810823 | bibcode = 2013Senso..1310823M | doi-access = free }}</ref><ref>{{cite journal | vauthors = Martin PD, Malley DF, Manning G, Fuller L |date=2002|title=Determination of soil organic carbon and nitrogen at thefield level using near-infrared spectroscopy |journal=Canadian Journal of Soil Science |volume=82|issue=4|pages=413–422 |doi=10.4141/S01-054 }}</ref>

== See also ==
{{Portal|Biology|Chemistry}}
{{div col|colwidth=20em}}
* [[Amino acid dating]]
* [[Beta-peptide]]
* [[Degron]]
* [[Erepsin]]
* [[Homochirality]]
* [[Hyperaminoacidemia]]
* [[Leucines]]
* [[Miller–Urey experiment]]
* [[Nucleic acid sequence]]
* [[RNA codon table]]
{{div col end}}

== Notes ==
{{notelist}}

== References ==
{{Reflist}}

== Further reading ==
{{refbegin}}
* {{cite book | last = Tymoczko | first = John L. | name-list-style = vanc | year = 2012 | title = Biochemistry | url = https://archive.org/details/biochemistryseve00berg | url-access = limited | publisher = W. H. Freeman and company | location = New York | chapter = Protein Composition and Structure | pages = 28–31 | chapter-url = https://archive.org/details/biochemistryseve00berg/page/n61 | isbn = 9781429229364}}
* {{cite book | vauthors = Doolittle RF | author-link = Russell Doolittle | veditors = Fasman GD | year = 1989 | title = Predictions of Protein Structure and the Principles of Protein Conformation | publisher = [[Plenum Press]] | location = New York | chapter = Redundancies in protein sequences | pages = 599–623 | isbn = 978-0-306-43131-9 | lccn = 89008555}}
* {{cite book | last1 = Nelson | first1 = David L. | last2 = Cox | first2 = Michael M. | name-list-style = vanc | year = 2000 | title = Lehninger Principles of Biochemistry | publisher = [[Worth Publishers]] | edition = 3rd | isbn = 978-1-57259-153-0 | lccn = 99049137 | url-access = registration | url = https://archive.org/details/lehningerprincip01lehn}}
* {{cite book | last = Meierhenrich | first = Uwe | name-list-style = vanc | author-link = Uwe Meierhenrich | year = 2008 | title = Amino acids and the asymmetry of life | publisher = [[Springer Verlag]] | location = Berlin | isbn = 978-3-540-76885-2 | lccn = 2008930865 | url = http://rogov.zwz.ru/Macroevolution/amino.pdf | url-status = dead | archive-url = https://web.archive.org/web/20120112005425/http://rogov.zwz.ru/Macroevolution/amino.pdf | archive-date = 12 January 2012}}
{{refend}}

== External links ==
* {{Commons-inline}}

{{Amino acids}}
{{Chemical bonds}}
{{Protein primary structure}}
{{Amino acid metabolism enzymes}}
{{Authority control}}


{{Good article}}

{{DEFAULTSORT:Amino Acid}}
[[Category:Amino acids| ]]
[[Category:Nitrogen cycle]]
[[Category:Zwitterions]]