{{Short description|Protein(s) forming a major part of an organism's immune system}}
{{cs1 config|name-list-style=vanc|display-authors=6}}
{{About|the class of proteins}}
{{Use dmy dates|date=March 2020}}
[[File:Antibody.svg|thumb|upright=1.2|Each antibody binds to a specific [[antigen]] in a highly specific interaction analogous to a lock and key.]]

An '''antibody''' ('''Ab''') is the secreted form of a [[B cell]] receptor; the term '''immunoglobulin''' ('''Ig''') can refer to either the membrane-bound form or the secreted form of the B cell receptor, but they are, broadly speaking, the same protein, and so the terms are often treated as synonymous.<ref name="Rhoades">{{cite book|url=https://archive.org/details/humanphysiologyw00rodn/page/584|title=Human Physiology|vauthors=Rhoades RA, Pflanzer RG|publisher=Thomson Learning|year=2002|isbn=978-0-534-42174-8|edition=5th|page=[https://archive.org/details/humanphysiologyw00rodn/page/584 584]|url-access=registration}}</ref> Antibodies are large, Y-shaped [[protein]]s belonging to the [[immunoglobulin superfamily]] which are used by the [[immune system]] to identify and neutralize [[Antigen|antigens]] such as [[pathogenic bacteria|bacteria]] and [[virus]]es, including those that cause disease. Antibodies can recognize virtually any size antigen with diverse chemical compositions from molecules.<ref>{{cite journal |last1=Wilson |first1=Ian A. |last2=Stanfield |first2=Robyn L. |title=50 Years of structural immunology |journal=The Journal of Biological Chemistry |date=3 May 2021 |volume=296 |pages=100745 |doi=10.1016/j.jbc.2021.100745 |doi-access=free |pmid=33957119 |pmc=8163984 |issn=0021-9258 |quote=Antibodies (A–D) can recognize virtually any antigen whether large or small, and which can have diverse chemical compositions from small molecules (A) to carbohydrates to lipids to peptides (B) to proteins (C and D) and combinations thereof.}}</ref> Each antibody recognizes one or more specific [[antigen]]s.<ref name="Janeway5">{{cite book|url=https://archive.org/details/immunobiology00char|title=Immunobiology| vauthors = Janeway C |publisher=Garland Publishing|year=2001|isbn=978-0-8153-3642-6|edition=5th|url-access=registration}}</ref><ref name="pmid8450761">{{cite journal | vauthors = Litman GW, Rast JP, Shamblott MJ, Haire RN, Hulst M, Roess W, Litman RT, Hinds-Frey KR, Zilch A, Amemiya CT | title = Phylogenetic diversification of immunoglobulin genes and the antibody repertoire | journal = Molecular Biology and Evolution | volume = 10 | issue = 1 | pages = 60–72 | date = January 1993 | pmid = 8450761 | doi = 10.1093/oxfordjournals.molbev.a040000 | doi-access = free }}</ref> Antigen literally means "antibody generator", as it is the presence of an antigen that drives the formation of an antigen-specific antibody. Each tip of the "Y" of an antibody contains a [[paratope]] that specifically binds to one particular [[epitope]] on an antigen, allowing the two molecules to bind together with precision. Using this mechanism, antibodies can effectively "tag" a [[microbe]] or an infected cell for attack by other parts of the immune system, or can neutralize it directly (for example, by blocking a part of a virus that is essential for its invasion).

To allow the immune system to recognize millions of different antigens, the antigen-binding sites at both tips of the antibody come in an equally wide variety. The rest of the antibody structure is relatively generic. In humans, antibodies occur in five classes, sometimes called isotypes: [[IgA]], [[IgD]], [[IgE]], [[IgG]], and [[IgM]]. Human IgG and IgA antibodies are also divided into discrete subclasses (IgG1, IgG2, IgG3, IgG4; IgA1 and IgA2). The class refers to the functions triggered by the antibody (also known as effector functions), in addition to some other structural features. Antibodies from different classes also differ in where they are released in the body and at what stage of an immune response. Importantly, while classes and subclasses of antibodies may be shared between species (at least in name), their functions and distribution throughout the body may be different. For example, mouse IgG1 is closer to human IgG2 than human IgG1 in terms of its function.

The term [[humoral immunity]] is often treated as synonymous with the antibody response, describing the function of the immune system that exists in the body's humors (fluids) in the form of soluble proteins, as distinct from [[cell-mediated immunity]], which generally describes the responses of [[T cell]]s (especially cytotoxic T cells). In general, antibodies are considered part of the [[adaptive immune system]], though this classification can become complicated. For example, natural IgM,<ref>{{Cite journal |last1=Ehrenstein |first1=Michael R. |last2=Notley |first2=Clare A. |date=2010-10-15 |title=The importance of natural IgM: scavenger, protector and regulator |url=http://dx.doi.org/10.1038/nri2849 |journal=Nature Reviews Immunology |volume=10 |issue=11 |pages=778–786 |doi=10.1038/nri2849 |pmid=20948548 |s2cid=35784099 |issn=1474-1733}}</ref> which are made by B-1 lineage cells that have properties more similar to innate immune cells than adaptive, refers to IgM antibodies made independently of an immune response that demonstrate polyreactivity- they recognize multiple distinct (unrelated) antigens. These can work with the [[complement system]] in the earliest phases of an immune response to help facilitate clearance of the offending antigen and delivery of the resulting [[Immune complex|immune complexes]] to the [[Lymph node|lymph nodes]] or [[spleen]] for initiation of an immune response. Hence in this capacity, the function of antibodies is more akin to that of innate immunity than adaptive. Nonetheless, in general antibodies are regarded as part of the adaptive immune system because they demonstrate exceptional specificity (with some exception), are produced through genetic rearrangements (rather than being encoded directly in [[germline]]), and are a manifestation of immunological memory.

In the course of an immune response, B cells can progressively [[Cellular differentiation|differentiate]] into antibody-secreting cells (B cells themselves do not secrete antibody; B cells do, however, express B cell receptors, the membrane-bound form of the antibody, on their surface) or memory B cells.<ref>{{Cite journal |last1=Akkaya |first1=Munir |last2=Kwak |first2=Kihyuck |last3=Pierce |first3=Susan K. |date=April 2020 |title=B cell memory: building two walls of protection against pathogens |journal=Nature Reviews Immunology |language=en |volume=20 |issue=4 |pages=229–238 |doi=10.1038/s41577-019-0244-2 |pmid=31836872 |pmc=7223087 |issn=1474-1741}}</ref> Antibody-secreting cells comprise plasmablasts and [[Plasma cell|plasma cells]], which differ mainly in the degree to which they secrete antibody, their lifespan, metabolic adaptations, and surface markers.<ref>{{Cite journal |last1=Tellier |first1=Julie |last2=Nutt |first2=Stephen L |date=2018-10-15 |title=Plasma cells: The programming of an antibody-secreting machine |url=http://dx.doi.org/10.1002/eji.201847517 |journal=European Journal of Immunology |volume=49 |issue=1 |pages=30–37 |doi=10.1002/eji.201847517 |pmid=30273443 |issn=0014-2980|hdl=11343/284565 |hdl-access=free }}</ref> Plasmablasts are rapidly proliferating, short-lived cells produced in the early phases of the immune response (classically described as arising extrafollicularly rather than from the [[germinal center]]) which have the potential to differentiate further into plasma cells.<ref>{{Citation |title=B Cell Memory and Plasma Cell Development |date=2015 |work=Molecular Biology of B Cells |pages=227–249 |url=https://linkinghub.elsevier.com/retrieve/pii/B978012397933900014X |access-date=2024-01-24 |publisher=Elsevier |language=en |doi=10.1016/b978-0-12-397933-9.00014-x |isbn=978-0-12-397933-9}}</ref> The literature is sloppy at times and often describes plasmablasts as just short-lived plasma cells- formally this is incorrect. Plasma cells, in contrast, do not divide (they are [[terminally differentiated]]), and rely on survival niches comprising specific cell types and cytokines to persist.<ref>{{Cite journal |last1=Chu |first1=Van T. |last2=Berek |first2=Claudia |date=2012-12-19 |title=The establishment of the plasma cell survival niche in the bone marrow |url=http://dx.doi.org/10.1111/imr.12011 |journal=Immunological Reviews |volume=251 |issue=1 |pages=177–188 |doi=10.1111/imr.12011 |pmid=23278749 |s2cid=205212187 |issn=0105-2896}}</ref> Plasma cells will secrete huge quantities of antibody regardless of whether or not their cognate antigen is present, ensuring that antibody levels to the antigen in question do not fall to 0, provided the plasma cell stays alive. The rate of antibody secretion, however, can be regulated, for example, by the presence of adjuvant molecules that stimulate the immune response such as [[Toll-like receptor|TLR]] ligands.<ref>{{Cite journal |last1=Dorner |first1=Marcus |last2=Brandt |first2=Simone |last3=Tinguely |first3=Marianne |last4=Zucol |first4=Franziska |last5=Bourquin |first5=Jean-Pierre |last6=Zauner |first6=Ludwig |last7=Berger |first7=Christoph |last8=Bernasconi |first8=Michele |last9=Speck |first9=Roberto F. |last10=Nadal |first10=David |date=2009-11-06 |title=Plasma cell toll-like receptor (TLR) expression differs from that of B cells, and plasma cell TLR triggering enhances immunoglobulin production |url=http://dx.doi.org/10.1111/j.1365-2567.2009.03143.x |journal=Immunology |volume=128 |issue=4 |pages=573–579 |doi=10.1111/j.1365-2567.2009.03143.x |pmid=19950420 |pmc=2792141 |issn=0019-2805}}</ref> Long-lived plasma cells can live for potentially the entire lifetime of the organism.<ref>{{Cite journal |last1=Joyner |first1=Chester J. |last2=Ley |first2=Ariel M. |last3=Nguyen |first3=Doan C. |last4=Ali |first4=Mohammad |last5=Corrado |first5=Alessia |last6=Tipton |first6=Christopher |last7=Scharer |first7=Christopher D. |last8=Mi |first8=Tian |last9=Woodruff |first9=Matthew C. |last10=Hom |first10=Jennifer |last11=Boss |first11=Jeremy M. |last12=Duan |first12=Meixue |last13=Gibson |first13=Greg |last14=Roberts |first14=Danielle |last15=Andrews |first15=Joel |date=March 2022 |title=Generation of human long-lived plasma cells by developmentally regulated epigenetic imprinting |journal=Life Science Alliance |volume=5 |issue=3 |pages=e202101285 |doi=10.26508/lsa.202101285 |issn=2575-1077 |pmc=8739272 |pmid=34952892}}</ref> Classically, the survival niches that house long-lived plasma cells reside in the bone marrow,<ref>{{Cite journal |last1=Halliley |first1=Jessica L. |last2=Tipton |first2=Christopher M. |last3=Liesveld |first3=Jane |last4=Rosenberg |first4=Alexander F. |last5=Darce |first5=Jaime |last6=Gregoretti |first6=Ivan V. |last7=Popova |first7=Lana |last8=Kaminiski |first8=Denise |last9=Fucile |first9=Christopher F. |last10=Albizua |first10=Igor |last11=Kyu |first11=Shuya |last12=Chiang |first12=Kuang-Yueh |last13=Bradley |first13=Kyle T. |last14=Burack |first14=Richard |last15=Slifka |first15=Mark |date=July 2015 |title=Long-Lived Plasma Cells Are Contained within the CD19−CD38hiCD138+ Subset in Human Bone Marrow |journal=Immunity |language=en |volume=43 |issue=1 |pages=132–145 |doi=10.1016/j.immuni.2015.06.016 |pmc=4680845 |pmid=26187412}}</ref> though it cannot be assumed that any given plasma cell in the bone marrow will be long-lived. However, other work indicates that survival niches can readily be established within the mucosal tissues- though the classes of antibodies involved show a different hierarchy from those in the bone marrow.<ref>{{Cite journal |last1=Tellier |first1=Julie |last2=Tarasova |first2=Ilariya |last3=Nie |first3=Junli |last4=Smillie |first4=Christopher S. |last5=Fedele |first5=Pasquale L. |last6=Cao |first6=Wang H. J. |last7=Groom |first7=Joanna R. |last8=Belz |first8=Gabrielle T. |last9=Bhattacharya |first9=Deepta |last10=Smyth |first10=Gordon K. |last11=Nutt |first11=Stephen L. |date=2024-01-03 |title=Unraveling the diversity and functions of tissue-resident plasma cells |url=http://dx.doi.org/10.1038/s41590-023-01712-w |journal=Nature Immunology |volume=25 |issue=2 |pages=330–342 |doi=10.1038/s41590-023-01712-w |pmid=38172260 |s2cid=266752931 |issn=1529-2908}}</ref><ref>{{Cite journal |last1=Landsverk |first1=Ole J. B. |last2=Snir |first2=Omri |last3=Casado |first3=Raquel Bartolomé |last4=Richter |first4=Lisa |last5=Mold |first5=Jeff E. |last6=Réu |first6=Pedro |last7=Horneland |first7=Rune |last8=Paulsen |first8=Vemund |last9=Yaqub |first9=Sheraz |last10=Aandahl |first10=Einar Martin |last11=Øyen |first11=Ole M. |last12=Thorarensen |first12=Hildur Sif |last13=Salehpour |first13=Mehran |last14=Possnert |first14=Göran |last15=Frisén |first15=Jonas |date=February 2017 |title=Antibody-secreting plasma cells persist for decades in human intestine |journal=The Journal of Experimental Medicine |volume=214 |issue=2 |pages=309–317 |doi=10.1084/jem.20161590 |issn=1540-9538 |pmc=5294861 |pmid=28104812}}</ref> B cells can also differentiate into memory B cells which can persist for decades similarly to long-lived plasma cells. These cells can be rapidly recalled in a secondary immune response, undergoing class switching, affinity maturation, and differentiating into antibody-secreting cells.

Antibodies are central to the immune protection elicited by most vaccines and infections (although other components of the immune system certainly participate and for some diseases are considerably more important than antibodies in generating an immune response, e.g. [[Shingles|herpes zoster]]).<ref>{{Cite journal |last=Plotkin |first=Stanley A. |date=2022 |title=Recent updates on correlates of vaccine-induced protection |journal=Frontiers in Immunology |volume=13 |pages=1081107 |doi=10.3389/fimmu.2022.1081107 |doi-access=free |issn=1664-3224 |pmc=9912984 |pmid=36776392}}</ref> Durable protection from infections caused by a given microbe – that is, the ability of the microbe to enter the body and begin to replicate (not necessarily to cause disease) – depends on sustained production of large quantities of antibodies, meaning that effective vaccines ideally elicit persistent high levels of antibody, which relies on long-lived plasma cells. At the same time, many microbes of medical importance have the ability to mutate to escape antibodies elicited by prior infections, and long-lived plasma cells cannot undergo affinity maturation or class switching. This is compensated for through memory B cells: novel variants of a microbe that still retain structural features of previously encountered antigens can elicit memory B cell responses that adapt to those changes. It has been suggested that long-lived plasma cells secrete B cell receptors with higher affinity than those on the surfaces of memory B cells, but findings are not entirely consistent on this point.<ref>{{Cite journal |last1=Sutton |first1=Henry J. |last2=Gao |first2=Xin |last3=Kelly |first3=Hannah G. |last4=Parker |first4=Brian J. |last5=Lofgren |first5=Mariah |last6=Dacon |first6=Cherrelle |last7=Chatterjee |first7=Deepyan |last8=Seder |first8=Robert A. |last9=Tan |first9=Joshua |last10=Idris |first10=Azza H. |last11=Neeman |first11=Teresa |last12=Cockburn |first12=Ian A. |date=2024-01-12 |title=Lack of affinity signature for germinal center cells that have initiated plasma cell differentiation |url=https://pubmed.ncbi.nlm.nih.gov/38228150 |journal=Immunity |volume=57 |issue=2 |pages=S1074–7613(23)00541–1 |doi=10.1016/j.immuni.2023.12.010 |issn=1097-4180 |pmid=38228150|pmc=10922795 |pmc-embargo-date=February 13, 2025 }}</ref>

== Structure ==
{{anchor|CDRs, Fv, Fab and Fc Regions}}
[[File:Antibody basic unit.svg|thumb|upright=1.2|right|Schematic structure of an antibody: two heavy chains (blue, yellow) and the two light chains (green, pink). The antigen binding site is circled.]]
{{multiple image
 | direction = vertical
 | footer = A more accurate depiction of an antibody (3D structure at [https://www.rcsb.org/3d-view/1IGY/1 RCSB PDB]). [[Glycans]] in the Fc region are shown in black.
 | image1 = Antibody IgG1 structure.png
 | alt1 = Model of an antibody showing beta strands
 | image2 = Antibody IgG1 surface.png
 | alt2 = Surface model of an antibody at the molecular level
}}
Antibodies are heavy (~150&nbsp;k[[Dalton (unit)|Da]]) [[protein]]s of about 10 [[Nanometre|nm]] in size,<ref>{{cite journal | vauthors = Reth M | title = Matching cellular dimensions with molecular sizes | journal = Nature Immunology | volume = 14 | issue = 8 | pages = 765–7 | date = August 2013 | pmid = 23867923 | doi = 10.1038/ni.2621 | s2cid = 24333875 | url = http://www.slas.ac.cn/upload/20130815-4.pdf | access-date = 1 May 2018 | archive-date = 2 May 2018 | archive-url = https://web.archive.org/web/20180502064449/http://www.slas.ac.cn/upload/20130815-4.pdf | url-status = dead }}</ref>
arranged in three [[globular protein|globular]] regions that roughly form a Y shape.

In humans and most other [[mammal]]s, an antibody unit consists of four [[polypeptide chain]]s; two identical ''[[Immunoglobulin heavy chain|heavy chains]]'' and two identical ''[[Immunoglobulin light chain|light chains]]'' connected by [[disulfide bond]]s.<ref name = woof/>
Each chain is a series of [[protein domain|domains]]: somewhat similar sequences of about 110 [[amino acid]]s each.
These domains are usually represented in simplified schematics as rectangles.
Light chains consist of one variable domain V<sub>L</sub> and one constant domain C<sub>L</sub>, while heavy chains contain one variable domain V<sub>H</sub> and three to four constant domains C<sub>H</sub>1, C<sub>H</sub>2, ...<ref>{{cite journal | vauthors = Barclay AN | title = Membrane proteins with immunoglobulin-like domains—a master superfamily of interaction molecules | journal = Seminars in Immunology | volume = 15 | issue = 4 | pages = 215–23 | date = August 2003 | pmid = 14690046 | doi = 10.1016/S1044-5323(03)00047-2 }}</ref>

Structurally an antibody is also partitioned into two [[Fragment antigen-binding|antigen-binding fragments]] (Fab), containing one V<sub>L</sub>, V<sub>H</sub>, C<sub>L</sub>, and C<sub>H</sub>1 domain each, as well as the [[Fragment crystallizable region|crystallisable fragment]] (Fc), forming the trunk of the Y shape.<ref name= putnam79>{{cite journal | vauthors = Putnam FW, Liu YS, Low TL | title = Primary structure of a human IgA1 immunoglobulin. IV. Streptococcal IgA1 protease, digestion, Fab and Fc fragments, and the complete amino acid sequence of the alpha 1 heavy chain | journal = The Journal of Biological Chemistry | volume = 254 | issue = 8 | pages = 2865–74 | date = April 1979 | doi = 10.1016/S0021-9258(17)30153-9 | pmid = 107164 | doi-access = free }}</ref>
In between them is a hinge region of the heavy chains, whose flexibility allows antibodies to bind to pairs of epitopes at various distances, to form complexes ([[protein dimer|dimer]]s, trimers, etc.), and to bind effector molecules more easily.<ref name="Roitt">{{Cite book | vauthors = Delves PJ, Martin SJ, Burton DR, Roitt IM |url=https://www.worldcat.org/oclc/949912256 |title=Roitt's essential immunology |date=2017 |isbn=978-1-118-41577-1 |edition=13th |location=Chichester, West Sussex |language=en |oclc=949912256}}</ref>

In an [[Serum protein electrophoresis|electrophoresis]] test of [[blood proteins]], antibodies mostly migrate to the last, [[gamma globulin]] fraction.
Conversely, most gamma-globulins are antibodies, which is why the two terms were historically used as synonyms, as were the symbols Ig and [[gamma|γ]].
This variant terminology fell out of use due to the correspondence being inexact and due to confusion with γ (gamma) [[Immunoglobulin heavy chain|heavy chains]] which characterize the [[IgG]] class of antibodies.<ref>{{Cite web |title=MeSH Browser – gamma-Globulins |url=https://meshb.nlm.nih.gov/record/ui?ui=D005719 |access-date=2020-10-18 |website=meshb.nlm.nih.gov}}</ref><ref>{{cite journal | title = Recommendations for the nomenclature of human immunoglobulins | journal = Journal of Immunology | volume = 108 | issue = 6 | pages = 1733–4 | date = June 1972 | doi = 10.4049/jimmunol.108.6.1733 | pmid = 5031329 | doi-access = free }}</ref>

===Antigen-binding site===
The variable domains can also be referred to as the F<sub>V</sub> region. It is the subregion of Fab that binds to an antigen.
More specifically, each variable domain contains three ''hypervariable regions'' – the amino acids seen there vary the most from antibody to antibody.
When the protein folds, these regions give rise to three loops of [[Beta sheet|β-strand]]s, localized near one another on the surface of the antibody.
These loops are referred to as the [[complementarity-determining region]]s (CDRs), since their shape complements that of an antigen.
Three CDRs from each of the heavy and light chains together form an antibody-binding site whose shape can be anything from a pocket to which a smaller antigen binds, to a larger surface, to a protrusion that sticks out into a groove in an antigen.
Typically however only a few residues contribute to most of the binding energy.<ref name="Janeway5"/>

The existence of two identical antibody-binding sites allows antibody molecules to bind strongly to multivalent antigen (repeating sites such as [[polysaccharide]]s in [[bacterial cell wall]]s, or other sites at some distance apart), as well as to form antibody complexes and larger [[antigen-antibody complex]]es.<ref name="Janeway5"/> The resulting cross-linking plays a role in activating other parts of the immune system.{{citation needed|date=March 2023}}

The structures of CDRs have been clustered and classified by Chothia et al.<ref name= Chothia1997>
{{cite journal | vauthors = Al-Lazikani B, Lesk AM, Chothia C | title = Standard conformations for the canonical structures of immunoglobulins | journal = Journal of Molecular Biology | volume = 273 | issue = 4 | pages = 927–48 | date = November 1997 | pmid = 9367782 | doi = 10.1006/jmbi.1997.1354 }}</ref>
and more recently by North et al.<ref name= North2010>{{cite journal | vauthors = North B, Lehmann A, Dunbrack RL | title = A new clustering of antibody CDR loop conformations | journal = Journal of Molecular Biology | volume = 406 | issue = 2 | pages = 228–56 | date = February 2011 | pmid = 21035459 | pmc = 3065967 | doi = 10.1016/j.jmb.2010.10.030 }}</ref>
and Nikoloudis et al.<ref name= Nikoloudis2014>{{cite journal | vauthors = Nikoloudis D, Pitts JE, Saldanha JW | title = A complete, multi-level conformational clustering of antibody complementarity-determining regions | journal = PeerJ | volume = 2 | issue = e456 | pages = e456 | year = 2014 | pmid = 25071986 | pmc = 4103072 | doi = 10.7717/peerj.456 | doi-access = free }}</ref> However, describing an antibody's binding site using only one single static structure limits the understanding and characterization of the antibody's function and properties. To improve antibody structure prediction and to take the strongly correlated CDR loop and interface movements into account, antibody paratopes should be described as interconverting states in solution with varying probabilities.<ref name="Fernandez-Quintero2021">{{cite journal | vauthors = Fernández-Quintero ML, Georges G, Varga JM, Liedl KR | title = Ensembles in solution as a new paradigm for antibody structure prediction and design | journal = mAbs | volume = 13 | issue = 1 | pages = 1923122 | year = 2021 | pmid = 34030577 | pmc = 8158028 | doi = 10.1080/19420862.2021.1923122 }}</ref>

In the framework of the [[immune network theory]], CDRs are also called idiotypes. According to immune network theory, the adaptive immune system is regulated by interactions between idiotypes.

===Fc region===
{{main|Fragment crystallizable region}}
The [[Fc region]] (the trunk of the Y shape) is composed of constant domains from the heavy chains. Its role is in modulating immune cell activity: it is where effector molecules bind to, triggering various effects after the antibody Fab region binds to an antigen.<ref name=Janeway5/><ref name=Roitt/>
[[Effector cell]]s (such as [[macrophage]]s or [[natural killer cell]]s) bind via their [[Fc receptor]]s (FcR) to the Fc region of an antibody, while the [[complement system]] is activated by binding the [[C1q]] protein complex. IgG or IgM can bind to C1q, but IgA cannot, therefore IgA does not activate the [[classical complement pathway]].<ref name="pmid21937984">{{cite journal | vauthors = Woof JM, Russell RW | title = Structure and function relationships in IgA | journal = [[Mucosal Immunology (journal)|Mucosal Immunology]] | volume = 4 | issue=6 | pages = 590–597 | date=2011 | doi = 10.1038/mi.2011.39 | pmid = 21937984| doi-access = free }}</ref>

Another role of the Fc region is to selectively distribute different antibody classes across the body. In particular, the [[neonatal Fc receptor]] (FcRn) binds to the Fc region of IgG antibodies to transport it across the placenta, from the mother to the fetus. In addition to this, binding to FcRn endows IgG with an exceptionally long half-life relative to other plasma proteins of 3-4 weeks. IgG3 in most cases (depending on allotype) has mutations at the FcRn binding site which lower affinity for FcRn, which are thought to have evolved to limit the highly inflammatory effects of this subclass.<ref>{{Cite journal |last1=Damelang |first1=Timon |last2=Rogerson |first2=Stephen J. |last3=Kent |first3=Stephen J. |last4=Chung |first4=Amy W. |date=March 2019 |title=Role of IgG3 in Infectious Diseases |url=https://doi.org/10.1016/j.it.2019.01.005 |journal=Trends in Immunology |volume=40 |issue=3 |pages=197–211 |doi=10.1016/j.it.2019.01.005 |pmid=30745265 |hdl=11343/284299 |s2cid=73419807 |issn=1471-4906|hdl-access=free }}</ref> 

Antibodies are [[glycoprotein]]s,<ref name = "immune_glycan"/> that is, they have carbohydrates (glycans) added to conserved [[amino acid]] residues.<ref name = "immune_glycan"/><ref>{{cite journal | vauthors = Mattu TS, Pleass RJ, Willis AC, Kilian M, Wormald MR, Lellouch AC, Rudd PM, Woof JM, Dwek RA | title = The glycosylation and structure of human serum IgA1, Fab, and Fc regions and the role of N-glycosylation on Fcα receptor interactions | journal = The Journal of Biological Chemistry | volume = 273 | issue = 4 | pages = 2260–72 | date = January 1998 | pmid = 9442070 | doi = 10.1074/jbc.273.4.2260 | doi-access = free }}</ref>
These conserved [[glycosylation]] sites occur in the Fc region and influence interactions with effector molecules.<ref name = "immune_glycan">{{cite journal | vauthors = Maverakis E, Kim K, Shimoda M, Gershwin ME, Patel F, Wilken R, Raychaudhuri S, Ruhaak LR, Lebrilla CB | title = Glycans in the immune system and The Altered Glycan Theory of Autoimmunity: a critical review | journal = Journal of Autoimmunity | volume = 57 | issue = 6 | pages = 1–13 | date = February 2015 | pmid = 25578468 | pmc = 4340844 | doi = 10.1016/j.jaut.2014.12.002 }}</ref><ref>{{cite journal | vauthors = Cobb BA | title = The history of IgG glycosylation and where we are now | journal = Glycobiology | volume = 30 | issue = 4 | pages = 202–213 | date = March 2020 | pmid = 31504525 | pmc = 7109348 | doi = 10.1093/glycob/cwz065 }}</ref>

===Protein structure===
The [[N-terminus]] of each chain is situated at the tip.
Each [[immunoglobulin domain]] has a similar structure, characteristic of all the members of the [[immunoglobulin superfamily]]:
it is composed of between 7 (for constant domains) and 9 (for variable domains) [[β-strand]]s, forming two [[beta sheet]]s in a [[Beta sheet#Greek key motif|Greek key motif]].
The sheets create a "sandwich" shape, the [[immunoglobulin fold]], held together by a disulfide bond.

===Antibody complexes===
[[File:Mono-und-Polymere.svg|thumb|upright|Some antibodies form [[protein structure|complexes]] that bind to multiple antigen molecules.]]
Secreted antibodies can occur as a single Y-shaped unit, a [[monomer]].
However, some antibody classes also form [[protein dimer|dimers]] with two Ig units (as with IgA), [[tetramer protein|tetramer]]s with four Ig units (like [[teleost fish]] IgM), or [[pentamer]]s with five Ig units (like shark IgW or mammalian IgM, which occasionally forms [[hexamer]]s as well, with six units).<ref name = roux>{{cite journal | vauthors = Roux KH | title = Immunoglobulin structure and function as revealed by electron microscopy | journal = International Archives of Allergy and Immunology | volume = 120 | issue = 2 | pages = 85–99 | date = October 1999 | pmid = 10545762 | doi = 10.1159/000024226 | s2cid = 12187510 }}</ref> IgG can also form hexamers, though no J chain is required.<ref>{{Cite journal |last1=Diebolder |first1=Christoph A. |last2=Beurskens |first2=Frank J. |last3=de Jong |first3=Rob N. |last4=Koning |first4=Roman I. |last5=Strumane |first5=Kristin |last6=Lindorfer |first6=Margaret A. |last7=Voorhorst |first7=Marleen |last8=Ugurlar |first8=Deniz |last9=Rosati |first9=Sara |last10=Heck |first10=Albert J. R. |last11=van de Winkel |first11=Jan G. J. |last12=Wilson |first12=Ian A. |last13=Koster |first13=Abraham J. |last14=Taylor |first14=Ronald P. |last15=Ollmann Saphire |first15=Erica |date=2014-03-14 |title=Complement Is Activated by IgG Hexamers Assembled at the Cell Surface |journal=Science |language=en |volume=343 |issue=6176 |pages=1260–1263 |doi=10.1126/science.1248943 |issn=0036-8075 |pmc=4250092 |pmid=24626930|bibcode=2014Sci...343.1260D }}</ref> IgA tetramers and pentamers have also been reported.<ref>{{Cite journal |last1=Kumar |first1=Nikit |last2=Arthur |first2=Christopher P. |last3=Ciferri |first3=Claudio |last4=Matsumoto |first4=Marissa L. |date=2020-02-28 |title=Structure of the secretory immunoglobulin A core |url=https://www.science.org/doi/10.1126/science.aaz5807 |journal=Science |language=en |volume=367 |issue=6481 |pages=1008–1014 |doi=10.1126/science.aaz5807 |pmid=32029686 |bibcode=2020Sci...367.1008K |issn=0036-8075}}</ref>

Antibodies also form complexes by binding to antigen: this is called an [[antigen-antibody complex]] or ''immune complex''.
Small antigens can cross-link two antibodies, also leading to the formation of antibody dimers, trimers, tetramers, etc.
Multivalent antigens (e.g., cells with multiple epitopes) can form larger complexes with antibodies.
An extreme example is the clumping, or [[Agglutination (biology)|agglutination]], of [[red blood cell]]s with antibodies in the [[Coombs test]] to determine [[blood group]]s: the large clumps become insoluble, leading to visually apparent [[precipitation (chemistry)|precipitation]].

===B cell receptors===
{{main|B-cell receptor}}
The membrane-bound form of an antibody may be called a ''surface immunoglobulin'' (sIg) or a ''membrane immunoglobulin'' (mIg). It is part of the ''B cell receptor'' (BCR), which allows a B cell to detect when a specific antigen is present in the body and triggers B cell activation.<ref name="T cell-dependent B cell activation">{{cite journal|vauthors=Parker DC|year=1993|title=T cell-dependent B cell activation|journal=Annual Review of Immunology|volume=11|issue=1|pages=331–60|doi=10.1146/annurev.iy.11.040193.001555|pmid=8476565}}</ref> The BCR is composed of surface-bound IgD or IgM antibodies and associated Ig-α and Ig-β [[heterodimer]]s, which are capable of [[signal transduction]].<ref name="Wintrobe">{{cite book | vauthors = Wintrobe MM |author-link= Maxwell Wintrobe| veditors = Greer JG, Foerster F, Lukens JN, Rodgers GM, Paraskevas F |title=Wintrobe's clinical hematology|edition=11|publisher=Lippincott Williams & Wilkins|location=Hagerstown, MD|year=2004|pages=453–456|isbn=978-0-7817-3650-3}}</ref> A typical human B cell will have 50,000 to 100,000 antibodies bound to its surface.<ref name="Wintrobe" /> Upon antigen binding, they cluster in large patches, which can exceed 1 micrometer in diameter, on lipid rafts that isolate the BCRs from most other [[cell signaling]] receptors.<ref name="Wintrobe" />
These patches may improve the efficiency of the [[Cell-mediated immunity|cellular immune response]].<ref name="pmid18275475">{{cite journal | vauthors = Tolar P, Sohn HW, Pierce SK | title = Viewing the antigen-induced initiation of B-cell activation in living cells | journal = Immunological Reviews | volume = 221 | issue = 1 | pages = 64–76 | date = February 2008 | pmid = 18275475 | doi = 10.1111/j.1600-065X.2008.00583.x | s2cid = 38464264 | url = https://zenodo.org/record/1230708 }}</ref> In humans, the cell surface is bare around the B cell receptors for several hundred nanometers,<ref name="Wintrobe"/> which further isolates the BCRs from competing influences.

==Classes==
Antibodies can come in different varieties known as ''[[Isotype (immunology)|isotypes]]'' or ''classes''. In humans there are five antibody classes known as IgA, IgD, IgE, IgG, and IgM, which are further subdivided into subclasses such as IgA1, IgA2.
The prefix "Ig" stands for ''immunoglobulin'', while the suffix denotes the type of heavy chain the antibody contains: the heavy chain types α (alpha), γ (gamma), δ (delta), ε (epsilon), μ (mu) give rise to IgA, IgG, IgD, IgE, IgM, respectively.
The distinctive features of each class are determined by the part of the heavy chain within the hinge and Fc region.<ref name="Janeway5"/>

The classes differ in their biological properties, functional locations and ability to deal with different antigens, as depicted in the table.<ref name = woof>{{cite journal | vauthors = Woof JM, Burton DR | title = Human antibody-Fc receptor interactions illuminated by crystal structures | journal = Nature Reviews. Immunology | volume = 4 | issue = 2 | pages = 89–99 | date = February 2004 | pmid = 15040582 | doi = 10.1038/nri1266 | s2cid = 30584218 }}</ref>
For example, [[IgE]] antibodies are responsible for an [[allergic]] response consisting of [[histamine]] release from [[mast cell]]s, often a sole contributor to [[asthma]] (though other pathways exist as do exist symptoms very similar to yet not technically asthma). The antibody's variable region binds to allergic antigen, for example [[house dust mite]] particles, while its Fc region (in the ε heavy chains) binds to [[FcεRI|Fc receptor ε]] on a mast cell, triggering its [[degranulation]]: the release of molecules stored in its granules.<ref>{{cite journal | vauthors = Williams CM, Galli SJ | title = The diverse potential effector and immunoregulatory roles of mast cells in allergic disease | journal = The Journal of Allergy and Clinical Immunology | volume = 105 | issue = 5 | pages = 847–59 | date = May 2000 | pmid = 10808163 | doi = 10.1067/mai.2000.106485 |doi-access=free }}</ref>

{| class="wikitable" style="width:100%; text-align:center;"
|+ Antibody isotypes of humans
! Class !! Subclasses !! Description
|-
| [[IgA]] || 2
| style="text-align:left;" | Found in [[mucosal]] areas, such as the [[Gut (zoology)|gut]], [[respiratory tract]] and [[urogenital tract]], and prevents colonization by [[pathogen]]s.<ref>{{cite journal | vauthors = Underdown BJ, Schiff JM | title = Immunoglobulin A: strategic defense initiative at the mucosal surface | journal = Annual Review of Immunology | volume = 4 | issue = 1 | pages = 389–417 | year = 1986 | pmid = 3518747 | doi = 10.1146/annurev.iy.04.040186.002133 }}</ref> Also found in saliva, tears, and breast milk.
|-
| [[IgD]] || 1
| style="text-align:left;" | Functions mainly as an antigen receptor on B cells that have not been exposed to antigens.<ref name=Geisberger>{{cite journal | vauthors = Geisberger R, Lamers M, Achatz G | title = The riddle of the dual expression of IgM and IgD | journal = Immunology | volume = 118 | issue = 4 | pages = 429–37 | date = August 2006 | pmid = 16895553 | pmc = 1782314 | doi = 10.1111/j.1365-2567.2006.02386.x }}</ref> It has been shown to activate [[basophil]]s and [[mast cell]]s to produce [[antimicrobial]] factors.<ref name=Chen>{{cite journal | vauthors = Chen K, Xu W, Wilson M, He B, Miller NW, Bengtén E, Edholm ES, Santini PA, Rath P, Chiu A, Cattalini M, Litzman J, B Bussel J, Huang B, Meini A, Riesbeck K, Cunningham-Rundles C, Plebani A, Cerutti A | title = Immunoglobulin D enhances immune surveillance by activating antimicrobial, proinflammatory and B cell-stimulating programs in basophils | journal = Nature Immunology | volume = 10 | issue = 8 | pages = 889–98 | date = August 2009 | pmid = 19561614 | pmc = 2785232 | doi = 10.1038/ni.1748 }}</ref>
|-
| [[IgE]] || 1
| style="text-align:left;" | Binds to [[allergen]]s and triggers [[histamine]] release from [[mast cell]]s and [[basophil]]s, and is involved in [[allergy]]. Humans and other animals evolved IgE to protect against [[parasitic worm]]s, though in the present, IgE is primarily related to allergies and asthma.<ref name="Pier">{{cite book |title=Immunology, Infection, and Immunity |vauthors=Pier GB, Lyczak JB, Wetzler LM |publisher=ASM Press |year=2004 |isbn=978-1-55581-246-1}}</ref>
|-
| [[IgG]] || 4
| style="text-align:left;" | In its four forms, provides the majority of antibody-based immunity against invading pathogens.<ref name=Pier/> The only antibody capable of crossing the [[placenta]] to give passive immunity to the [[fetus]].
|-
| [[IgM]]
| 1
| style="text-align:left;" | Expressed on the surface of B cells (monomer) and in a secreted form (pentamer) with very high [[avidity]]. Eliminates pathogens in the early stages of B cell-mediated (humoral) immunity before there is sufficient IgG.<ref name=Pier/><ref name=Geisberger/>
|}

The antibody isotype of a B cell changes during cell [[Pre-pre B cell|development]] and activation. Immature B cells, which have never been exposed to an antigen, express only the IgM isotype in a cell surface bound form. The B lymphocyte, in this ready-to-respond form, is known as a "[[Naive B cell|naive B lymphocyte]]." The naive B lymphocyte expresses both surface IgM and IgD. The co-expression of both of these immunoglobulin isotypes renders the B cell ready to respond to antigen.<ref name=Goding>{{cite book | vauthors = Goding JW | title = Contemporary Topics in Immunobiology | chapter = Allotypes of IgM and IgD Receptors in the Mouse: A Probe for Lymphocyte Differentiation | volume = 8 | pages = 203–43 | date = 1978 | pmid = 357078 | doi = 10.1007/978-1-4684-0922-2_7 |isbn = 978-1-4684-0924-6}}</ref> B cell activation follows engagement of the cell-bound antibody molecule with an antigen, causing the cell to divide and [[Cellular differentiation|differentiate]] into an antibody-producing cell called a [[plasma cell]]. In this activated form, the B cell starts to produce antibody in a [[Secretion|secreted]] form rather than a [[cell membrane|membrane]]-bound form. Some [[daughter cell]]s of the activated B cells undergo [[isotype switching]], a mechanism that causes the production of antibodies to change from IgM or IgD to the other antibody isotypes, IgE, IgA, or IgG, that have defined roles in the immune system.

===Light chain types===
{{Further|Immunoglobulin light chain}}
In mammals there are two types of [[immunoglobulin light chain]], which are called [[lambda]] (λ) and [[kappa]] (κ). However, there is no known functional difference between them, and both can occur with any of the five major types of heavy chains.<ref name=Janeway5/> Each antibody contains two identical light chains: both κ or both λ. Proportions of κ and λ types vary by species and can be used to detect abnormal proliferation of B cell clones. Other types of light chains, such as the [[iota]] (ι) chain, are found in other [[vertebrate]]s like sharks ([[Chondrichthyes]]) and bony fishes ([[Teleostei]]).

===In non-mammalian animals===
In most [[placental mammal]]s, the structure of antibodies is generally the same.
[[Jawed fish]] appear to be the most primitive animals that are able to make antibodies similar to those of mammals, although many features of their adaptive immunity appeared somewhat earlier.<ref>{{cite journal | vauthors = Litman GW, Rast JP, Fugmann SD | title = The origins of vertebrate adaptive immunity | journal = Nature Reviews. Immunology | volume = 10 | issue = 8 | pages = 543–53 | date = August 2010 | pmid = 20651744 | pmc = 2919748 | doi = 10.1038/nri2807 }}</ref>

[[Cartilaginous fish]] (such as sharks) produce [[Heavy-chain antibody|heavy-chain-only antibodies]] (i.e., lacking light chains) which moreover feature longer chain [[pentamer]]s (with five constant units per molecule). [[Camelids]] (such as camels, llamas, alpacas) are also notable for producing heavy-chain-only antibodies.<ref name=Janeway5/><ref>{{cite journal | vauthors = Litman GW, Rast JP, Fugmann SD | title = The origins of vertebrate adaptive immunity | journal = Nature Reviews. Immunology | volume = 10 | issue = 8 | pages = 543–53 | date = August 2010 | pmid = 20651744 | pmc = 2919748 | doi = 10.1002/9783527699124.ch4 | publisher = John Wiley & Sons, Ltd | isbn = 978-3-527-69912-4 }}</ref>

{| class="wikitable"
|+ Antibody classes not found in mammals
|-
! Class !! Types !! Description
|-
| [[Immunoglobulin Y|IgY]] || || Found in [[bird]]s and [[reptile]]s; related to mammalian IgG.<ref>{{cite journal | vauthors = Lundqvist ML, Middleton DL, Radford C, Warr GW, Magor KE | title = Immunoglobulins of the non-galliform birds: antibody expression and repertoire in the duck | journal = Developmental and Comparative Immunology | volume = 30 | issue = 1–2 | pages = 93–100 | date = 2006 | pmid = 16150486 | pmc = 1317265 | doi = 10.1016/j.dci.2005.06.019 }}</ref>
|-
| IgW || || Found in [[Elasmobranchii|sharks and skates]]; related to mammalian IgD.<ref>{{cite journal | vauthors = Berstein RM, Schluter SF, Shen S, Marchalonis JJ | title = A new high molecular weight immunoglobulin class from the carcharhine shark: implications for the properties of the primordial immunoglobulin | journal = Proceedings of the National Academy of Sciences of the United States of America | volume = 93 | issue = 8 | pages = 3289–93 | date = April 1996 | pmid = 8622930 | pmc = 39599 | doi = 10.1073/pnas.93.8.3289 | bibcode = 1996PNAS...93.3289B | doi-access = free }}</ref>
|-
| IgT/Z || || Found in [[Teleost|teleost fish]]<ref>Salinas, I., & Parra, D. (2015). Fish mucosal immunity: Intestine. In Mucosal Health in Aquaculture. Elsevier Inc. https://doi.org/10.1016/B978-0-12-417186-2.00006-6</ref>
|}

==Antibody–antigen interactions==
The antibody's paratope interacts with the antigen's epitope. An antigen usually contains different epitopes along its surface arranged discontinuously, and dominant epitopes on a given antigen are called determinants.

Antibody and antigen interact by spatial complementarity (lock and key). The molecular forces involved in the Fab-epitope interaction are weak and non-specific – for example [[Coulomb's law|electrostatic forces]], [[hydrogen bond]]s, [[hydrophobic interactions]], and [[van der Waals force]]s. This means binding between antibody and antigen is reversible, and the antibody's [[Affinity (pharmacology)|affinity]] towards an antigen is relative rather than absolute. Relatively weak binding also means it is possible for an antibody to [[Cross-reactivity|cross-react]] with different antigens of different relative affinities.

== Function ==
{{Further|Immune system}}
[[File:Antibody Opsonization.svg|thumb|{{ordered list
| list_style=margin-left:1em;
| Antibodies (A) and pathogens (B) free roam in the blood.
| The antibodies bind to pathogens, and can do so in different formations such as:{{ordered list
 | list_style=margin-left:2em;
 | list-style-type=lower-alpha
 | opsonization,
 | neutralisation, and
 | agglutination.
 }}
| A phagocyte (C) approaches the pathogen, and the Fc region (D) of the antibody binds to one of the Fc receptors (E) of the phagocyte.
| Phagocytosis occurs as the pathogen is ingested.
}}]]

The main categories of antibody action include the following:

* [[Neutralisation (immunology)|Neutralisation]], in which [[neutralizing antibody|neutralizing antibodies]] block parts of the surface of a bacterial cell or virion to render its attack ineffective
* [[Agglutination (biology)|Agglutination]], in which antibodies "glue together" foreign cells into clumps that are attractive targets for [[phagocytosis]]
* [[Precipitation (chemistry)|Precipitation]], in which antibodies "glue together" [[blood serum|serum]]-soluble antigens, forcing them to precipitate out of solution in clumps that are attractive targets for [[phagocytosis]]
* [[Complement system#Overview|Complement activation]] (fixation), in which antibodies that are latched onto a foreign cell encourage complement to attack it with a [[membrane attack complex]], which leads to the following:
** [[Lysis]] of the foreign cell
** Encouragement of [[inflammation]] by [[chemotaxis|chemotactically]] attracting inflammatory cells

More indirectly, an antibody can signal immune cells to present antibody fragments to [[T cell]]s, or [[downregulate]] other immune cells to avoid [[autoimmunity]].

Activated B cells [[cellular differentiation|differentiate]] into either antibody-producing cells called [[plasma cell]]s that secrete soluble antibody or [[memory B cell|memory cells]] that survive in the body for years afterward in order to allow the immune system to remember an antigen and respond faster upon future exposures.<ref name="pmid17337763">{{cite journal |vauthors=Borghesi L, Milcarek C |year=2006 |title=From B cell to plasma cell: regulation of V(D)J recombination and antibody secretion |journal=Immunologic Research |volume=36 |issue=1–3 |pages=27–32 |doi=10.1385/IR:36:1:27 |pmid=17337763 |s2cid=27041937 |doi-access=free}}</ref>

At the [[prenatal]] and neonatal stages of life, the presence of antibodies is provided by [[passive immunization]] from the mother. Early endogenous antibody production varies for different kinds of antibodies, and usually appear within the first years of life. Since antibodies exist freely in the bloodstream, they are said to be part of the [[humoral immune system]]. Circulating antibodies are produced by clonal B cells that specifically respond to only one [[antigen]] (an example is a [[virus]] [[capsid|capsid protein]] fragment). Antibodies contribute to [[immunity (medical)|immunity]] in three ways: They prevent pathogens from entering or damaging cells by binding to them; they stimulate removal of pathogens by [[macrophages]] and other cells by coating the pathogen; and they trigger destruction of pathogens by stimulating other [[immune response]]s such as the [[complement system|complement pathway]].<ref name=Ravetch>{{cite journal | vauthors = Ravetch JV, Bolland S | title = IgG Fc receptors | journal = Annual Review of Immunology | volume = 19 | issue = 1 | pages = 275–90 | year = 2001 | pmid = 11244038 | doi = 10.1146/annurev.immunol.19.1.275 }}</ref> Antibodies will also trigger vasoactive amine degranulation to contribute to immunity against certain types of antigens (helminths, allergens).

[[File:IgM white background.png|thumb|left|The secreted mammalian [[IgM]] has five Ig units. Each Ig unit (labeled 1) has two epitope binding [[Fab region]]s, so IgM is capable of binding up to 10 epitopes.]]

===Activation of complement===
Antibodies that bind to surface antigens (for example, on bacteria) will attract the first component of the [[complement cascade]] with their [[Fv region|Fc region]] and initiate activation of the "classical" complement system.<ref name=Ravetch/> This results in the killing of bacteria in two ways.<ref name=Pier/> First, the binding of the antibody and complement molecules marks the microbe for ingestion by [[phagocyte]]s in a process called [[opsonization]]; these phagocytes are attracted by certain complement molecules generated in the complement cascade. Second, some complement system components form a [[Complement membrane attack complex|membrane attack complex]] to assist antibodies to kill the bacterium directly (bacteriolysis).<ref>{{cite journal | vauthors = Rus H, Cudrici C, Niculescu F | title = The role of the complement system in innate immunity | journal = Immunologic Research | volume = 33 | issue = 2 | pages = 103–12 | year = 2005 | pmid = 16234578 | doi = 10.1385/IR:33:2:103 | s2cid = 46096567 }}</ref>

===Activation of effector cells===
To combat pathogens that replicate outside cells, antibodies bind to pathogens to link them together, causing them to [[Agglutination (biology)|agglutinate]]. Since an antibody has at least two paratopes, it can bind more than one antigen by binding identical epitopes carried on the surfaces of these antigens. By coating the pathogen, antibodies stimulate effector functions against the pathogen in cells that recognize their Fc region.<ref name=Pier/>

Those cells that recognize coated pathogens have Fc receptors, which, as the name suggests, interact with the [[Fc region]] of IgA, IgG, and IgE antibodies. The engagement of a particular antibody with the Fc receptor on a particular cell triggers an effector function of that cell; phagocytes will [[phagocytosis|phagocytose]], [[mast cell]]s and [[neutrophil]]s will [[degranulation|degranulate]], [[natural killer cell]]s will release [[cytokine]]s and [[cytotoxic]] molecules; that will ultimately result in destruction of the invading microbe. The activation of natural killer cells by antibodies initiates a cytotoxic mechanism known as [[antibody-dependent cell-mediated cytotoxicity]] (ADCC) – this process may explain the efficacy of [[Monoclonal antibody|monoclonal antibodies]] used in [[biopharmaceutical|biological]] therapies against [[cancer]]. The Fc receptors are isotype-specific, which gives greater flexibility to the immune system, invoking only the appropriate immune mechanisms for distinct pathogens.<ref name=Janeway5/>

===Natural antibodies===
Humans and higher primates also produce "natural antibodies" that are present in serum before viral infection. Natural antibodies have been defined as antibodies that are produced without any previous infection, [[vaccination]], other foreign antigen exposure or [[passive immunization]]. These antibodies can activate the classical complement pathway leading to lysis of enveloped virus particles long before the adaptive immune response is activated. Many natural antibodies are directed against the disaccharide [[galactose]] α(1,3)-galactose (α-Gal), which is found as a terminal sugar on [[Glycosylation|glycosylated]] cell surface proteins, and generated in response to production of this sugar by bacteria contained in the human gut.<ref>{{cite news|author=Racaniello, Vincent |url=http://www.virology.ws/2009/10/06/natural-antibody-protects-against-viral-infection/ |date=6 October 2009 |title=Natural antibody protects against viral infection |website=Virology Blog |access-date=22 January 2010 |archive-url=https://web.archive.org/web/20100220015318/http://www.virology.ws/2009/10/06/natural-antibody-protects-against-viral-infection/ |archive-date=20 February 2010 |url-status=live}}</ref> Rejection of [[Organ xenotransplantation|xenotransplantated organs]] is thought to be, in part, the result of natural antibodies circulating in the serum of the recipient binding to α-Gal antigens expressed on the donor tissue.<ref>{{cite journal | vauthors = Milland J, Sandrin MS | title = ABO blood group and related antigens, natural antibodies and transplantation | journal = Tissue Antigens | volume = 68 | issue = 6 | pages = 459–66 | date = December 2006 | pmid = 17176435 | doi = 10.1111/j.1399-0039.2006.00721.x }}</ref>

==Immunoglobulin diversity==
Virtually all microbes can trigger an antibody response. Successful recognition and eradication of many different types of microbes requires diversity among antibodies; their amino acid composition varies allowing them to interact with many different antigens.<ref>{{cite journal | vauthors = Mian IS, Bradwell AR, Olson AJ | title = Structure, function and properties of antibody binding sites | journal = Journal of Molecular Biology | volume = 217 | issue = 1 | pages = 133–51 | date = January 1991 | pmid = 1988675 | doi = 10.1016/0022-2836(91)90617-F }}</ref> It has been estimated that humans generate about 10 billion different antibodies, each capable of binding a distinct epitope of an antigen.<ref name="pmid8612345">{{cite journal | vauthors = Fanning LJ, Connor AM, Wu GE | title = Development of the immunoglobulin repertoire | journal = Clinical Immunology and Immunopathology | volume = 79 | issue = 1 | pages = 1–14 | date = April 1996 | pmid = 8612345 | doi = 10.1006/clin.1996.0044 }}</ref> Although a huge repertoire of different antibodies is generated in a single individual, the number of [[gene]]s available to make these proteins is limited by the size of the human genome. Several complex genetic mechanisms have evolved that allow vertebrate B cells to generate a diverse pool of antibodies from a relatively small number of antibody genes.<ref name = namazee>{{cite journal | vauthors = Nemazee D | title = Receptor editing in lymphocyte development and central tolerance | journal = Nature Reviews. Immunology | volume = 6 | issue = 10 | pages = 728–40 | date = October 2006 | pmid = 16998507 | doi = 10.1038/nri1939 | s2cid = 2234228 }}</ref>

===Domain variability===
[[File:Complementarity determining regions.PNG|thumb|upright=1.25|The complementarity determining regions of the heavy chain are shown in red ({{PDB|1IGT}})]]
The chromosomal region that encodes an antibody is large and contains several distinct gene loci for each domain of the antibody—the chromosome region containing heavy chain genes ([[IGH@]]) is found on [[chromosome 14]], and the loci containing lambda and kappa light chain genes ([[IGL@]] and [[IGK@]]) are found on chromosomes [[chromosome 22|22]] and [[chromosome 2|2]] in humans. One of these domains is called the variable domain, which is present in each heavy and light chain of every antibody, but can differ in different antibodies generated from distinct B cells. Differences between the variable domains are located on three loops known as hypervariable regions (HV-1, HV-2 and HV-3) or [[complementarity-determining region]]s (CDR1, CDR2 and CDR3). CDRs are supported within the variable domains by conserved framework regions. The heavy chain locus contains about 65 different variable domain genes that all differ in their CDRs. Combining these genes with an array of genes for other domains of the antibody generates a large cavalry of antibodies with a high degree of variability. This combination is called V(D)J recombination discussed below.<ref>Peter Parham. ''The Immune System''. 2nd ed. Garland Science: New York, 2005. pg.47–62</ref>

===V(D)J recombination===
{{Further|V%28D%29J recombination}}
[[File:VDJ recombination.png|thumb|upright=1.25|Simplified overview of V(D)J recombination of immunoglobulin heavy chains]]
Somatic recombination of immunoglobulins, also known as ''V(D)J recombination'', involves the generation of a unique immunoglobulin variable region. The variable region of each immunoglobulin heavy or light chain is encoded in several pieces—known as gene segments (subgenes). These segments are called variable (V), diversity (D) and joining (J) segments.<ref name = namazee/> V, D and J segments are found in [[immunoglobulin heavy chain|Ig heavy chains]], but only V and J segments are found in [[Immunoglobulin light chain|Ig light chains]]. Multiple copies of the V, D and J gene segments exist, and are tandemly arranged in the [[genome]]s of [[mammal]]s. In the bone marrow, each developing B cell will assemble an immunoglobulin variable region by randomly selecting and combining one V, one D and one J gene segment (or one V and one J segment in the light chain). As there are multiple copies of each type of gene segment, and different combinations of gene segments can be used to generate each immunoglobulin variable region, this process generates a huge number of antibodies, each with different [[wikt:paratope|paratopes]], and thus different antigen specificities.<ref name="Market">{{cite journal | vauthors = Market E, Papavasiliou FN | title = V(D)J recombination and the evolution of the adaptive immune system | journal = PLOS Biology | volume = 1 | issue = 1 | pages = E16 | date = October 2003 | pmid = 14551913 | pmc = 212695 | doi = 10.1371/journal.pbio.0000016 | doi-access = free }}</ref> The rearrangement of several subgenes (i.e. V2 family) for lambda light chain immunoglobulin is coupled with the activation of microRNA miR-650, which further influences biology of B-cells.

[[Recombination-activating gene|RAG]] proteins play an important role with V(D)J recombination in cutting DNA at a particular region.<ref name="Market"/> Without the presence of these proteins, V(D)J recombination would not occur.<ref name="Market"/>

After a B cell produces a functional immunoglobulin gene during V(D)J recombination, it cannot express any other variable region (a process known as [[allelic exclusion]]) thus each B cell can produce antibodies containing only one kind of variable chain.<ref name=Janeway5/><ref>{{cite journal | vauthors = Bergman Y, Cedar H | title = A stepwise epigenetic process controls immunoglobulin allelic exclusion | journal = Nature Reviews. Immunology | volume = 4 | issue = 10 | pages = 753–61 | date = October 2004 | pmid = 15459667 | doi = 10.1038/nri1458 | s2cid = 8579156 }}</ref>

===Somatic hypermutation and affinity maturation===
{{Further|Somatic hypermutation|Affinity maturation}}

Following activation with antigen, B cells begin to [[Cell division|proliferate]] rapidly. In these rapidly dividing cells, the genes encoding the variable domains of the heavy and light chains undergo a high rate of [[point mutation]], by a process called ''somatic hypermutation'' (SHM). SHM results in approximately one [[nucleotide]] change per variable gene, per cell division.<ref name=diaz>{{cite journal | vauthors = Diaz M, Casali P | title = Somatic immunoglobulin hypermutation | journal = Current Opinion in Immunology | volume = 14 | issue = 2 | pages = 235–40 | date = April 2002 | pmid = 11869898 | pmc = 4621002 | doi = 10.1016/S0952-7915(02)00327-8 }}</ref> As a consequence, any daughter B cells will acquire slight [[amino acid]] differences in the variable domains of their antibody chains.

This serves to increase the diversity of the antibody pool and impacts the antibody's antigen-binding [[Chemical affinity|affinity]].<ref>{{cite journal | vauthors = Honjo T, Habu S | title = Origin of immune diversity: genetic variation and selection | journal = Annual Review of Biochemistry | volume = 54 | issue = 1 | pages = 803–30 | year = 1985 | pmid = 3927822 | doi = 10.1146/annurev.bi.54.070185.004103 }}</ref> Some point mutations will result in the production of antibodies that have a weaker interaction (low affinity) with their antigen than the original antibody, and some mutations will generate antibodies with a stronger interaction (high affinity).<ref name=orguil>{{cite journal | vauthors = Or-Guil M, Wittenbrink N, Weiser AA, Schuchhardt J | title = Recirculation of germinal center B cells: a multilevel selection strategy for antibody maturation | journal = Immunological Reviews | volume = 216 | pages = 130–41 | date = April 2007 | pmid = 17367339 | doi = 10.1111/j.1600-065X.2007.00507.x | s2cid = 37636392 }}</ref> B cells that express high affinity antibodies on their surface will receive a strong survival signal during interactions with other cells, whereas those with low affinity antibodies will not, and will die by [[apoptosis]].<ref name=orguil/> Thus, B cells expressing antibodies with a higher affinity for the antigen will outcompete those with weaker affinities for function and survival allowing the average affinity of antibodies to increase over time. The process of generating antibodies with increased binding affinities is called ''affinity maturation''. Affinity maturation occurs in mature B cells after V(D)J recombination, and is dependent on help from [[helper T cell]]s.<ref>{{cite journal | vauthors = Neuberger MS, Ehrenstein MR, Rada C, Sale J, Batista FD, Williams G, Milstein C | title = Memory in the B-cell compartment: antibody affinity maturation | journal = Philosophical Transactions of the Royal Society of London. Series B, Biological Sciences | volume = 355 | issue = 1395 | pages = 357–60 | date = March 2000 | pmid = 10794054 | pmc = 1692737 | doi = 10.1098/rstb.2000.0573 }}</ref>

===Class switching===
[[File:Class switch recombination.png|thumb|upright=1.25|Mechanism of class switch recombination that allows isotype switching in activated B cells]]
[[Immunoglobulin class switching|Isotype or class switching]] is a [[biological process]] occurring after activation of the B cell, which allows the cell to produce different classes of antibody (IgA, IgE, or IgG).<ref name=Market/> The different classes of antibody, and thus effector functions, are defined by the constant (C) regions of the immunoglobulin heavy chain. Initially, naive B cells express only cell-surface IgM and IgD with identical antigen binding regions. Each isotype is adapted for a distinct function; therefore, after activation, an antibody with an IgG, IgA, or IgE effector function might be required to effectively eliminate an antigen. Class switching allows different daughter cells from the same activated B cell to produce antibodies of different isotypes. Only the constant region of the antibody heavy chain changes during class switching; the variable regions, and therefore antigen specificity, remain unchanged. Thus the progeny of a single B cell can produce antibodies, all specific for the same antigen, but with the ability to produce the effector function appropriate for each antigenic challenge. Class switching is triggered by cytokines; the isotype generated depends on which cytokines are present in the B cell environment.<ref>{{cite journal | vauthors = Stavnezer J, Amemiya CT | title = Evolution of isotype switching | journal = Seminars in Immunology | volume = 16 | issue = 4 | pages = 257–75 | date = August 2004 | pmid = 15522624 | doi = 10.1016/j.smim.2004.08.005 }}</ref>

Class switching occurs in the heavy chain gene [[Locus (genetics)|locus]] by a mechanism called class switch recombination (CSR). This mechanism relies on conserved [[nucleotide]] motifs, called ''switch (S) regions'', found in [[DNA]] upstream of each constant region gene (except in the δ-chain). The DNA strand is broken by the activity of a series of [[enzyme]]s at two selected S-regions.<ref>{{cite journal | vauthors = Durandy A | title = Activation-induced cytidine deaminase: a dual role in class-switch recombination and somatic hypermutation | journal = European Journal of Immunology | volume = 33 | issue = 8 | pages = 2069–73 | date = August 2003 | pmid = 12884279 | doi = 10.1002/eji.200324133 | s2cid = 32059768 }}</ref><ref>{{cite journal | vauthors = Casali P, Zan H | title = Class switching and Myc translocation: how does DNA break? | journal = Nature Immunology | volume = 5 | issue = 11 | pages = 1101–3 | date = November 2004 | pmid = 15496946 | pmc = 4625794 | doi = 10.1038/ni1104-1101 }}</ref> The variable domain [[exon]] is rejoined through a process called [[non-homologous end joining]] (NHEJ) to the desired constant region (γ, α or ε). This process results in an immunoglobulin gene that encodes an antibody of a different isotype.<ref>{{cite journal | vauthors = Lieber MR, Yu K, Raghavan SC | title = Roles of nonhomologous DNA end joining, V(D)J recombination, and class switch recombination in chromosomal translocations | journal = DNA Repair | volume = 5 | issue = 9–10 | pages = 1234–45 | date = September 2006 | pmid = 16793349 | doi = 10.1016/j.dnarep.2006.05.013 }}</ref>

===Specificity designations===
{{anchor|valence}}An antibody can be called ''monospecific'' if it has specificity for a single antigen or epitope,<ref>[https://books.google.com/books?id=TfW5sUfeM5gC&pg=PA22 p. 22] in: {{Cite book | vauthors = Shoenfeld Y, Meroni PL, Gershwin ME | title = Autoantibodie | year = 2007 | publisher = Elsevier | location = Amsterdam; Boston | isbn = 978-0-444-52763-9 }}
</ref> or bispecific if it has affinity for two different antigens or two different epitopes on the same antigen.<ref>{{cite journal | vauthors = Spiess C, Zhai Q, Carter PJ | title = Alternative molecular formats and therapeutic applications for bispecific antibodies | journal = Molecular Immunology | volume = 67 | issue = 2 Pt A | pages = 95–106 | date = October 2015 | pmid = 25637431 | doi = 10.1016/j.molimm.2015.01.003 | doi-access = free }}</ref> A group of antibodies can be called ''polyvalent'' (or ''unspecific'') if they have affinity for various antigens<ref name=farlex-polyvalent/> or microorganisms.<ref name=farlex-polyvalent>[http://medical-dictionary.thefreedictionary.com/polyvalent Farlex dictionary > polyvalent] Citing: The American Heritage Medical Dictionary. 2004</ref> [[Intravenous immunoglobulin]], if not otherwise noted, consists of a variety of different IgG (polyclonal IgG). In contrast, [[monoclonal antibodies]] are identical antibodies produced by a single B cell.

===Asymmetrical antibodies===
Heterodimeric antibodies, which are also asymmetrical antibodies, allow for greater flexibility and new formats for attaching a variety of drugs to the antibody arms. One of the general formats for a heterodimeric antibody is the "knobs-into-holes" format. This format is specific to the heavy chain part of the constant region in antibodies. The "knobs" part is engineered by replacing a small amino acid with a larger one. It fits into the "hole", which is engineered by replacing a large amino acid with a smaller one. What connects the "knobs" to the "holes" are the disulfide bonds between each chain. The "knobs-into-holes" shape facilitates antibody dependent cell mediated cytotoxicity. [[Single-chain variable fragment|Single-chain variable fragments]] ([[scFv]]) are connected to the variable domain of the heavy and light chain via a short linker peptide. The linker is rich in glycine, which gives it more flexibility, and serine/threonine, which gives it specificity. Two different scFv fragments can be connected together, via a hinge region, to the constant domain of the heavy chain or the constant domain of the light chain.<ref>{{cite journal | vauthors = Gunasekaran K, Pentony M, Shen M, Garrett L, Forte C, Woodward A, Ng SB, Born T, Retter M, Manchulenko K, Sweet H, Foltz IN, Wittekind M, Yan W | title = Enhancing antibody Fc heterodimer formation through electrostatic steering effects: applications to bispecific molecules and monovalent IgG | journal = The Journal of Biological Chemistry | volume = 285 | issue = 25 | pages = 19637–46 | date = June 2010 | pmid = 20400508 | pmc = 2885242 | doi = 10.1074/jbc.M110.117382 | doi-access = free }}</ref> This gives the antibody bispecificity, allowing for the binding specificities of two different antigens.<ref>{{cite journal | vauthors = Muller KM |title=The first constant domain (CH1 and CL) of an antibody used as heterodimerization domain for bispecific miniantibodies |journal=FEBS Letters |volume=422 |issue=2 |pages=259–264 |year=1998 | doi = 10.1016/s0014-5793(98)00021-0 |pmid=9490020 |s2cid=35243494 |doi-access=free }}</ref> The "knobs-into-holes" format enhances heterodimer formation but does not suppress homodimer formation.

To further improve the function of heterodimeric antibodies, many scientists are looking towards artificial constructs. Artificial antibodies are largely diverse protein motifs that use the functional strategy of the antibody molecule, but are not limited by the loop and framework structural constraints of the natural antibody.<ref>{{cite journal | vauthors = Gao C, Mao S, Lo CH, Wirsching P, Lerner RA, Janda KD | title = Making artificial antibodies: a format for phage display of combinatorial heterodimeric arrays | journal = Proceedings of the National Academy of Sciences of the United States of America | volume = 96 | issue = 11 | pages = 6025–30 | date = May 1999 | pmid = 10339535 | pmc = 26829 | doi = 10.1073/pnas.96.11.6025 | bibcode = 1999PNAS...96.6025G | doi-access = free }}</ref> Being able to control the combinational design of the sequence and three-dimensional space could transcend the natural design and allow for the attachment of different combinations of drugs to the arms.

Heterodimeric antibodies have a greater range in shapes they can take and the drugs that are attached to the arms do not have to be the same on each arm, allowing for different combinations of drugs to be used in cancer treatment. Pharmaceuticals are able to produce highly functional bispecific, and even multispecific, antibodies. The degree to which they can function is impressive given that such a change of shape from the natural form should lead to decreased functionality.

=== Interchromosomal DNA Transposition ===
Antibody diversification typically occurs through somatic hypermutation, class switching, and affinity maturation targeting the BCR gene loci, but on occasion more unconventional forms of diversification have been documented.<ref>{{Cite journal |last1=Kanyavuz |first1=Alexia |last2=Marey-Jarossay |first2=Annaelle |last3=Lacroix-Desmazes |first3=Sébastien |last4=Dimitrov |first4=Jordan D. |date=June 2019 |title=Breaking the law: unconventional strategies for antibody diversification |url=https://www.nature.com/articles/s41577-019-0126-7 |journal=Nature Reviews Immunology |language=en |volume=19 |issue=6 |pages=355–368 |doi=10.1038/s41577-019-0126-7 |pmid=30718829 |s2cid=59603663 |issn=1474-1741}}</ref> For example, in the case of [[malaria]] caused by ''[[Plasmodium falciparum]],'' some antibodies from those who had been infected demonstrated an insertion from chromosome 19 containing a 98-amino acid stretch from leukocyte-associated immunoglobulin-like receptor 1, [[LAIR1]], in the elbow joint. This represents a form of interchromosomal transposition. LAIR1 normally binds collagen, but can recognize repetitive interspersed families of polypeptides (RIFIN) family members that are highly expressed on the surface of ''P. falciparum''-infected red blood cells. In fact, these antibodies underwent affinity maturation that enhanced affinity for RIFIN but abolished affinity for collagen. These "LAIR1-containing" antibodies have been found in 5-10% of donors from Tanzania and Mali, though not in European donors.<ref>{{Cite journal |last1=Pieper |first1=Kathrin |last2=Tan |first2=Joshua |last3=Piccoli |first3=Luca |last4=Foglierini |first4=Mathilde |last5=Barbieri |first5=Sonia |last6=Chen |first6=Yiwei |last7=Silacci-Fregni |first7=Chiara |last8=Wolf |first8=Tobias |last9=Jarrossay |first9=David |last10=Anderle |first10=Marica |last11=Abdi |first11=Abdirahman |last12=Ndungu |first12=Francis M. |last13=Doumbo |first13=Ogobara K. |last14=Traore |first14=Boubacar |last15=Tran |first15=Tuan M. |date=August 2017 |title=Public antibodies to malaria antigens generated by two LAIR1 insertion modalities |journal=Nature |language=en |volume=548 |issue=7669 |pages=597–601 |doi=10.1038/nature23670 |issn=0028-0836 |pmc=5635981 |pmid=28847005|bibcode=2017Natur.548..597P }}</ref> European donors did show 100-1000 nucleotide stretches inside the elbow joints as well, however. This particular phenomenon  may be specific to malaria, as infection is known to induce genomic instability.<ref>{{Cite journal |last1=Robbiani |first1=Davide F. |last2=Deroubaix |first2=Stephanie |last3=Feldhahn |first3=Niklas |last4=Oliveira |first4=Thiago Y. |last5=Callen |first5=Elsa |last6=Wang |first6=Qiao |last7=Jankovic |first7=Mila |last8=Silva |first8=Israel T. |last9=Rommel |first9=Philipp C. |last10=Bosque |first10=David |last11=Eisenreich |first11=Tom |last12=Nussenzweig |first12=André |last13=Nussenzweig |first13=Michel C. |date=August 2015 |title=Plasmodium Infection Promotes Genomic Instability and AID-Dependent B Cell Lymphoma |journal=Cell |language=en |volume=162 |issue=4 |pages=727–737 |doi=10.1016/j.cell.2015.07.019 |pmc=4538708 |pmid=26276629}}</ref> 

==History==
{{See also|History of immunology}}

The first use of the term "antibody" occurred in a text by [[Paul Ehrlich]]. The term ''Antikörper'' (the German word for ''antibody'') appears in the conclusion of his article "Experimental Studies on Immunity", published in October 1891, which states that, "if two substances give rise to two different ''Antikörper'', then they themselves must be different".<ref name="Lindenmann">{{cite journal|vauthors=Lindenmann J|date=April 1984|title=Origin of the terms 'antibody' and 'antigen'|journal=Scandinavian Journal of Immunology|volume=19|issue=4|pages=281–5|doi=10.1111/j.1365-3083.1984.tb00931.x|pmid=6374880|s2cid=222200504}}</ref> However, the term was not accepted immediately and several other terms for antibody were proposed; these included ''Immunkörper'', ''Amboceptor'', ''Zwischenkörper'', ''substance sensibilisatrice'', ''copula'', ''Desmon'', ''philocytase'', ''fixateur'', and ''Immunisin''.<ref name=Lindenmann/> The word ''antibody'' has formal analogy to the word ''[[antitoxin]]'' and a similar concept to ''Immunkörper'' (''immune body'' in English).<ref name=Lindenmann/> As such, the original construction of the word contains a logical flaw; the antitoxin is something directed against a toxin, while the antibody is a body directed against something.<ref name=Lindenmann/>

[[File:AngeloftheWest.jpg|thumb|left|''[[Angel of the West]]'' (2008) by [[Julian Voss-Andreae]] is a sculpture based on the antibody structure published by E. Padlan.<ref>{{cite journal | vauthors = Padlan EA | title = Anatomy of the antibody molecule | journal = Molecular Immunology | volume = 31 | issue = 3 | pages = 169–217 | date = February 1994 | pmid = 8114766 | doi = 10.1016/0161-5890(94)90001-9 | url = https://zenodo.org/record/1258337 }}</ref> Created for the Florida campus of [[the Scripps Research Institute]],<ref>{{cite magazine| vauthors = Sauter E |date=10 November 2018|title=New Sculpture Portraying Human Antibody as Protective Angel Installed on Scripps Florida Campus|url=http://www.scripps.edu/newsandviews/e_20081110/sculpture.html|magazine=News & Views|publisher=The Scripps Research Institute|volume=8|issue=34|archive-url=https://web.archive.org/web/20110110070639/http://www.scripps.edu/newsandviews/e_20081110/sculpture.html|archive-date=10 January 2011|access-date=12 December 2008|url-status=live}}</ref> the antibody is placed into a ring referencing [[Leonardo da Vinci|Leonardo da Vinci's]] ''[[Vitruvian Man]]'' thus highlighting the similarity of the antibody and the human body.<ref>{{cite web|url=http://www.boingboing.net/2008/10/22/protein-sculpture-in.html|title=Protein sculpture inspired by Vitruvian Man| vauthors = Pescovitz D |date=22 October 2008|website=boingboing|type=Blog|url-status=live|archive-url=https://web.archive.org/web/20101104033646/http://boingboing.net/2008/10/22/protein-sculpture-in.html|archive-date=4 November 2010|access-date=12 December 2008}}</ref>]]

The study of antibodies began in 1890 when [[Emil von Behring]] and [[Kitasato Shibasaburō]] described antibody activity against [[diphtheria]] and [[tetanus toxin]]s. Von Behring and Kitasato put forward the theory of [[humoral immunity]], proposing that a mediator in serum could react with a foreign antigen.<ref>Emil von Behring – Biographical. NobelPrize.org. Nobel Media AB 2020. Mon. 20 January 2020. <https://www.nobelprize.org/prizes/medicine/1901/behring/biographical/><!--MLA citation from bottom of cited page--></ref><ref>{{cite journal | vauthors = AGN | title = The Late Baron Shibasaburo Kitasato | journal = Canadian Medical Association Journal | volume = 25 | issue = 2 | pages = 206 | date = August 1931 | pmid = 20318414 | pmc = 382621 }}</ref> His idea prompted Paul Ehrlich to propose the [[side-chain theory]] for antibody and antigen interaction in 1897, when he hypothesized that receptors (described as "side-chains") on the surface of cells could bind specifically to [[toxin]]s&nbsp;– in a "lock-and-key" interaction&nbsp;– and that this binding reaction is the trigger for the production of antibodies.<ref>{{cite journal|vauthors=Winau F, Westphal O, Winau R|date=July 2004|title=Paul Ehrlich—in search of the magic bullet|journal=Microbes and Infection|volume=6|issue=8|pages=786–9|doi=10.1016/j.micinf.2004.04.003|pmid=15207826|doi-access=free}}</ref> Other researchers believed that antibodies existed freely in the blood and, in 1904, [[Almroth Wright]] suggested that soluble antibodies coated [[bacteria]] to label them for [[phagocytosis]] and killing; a process that he named [[opsonin]]ization.<ref>{{cite journal|vauthors=Silverstein AM|date=May 2003|title=Cellular versus humoral immunology: a century-long dispute|journal=Nature Immunology|volume=4|issue=5|pages=425–8|doi=10.1038/ni0503-425|pmid=12719732|s2cid=31571243|doi-access=free}}</ref>
[[File:Michael Heidelberger 1954.jpg|thumb|[[Michael Heidelberger]]]]
In the 1920s, [[Michael Heidelberger]] and [[Oswald Avery]] observed that antigens could be precipitated by antibodies and went on to show that antibodies are made of protein.<ref>{{cite journal | vauthors = Van Epps HL | title = Michael Heidelberger and the demystification of antibodies | journal = The Journal of Experimental Medicine | volume = 203 | issue = 1 | pages = 5 | date = January 2006 | pmid = 16523537 | pmc = 2118068 | doi = 10.1084/jem.2031fta }}</ref> The biochemical properties of antigen-antibody-binding interactions were examined in more detail in the late 1930s by [[John Marrack]].<ref>{{cite book | vauthors = Marrack JR | title = Chemistry of antigens and antibodies | edition = 2nd | year = 1938 | publisher = His Majesty's Stationery Office | location = London | oclc=3220539}}</ref> The next major advance was in the 1940s, when [[Linus Pauling]] confirmed the lock-and-key theory proposed by [[Paul Ehrlich|Ehrlich]] by showing that the interactions between antibodies and antigens depend more on their shape than their chemical composition.<ref>{{cite web|url=http://profiles.nlm.nih.gov/MM/Views/Exhibit/narrative/specificity.html |title=The Linus Pauling Papers: How Antibodies and Enzymes Work |access-date=5 June 2007 |archive-url=https://web.archive.org/web/20101205061247/http://profiles.nlm.nih.gov/MM/Views/Exhibit/narrative/specificity.html |archive-date=5 December 2010 |url-status=live}}</ref> In 1948, [[Astrid Fagraeus]] discovered that [[B cell]]s, in the form of [[plasma cell]]s, were responsible for generating antibodies.<ref>{{cite journal | vauthors = Silverstein AM | title = Labeled antigens and antibodies: the evolution of magic markers and magic bullets | journal = Nature Immunology | volume = 5 | issue = 12 | pages = 1211–7 | date = December 2004 | pmid = 15549122 | doi = 10.1038/ni1140 | s2cid = 40595920 | url = http://users.path.ox.ac.uk/~seminars/halelibrary/Paper%2018.pdf | archive-url = https://web.archive.org/web/20090325001032/http://users.path.ox.ac.uk/~seminars/halelibrary/Paper%2018.pdf| url-status = dead | archive-date = 25 March 2009 }}</ref>

Further work concentrated on characterizing the structures of the antibody proteins. A major advance in these structural studies was the discovery in the early 1960s by [[Gerald Edelman]] and Joseph Gally of the antibody [[Immunoglobulin light chain|light chain]],<ref>{{cite journal | vauthors = Edelman GM, Gally JA | title = The nature of Bence-Jones proteins. Chemical similarities to polypetide chains of myeloma globulins and normal gamma-globulins | journal = The Journal of Experimental Medicine | volume = 116 | issue = 2 | pages = 207–27 | date = August 1962 | pmid = 13889153 | pmc = 2137388 | doi = 10.1084/jem.116.2.207 }}</ref> and their realization that this protein is the same as the [[Bence-Jones protein]] described in 1845 by [[Henry Bence Jones]].<ref>{{cite journal | vauthors = Stevens FJ, Solomon A, Schiffer M | title = Bence Jones proteins: a powerful tool for the fundamental study of protein chemistry and pathophysiology | journal = Biochemistry | volume = 30 | issue = 28 | pages = 6803–5 | date = July 1991 | pmid = 2069946 | doi = 10.1021/bi00242a001 | url = https://digital.library.unt.edu/ark:/67531/metadc1400136/ }}</ref> Edelman went on to discover that antibodies are composed of [[disulfide bond]]-linked heavy and light chains. Around the same time, antibody-binding (Fab) and antibody tail (Fc) regions of [[Immunoglobulin G|IgG]] were characterized by [[Rodney Porter]].<ref name=edel>{{cite journal | vauthors = Raju TN | title = The Nobel chronicles. 1972: Gerald M Edelman (b 1929) and Rodney R Porter (1917–85) | journal = Lancet | volume = 354 | issue = 9183 | pages = 1040 | date = September 1999 | pmid = 10501404 | doi = 10.1016/S0140-6736(05)76658-7 | s2cid = 54380536 }}</ref> Together, these scientists deduced the structure and complete [[amino acid]] sequence of IgG, a feat for which they were jointly awarded the 1972 [[Nobel Prize in Physiology or Medicine]].<ref name=edel/> The Fv fragment was prepared and characterized by David Givol.<ref>{{cite journal | vauthors = Hochman J, Inbar D, Givol D | title = An active antibody fragment (Fv) composed of the variable portions of heavy and light chains | journal = Biochemistry | volume = 12 | issue = 6 | pages = 1130–5 | date = March 1973 | pmid = 4569769 | doi = 10.1021/bi00730a018 }}</ref> While most of these early studies focused on IgM and IgG, other immunoglobulin isotypes were identified in the 1960s: Thomas Tomasi discovered secretory antibody ([[IgA]]);<ref>{{cite journal | vauthors = Tomasi TB | title = The discovery of secretory IgA and the mucosal immune system | journal = Immunology Today | volume = 13 | issue = 10 | pages = 416–8 | date = October 1992 | pmid = 1343085 | doi = 10.1016/0167-5699(92)90093-M }}</ref> David S. Rowe and John L. Fahey discovered IgD;<ref>{{cite journal | vauthors = Preud'homme JL, Petit I, Barra A, Morel F, Lecron JC, Lelièvre E | title = Structural and functional properties of membrane and secreted IgD | journal = Molecular Immunology | volume = 37 | issue = 15 | pages = 871–87 | date = October 2000 | pmid = 11282392 | doi = 10.1016/S0161-5890(01)00006-2 }}</ref> and [[Kimishige Ishizaka]] and [[Teruko Ishizaka]] discovered [[IgE]] and showed it was a class of antibodies involved in [[allergy|allergic]] reactions.<ref>{{cite journal | vauthors = Johansson SG | title = The discovery of immunoglobulin E | journal = Allergy and Asthma Proceedings | volume = 27 | issue = 2 Suppl 1 | pages = S3–6 | year = 2006 | pmid = 16722325 }}</ref> In a landmark series of experiments beginning in 1976, [[Susumu Tonegawa]] showed that genetic material can rearrange itself to form the vast array of available antibodies.<ref>{{cite journal | vauthors = Hozumi N, Tonegawa S | title = Evidence for somatic rearrangement of immunoglobulin genes coding for variable and constant regions | journal = Proceedings of the National Academy of Sciences of the United States of America | volume = 73 | issue = 10 | pages = 3628–32 | date = October 1976 | pmid = 824647 | pmc = 431171 | doi = 10.1073/pnas.73.10.3628 | bibcode = 1976PNAS...73.3628H | doi-access = free }}</ref>

==Medical applications==
===Disease diagnosis===
Detection of particular antibodies is a very common form of medical [[medical diagnosis|diagnostics]], and applications such as [[serology]] depend on these methods.<ref>{{cite web |url=http://www.immunospot.eu/elisa-animation.html |title=Animated depictions of how antibodies are used in ELISA assays |access-date=8 May 2007 |website=Cellular Technology Ltd.—Europe |archive-url=https://web.archive.org/web/20110614091640/http://www.elispot-analyzers.de/english/elisa-animation.html |archive-date=14 June 2011 |url-status=dead}}</ref> For example, in biochemical assays for disease diagnosis,<ref>{{cite web |url=http://www.immunospot.eu/elispot-animation.html |title=Animated depictions of how antibodies are used in ELISPOT assays |access-date=8 May 2007 |website=Cellular Technology Ltd.—Europe |archive-url=https://web.archive.org/web/20110516142529/http://www.elispot-analyzers.de/english/elispot-animation.html |archive-date=16 May 2011 |url-status=dead}}</ref> a [[titer]] of antibodies directed against [[Epstein-Barr virus]] or [[Lyme disease]] is estimated from the blood. If those antibodies are not present, either the person is not infected or the infection occurred a ''very'' long time ago, and the B cells generating these specific antibodies have naturally decayed.

In [[clinical immunology]], levels of individual classes of immunoglobulins are measured by [[nephelometry]] (or [[turbidimetry]]) to characterize the antibody profile of patient.<ref>{{cite journal |author=Stern P |title=Current possibilities of turbidimetry and nephelometry |journal=Klin Biochem Metab |volume=14 |issue=3 |pages=146–151 |year=2006 |url=http://www.clsjep.cz/odkazy/kbm0603-146.pdf |archive-url=https://web.archive.org/web/20080410032918/http://www.clsjep.cz/odkazy/kbm0603-146.pdf |archive-date=10 April 2008 |url-status=dead}}</ref> Elevations in different classes of immunoglobulins are sometimes useful in determining the cause of [[liver]] damage in patients for whom the diagnosis is unclear.<ref name=Rhoades/> For example, elevated IgA indicates alcoholic [[cirrhosis]], elevated IgM indicates [[viral hepatitis]] and [[primary biliary cirrhosis]], while IgG is elevated in viral hepatitis, [[autoimmune hepatitis]] and cirrhosis.

[[Autoimmune disorder]]s can often be traced to antibodies that bind the body's own [[epitope]]s; many can be detected through [[blood test]]s. Antibodies directed against [[red blood cell]] surface antigens in immune mediated [[hemolytic anemia]] are detected with the [[Coombs test]].<ref name=Dean>{{cite book | vauthors = Dean L |title= Blood Groups and Red Cell Antigens| year= 2005|publisher=National Library of Medicine (US) |location=NCBI Bethesda (MD)|chapter= Chapter 4: Hemolytic disease of the newborn |chapter-url= https://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=rbcantigen.chapter.ch4}}</ref> The Coombs test is also used for antibody screening in [[blood transfusion]] preparation and also for antibody screening in [[antenatal]] women.<ref name=Dean/>

Practically, several immunodiagnostic methods based on detection of complex antigen-antibody are used to diagnose infectious diseases, for example [[ELISA]], [[immunofluorescence]], [[Western blot]], [[immunodiffusion]], [[immunoelectrophoresis]], and [[magnetic immunoassay]].<ref>{{cite journal |last1=Sullivan |first1=Mark V |last2=Stockburn |first2=William J |last3=Hawes |first3=Philippa C |last4=Mercer |first4=Tim |last5=Reddy |first5=Subrayal M |title=Green synthesis as a simple and rapid route to protein modified magnetic nanoparticles for use in the development of a fluorometric molecularly imprinted polymer-based assay for detection of myoglobin |journal=Nanotechnology |date=26 February 2021 |volume=32 |issue=9 |pages=095502 |doi=10.1088/1361-6528/abce2d|pmid=33242844 |bibcode=2021Nanot..32i5502S |doi-access=free |pmc=8314874 }}</ref> Antibodies raised against human [[chorion]]ic [[gonadotropin]] are used in over the counter pregnancy tests.

New dioxaborolane chemistry enables radioactive [[fluoride]] ([[Fluorine-18|<sup>18</sup>F]]) labeling of antibodies, which allows for [[positron emission tomography]] (PET) imaging of [[cancer]].<ref>{{cite journal | vauthors = Rodriguez EA, Wang Y, Crisp JL, Vera DR, Tsien RY, Ting R | title = New Dioxaborolane Chemistry Enables [(18)F]-Positron-Emitting, Fluorescent [(18)F]-Multimodality Biomolecule Generation from the Solid Phase | language = EN | journal = Bioconjugate Chemistry | volume = 27 | issue = 5 | pages = 1390–1399 | date = May 2016 | pmid = 27064381 | pmc = 4916912 | doi = 10.1021/acs.bioconjchem.6b00164 }}</ref>

===Disease therapy===
Targeted [[monoclonal antibody therapy]] is employed to treat diseases such as [[rheumatoid arthritis]],<ref>{{cite journal | vauthors = Feldmann M, Maini RN | title = Anti-TNF alpha therapy of rheumatoid arthritis: what have we learned? | journal = Annual Review of Immunology | volume = 19 | issue = 1 | pages = 163–96 | year = 2001 | pmid = 11244034 | doi = 10.1146/annurev.immunol.19.1.163 }}</ref> [[multiple sclerosis]],<ref>{{cite journal | vauthors = Doggrell SA | title = Is natalizumab a breakthrough in the treatment of multiple sclerosis? | journal = Expert Opinion on Pharmacotherapy | volume = 4 | issue = 6 | pages = 999–1001 | date = June 2003 | pmid = 12783595 | doi = 10.1517/14656566.4.6.999 | s2cid = 16104816 }}</ref> [[psoriasis]],<ref>{{cite journal | vauthors = Krueger GG, Langley RG, Leonardi C, Yeilding N, Guzzo C, Wang Y, Dooley LT, Lebwohl M | title = A human interleukin-12/23 monoclonal antibody for the treatment of psoriasis | journal = The New England Journal of Medicine | volume = 356 | issue = 6 | pages = 580–92 | date = February 2007 | pmid = 17287478 | doi = 10.1056/NEJMoa062382 | doi-access = free }}</ref> and many forms of [[cancer]] including [[non-Hodgkin's lymphoma]],<ref>{{cite journal | vauthors = Plosker GL, Figgitt DP | title = Rituximab: a review of its use in non-Hodgkin's lymphoma and chronic lymphocytic leukaemia | journal = Drugs | volume = 63 | issue = 8 | pages = 803–43 | year = 2003 | pmid = 12662126 | doi = 10.2165/00003495-200363080-00005 }}</ref> [[colorectal cancer]], [[head and neck cancer]] and [[breast cancer]].<ref>{{cite journal | vauthors = Vogel CL, Cobleigh MA, Tripathy D, Gutheil JC, Harris LN, Fehrenbacher L, Slamon DJ, Murphy M, Novotny WF, Burchmore M, Shak S, Stewart SJ | title = First-line Herceptin monotherapy in metastatic breast cancer | journal = Oncology | volume = 61| issue = Suppl. 2 | pages = 37–42 | year = 2001 | pmid = 11694786 | doi = 10.1159/000055400 | series = 61 | s2cid = 24924864 }}</ref>

Some immune deficiencies, such as [[X-linked agammaglobulinemia]] and [[hypogammaglobulinemia]], result in partial or complete lack of antibodies.<ref>{{cite journal | vauthors = LeBien TW | title = Fates of human B-cell precursors | journal = Blood | volume = 96 | issue = 1 | pages = 9–23 | date = July 2000 | pmid = 10891425 | url = http://bloodjournal.hematologylibrary.org/cgi/content/full/96/1/9 | archive-url = https://web.archive.org/web/20100429220000/http://bloodjournal.hematologylibrary.org/cgi/content/full/96/1/9 | url-status = dead | archive-date = 29 April 2010 | doi = 10.1182/blood.V96.1.9 | access-date = 31 March 2007 }}</ref> These diseases are often treated by inducing a short-term form of [[immunity (medical)|immunity]] called [[passive immunity]]. Passive immunity is achieved through the transfer of ready-made antibodies in the form of human or animal [[blood plasma|serum]], pooled immunoglobulin or [[Monoclonal antibody|monoclonal antibodies]], into the affected individual.<ref name=USC>{{cite web|author=Ghaffer A |title=Immunization |website=Immunology&nbsp;— Chapter 14 |publisher=University of South Carolina School of Medicine |url=http://pathmicro.med.sc.edu/ghaffar/immunization.htm |date=26 March 2006 |access-date=6 June 2007 |archive-url=https://web.archive.org/web/20101018004057/http://pathmicro.med.sc.edu/ghaffar/immunization.htm |archive-date=18 October 2010 |url-status=live}}</ref>

===Prenatal therapy===
[[Rh factor]], also known as Rh D antigen, is an antigen found on [[red blood cell]]s; individuals that are Rh-positive (Rh+) have this antigen on their red blood cells and individuals that are Rh-negative (Rh–) do not. During normal [[childbirth]], delivery trauma or complications during pregnancy, blood from a [[fetus]] can enter the mother's system. In the case of an Rh-incompatible mother and child, consequential blood mixing may sensitize an Rh- mother to the Rh antigen on the blood cells of the Rh+ child, putting the remainder of the [[pregnancy]], and any subsequent pregnancies, at risk for [[hemolytic disease of the newborn]].<ref>{{cite journal | vauthors = Urbaniak SJ, Greiss MA | title = RhD haemolytic disease of the fetus and the newborn | journal = Blood Reviews | volume = 14 | issue = 1 | pages = 44–61 | date = March 2000 | pmid = 10805260 | doi = 10.1054/blre.1999.0123 }}</ref>

[[Rho(D) immune globulin]] antibodies are specific for human RhD antigen.<ref name= Fung>{{cite journal | vauthors = Fung Kee Fung K, Eason E, Crane J, Armson A, De La Ronde S, Farine D, Keenan-Lindsay L, Leduc L, Reid GJ, Aerde JV, Wilson RD, Davies G, Désilets VA, Summers A, Wyatt P, Young DC | title = Prevention of Rh alloimmunization | journal = Journal of Obstetrics and Gynaecology Canada | volume = 25 | issue = 9 | pages = 765–73 | date = September 2003 | pmid = 12970812 | doi = 10.1016/S1701-2163(16)31006-4 }}</ref> Anti-RhD antibodies are administered as part of a [[prenatal care|prenatal treatment regimen]] to prevent sensitization that may occur when a Rh-negative mother has a Rh-positive fetus. Treatment of a mother with Anti-RhD antibodies prior to and immediately after trauma and delivery destroys Rh antigen in the mother's system from the fetus. This occurs before the antigen can stimulate maternal B cells to "remember" Rh antigen by generating memory B cells. Therefore, her humoral immune system will not make anti-Rh antibodies, and will not attack the Rh antigens of the current or subsequent babies. Rho(D) Immune Globulin treatment prevents sensitization that can lead to [[Rh disease]], but does not prevent or treat the underlying disease itself.<ref name= Fung/>

==Research applications==
[[File:FluorescentCells.jpg|thumb|[[Immunofluorescence]] image of the [[Eukaryote|eukaryotic]] [[cytoskeleton]]. [[Microtubule]]s as shown in green, are marked by an antibody conjugated to a green fluorescing molecule, [[Fluorescein isothiocyanate|FITC]].]]

Specific antibodies are produced by injecting an [[antigen]] into a [[mammal]], such as a [[mouse]], [[rat]], [[rabbit]], [[goat]], [[sheep]], or [[horse]] for large quantities of antibody. Blood isolated from these animals contains ''[[polyclonal antibodies]]''—multiple antibodies that bind to the same antigen—in the [[Blood plasma|serum]], which can now be called [[antiserum]]. Antigens are also injected into [[chicken]]s for generation of polyclonal antibodies in [[egg yolk]].<ref>{{cite journal | vauthors = Tini M, Jewell UR, Camenisch G, Chilov D, Gassmann M | title = Generation and application of chicken egg-yolk antibodies | journal = Comparative Biochemistry and Physiology. Part A, Molecular & Integrative Physiology | volume = 131 | issue = 3 | pages = 569–74 | date = March 2002 | pmid = 11867282 | doi = 10.1016/S1095-6433(01)00508-6 }}</ref> To obtain antibody that is specific for a single epitope of an antigen, antibody-secreting [[lymphocyte]]s are isolated from the animal and [[Biological immortality|immortalized]] by fusing them with a cancer cell line. The fused cells are called [[hybridoma]]s, and will continually grow and secrete antibody in culture. Single hybridoma cells are isolated by [[dilution cloning]] to generate [[cloning#Cellular cloning|cell clones]] that all produce the same antibody; these antibodies are called ''[[monoclonal antibodies]]''.<ref>{{cite journal | vauthors = Cole SP, Campling BG, Atlaw T, Kozbor D, Roder JC | title = Human monoclonal antibodies | journal = Molecular and Cellular Biochemistry | volume = 62 | issue = 2 | pages = 109–20 | date = June 1984 | pmid = 6087121 | doi = 10.1007/BF00223301 | s2cid = 12616168 }}</ref> Polyclonal and monoclonal antibodies are often purified using [[Protein A/G]] or [[Affinity chromatography|antigen-affinity chromatography]].<ref>{{cite journal | vauthors = Kabir S | title = Immunoglobulin purification by affinity chromatography using protein A mimetic ligands prepared by combinatorial chemical synthesis | journal = Immunological Investigations | volume = 31 | issue = 3–4 | pages = 263–78 | year = 2002 | pmid = 12472184 | doi = 10.1081/IMM-120016245 | s2cid = 12785078 }}</ref>

In research, purified antibodies are used in many applications. Antibodies for research applications can be found directly from antibody suppliers, or through use of a specialist search engine. Research antibodies are most commonly used to identify and locate [[intracellular]] and [[extracellular]] proteins. Antibodies are used in [[flow cytometry]] to differentiate cell types by the proteins they express; different types of cells express different combinations of [[cluster of differentiation]] molecules on their surface, and produce different intracellular and secretable proteins.<ref name=Stecher>{{cite journal | vauthors = Brehm-Stecher BF, Johnson EA | title = Single-cell microbiology: tools, technologies, and applications | journal = Microbiology and Molecular Biology Reviews | volume = 68 | issue = 3 | pages = 538–59, table of contents | date = September 2004 | pmid = 15353569 | pmc = 515252 | doi = 10.1128/MMBR.68.3.538-559.2004 }}</ref> They are also used in [[immunoprecipitation]] to separate proteins and anything bound to them (co-immunoprecipitation) from other molecules in a [[cell lysate]],<ref>{{Cite book | vauthors = Williams NE | chapter = Chapter 23 Immunoprecipitation Procedures | title = Methods in Cell Biology Volume 62 | volume = 62 | pages = [https://archive.org/details/tetrahymenatherm62acad/page/449 449–53] | year = 2000 | pmid = 10503210 | doi = 10.1016/S0091-679X(08)61549-6 | isbn = 978-0-12-544164-3 | chapter-url = https://archive.org/details/tetrahymenatherm62acad/page/449 | publisher = San Diego, CA : Academic Press }}</ref> in [[Western blot]] analyses to identify proteins separated by [[electrophoresis]],<ref>{{cite journal | vauthors = Kurien BT, Scofield RH | title = Western blotting | journal = Methods | volume = 38 | issue = 4 | pages = 283–93 | date = April 2006 | pmid = 16483794 | doi = 10.1016/j.ymeth.2005.11.007 }}</ref> and in [[immunohistochemistry]] or [[immunofluorescence]] to examine protein expression in tissue sections or to locate proteins within cells with the assistance of a [[microscope]].<ref name=Stecher/><ref>{{Cite book | vauthors = Scanziani E | title = Mycoplasma Protocols | chapter = Immunohistochemical staining of fixed tissues | series = Methods in Molecular Biology | volume = 104 | pages = [https://archive.org/details/mycoplasmaprotoc00roge/page/133 133–40] | year = 1998 | pmid = 9711649 | doi = 10.1385/0-89603-525-5:133 | isbn = 978-0-89603-525-6 | chapter-url = https://archive.org/details/mycoplasmaprotoc00roge/page/133 | publisher = Totowa, N.J. : Humana Press }}</ref> Proteins can also be detected and quantified with antibodies, using [[ELISA]] and [[ELISpot]] techniques.<ref>{{Cite book| vauthors = Reen DJ | title = Basic Protein and Peptide Protocols | chapter = Enzyme-linked immunosorbent assay (ELISA) | series = Methods in Molecular Biology | volume = 32 | pages = 461–6 | year = 1994 | pmid = 7951745 | doi = 10.1385/0-89603-268-X:461 | isbn = 978-0-89603-268-2 | pmc = 2366430 }}</ref><ref>{{Cite book| vauthors = Kalyuzhny AE | title = Handbook of ELISPOT | chapter = Chemistry and biology of the ELISPOT assay | series = Methods in Molecular Biology | volume = 302 | pages = 15–31 | year = 2005 | pmid = 15937343 | doi = 10.1385/1-59259-903-6:015 | isbn = 978-1-59259-903-5 }}</ref>

Antibodies used in research are some of the most powerful, yet most problematic reagents with a tremendous number of factors that must be controlled in any experiment including cross reactivity, or the antibody recognizing multiple epitopes and affinity, which can vary widely depending on experimental conditions such as pH, solvent, state of tissue etc. Multiple attempts have been made to improve both the way that researchers validate antibodies<ref>{{cite journal | vauthors = Saper CB | title = An open letter to our readers on the use of antibodies | journal = The Journal of Comparative Neurology | volume = 493 | issue = 4 | pages = 477–8 | date = December 2005 | pmid = 16304632 | doi = 10.1002/cne.20839 | s2cid = 14082678 | doi-access = free }}</ref><ref>{{Cite web|url=https://grants.nih.gov/grants/guide/notice-files/NOT-OD-16-011.html|title=NOT-OD-16-011: Implementing Rigor and Transparency in NIH & AHRQ Research Grant Applications|website=grants.nih.gov}}</ref> and ways in which they report on antibodies. Researchers using antibodies in their work need to record them correctly in order to allow their research to be reproducible (and therefore tested, and qualified by other researchers). Less than half of research antibodies referenced in academic papers can be easily identified.<ref>{{cite journal | vauthors = Vasilevsky NA, Brush MH, Paddock H, Ponting L, Tripathy SJ, Larocca GM, Haendel MA | title = On the reproducibility of science: unique identification of research resources in the biomedical literature | journal = PeerJ | volume = 1 | pages = e148 | date = 2 September 2013 | pmid = 24032093 | pmc = 3771067 | doi = 10.7717/peerj.148 | author-link7 = Melissa Haendel | doi-access = free }}</ref> Papers published in [[Faculty of 1000|F1000]] in 2014 and 2015 provide researchers with a guide for reporting research antibody use.<ref>{{cite journal | vauthors = Bandrowski A, Brush M, Grethe JS, Haendel MA, Kennedy DN, Hill S, Hof PR, Martone ME, Pols M, Tan S, Washington N, Zudilova-Seinstra E, Vasilevsky N | title = The Resource Identification Initiative: A cultural shift in publishing | journal = F1000Research | volume = 4 | pages = 134 | year = 2015 | pmid = 26594330 | pmc = 4648211 | doi = 10.12688/f1000research.6555.2 | doi-access = free }}</ref><ref>{{cite journal | vauthors = Helsby MA, Fenn JR, Chalmers AD | title = Reporting research antibody use: how to increase experimental reproducibility | journal = F1000Research | volume = 2 | pages = 153 | date = 23 August 2013 | pmid = 24358895 | pmc = 3829129 | doi = 10.12688/f1000research.2-153.v2 | doi-access = free }}</ref> The RRID paper, is co-published in 4 journals that implemented the [[RRIDs]] Standard for research resource citation, which draws data from the antibodyregistry.org as the source of antibody identifiers<ref>{{Cite web|url=https://antibodyregistry.org/|title=The Antibody Registry|website=antibodyregistry.org}}</ref> (see also group at [[FORCE11|Force11]]<ref>{{cite web|title=Resource Identification Initiative|url=https://www.force11.org/group/resource-identification-initiative|website=FORCE11|access-date=18 April 2016|date=14 August 2013}}</ref>).

Antibody regions can be used to further biomedical research by acting as a guide for drugs to reach their target. Several application involve using bacterial plasmids to tag plasmids with the Fc region of the antibody such as [[pFUSE-Fc plasmid]].

==Regulations==
===Production and testing===
There are several ways to obtain antibodies, including in vivo techniques like animal immunization and various in vitro approaches, such as the phage display method.<ref>{{Cite web |last=Eberle |first=Christian |date=February 20, 2023 |title=Antibody Production simply explained |url=https://www.evitria.com/journal/antibodies/antibody-production/ |access-date=December 7, 2023}}</ref> Traditionally, most antibodies are produced by hybridoma [[cell (biology)|cell]] lines through immortalization of antibody-producing cells by chemically induced fusion with [[Multiple myeloma|myeloma]] cells. In some cases, additional fusions with other lines have created "[[Trifunctional antibody|triomas]]" and "[[Trifunctional antibody|quadromas]]". The manufacturing process should be appropriately described and validated. Validation studies should at least include:
* The demonstration that the process is able to produce in good quality (the process should be validated)
* The [[efficiency]] of the antibody purification (all [[impurities]] and [[virus]] must be eliminated)
* The characterization of purified antibody ([[physical chemistry|physicochemical]] characterization, [[immunological]] properties, [[biological]] activities, contaminants, ...)
* Determination of the virus clearance studies

===Before clinical trials===
* Product safety testing: Sterility ([[bacteria]] and [[Fungus|fungi]]), [[in vitro]] and [[in vivo]] testing for adventitious viruses, [[Murinae|murine]] [[retrovirus]] testing..., product safety data needed before the initiation of feasibility trials in serious or immediately life-threatening conditions, it serves to evaluate dangerous potential of the product.
* Feasibility testing: These are pilot studies whose objectives include, among others, early characterization of safety and initial proof of concept in a small specific patient population (in vitro or in vivo testing).

===Preclinical studies===
* Testing [[cross-reactivity]] of antibody: to highlight unwanted interactions (toxicity) of antibodies with previously characterized tissues. This study can be performed in vitro (reactivity of the antibody or immunoconjugate should be determined with a quick-frozen adult tissues) or in vivo (with appropriates animal models).
* [[Phases of clinical research|Preclinical]] [[pharmacology]] and [[toxicity]] testing: [[preclinical]] safety testing of antibody is designed to identify possible toxicity in humans, to estimate the likelihood and severity of potential adverse events in humans, and to identify a safe starting dose and dose escalation, when possible.
* Animal toxicity studies: [[Acute toxicity]] testing, repeat-dose toxicity testing, [[Chronic toxicity|long-term toxicity]] testing
* [[Pharmacokinetics]] and [[pharmacodynamics]] testing: Use for determinate clinical dosages, antibody activities, evaluation of the potential clinical effects

==Structure prediction and computational antibody design==
The importance of antibodies in health care and the [[biotechnology]] industry demands knowledge of their structures at [[Image resolution|high resolution]]. This information is used for [[protein engineering]], modifying the antigen binding affinity, and identifying an epitope, of a given antibody. [[X-ray crystallography]] is one commonly used method for determining antibody structures. However, crystallizing an antibody is often laborious and time-consuming. Computational approaches provide a cheaper and faster alternative to crystallography, but their results are more equivocal, since they do not produce empirical structures. Online web servers such as ''Web Antibody Modeling'' (WAM)<ref>{{webarchive |url=https://web.archive.org/web/20110717212251/http://antibody.bath.ac.uk/abmod.html |date=17 July 2011 }}<br /> [http://antibody.bath.ac.uk/abmod.html WAM]</ref> and ''Prediction of Immunoglobulin Structure'' (PIGS)<ref>{{cite journal | vauthors = Marcatili P, Rosi A, Tramontano A | title = PIGS: automatic prediction of antibody structures | journal = Bioinformatics | volume = 24 | issue = 17 | pages = 1953–4 | date = September 2008 | pmid = 18641403 | doi = 10.1093/bioinformatics/btn341 | url = http://biocomputing.it/pigs/ | archive-url = https://web.archive.org/web/20101126235204/http://arianna.bio.uniroma1.it/pigs/ | url-status = live | archive-date = 26 November 2010 | doi-access = free }}<br /> [http://biocomputing.it/pigs/ Prediction of Immunoglobulin Structure (PIGS)] {{Webarchive|url=https://web.archive.org/web/20101126235204/http://arianna.bio.uniroma1.it/pigs/ |date=26 November 2010 }}</ref> enable computational modeling of antibody variable regions. Rosetta Antibody is a novel antibody F<sub>V</sub> region structure prediction [[Server (computing)|server]], which incorporates sophisticated techniques to minimize CDR loops and optimize the relative orientation of the light and heavy chains, as well as [[homology (biology)|homology]] models that predict successful docking of antibodies with their unique antigen.<ref>{{webarchive|url=https://web.archive.org/web/20110719215959/http://antibody.graylab.jhu.edu/ |date=19 July 2011 }}<br />[http://antibody.graylab.jhu.edu RosettaAntibody]</ref> However, describing an antibody's binding site using only one single static structure limits the understanding and characterization of the antibody's function and properties. To improve antibody structure prediction and to take the strongly correlated CDR loop and interface movements into account, antibody paratopes should be described as interconverting states in solution with varying probabilities.<ref name="Fernandez-Quintero2021" />

The ability to describe the antibody through binding affinity to the antigen is supplemented by information on antibody structure and amino acid sequences for the purpose of patent claims.<ref>{{cite web|title= Written Description Problems of the Monoclonal Antibody Patents after Centocor v. Abbott| vauthors = Park H |url= http://jolt.law.harvard.edu/digest/patent/written-description-problems-of-the-monoclonal-antibody-patents-after-centocor-v-abbott|website= jolt.law.harvard.edu|access-date= 12 December 2014|archive-url= https://web.archive.org/web/20141213031525/http://jolt.law.harvard.edu/digest/patent/written-description-problems-of-the-monoclonal-antibody-patents-after-centocor-v-abbott|archive-date= 13 December 2014|url-status= dead}}</ref> Several methods have been presented for computational design of antibodies based on the structural [[bioinformatics]] studies of antibody CDRs.<ref>{{cite journal | vauthors = Adolf-Bryfogle J, Kalyuzhniy O, Kubitz M, Weitzner BD, Hu X, Adachi Y, Schief WR, Dunbrack RL | title = RosettaAntibodyDesign (RAbD): A general framework for computational antibody design | journal = PLOS Computational Biology | volume = 14 | issue = 4 | pages = e1006112 | date = April 2018 | pmid = 29702641 | pmc = 5942852 | doi = 10.1371/journal.pcbi.1006112 | bibcode = 2018PLSCB..14E6112A | doi-access = free }}</ref><ref>{{cite journal | vauthors = Lapidoth GD, Baran D, Pszolla GM, Norn C, Alon A, Tyka MD, Fleishman SJ | title = AbDesign: An algorithm for combinatorial backbone design guided by natural conformations and sequences | journal = Proteins | volume = 83 | issue = 8 | pages = 1385–406 | date = August 2015 | pmid = 25670500 | pmc = 4881815 | doi = 10.1002/prot.24779 }}</ref><ref>{{cite journal | vauthors = Li T, Pantazes RJ, Maranas CD | title = OptMAVEn--a new framework for the de novo design of antibody variable region models targeting specific antigen epitopes | journal = PLOS ONE | volume = 9 | issue = 8 | pages = e105954 | date = 2014 | pmid = 25153121 | pmc = 4143332 | doi = 10.1371/journal.pone.0105954 | bibcode = 2014PLoSO...9j5954L | doi-access = free }}</ref>

There are a variety of methods used to sequence an antibody including [[Edman degradation]], [[cDNA]], etc.; albeit one of the most common modern uses for peptide/protein identification is liquid [[chromatography]] coupled with [[tandem mass spectrometry]] (LC-MS/MS).<ref>{{cite journal | vauthors = Pham V, Henzel WJ, Arnott D, Hymowitz S, Sandoval WN, Truong BT, Lowman H, Lill JR | title = De novo proteomic sequencing of a monoclonal antibody raised against OX40 ligand | journal = Analytical Biochemistry | volume = 352 | issue = 1 | pages = 77–86 | date = May 2006 | pmid = 16545334 | doi = 10.1016/j.ab.2006.02.001 }}</ref> High volume antibody sequencing methods require computational approaches for the data analysis, including [[de novo sequencing]] directly from tandem mass spectra<ref>{{cite journal | vauthors = Ma B, Zhang K, Hendrie C, Liang C, Li M, Doherty-Kirby A, Lajoie G | title = PEAKS: powerful software for peptide de novo sequencing by tandem mass spectrometry | journal = Rapid Communications in Mass Spectrometry | volume = 17 | issue = 20 | pages = 2337–42 | year = 2003 | pmid = 14558135 | doi = 10.1002/rcm.1196 | bibcode = 2003RCMS...17.2337M }}</ref> and database search methods that use existing [[protein sequence]] databases.<ref>{{cite journal | vauthors = Zhang J, Xin L, Shan B, Chen W, Xie M, Yuen D, Zhang W, Zhang Z, Lajoie GA, Ma B | title = PEAKS DB: de novo sequencing assisted database search for sensitive and accurate peptide identification | journal = Molecular & Cellular Proteomics | volume = 11 | issue = 4 | pages = M111.010587 | date = April 2012 | pmid = 22186715 | pmc = 3322562 | doi = 10.1074/mcp.M111.010587  |doi-access=free }}</ref><ref>{{cite journal | vauthors = Perkins DN, Pappin DJ, Creasy DM, Cottrell JS | title = Probability-based protein identification by searching sequence databases using mass spectrometry data | journal = Electrophoresis | volume = 20 | issue = 18 | pages = 3551–67 | date = December 1999 | pmid = 10612281 | doi = 10.1002/(SICI)1522-2683(19991201)20:18<3551::AID-ELPS3551>3.0.CO;2-2 | s2cid = 42423655 }}</ref> Many versions of shotgun protein sequencing are able to increase the coverage by utilizing CID/HCD/ETD<ref>{{cite journal | vauthors = Bandeira N, Tang H, Bafna V, Pevzner P | title = Shotgun protein sequencing by tandem mass spectra assembly | journal = Analytical Chemistry | volume = 76 | issue = 24 | pages = 7221–33 | date = December 2004 | pmid = 15595863 | doi = 10.1021/ac0489162 }}</ref> fragmentation methods and other techniques, and they have achieved substantial progress in attempt to fully sequence [[proteins]], especially antibodies. Other methods have assumed the existence of similar proteins,<ref>{{cite journal | vauthors = Liu X, Han Y, Yuen D, Ma B | title = Automated protein (re)sequencing with MS/MS and a homologous database yields almost full coverage and accuracy | journal = Bioinformatics | volume = 25 | issue = 17 | pages = 2174–80 | date = September 2009 | pmid = 19535534 | doi = 10.1093/bioinformatics/btp366 | doi-access = free }}</ref> a known [[genome sequence]],<ref>{{cite journal | vauthors = Castellana NE, Pham V, Arnott D, Lill JR, Bafna V | title = Template proteogenomics: sequencing whole proteins using an imperfect database | journal = Molecular & Cellular Proteomics | volume = 9 | issue = 6 | pages = 1260–70 | date = June 2010 | pmid = 20164058 | pmc = 2877985 | doi = 10.1074/mcp.M900504-MCP200  |doi-access=free }}</ref> or combined top-down and bottom up approaches.<ref>{{cite journal | vauthors = Liu X, Dekker LJ, Wu S, Vanduijn MM, Luider TM, Tolić N, Kou Q, Dvorkin M, Alexandrova S, Vyatkina K, Paša-Tolić L, Pevzner PA | title = De novo protein sequencing by combining top-down and bottom-up tandem mass spectra | journal = Journal of Proteome Research | volume = 13 | issue = 7 | pages = 3241–8 | date = July 2014 | pmid = 24874765 | doi = 10.1021/pr401300m }}</ref> Current technologies have the ability to assemble [[protein sequences]] with high accuracy by integrating [[de novo sequencing]] [[peptides]], intensity, and positional confidence scores from database and [[Homology (biology)|homology]] searches.<ref>{{cite journal | vauthors = Tran NH, Rahman MZ, He L, Xin L, Shan B, Li M | title = Complete De Novo Assembly of Monoclonal Antibody Sequences | journal = Scientific Reports | volume = 6 | pages = 31730 | date = August 2016 | pmid = 27562653 | pmc = 4999880 | doi = 10.1038/srep31730 | bibcode = 2016NatSR...631730T }}</ref>

==Antibody mimetic==
[[Antibody mimetic]]s are organic compounds, like antibodies, that can specifically bind antigens. They consist of artificial peptides or proteins, or [[aptamer]]-based nucleic acid molecules with a molar mass of about 3 to 20 [[Dalton (unit)|kDa]]. Antibody fragments, such as [[Fragment antigen-binding|Fab]] and [[Single-domain antibody|nanobodies]] are not considered as [[antibody mimetic]]s. Common advantages over antibodies are better solubility, tissue penetration, stability towards heat and [[enzyme]]s, and comparatively low production costs. Antibody mimetics have being developed and commercialized as research, diagnostic and therapeutic agents.<ref>{{cite journal | vauthors = Gebauer M, Skerra A | title = Engineered protein scaffolds as next-generation antibody therapeutics | journal = Current Opinion in Chemical Biology | volume = 13 | issue = 3 | pages = 245–55 | date = June 2009 | pmid = 19501012 | doi = 10.1016/j.cbpa.2009.04.627 }}</ref>

=={{anchor|BAU}}Binding antibody unit==
BAU (binding antibody unit, often as BAU/mL) is a [[measurement unit]] defined by the [[WHO]] for the comparison of [[assay]]s detecting the same class of immunoglobulins with the same specificity.<ref>{{cite journal |title=WHO International Standard for anti-SARS-CoV-2 immunoglobulin |orig-date=2021-02-23 |date=2021-04-10 |volume=397 |number=10282 |pages=1347–1348 |doi=10.1016/S0140-6736(21)00527-4 |pmc=7987302 |pmid=33770519 |author-first1=Paul A. |author-last1=Kristiansen |author-first2=Mark |author-last2=Page |author-first3=Valentina |author-last3=Bernasconi |author-first4=Giada |author-last4=Mattiuzzo |author-first5=Peter |author-last5=Dull |author-first6=Karen |author-last6=Makar |author-first7=Stanley |author-last7=Plotkin |author-first8=Ivana |author-last8=Knezevic |journal=[[The Lancet]] |publisher=[[World Health Organization]] / [[Elsevier]] }}</ref><ref>{{cite journal |title=WHO International Standard for evaluation of the antibody response to COVID-19 vaccines: call for urgent action by the scientific community |author-first1=Ivana |author-last1=Knezevic |author-first2=Giada |author-last2=Mattiuzzo |author-first3=Mark |author-last3=Page |author-first4=Philip |author-last4=Minor |author-first5=Elwyn |author-last5=Griffiths |author-first6=Micha |author-last6=Nuebling |author-first7=Vasee |author-last7=Moorthy |date=2021-10-26 |journal=[[The Lancet Microbe]] |volume=3 |issue=3 |pages=e235–e240 |publisher=[[World Health Organization]] / [[Elsevier]] |doi=10.1016/S2666-5247(21)00266-4 |pmid=34723229 |pmc=8547804 }}</ref><ref>{{cite web |title=Training webinar for the calibration of quantitative serology assays using the WHO International Standard for anti-SARS-CoV-2 immunoglobulin |author-first=Ivana |author-last=Knezevic |date=2021-11-10 |url=https://cdn.who.int/media/docs/default-source/biologicals/covid-19/agenda-ppt-q-a-ik.pdf?sfvrsn=89c3ddfa_5 |access-date=2022-03-05 |url-status=live |archive-url=https://web.archive.org/web/20220218031435/https://cdn.who.int/media/docs/default-source/biologicals/covid-19/agenda-ppt-q-a-ik.pdf?sfvrsn=89c3ddfa_5 |archive-date=2022-02-18}} (68 pages)</ref>

== See also ==
{{div col|colwidth=20em}}
* [[Affimer]]
* [[Anti-mitochondrial antibodies]]
* [[Anti-nuclear antibodies]]
* [[Antibody mimetic]]
* [[Aptamer]]
* [[Colostrum]]
* [[ELISA]]
* [[Humoral immunity]]
* [[Immunology]]
* [[Immunosuppressive drug]]
* [[Intravenous immunoglobulin]] (IVIg)
* [[Magnetic immunoassay]]
* [[Microantibody]]
* [[Monoclonal antibody]]
* [[Neutralizing antibody]]
* [[Optimer Ligand]]
* [[Secondary antibodies]]
* [[Single-domain antibody]]
* [[Slope spectroscopy]]
* [[Surrobody]]
* [[Synthetic antibody]]
* [[Western blot normalization]]
{{div col end}}

== References ==
{{Reflist}}

== External links ==
{{Commons category|Antibodies}}
<!--===========================({{No more links}})===============================-->
<!--| DO ''not'' ADD MORE LINKS TO THIS ARTICLE. WIKIPEDIA IS ''not'' A COLLECTION OF |-->
<!--| LINKS. If you think that your link might be useful, do not add it here, |-->
<!--| but put it on this article's discussion page first or submit your link  |-->
<!--| to the appropriate category at the Open Directory Project (www.dmoz.org)|-->
<!--|                                                                         |http://pdl.com/technology-products/how-do-antibodies-work/-->
<!--===========================({{No more links}})===============================-->
* [https://web.archive.org/web/20070403013202/http://www.path.cam.ac.uk/~mrc7/mikeimages.html Mike's Immunoglobulin Structure/Function Page] at [[University of Cambridge]]
* [https://pdb101.rcsb.org/motm/21 Antibodies as the PDB molecule of the month] Discussion of the structure of antibodies at RCSB [[Protein Data Bank]]
* [https://web.archive.org/web/20070405170923/http://users.path.ox.ac.uk/~scobbold/tig/new1/mabth.html A hundred years of antibody therapy] History and applications of antibodies in the treatment of disease at [[University of Oxford]]
* [http://www.cellsalive.com/antibody.htm How Lymphocytes Produce Antibody] from Cells Alive!

{{Immune system}}
{{Globular proteins}}
{{Immune proteins}}
{{Autoantibodies}}
{{Authority control}}

[[Category:Antibodies| ]]
[[Category:Glycoproteins]]
[[Category:Immunology]]
[[Category:Reagents for biochemistry]]