{{Short description|Domain of microorganisms}}
{{About|the microorganisms|the genus|Bacterium (genus){{!}}''Bacterium'' (genus)|other uses}}
{{Featured article}}
{{cs1 config|name-list-style=vanc}}
{{pp-move}}
{{pp-semi-indef}}
{{Use British English|date=September 2016}}
{{Use dmy dates|date=March 2022}}
{{Automatic taxobox
| name = Bacteria
| fossil_range = [[Archean]]–[[Holocene|Present]] {{long fossil range|3500|0}} <ref>{{cite web |title=31. Ancient Life: Apex Chert Microfossils |url=https://www.lpi.usra.edu/publications/slidesets/marslife/slide_31.html |access-date=2022-03-12 |website=www.lpi.usra.edu }}</ref>
| image = E. coli Bacteria (7316101966).jpg
| image_caption = [[Scanning electron microscope|Scanning electron micrograph]] of ''[[Escherichia coli]]'' [[bacillus (shape)|rods]]
| taxon = Bacteria
| authority = [[Carl Woese|Woese]] et al. 1990
| subdivision_ranks = Phyla
| subdivision = See {{Section link|#Phyla}}
| synonyms = * "Bacteria" <small>([[Ferdinand Cohn|Cohn]] 1872) [[Cavalier-Smith]] 1983</small>
* "Bacteria" <small>[[Ernst Haeckel|Haeckel]] 1894</small>
* "Bacteria" <small>Cavalier-Smith 2002</small>
* "Bacteriaceae" <small>Cohn 1872a</small>
* "Bacteriobionta" <small>Möhn 1984</small>
* "Bacteriophyta" <small>Schussnig 1925</small>
* "Eubacteria" <small>Woese and Fox 1977</small>
* "Neobacteria" <small>Möhn 1984</small>
* "Schizomycetaceae" <small>de Toni and Trevisan 1889</small>
* "Schizomycetes" <small>Nägeli 1857</small>
}}

'''Bacteria''' ({{IPAc-en|audio=en-us-bacteria.ogg|b|æ|k|ˈ|t|ɪər|i|ə}}; {{Singular}}: bacterium) are ubiquitous, mostly free-living organisms often consisting of one [[Cell (biology)|biological cell]]. They constitute a large [[domain (biology)|domain]] of [[Prokaryote|prokaryotic]] [[microorganism]]s. Typically a few [[micrometre]]s in length, bacteria were among the first life forms to appear on [[Earth]], and are present in most of its [[habitat]]s. Bacteria inhabit soil, water, [[Hot spring|acidic hot springs]], [[radioactive waste]], and the [[deep biosphere]] of [[Earth's crust]]. Bacteria play a vital role in many stages of the [[nutrient cycle]] by recycling nutrients and the [[nitrogen fixation|fixation of nitrogen]] from the [[Earth's atmosphere|atmosphere]]. The nutrient cycle includes the [[decomposition]] of [[cadaver|dead bodies]]; bacteria are responsible for the [[putrefaction]] stage in this process. In the biological communities surrounding [[hydrothermal vent]]s and [[cold seep]]s, [[extremophile]] bacteria provide the nutrients needed to sustain life by converting dissolved compounds, such as [[hydrogen sulphide]] and [[methane]], to energy. Bacteria also live in [[mutualism (biology)|mutualistic]], [[commensal]] and [[parasitic]] relationships with plants and animals. Most bacteria have not been characterised and there are many species that cannot be [[microbiological culture|grown]] in the laboratory. The study of bacteria is known as [[bacteriology]], a branch of [[microbiology]].

Like all animals, humans carry vast numbers (approximately 10<sup>13</sup> to 10<sup>14</sup>) of bacteria.<ref>{{cite journal |vauthors=Sender R, Fuchs S, Milo R |date=2016-08-19 |title=Revised Estimates for the Number of Human and Bacteria Cells in the Body |journal=PLOS Biology |language=en |volume=14 |issue=8 |pages=e1002533 |doi=10.1371/journal.pbio.1002533 |issn=1545-7885 |pmc=4991899 |pmid=27541692 |doi-access=free }}</ref> Most are in the [[gut flora|gut]], though there are many on the skin. Most of the bacteria in and on the body are harmless or rendered so by the protective effects of the [[immune system]], and many are [[probiotic|beneficial]],<ref name="McCutcheon"/> particularly the ones in the gut. However, several species of bacteria are [[pathogenic bacteria|pathogenic]] and cause [[infectious disease]]s, including [[cholera]], [[syphilis]], [[anthrax]], [[leprosy]], [[tuberculosis]], [[tetanus]] and [[bubonic plague]]. The most common fatal bacterial diseases are [[respiratory infection]]s. [[Antibiotic]]s are used to treat [[Infection|bacterial infections]] and are also used in farming, making [[antibiotic resistance]] a growing problem. Bacteria are important in [[sewage treatment]] and the breakdown of [[oil spill]]s, the production of [[cheese]] and [[yogurt]] through [[fermentation (biochemistry)|fermentation]], the recovery of gold, [[palladium]], copper and other metals in the mining sector, as well as in [[biotechnology]], and the manufacture of antibiotics and other chemicals.

Once regarded as [[plant]]s constituting the class ''Schizomycetes'' ("fission fungi"), bacteria are now classified as [[prokaryote]]s. Unlike cells of animals and other [[eukaryote]]s, bacterial cells do not contain a [[cell nucleus|nucleus]] and rarely harbour [[cell membrane|membrane]]-bound [[organelle]]s. Although the term ''bacteria'' traditionally included all prokaryotes, the [[scientific classification]] changed after the discovery in the 1990s that prokaryotes consist of two very different groups of organisms that [[evolution|evolved]] from an [[last universal common ancestor|ancient common ancestor]]. These [[domain (biology)|evolutionary domains]] are called Bacteria and [[Archaea]].{{sfn|Hall|2008|p=145}}

== Etymology ==
[[File:Bacillus subtilis 2.jpg|right|thumb|Rod-shaped ''[[Bacillus subtilis]]'']]

The word ''bacteria'' is the plural of the [[Neo-Latin]] ''{{wikt-lang|la|bacterium}}'', which is the [[latinisation (literature)|Latinisation]] of the [[Ancient Greek]] {{wikt-lang|grc|βακτήριον}} (''{{grc-transl|βακτήριον}}''),<ref>{{LSJ|bakth/rion|βακτήριον|ref}}.</ref> the [[diminutive]] of {{wikt-lang|grc|βακτηρία}} (''{{grc-transl|βακτηρία}}''), meaning "staff, cane",<ref>{{LSJ|bakthri/a|βακτηρία|shortref}}.</ref> because the first ones to be discovered were [[bacillus (shape)|rod-shaped]].<ref>{{OEtymD|bacteria}}</ref>{{sfn|Krasner|2014|p=74}}

== Origin and early evolution ==
{{Main|Evolution of bacteria}}
{{further|Earliest known life forms|Evolutionary history of life|Timeline of evolution}}
[[File:Phylogenetic tree of life LUCA.svg|thumb|[[Phylogenetic tree]] of Bacteria, [[Archaea]] and [[Eukaryote|Eucarya]], with the [[last universal common ancestor]] (LUCA) at the root.<ref name="pmid2112744">{{cite journal |vauthors=Woese CR, Kandler O, Wheelis ML |title=Towards a natural system of organisms: proposal for the domains Archaea, Bacteria, and Eucarya |journal=Proceedings of the National Academy of Sciences of the United States of America |volume=87 |issue=12 |pages=4576–79 |date=June 1990 |pmid=2112744 |pmc=54159 |doi=10.1073/pnas.87.12.4576 |bibcode=1990PNAS...87.4576W |doi-access=free }}</ref>]]

The ancestors of bacteria were unicellular microorganisms that were the [[Earliest known life forms|first forms of life]] to appear on Earth, about 4&nbsp;billion years ago.{{sfn|Hall|2008|page=84}} For about 3&nbsp;billion years, most organisms were microscopic, and bacteria and archaea were the dominant forms of life.<ref name="pmid31475212">{{cite journal |vauthors=Godoy-Vitorino F |title=Human microbial ecology and the rising new medicine |journal=Annals of Translational Medicine |volume=7 |issue=14 |pages=342 |date=July 2019 |pmid=31475212 |pmc=6694241 |doi=10.21037/atm.2019.06.56 |doi-access=free }}</ref><ref>{{cite journal |vauthors=Schopf JW |title=Disparate rates, differing fates: tempo and mode of evolution changed from the Precambrian to the Phanerozoic |journal=Proceedings of the National Academy of Sciences of the United States of America |volume=91 |issue=15 |pages=6735–42 |date=July 1994 |pmid=8041691 |pmc=44277 |doi=10.1073/pnas.91.15.6735 |bibcode=1994PNAS...91.6735S |doi-access=free }}</ref><ref>{{cite journal |vauthors=DeLong EF, Pace NR |title=Environmental diversity of bacteria and archaea |journal=Systematic Biology |volume=50 |issue=4 |pages=470–78 |date=August 2001 |pmid=12116647 |doi=10.1080/106351501750435040 |citeseerx=10.1.1.321.8828 }}</ref> Although bacterial [[fossil]]s exist, such as [[stromatolite]]s, their lack of distinctive [[morphology (biology)|morphology]] prevents them from being used to examine the history of bacterial evolution, or to date the time of origin of a particular bacterial species. However, gene sequences can be used to reconstruct the bacterial [[phylogenetics|phylogeny]], and these studies indicate that bacteria diverged first from the archaeal/eukaryotic lineage.<ref>{{cite journal |vauthors=Brown JR, Doolittle WF |title=Archaea and the prokaryote-to-eukaryote transition |journal=Microbiology and Molecular Biology Reviews |volume=61 |issue=4 |pages=456–502 |date=December 1997 |doi=10.1128/mmbr.61.4.456-502.1997 |pmid=9409149 |pmc=232621 }}</ref> The [[most recent common ancestor]] (MRCA) of bacteria and archaea was probably a [[Thermophile|hyperthermophile]] that lived about 2.5&nbsp;billion–3.2&nbsp;billion years ago.<ref name="pmid29894297">{{cite journal |vauthors=Daum B, Gold V |title=Twitch or swim: towards the understanding of prokaryotic motion based on the type IV pilus blueprint |journal=Biological Chemistry |volume=399 |issue=7 |pages=799–808 |date=June 2018 |pmid=29894297 |doi=10.1515/hsz-2018-0157 |s2cid=48352675 |hdl=10871/33366 |hdl-access=free }}</ref><ref>{{cite journal |vauthors=Di Giulio M |title=The universal ancestor and the ancestor of bacteria were hyperthermophiles |journal=Journal of Molecular Evolution |volume=57 |issue=6 |pages=721–30 |date=December 2003 |pmid=14745541 |doi=10.1007/s00239-003-2522-6 |bibcode=2003JMolE..57..721D |s2cid=7041325 }}</ref><ref>{{cite journal |vauthors=Battistuzzi FU, Feijao A, Hedges SB |title=A genomic timescale of prokaryote evolution: insights into the origin of methanogenesis, phototrophy, and the colonization of land |journal=BMC Evolutionary Biology |volume=4 |page=44 |date=November 2004 |pmid=15535883 |pmc=533871 |doi=10.1186/1471-2148-4-44 |doi-access=free }}</ref> The earliest life on land may have been bacteria some 3.22&nbsp;billion years ago.<ref name="NG-20180723">{{cite journal |vauthors=Homann M, Sansjofre P, Van Zuilen M, Heubeck C, Gong J, Killingsworth B, Foster IS, Airo A, Van Kranendonk MJ, Ader M, Lalonde SV |display-authors=6 |title=Microbial life and biogeochemical cycling on land 3,220 million years ago |date=23 July 2018 |journal=[[Nature Geoscience]] |volume=11 |issue=9 |pages=665–671 |doi=10.1038/s41561-018-0190-9 |bibcode=2018NatGe..11..665H |s2cid=134935568 |url=https://hal.univ-brest.fr/hal-01901955/file/Homann%20et%20al.%202018%20-%20accepted-1.pdf }}</ref>

Bacteria were also involved in the second great evolutionary divergence, that of the archaea and eukaryotes.<ref name="Gabaldón">{{cite journal |vauthors=Gabaldón T |title=Origin and Early Evolution of the Eukaryotic Cell |journal=Annual Review of Microbiology |volume=75 |issue=1 |pages=631–647 |date=October 2021 |pmid=34343017 |doi=10.1146/annurev-micro-090817-062213 |url=https://www.annualreviews.org/doi/10.1146/annurev-micro-090817-062213 |access-date=19 August 2022 |url-status=dead |s2cid=236916203 |archive-url=https://web.archive.org/web/20220819211000/https://www.annualreviews.org/doi/10.1146/annurev-micro-090817-062213 |archive-date=19 August 2022 }}</ref><ref name="Callier">{{cite journal |vauthors=Callier V |title=Mitochondria and the origin of eukaryotes |journal=Knowable Magazine |date=8 June 2022 |doi=10.1146/knowable-060822-2 |doi-access=free |url=https://knowablemagazine.org/article/living-world/2022/mitochondria-origin-eukaryotes |access-date=19 August 2022}}</ref> Here, eukaryotes resulted from the entering of ancient bacteria into [[endosymbiont|endosymbiotic]] associations with the ancestors of eukaryotic cells, which were themselves possibly related to the Archaea.<ref>{{cite journal |vauthors=Poole AM, Penny D |title=Evaluating hypotheses for the origin of eukaryotes |journal=BioEssays |volume=29 |issue=1 |pages=74–84 |date=January 2007 |pmid=17187354 |doi=10.1002/bies.20516 }}</ref><ref name="Dyall">{{cite journal | vauthors = Dyall SD, Brown MT, Johnson PJ | title = Ancient invasions: from endosymbionts to organelles | journal = Science | volume = 304 | issue = 5668 | pages = 253–257 | date = April 2004 | pmid = 15073369 | doi = 10.1126/science.1094884 | s2cid = 19424594 | bibcode = 2004Sci...304..253D }}</ref> This involved the engulfment by proto-eukaryotic cells of [[alphaproteobacteria]]l [[symbiosis|symbionts]] to form either [[mitochondrion|mitochondria]] or [[hydrogenosome]]s, which are still found in all known Eukarya (sometimes in highly [[reduced form]], e.g. in ancient "amitochondrial" protozoa). Later, some eukaryotes that already contained mitochondria also engulfed [[cyanobacteria]]-like organisms, leading to the formation of [[chloroplast]]s in algae and plants. This is known as [[primary endosymbiosis]].<ref name="pmid34018613">{{cite journal | vauthors = Stephens TG, Gabr A, Calatrava V, Grossman AR, Bhattacharya D | title = Why is primary endosymbiosis so rare? | journal = The New Phytologist | volume = 231 | issue = 5 | pages = 1693–1699 | date = September 2021 | pmid = 34018613 | pmc = 8711089 | doi = 10.1111/nph.17478 | doi-access = free }}</ref>

== Habitat ==
Bacteria are ubiquitous, living in every possible habitat on the planet including soil, underwater, deep in Earth's crust and even such extreme environments as acidic hot springs and radioactive waste.<ref name="pmid17331729">{{cite journal | vauthors = Baker-Austin C, Dopson M | title = Life in acid: pH homeostasis in acidophiles | journal = Trends in Microbiology | volume = 15 | issue = 4 | pages = 165–171 | date = April 2007 | pmid = 17331729 | doi = 10.1016/j.tim.2007.02.005 }}</ref><ref name="pmid33114255">{{cite journal | vauthors = Jeong SW, Choi YJ | title = Extremophilic Microorganisms for the Treatment of Toxic Pollutants in the Environment | journal = Molecules | volume = 25 | issue = 21 | page = 4916 | date = October 2020 | pmid = 33114255 | pmc = 7660605 | doi = 10.3390/molecules25214916 | doi-access = free }}</ref> There are thought to be approximately 2×10<sup>30</sup> bacteria on Earth,<ref name="pmid30760902">{{cite journal | vauthors = Flemming HC, Wuertz S | title = Bacteria and archaea on Earth and their abundance in biofilms | journal = Nature Reviews. Microbiology | volume = 17 | issue = 4 | pages = 247–260 | date = April 2019 | pmid = 30760902 | doi = 10.1038/s41579-019-0158-9 | s2cid = 61155774 }}</ref> forming a [[biomass (ecology)|biomass]] that is only exceeded by plants.<ref name="Bar-On">{{cite journal | vauthors = Bar-On YM, Phillips R, Milo R | title = The biomass distribution on Earth | journal = Proceedings of the National Academy of Sciences of the United States of America | volume = 115 | issue = 25 | pages = 6506–6511 | date = June 2018 | pmid = 29784790 | pmc = 6016768 | doi = 10.1073/pnas.1711842115 | doi-access = free | bibcode = 2018PNAS..115.6506B }}</ref> They are abundant in lakes and oceans, in arctic ice, and [[geothermal springs]]{{sfn|Wheelis|2008|page=362}} where they provide the nutrients needed to sustain life by converting dissolved compounds, such as [[hydrogen sulphide]] and [[methane]], to energy.<ref name="pmid34203823">{{cite journal | vauthors = Kushkevych I, Procházka J, Gajdács M, Rittmann SK, Vítězová M | title = Molecular Physiology of Anaerobic Phototrophic Purple and Green Sulfur Bacteria | journal = International Journal of Molecular Sciences | volume = 22 | issue = 12 | page = 6398 | date = June 2021 | pmid = 34203823 | pmc = 8232776 | doi = 10.3390/ijms22126398 | doi-access = free }}</ref> They live on and in plants and animals. Most do not cause diseases, are beneficial to their environments, and are essential for life.<ref name="McCutcheon">{{cite journal | vauthors = McCutcheon JP | title = The Genomics and Cell Biology of Host-Beneficial Intracellular Infections | journal = Annual Review of Cell and Developmental Biology | volume = 37 | issue = 1 | pages = 115–142 | date = October 2021 | pmid = 34242059 | doi = 10.1146/annurev-cellbio-120219-024122 | s2cid = 235786110 | doi-access = free }}</ref>{{sfn|Wheelis|2008|page=6}} The soil is a rich source of bacteria and a few grams contain around a thousand million of them. They are all essential to soil ecology, breaking down toxic waste and recycling nutrients. They are even found in the atmosphere and one cubic metre of air holds around one hundred million bacterial cells. The oceans and seas harbour around 3 x 10<sup>26</sup> bacteria which provide up to 50% of the oxygen humans breathe.{{sfn|Pommerville|2014|p=3–6}} Only around 2% of bacterial species have been fully studied.{{sfn|Krasner|2014|page=38}}

{| class="wikitable" style="margin-left: auto; margin-right:auto"
|+ [[Extremophile]] bacteria
! Habitat !! Species !! Reference
|-
|Cold (minus 15&nbsp;°C Antarctica) || [[Cryptoendoliths]] ||{{sfn|Pommerville|2014|page=134}}
|-
|Hot (70–100&nbsp;°C [[geyser]]s) || ''[[Thermus aquaticus]]'' || {{sfn|Krasner|2014|page=38}}
|-
| Radiation, 5M[[Rad (unit)|Rad]] || ''[[Deinococcus radiodurans]]''|| {{sfn|Pommerville|2014|page=134}}
|-
| Saline, 47% salt ([[Dead Sea]], [[Great Salt Lake]]) || several species ||{{sfn|Krasner|2014|page=38}}{{sfn|Pommerville|2014|page=134}}
|-
|Acid [[pH]] 3 ||several species || <ref name="pmid17331729" />
|-
|Alkaline pH 12.8 || [[betaproteobacteria]]|| {{sfn|Pommerville|2014|page=134}}
|-
|Space (6 years on a [[NASA]] satellite) || ''[[Bacillus subtilis]]'' || {{sfn|Pommerville|2014|page=134}}
|-
|3.2&nbsp;km underground || several species || {{sfn|Pommerville|2014|page=134}}
|-
|High pressure ([[Mariana Trench]] – 1200 [[Standard atmosphere (unit)|atm]]) || Moritella, [[Shewanella]] and others || {{sfn|Pommerville|2014|page=134}}
|}

== Morphology ==
{{further|Bacterial cell structure#Cell morphology}}
[[File:Bacterial morphology diagram.svg|left|thumb|upright=1.4|alt=a diagram showing bacteria morphology|Bacteria display many cell [[morphology (biology)|morphologies]] and arrangements{{sfn | Krasner | 2014 | p=74}}]]
'''Size'''. Bacteria display a wide diversity of shapes and sizes. Bacterial cells are about one-tenth the size of eukaryotic cells and are typically 0.5–5.0&nbsp;[[micrometre]]s in length. However, a few species are visible to the unaided eye—for example, ''[[Thiomargarita namibiensis]]'' is up to half a millimetre long,<ref>{{cite journal | vauthors = Schulz HN, Jorgensen BB | title = Big bacteria | journal = Annual Review of Microbiology | volume = 55 | pages = 105–137 | year = 2001 | pmid = 11544351 | doi = 10.1146/annurev.micro.55.1.105 | s2cid = 18168018 }}</ref> ''[[Epulopiscium fishelsoni]]'' reaches 0.7&nbsp;mm,<ref name="Williams2011">{{cite journal | vauthors = Williams C | year=2011 |title=Who are you calling simple? |journal=New Scientist |volume=211 |issue=2821 |pages=38–41 |doi=10.1016/S0262-4079(11)61709-0 }}</ref> and ''[[Thiomargarita magnifica]]'' can reach even 2&nbsp;cm in length, which is 50 times larger than other known bacteria.<ref>{{cite journal |vauthors=Volland JM, Gonzalez-Rizzo S, Gros O, Tyml T, Ivanova N, Schulz F, Goudeau D, Elisabeth NH, Nath N, Udwary D, Malmstrom RR |date=18 February 2022 |title=A centimeter-long bacterium with DNA compartmentalized in membrane-bound organelles |url=https://www.biorxiv.org/content/10.1101/2022.02.16.480423v1 |journal=bioRxiv |type=preprint |doi=10.1101/2022.02.16.480423 |s2cid=246975579 }}</ref><ref>{{cite journal | vauthors = Sanderson K | title = Largest bacterium ever found is surprisingly complex | journal = Nature | date = June 2022 | pmid = 35750919 | doi = 10.1038/d41586-022-01757-1 | s2cid = 250022076 }}</ref> Among the smallest bacteria are members of the genus ''[[Mycoplasma]]'', which measure only 0.3&nbsp;micrometres, as small as the largest [[virus]]es.<ref>{{cite journal | vauthors = Robertson J, Gomersall M, Gill P | title = Mycoplasma hominis: growth, reproduction, and isolation of small viable cells | journal = Journal of Bacteriology | volume = 124 | issue = 2 | pages = 1007–1018 | date = November 1975 | pmid = 1102522 | pmc = 235991 | doi = 10.1128/JB.124.2.1007-1018.1975 }}</ref> Some bacteria may be even smaller, but these [[ultramicrobacteria]] are not well-studied.<ref name="Velimirov2001">{{cite journal | vauthors = Velimirov B | year = 2001 | title = Nanobacteria, Ultramicrobacteria and Starvation Forms: A Search for the Smallest Metabolizing Bacterium | journal = Microbes and Environments | volume = 16 | issue = 2 | pages = 67–77 | doi = 10.1264/jsme2.2001.67 | doi-access = free }}</ref>

'''Shape'''. Most bacterial species are either spherical, called ''[[coccus|cocci]]'' (''singular coccus'', from Greek ''kókkos'', grain, seed), or rod-shaped, called ''[[bacillus (shape)|bacilli]]'' (''sing''. bacillus, from [[Latin]] ''baculus'', stick).<ref>{{cite book |last=Dusenbery |first=David B |title=Living at Micro Scale |publisher=[[Harvard University Press]] |year=2009 |isbn=978-0-674-03116-6 |location=Cambridge, Massachusetts |pages=20–25 |language=en}}</ref> Some bacteria, called ''[[Spiral bacteria#Vibrio|vibrio]]'', are shaped like slightly curved rods or comma-shaped; others can be spiral-shaped, called ''[[Spiral bacteria#Spirillum|spirilla]]'', or tightly coiled, called ''[[Spiral bacteria#Spirochete|spirochaetes]]''. A small number of other unusual shapes have been described, such as star-shaped bacteria.<ref>{{cite journal | vauthors = Yang DC, Blair KM, Salama NR | title = Staying in Shape: the Impact of Cell Shape on Bacterial Survival in Diverse Environments | journal = Microbiology and Molecular Biology Reviews | volume = 80 | issue = 1 | pages = 187–203 | date = March 2016 | pmid = 26864431 | pmc = 4771367 | doi = 10.1128/MMBR.00031-15 }}</ref> This wide variety of shapes is determined by the bacterial [[cell wall]] and [[cytoskeleton]] and is important because it can influence the ability of bacteria to acquire nutrients, attach to surfaces, swim through liquids and escape [[predation|predators]].<ref>{{cite journal | vauthors = Cabeen MT, Jacobs-Wagner C | title = Bacterial cell shape | journal = Nature Reviews. Microbiology | volume = 3 | issue = 8 | pages = 601–10 | date = August 2005 | pmid = 16012516 | doi = 10.1038/nrmicro1205 | s2cid = 23938989 }}</ref><ref>{{cite journal | vauthors = Young KD | title = The selective value of bacterial shape | journal = Microbiology and Molecular Biology Reviews | volume = 70 | issue = 3 | pages = 660–703 | date = September 2006 | pmid = 16959965 | pmc = 1594593 | doi = 10.1128/MMBR.00001-06 }}</ref>
[[File:Relative scale.svg|thumb|The range of sizes shown by [[prokaryote]]s (Bacteria), relative to those of other organisms and [[biomolecule]]s.{{sfn | Crawford | 2007 | p=xi}}]]

'''Multicellularity'''. Most bacterial species exist as single cells; others associate in characteristic patterns: ''[[Neisseria]]'' forms diploids (pairs), [[Streptococcus|streptococci]] form chains, and [[Staphylococcus|staphylococci]] group together in "bunch of grapes" clusters. Bacteria can also group to form larger multicellular structures, such as the elongated [[filamentous bacteria|filaments]] of ''[[Actinomycetota]]'' species, the aggregates of ''[[Myxobacteria]]'' species, and the complex hyphae of ''[[Streptomyces]]'' species.<ref>{{cite journal | vauthors = Claessen D, Rozen DE, Kuipers OP, Søgaard-Andersen L, van Wezel GP | title = Bacterial solutions to multicellularity: a tale of biofilms, filaments and fruiting bodies | journal = Nature Reviews. Microbiology | volume = 12 | issue = 2 | pages = 115–24 | date = February 2014 | pmid = 24384602 | doi = 10.1038/nrmicro3178 | hdl = 11370/0db66a9c-72ef-4e11-a75d-9d1e5827573d | s2cid = 20154495 | url = https://www.rug.nl/research/portal/en/publications/bacterial-solutions-to-multicellularity(0db66a9c-72ef-4e11-a75d-9d1e5827573d).html | hdl-access = free }}</ref> These multicellular structures are often only seen in certain conditions. For example, when starved of amino acids, myxobacteria detect surrounding cells in a process known as [[quorum sensing]], migrate towards each other, and aggregate to form fruiting bodies up to 500&nbsp;micrometres long and containing approximately 100,000 bacterial cells.<ref>{{cite journal | vauthors = Shimkets LJ | title = Intercellular signaling during fruiting-body development of Myxococcus xanthus | journal = [[Annual Review of Microbiology]] | volume = 53 | pages = 525–49 | year = 1999 | pmid = 10547700 | doi = 10.1146/annurev.micro.53.1.525 }}</ref> In these fruiting bodies, the bacteria perform separate tasks; for example, about one in ten cells migrate to the top of a fruiting body and differentiate into a specialised dormant state called a myxospore, which is more resistant to drying and other adverse environmental conditions.<ref name=Kaiser>{{cite journal | vauthors = Kaiser D | title = Signaling in myxobacteria | journal = [[Annual Review of Microbiology]] | volume = 58 | pages = 75–98 | year = 2004 | pmid = 15487930 | doi = 10.1146/annurev.micro.58.030603.123620 }}</ref>

'''Biofilms'''. Bacteria often attach to surfaces and form dense aggregations called [[biofilm]]s{{sfn|Wheelis|2008|page=75}} and larger formations known as [[microbial mat]]s.<ref name="pmid34153859">{{cite journal |vauthors=Mandal A, Dutta A, Das R, Mukherjee J |title=Role of intertidal microbial communities in carbon dioxide sequestration and pollutant removal: A review |journal=Marine Pollution Bulletin |volume=170 |issue= |pages=112626 |date=June 2021 |pmid=34153859 |doi=10.1016/j.marpolbul.2021.112626 |bibcode=2021MarPB.17012626M }}</ref> These biofilms and mats can range from a few micrometres in thickness to up to half a metre in depth, and may contain multiple species of bacteria, [[protist]]s and archaea. Bacteria living in biofilms display a complex arrangement of cells and extracellular components, forming secondary structures, such as [[microcolony|microcolonies]], through which there are networks of channels to enable better diffusion of nutrients.<ref>{{cite journal | vauthors = Donlan RM | title = Biofilms: microbial life on surfaces | journal = Emerging Infectious Diseases | volume = 8 | issue = 9 | pages = 881–90 | date = September 2002 | pmid = 12194761 | pmc = 2732559 | doi = 10.3201/eid0809.020063 }}</ref><ref>{{cite journal | vauthors = Branda SS, Vik S, Friedman L, Kolter R | title = Biofilms: the matrix revisited | journal = Trends in Microbiology | volume = 13 | issue = 1 | pages = 20–26 | date = January 2005 | pmid = 15639628 | doi = 10.1016/j.tim.2004.11.006 }}</ref> In natural environments, such as soil or the surfaces of plants, the majority of bacteria are bound to surfaces in biofilms.<ref name=Davey>{{cite journal | vauthors = Davey ME, O'toole GA | title = Microbial biofilms: from ecology to molecular genetics | journal = Microbiology and Molecular Biology Reviews | volume = 64 | issue = 4 | pages = 847–67 | date = December 2000 | pmid = 11104821 | pmc = 99016 | doi = 10.1128/MMBR.64.4.847-867.2000 }}</ref> Biofilms are also important in medicine, as these structures are often present during chronic bacterial infections or in infections of [[implant (medicine)|implanted]] [[medical device]]s, and bacteria protected within biofilms are much harder to kill than individual isolated bacteria.<ref>{{cite journal | vauthors = Donlan RM, Costerton JW | title = Biofilms: survival mechanisms of clinically relevant microorganisms | journal = Clinical Microbiology Reviews | volume = 15 | issue = 2 | pages = 167–93 | date = April 2002 | pmid = 11932229 | pmc = 118068 | doi = 10.1128/CMR.15.2.167-193.2002 }}</ref>

== Cellular structure ==
{{further|Bacterial cell structure}}
[[File:Prokaryote cell.svg|thumb|right|alt=Prokaryote cell with structure and parts|Structure and contents of a typical [[Gram-positive]] bacterial cell (seen by the fact that only ''one'' cell membrane is present).]]

=== Intracellular structures ===
The bacterial cell is surrounded by a [[cell membrane]], which is made primarily of [[phospholipid]]s. This membrane encloses the contents of the cell and acts as a barrier to hold nutrients, [[protein]]s and other essential components of the [[cytoplasm]] within the cell.<ref>{{cite book |vauthors=Slonczewski JL, Foster JW |title=Microbiology : an Evolving Science |date=2013 |publisher=W W Norton |location=New York |isbn=978-0-393-12367-8 |edition=Third |page=82}}</ref> Unlike [[eukaryote|eukaryotic cells]], bacteria usually lack large membrane-bound structures in their cytoplasm such as a [[cell nucleus|nucleus]], [[mitochondrion|mitochondria]], [[chloroplast]]s and the other organelles present in eukaryotic cells.<ref name="pmid28664324">{{cite journal |vauthors=Feijoo-Siota L, Rama JL, Sánchez-Pérez A, Villa TG |title=Considerations on bacterial nucleoids |journal=Applied Microbiology and Biotechnology |volume=101 |issue=14 |pages=5591–602 |date=July 2017 |pmid=28664324 |doi=10.1007/s00253-017-8381-7 |s2cid=10173266 }}</ref> However, some bacteria have protein-bound organelles in the cytoplasm which compartmentalize aspects of bacterial metabolism,<ref name=Bobik2006>{{cite journal | vauthors = Bobik TA | title = Polyhedral organelles compartmenting bacterial metabolic processes | journal = Applied Microbiology and Biotechnology | volume = 70 | issue = 5 | pages = 517–25 | date = May 2006 | pmid = 16525780 | doi = 10.1007/s00253-005-0295-0 | s2cid = 8202321 }}</ref><ref>{{cite journal | vauthors = Yeates TO, Kerfeld CA, Heinhorst S, Cannon GC, Shively JM | title = Protein-based organelles in bacteria: carboxysomes and related microcompartments | journal = Nature Reviews. Microbiology | volume = 6 | issue = 9 | pages = 681–91 | date = September 2008 | pmid = 18679172 | doi = 10.1038/nrmicro1913 | s2cid = 22666203 }}</ref> such as the [[carboxysome]].<ref>{{cite journal | vauthors = Kerfeld CA, Sawaya MR, Tanaka S, Nguyen CV, Phillips M, Beeby M, Yeates TO | title = Protein structures forming the shell of primitive bacterial organelles | journal = Science | volume = 309 | issue = 5736 | pages = 936–38 | date = August 2005 | pmid = 16081736 | doi = 10.1126/science.1113397 | bibcode = 2005Sci...309..936K | citeseerx = 10.1.1.1026.896 | s2cid = 24561197 }}</ref> Additionally, bacteria have a multi-component [[prokaryotic cytoskeleton|cytoskeleton]] to control the localisation of proteins and nucleic acids within the cell, and to manage the process of [[cell division]].<ref name="Gitai Z 2005 577–86">{{cite journal | vauthors = Gitai Z | title = The new bacterial cell biology: moving parts and subcellular architecture | journal = Cell | volume = 120 | issue = 5 | pages = 577–86 | date = March 2005 | pmid = 15766522 | doi = 10.1016/j.cell.2005.02.026 | s2cid = 8894304 | doi-access = free }}</ref><ref>{{cite journal | vauthors = Shih YL, Rothfield L | title = The bacterial cytoskeleton | journal = Microbiology and Molecular Biology Reviews | volume = 70 | issue = 3 | pages = 729–54 | date = September 2006 | pmid = 16959967 | pmc = 1594594 | doi = 10.1128/MMBR.00017-06 }}</ref><ref>{{cite journal | vauthors = Norris V, den Blaauwen T, Cabin-Flaman A, Doi RH, Harshey R, Janniere L, Jimenez-Sanchez A, Jin DJ, Levin PA, Mileykovskaya E, Minsky A, Saier M, Skarstad K | title = Functional taxonomy of bacterial hyperstructures | journal = Microbiology and Molecular Biology Reviews | volume = 71 | issue = 1 | pages = 230–53 | date = March 2007 | pmid = 17347523 | pmc = 1847379 | doi = 10.1128/MMBR.00035-06 }}</ref>

Many important [[biochemistry|biochemical]] reactions, such as energy generation, occur due to [[diffusion|concentration gradients]] across membranes, creating a [[Electrochemical potential|potential]] difference analogous to a battery. The general lack of internal membranes in bacteria means these reactions, such as [[electron transport chain|electron transport]], occur across the cell membrane between the cytoplasm and the outside of the cell or [[periplasm]].{{sfn|Pommerville|2014|pages=120–121}} However, in many photosynthetic bacteria, the plasma membrane is highly folded and fills most of the cell with layers of light-gathering membrane.<ref name=bryantfrigaard>{{cite journal | vauthors = Bryant DA, Frigaard NU | title = Prokaryotic photosynthesis and phototrophy illuminated | journal = Trends in Microbiology | volume = 14 | issue = 11 | pages = 488–96 | date = November 2006 | pmid = 16997562 | doi = 10.1016/j.tim.2006.09.001 }}</ref> These light-gathering complexes may even form lipid-enclosed structures called [[chlorosome]]s in [[green sulfur bacteria]].<ref>{{cite journal | vauthors = Psencík J, Ikonen TP, Laurinmäki P, Merckel MC, Butcher SJ, Serimaa RE, Tuma R | title = Lamellar organization of pigments in chlorosomes, the light harvesting complexes of green photosynthetic bacteria | journal = Biophysical Journal | volume = 87 | issue = 2 | pages = 1165–72 | date = August 2004 | pmid = 15298919 | pmc = 1304455 | doi = 10.1529/biophysj.104.040956 | bibcode = 2004BpJ....87.1165P }}</ref>

[[File:Carboxysomes EM ptA.jpg|thumb|An [[electron microscopy|electron micrograph]] of ''[[Halothiobacillus neapolitanus]]'' cells with [[carboxysome]]s inside, with arrows highlighting visible carboxysomes. Scale bars indicate 100 nm.]]
Bacteria do not have a membrane-bound nucleus, and their [[gene]]tic material is typically a single [[circular bacterial chromosome]] of [[DNA]] located in the cytoplasm in an irregularly shaped body called the [[nucleoid]].<ref>{{cite journal | vauthors = Thanbichler M, Wang SC, Shapiro L | title = The bacterial nucleoid: a highly organized and dynamic structure | journal = Journal of Cellular Biochemistry | volume = 96 | issue = 3 | pages = 506–21 | date = October 2005 | pmid = 15988757 | doi = 10.1002/jcb.20519 | s2cid = 25355087 | doi-access = free }}</ref> The nucleoid contains the [[chromosome]] with its associated proteins and [[RNA]]. Like all other [[organism]]s, bacteria contain [[ribosome]]s for the production of proteins, but the structure of the bacterial ribosome is different from that of [[eukaryote]]s and archaea.<ref>{{cite journal | vauthors = Poehlsgaard J, Douthwaite S | title = The bacterial ribosome as a target for antibiotics | journal = Nature Reviews. Microbiology | volume = 3 | issue = 11 | pages = 870–81 | date = November 2005 | pmid = 16261170 | doi = 10.1038/nrmicro1265 | s2cid = 7521924 }}</ref>

Some bacteria produce intracellular nutrient storage granules, such as [[glycogen]],<ref>{{cite journal | vauthors = Yeo M, Chater K | title = The interplay of glycogen metabolism and differentiation provides an insight into the developmental biology of Streptomyces coelicolor | journal = Microbiology | volume = 151 | issue = Pt 3 | pages = 855–61 | date = March 2005 | pmid = 15758231 | doi = 10.1099/mic.0.27428-0 | url = http://mic.sgmjournals.org/cgi/content/full/151/3/855?view=long&pmid=15758231 | url-status = live | archive-url = https://web.archive.org/web/20070929092242/http://mic.sgmjournals.org/cgi/content/full/151/3/855?view=long&pmid=15758231 | archive-date = 29 September 2007 | df = dmy-all | doi-access = free }}</ref> [[polyphosphate]],<ref>{{cite journal | vauthors = Shiba T, Tsutsumi K, Ishige K, Noguchi T | title = Inorganic polyphosphate and polyphosphate kinase: their novel biological functions and applications | journal = Biochemistry. Biokhimiia | volume = 65 | issue = 3 | pages = 315–23 | date = March 2000 | pmid = 10739474 | url = http://protein.bio.msu.ru/biokhimiya/contents/v65/full/65030375.html | url-status = live | archive-url = https://web.archive.org/web/20060925070012/http://protein.bio.msu.ru/biokhimiya/contents/v65/full/65030375.html | archive-date = 25 September 2006 | df = dmy-all }}</ref> [[sulfur]]<ref>{{cite journal | vauthors = Brune DC | title = Isolation and characterization of sulfur globule proteins from Chromatium vinosum and Thiocapsa roseopersicina | journal = Archives of Microbiology | volume = 163 | issue = 6 | pages = 391–99 | date = June 1995 | pmid = 7575095 | doi = 10.1007/BF00272127 | bibcode = 1995ArMic.163..391B | s2cid = 22279133 }}</ref> or [[polyhydroxyalkanoates]].<ref>{{cite journal | vauthors = Kadouri D, Jurkevitch E, Okon Y, Castro-Sowinski S | title = Ecological and agricultural significance of bacterial polyhydroxyalkanoates | journal = Critical Reviews in Microbiology | volume = 31 | issue = 2 | pages = 55–67 | year = 2005 | pmid = 15986831 | doi = 10.1080/10408410590899228 | s2cid = 4098268 }}</ref> Bacteria such as the [[Photosynthesis|photosynthetic]] [[cyanobacteria]], produce internal [[Vacuole#Bacteria|gas vacuoles]], which they use to regulate their buoyancy, allowing them to move up or down into water layers with different light intensities and nutrient levels.<ref>{{cite journal | vauthors = Walsby AE | title = Gas vesicles | journal = Microbiological Reviews | volume = 58 | issue = 1 | pages = 94–144 | date = March 1994 | pmid = 8177173 | pmc = 372955 | doi = 10.1128/MMBR.58.1.94-144.1994 }}</ref>

=== Extracellular structures ===
{{further|Cell envelope}}
Around the outside of the cell membrane is the [[cell wall]]. Bacterial cell walls are made of [[peptidoglycan]] (also called murein), which is made from [[polysaccharide]] chains cross-linked by [[peptide]]s containing D-[[amino acid]]s.<ref>{{cite journal | vauthors = van Heijenoort J | s2cid = 46066256 | title = Formation of the glycan chains in the synthesis of bacterial peptidoglycan | journal = Glycobiology | volume = 11 | issue = 3 | pages = 25R–36R | date = March 2001 | pmid = 11320055 | doi = 10.1093/glycob/11.3.25R | doi-access = free }}</ref> Bacterial cell walls are different from the cell walls of [[plant]]s and [[fungus|fungi]], which are made of [[cellulose]] and [[chitin]], respectively.<ref name=Koch>{{cite journal | vauthors = Koch AL | title = Bacterial wall as target for attack: past, present, and future research | journal = Clinical Microbiology Reviews | volume = 16 | issue = 4 | pages = 673–87 | date = October 2003 | pmid = 14557293 | pmc = 207114 | doi = 10.1128/CMR.16.4.673-687.2003 }}</ref> The cell wall of bacteria is also distinct from that of achaea, which do not contain peptidoglycan. The cell wall is essential to the survival of many bacteria, and the antibiotic [[penicillin]] (produced by a fungus called ''[[Penicillium]]'') is able to kill bacteria by inhibiting a step in the synthesis of peptidoglycan.<ref name=Koch />

There are broadly speaking two different types of cell wall in bacteria, that classify bacteria into [[Gram-positive bacteria]] and [[Gram-negative bacteria]]. The names originate from the reaction of cells to the [[Gram stain]], a long-standing test for the classification of bacterial species.<ref name=Gram>{{cite journal| vauthors = Gram HC |author-link = Hans Christian Gram |year=1884 |title=Über die isolierte Färbung der Schizomyceten in Schnitt- und Trockenpräparaten |journal=Fortschr. Med. |volume=2 |pages=185–89}}</ref>

Gram-positive bacteria possess a thick cell wall containing many layers of peptidoglycan and [[teichoic acid]]s. In contrast, Gram-negative bacteria have a relatively thin cell wall consisting of a few layers of peptidoglycan surrounded by a second [[Lipid bilayer|lipid membrane]] containing [[lipopolysaccharide]]s and [[lipoprotein]]s. Most bacteria have the Gram-negative cell wall, and only members of the ''[[Bacillota]]'' group and [[actinomycetota]] (previously known as the low G+C and high G+C Gram-positive bacteria, respectively) have the alternative Gram-positive arrangement.<ref>{{cite journal | vauthors = Hugenholtz P | title = Exploring prokaryotic diversity in the genomic era | journal = Genome Biology | volume = 3 | issue = 2 | page = REVIEWS0003 | year = 2002 | pmid = 11864374 | pmc = 139013 | doi = 10.1186/gb-2002-3-2-reviews0003 | doi-access = free }}</ref> These differences in structure can produce differences in antibiotic susceptibility; for instance, [[vancomycin]] can kill only Gram-positive bacteria and is ineffective against Gram-negative [[pathogen]]s, such as ''[[Haemophilus influenzae]]'' or ''[[Pseudomonas aeruginosa]]''.<ref>{{cite journal | vauthors = Walsh FM, Amyes SG | title = Microbiology and drug resistance mechanisms of fully resistant pathogens | journal = Current Opinion in Microbiology | volume = 7 | issue = 5 | pages = 439–44 | date = October 2004 | pmid = 15451497 | doi = 10.1016/j.mib.2004.08.007 | url = http://mural.maynoothuniversity.ie/13551/1/FW-Microbiology-2004.pdf }}</ref> Some bacteria have cell wall structures that are neither classically Gram-positive or Gram-negative. This includes clinically important bacteria such as [[mycobacteria]] which have a thick peptidoglycan cell wall like a Gram-positive bacterium, but also a second outer layer of lipids.<ref>{{cite journal | vauthors = Alderwick LJ, Harrison J, Lloyd GS, Birch HL | title = The Mycobacterial Cell Wall – Peptidoglycan and Arabinogalactan | journal = Cold Spring Harbor Perspectives in Medicine | volume = 5 | issue = 8 | page = a021113 | date = March 2015 | pmid = 25818664 | pmc = 4526729 | doi = 10.1101/cshperspect.a021113 }}</ref>

In many bacteria, an [[S-layer]] of rigidly arrayed protein molecules covers the outside of the cell.<ref name="pmid24509785">{{cite journal |vauthors=Fagan RP, Fairweather NF |title=Biogenesis and functions of bacterial S-layers |journal=Nature Reviews. Microbiology |volume=12 |issue=3 |pages=211–22 |date=March 2014 |pmid=24509785 |doi=10.1038/nrmicro3213 |s2cid=24112697 |url=http://eprints.whiterose.ac.uk/97080/1/Fagan%20and%20Fairweather_for%20deposit.pdf }}</ref> This layer provides chemical and physical protection for the cell surface and can act as a [[macromolecule|macromolecular]] [[diffusion barrier]]. S-layers have diverse functions and are known to act as virulence factors in ''[[Campylobacter]]'' species and contain surface [[enzyme]]s in ''[[Bacillus stearothermophilus]]''.<ref name="pmid16013216">{{cite journal |vauthors=Thompson SA |title=Campylobacter surface-layers (S-layers) and immune evasion |journal=Annals of Periodontology |volume=7 |issue=1 |pages=43–53 |date=December 2002 |pmid=16013216 |pmc=2763180 |doi=10.1902/annals.2002.7.1.43 }}</ref><ref>{{cite journal | vauthors = Beveridge TJ, Pouwels PH, Sára M, Kotiranta A, Lounatmaa K, Kari K, Kerosuo E, Haapasalo M, Egelseer EM, Schocher I, Sleytr UB, Morelli L, Callegari ML, Nomellini JF, Bingle WH, Smit J, Leibovitz E, Lemaire M, Miras I, Salamitou S, Béguin P, Ohayon H, Gounon P, Matuschek M, Koval SF | title = Functions of S-layers | journal = FEMS Microbiology Reviews | volume = 20 | issue = 1–2 | pages = 99–149 | date = June 1997 | pmid = 9276929 | doi = 10.1111/j.1574-6976.1997.tb00305.x }}</ref>
[[File:EMpylori.jpg|thumb|left|alt=Helicobacter pylori electron micrograph, showing multiple flagella on the cell surface|''[[Helicobacter pylori]]'' electron micrograph, showing multiple flagella on the cell surface]]

[[Flagellum|Flagella]] are rigid protein structures, about 20&nbsp;nanometres in diameter and up to 20&nbsp;micrometres in length, that are used for [[motility]]. Flagella are driven by the energy released by the transfer of [[ion]]s down an [[electrochemical gradient]] across the cell membrane.<ref>{{cite book | vauthors = Kojima S, Blair DF | title = The Bacterial Flagellar Motor: Structure and Function of a Complex Molecular Machine | volume = 233 | pages = 93–134 | year = 2004 | pmid = 15037363 | doi = 10.1016/S0074-7696(04)33003-2 | isbn = 978-0-12-364637-8 | series = International Review of Cytology }}</ref>

[[Fimbria (bacteriology)|Fimbriae]] (sometimes called "[[pilus#Fimbriae|attachment pili]]") are fine filaments of protein, usually 2–10&nbsp;nanometres in diameter and up to several micrometres in length. They are distributed over the surface of the cell, and resemble fine hairs when seen under the [[electron microscope]].{{sfn|Wheelis|2008|page=76}} Fimbriae are believed to be involved in attachment to solid surfaces or to other cells, and are essential for the virulence of some bacterial pathogens.<ref name="pmid33614531">{{cite journal |vauthors=Cheng RA, Wiedmann M |title=Recent Advances in Our Understanding of the Diversity and Roles of Chaperone-Usher Fimbriae in Facilitating Salmonella Host and Tissue Tropism |journal=Frontiers in Cellular and Infection Microbiology |volume=10 |issue= |pages=628043 |date=2020 |pmid=33614531 |pmc=7886704 |doi=10.3389/fcimb.2020.628043 |doi-access=free }}</ref> [[Pilus|Pili]] (''sing''. pilus) are cellular appendages, slightly larger than fimbriae, that can transfer [[genetic material]] between bacterial cells in a process called [[bacterial conjugation|conjugation]] where they are called [[Pilus#Conjugative pili|conjugation pili]] or sex pili (see bacterial genetics, below).<ref>{{cite journal | vauthors = Silverman PM | title = Towards a structural biology of bacterial conjugation | journal = Molecular Microbiology | volume = 23 | issue = 3 | pages = 423–29 | date = February 1997 | pmid = 9044277 | doi = 10.1046/j.1365-2958.1997.2411604.x | s2cid = 24126399 | doi-access = free }}</ref> They can also generate movement where they are called [[pilus#Type IV pili|type IV pili]].<ref>{{cite journal | vauthors = Costa TR, Felisberto-Rodrigues C, Meir A, Prevost MS, Redzej A, Trokter M, Waksman G | title = Secretion systems in Gram-negative bacteria: structural and mechanistic insights | journal = Nature Reviews. Microbiology | volume = 13 | issue = 6 | pages = 343–59 | date = June 2015 | pmid = 25978706 | doi = 10.1038/nrmicro3456 | s2cid = 8664247 }}</ref>

[[Glycocalyx#Glycocalyx in Bacteria and Nature|Glycocalyx]] is produced by many bacteria to surround their cells,<ref name="pmid34130219">{{cite journal |vauthors=Luong P, Dube DH |title=Dismantling the bacterial glycocalyx: Chemical tools to probe, perturb, and image bacterial glycans |journal=Bioorganic & Medicinal Chemistry |volume=42 |issue= |pages=116268 |date=July 2021 |pmid=34130219 |doi=10.1016/j.bmc.2021.116268 | issn=0968-0896 |pmc=8276522 }}</ref> and varies in structural complexity: ranging from a disorganised [[slime layer]] of [[extracellular polymeric substance]]s to a highly structured [[bacterial capsule|capsule]]. These structures can protect cells from engulfment by eukaryotic cells such as [[macrophage]]s (part of the human [[immune system]]).<ref>{{cite journal | vauthors = Stokes RW, Norris-Jones R, Brooks DE, Beveridge TJ, Doxsee D, Thorson LM | title = The glycan-rich outer layer of the cell wall of Mycobacterium tuberculosis acts as an antiphagocytic capsule limiting the association of the bacterium with macrophages | journal = Infection and Immunity | volume = 72 | issue = 10 | pages = 5676–86 | date = October 2004 | pmid = 15385466 | pmc = 517526 | doi = 10.1128/IAI.72.10.5676-5686.2004 }}</ref> They can also act as [[antigen]]s and be involved in cell recognition, as well as aiding attachment to surfaces and the formation of biofilms.<ref name="pmid31320388">{{cite journal |vauthors=Kalscheuer R, Palacios A, Anso I, Cifuente J, Anguita J, Jacobs WR, Guerin ME, Prados-Rosales R |title=The Mycobacterium tuberculosis capsule: a cell structure with key implications in pathogenesis |journal=The Biochemical Journal |volume=476 |issue=14 |pages=1995–2016 |date=July 2019 |pmid=31320388 |pmc=6698057 |doi=10.1042/BCJ20190324}}</ref>

The assembly of these extracellular structures is dependent on [[bacterial secretion system]]s. These transfer proteins from the cytoplasm into the periplasm or into the environment around the cell. Many types of secretion systems are known and these structures are often essential for the [[virulence]] of pathogens, so are intensively studied.<ref name="pmid31320388" />

=== Endospores ===
{{further|Endospore}}
[[File:Gram Stain Anthrax.jpg|thumb|right|alt=Anthrax stained purple|''[[Bacillus anthracis]]'' (stained purple) growing in [[cerebrospinal fluid]]<ref name="pmid11747719">{{cite journal |vauthors=Jernigan JA, Stephens DS, Ashford DA, Omenaca C, Topiel MS, Galbraith M, Tapper M, Fisk TL, Zaki S, Popovic T, Meyer RF, Quinn CP, Harper SA, Fridkin SK, Sejvar JJ, Shepard CW, McConnell M, Guarner J, Shieh WJ, Malecki JM, Gerberding JL, Hughes JM, Perkins BA |title=Bioterrorism-related inhalational anthrax: the first 10 cases reported in the United States |journal=Emerging Infectious Diseases |volume=7 |issue=6 |pages=933–44 |date=2001 |pmid=11747719 |pmc=2631903 |doi=10.3201/eid0706.010604 }}</ref>]]

Some [[Genus|genera]] of Gram-positive bacteria, such as ''[[Bacillus]]'', ''[[Clostridium]]'', ''[[Sporohalobacter]]'', ''[[Anaerobacter]]'', and ''[[Heliobacteria|Heliobacterium]]'', can form highly resistant, dormant structures called ''[[endospore]]s''.<ref>{{cite journal | vauthors = Nicholson WL, Munakata N, Horneck G, Melosh HJ, Setlow P | title = Resistance of Bacillus endospores to extreme terrestrial and extraterrestrial environments | journal = Microbiology and Molecular Biology Reviews | volume = 64 | issue = 3 | pages = 548–72 | date = September 2000 | pmid = 10974126 | pmc = 99004 | doi = 10.1128/MMBR.64.3.548-572.2000 }}</ref> Endospores develop within the cytoplasm of the cell; generally, a single endospore develops in each cell.<ref name="McKenney">{{cite journal | vauthors = McKenney PT, Driks A, Eichenberger P | title = The Bacillus subtilis endospore: assembly and functions of the multilayered coat | journal = Nature Reviews. Microbiology | volume = 11 | issue = 1 | pages = 33–44 | date = January 2013 | pmid = 23202530 | doi = 10.1038/nrmicro2921 | pmc = 9910062 | s2cid = 205498395 | doi-access = free }}</ref> Each endospore contains a core of [[DNA]] and [[ribosome]]s surrounded by a cortex layer and protected by a multilayer rigid coat composed of peptidoglycan and a variety of proteins.<ref name="McKenney" />

Endospores show no detectable [[metabolism]] and can survive extreme physical and chemical stresses, such as high levels of [[ultraviolet|UV light]], [[gamma ray|gamma radiation]], [[detergent]]s, [[disinfectant]]s, heat, freezing, pressure, and [[desiccation]].<ref>{{cite journal | vauthors = Nicholson WL, Fajardo-Cavazos P, Rebeil R, Slieman TA, Riesenman PJ, Law JF, Xue Y | title = Bacterial endospores and their significance in stress resistance | journal = Antonie van Leeuwenhoek | volume = 81 | issue = 1–4 | pages = 27–32 | date = August 2002 | pmid = 12448702 | doi = 10.1023/A:1020561122764 | s2cid = 30639022 }}</ref> In this dormant state, these organisms may remain viable for millions of years.<ref>{{cite journal | vauthors = Vreeland RH, Rosenzweig WD, Powers DW | title = Isolation of a 250 million-year-old halotolerant bacterium from a primary salt crystal | journal = Nature | volume = 407 | issue = 6806 | pages = 897–900 | date = October 2000 | pmid = 11057666 | doi = 10.1038/35038060 | bibcode = 2000Natur.407..897V | s2cid = 9879073 }}</ref><ref>{{cite journal | vauthors = Cano RJ, Borucki MK | title = Revival and identification of bacterial spores in 25- to 40-million-year-old Dominican amber | journal = Science | volume = 268 | issue = 5213 | pages = 1060–64 | date = May 1995 | pmid = 7538699 | doi = 10.1126/science.7538699 | bibcode = 1995Sci...268.1060C }}</ref><ref>{{cite news |url=http://news.bbc.co.uk/2/hi/science/nature/1375505.stm |title=Row over ancient bacteria |date=7 June 2001 |work=BBC News |access-date=26 April 2020 |language=en-GB}}</ref> Endospores even allow bacteria to survive exposure to the [[Hard vacuum|vacuum]] and radiation of [[outer space]], leading to the possibility that bacteria could be distributed throughout the [[Universe]] by [[space dust]], [[meteoroid]]s, [[asteroids]], [[comets]], [[Small Solar System body|planetoids]], or [[directed panspermia]].<ref>{{cite journal | vauthors = Nicholson WL, Schuerger AC, Setlow P | title = The solar UV environment and bacterial spore UV resistance: considerations for Earth-to-Mars transport by natural processes and human spaceflight | journal = Mutation Research | volume = 571 | issue = 1–2 | pages = 249–64 | date = April 2005 | pmid = 15748651 | doi = 10.1016/j.mrfmmm.2004.10.012 }}</ref><ref>{{cite news|url=https://www.economist.com/science-and-technology/2018/04/12/colonising-the-galaxy-is-hard-why-not-send-bacteria-instead|title=Colonising the galaxy is hard. Why not send bacteria instead?|date=12 April 2018|newspaper=The Economist|access-date=26 April 2020|issn=0013-0613}}</ref>

Endospore-forming bacteria can cause disease; for example, [[anthrax]] can be contracted by the inhalation of ''[[Bacillus anthracis]]'' endospores, and contamination of deep puncture wounds with ''[[Clostridium tetani]]'' endospores causes [[tetanus]], which, like [[botulism]], is caused by a toxin released by the bacteria that grow from the spores.<ref name="pmid31111816">{{cite journal |vauthors=Revitt-Mills SA, Vidor CJ, Watts TD, Lyras D, Rood JI, Adams V |title=Virulence Plasmids of the Pathogenic Clostridia |journal=Microbiology Spectrum |volume=7 |issue=3 |pages= |date=May 2019 |pmid=31111816 |doi=10.1128/microbiolspec.GPP3-0034-2018 |s2cid=160013108 |url=https://research.monash.edu/en/publications/c40bc3f1-8952-46b6-b834-841642dd3281 }}</ref> [[Clostridioides difficile infection|''Clostridioides difficile'' infection]], a common problem in healthcare settings, is caused by spore-forming bacteria.<ref name="pmid34245901">{{cite journal |vauthors=Reigadas E, van Prehn J, Falcone M, Fitzpatrick F, Vehreschild MJ, Kuijper EJ, Bouza E |title=How to: prophylactic interventions for prevention of Clostridioides difficile infection |journal=Clinical Microbiology and Infection |volume=27 |issue=12 |pages=1777–1783 |date=July 2021 |pmid=34245901 |doi=10.1016/j.cmi.2021.06.037 |doi-access=free |hdl=1887/3249077 |hdl-access=free }}</ref>

== Metabolism ==
{{further|Microbial metabolism}}
Bacteria exhibit an extremely wide variety of [[metabolism|metabolic]] types.<ref>{{cite journal | vauthors = Nealson KH | title = Post-Viking microbiology: new approaches, new data, new insights | journal = Origins of Life and Evolution of the Biosphere | volume = 29 | issue = 1 | pages = 73–93 | date = January 1999 | pmid = 11536899 | doi = 10.1023/A:1006515817767 | bibcode = 1999OLEB...29...73N | s2cid = 12289639 }}</ref> The distribution of metabolic traits within a group of bacteria has traditionally been used to define their [[Taxonomy (biology)|taxonomy]], but these traits often do not correspond with modern genetic classifications.<ref>{{cite journal | vauthors = Xu J | title = Microbial ecology in the age of genomics and metagenomics: concepts, tools, and recent advances | journal = Molecular Ecology | volume = 15 | issue = 7 | pages = 1713–31 | date = June 2006 | pmid = 16689892 | doi = 10.1111/j.1365-294X.2006.02882.x | s2cid = 16374800 | doi-access = free }}</ref> Bacterial metabolism is classified into [[primary nutritional groups|nutritional groups]] on the basis of three major criteria: the source of [[Energy (biology)|energy]], the [[electron donor]]s used, and the source of [[carbon]] used for growth.<ref>{{cite journal | vauthors = Zillig W | title = Comparative biochemistry of Archaea and Bacteria | journal = Current Opinion in Genetics & Development | volume = 1 | issue = 4 | pages = 544–51 | date = December 1991 | pmid = 1822288 | doi = 10.1016/S0959-437X(05)80206-0 }}</ref>

[[Phototrophy|Phototrophic]] bacteria derive energy from light using [[photosynthesis]], while [[chemotrophy|chemotrophic]] bacteria breaking down chemical compounds through [[oxidation]],<ref name=MicroMetab>{{cite book |title=Microbiology: An Evolving Science |vauthors=Slonczewski JL, Foster JW |edition=3 |publisher=WW Norton & Company |pages=491–44}}</ref> driving metabolism by transferring electrons from a given [[electron donor]] to a [[terminal electron acceptor]] in a [[redox|redox reaction]]. Chemotrophs are further divided by the types of compounds they use to transfer electrons. Bacteria that derive electrons from inorganic compounds such as hydrogen, [[carbon monoxide]], or [[ammonia]] are called [[lithotroph]]s, while those that use organic compounds are called [[organotroph]]s.<ref name=MicroMetab /> Still, more specifically, [[aerobic organism]]s use [[oxygen]] as the terminal electron acceptor, while [[anaerobic organism]]s use other compounds such as [[nitrate]], [[sulfate]], or carbon dioxide.<ref name=MicroMetab />

Many bacteria, called [[heterotroph]]s, derive their carbon from other [[organic compound|organic carbon]]. Others, such as [[cyanobacteria]] and some [[purple bacteria]], are [[autotroph]]ic, meaning they obtain cellular carbon by [[Carbon fixation|fixing]] [[carbon dioxide]].<ref>{{cite journal | vauthors = Hellingwerf KJ, Crielaard W, Hoff WD, Matthijs HC, Mur LR, van Rotterdam BJ | title = Photobiology of bacteria | journal = Antonie van Leeuwenhoek | volume = 65 | issue = 4 | pages = 331–47 | year = 1994 | pmid = 7832590 | doi = 10.1007/BF00872217 | s2cid = 23438926 | url = http://dare.uva.nl/personal/pure/en/publications/photobiology-of-bacteria(61d4ae31-4ab8-4c2c-aeed-f9d9143155ca).html | type = Submitted manuscript }}</ref> In unusual circumstances, the gas [[methane]] can be used by [[methanotroph]]ic bacteria as both a source of [[electron]]s and a substrate for carbon [[anabolism]].<ref>{{cite journal | vauthors = Dalton H | title = The Leeuwenhoek Lecture 2000 the natural and unnatural history of methane-oxidizing bacteria | journal = Philosophical Transactions of the Royal Society of London. Series B, Biological Sciences | volume = 360 | issue = 1458 | pages = 1207–22 | date = June 2005 | pmid = 16147517 | pmc = 1569495 | doi = 10.1098/rstb.2005.1657 }}</ref>

{| class="wikitable" style="margin-left: auto; margin-right: auto"
|+ Nutritional types in bacterial metabolism
|-
!Nutritional type
!Source of energy
!Source of carbon
!Examples
|-
|&nbsp;[[Phototroph]]s&nbsp;
| style="text-align:center;"|Sunlight
| style="text-align:center;"|&nbsp;Organic compounds (photoheterotrophs) or carbon fixation (photoautotrophs)
|&nbsp;[[Cyanobacteria]], [[Green sulfur bacteria]], [[Chloroflexota]], or [[Purple bacteria]]&nbsp;
|-
|&nbsp;[[Lithotroph]]s
| style="text-align:center;"|Inorganic compounds
| style="text-align:center;"|&nbsp;Organic compounds (lithoheterotrophs) or carbon fixation (lithoautotrophs)
|&nbsp;[[Thermodesulfobacteriota]], [[Hydrogenophilaceae]], or [[Nitrospirota]]&nbsp;
|-
|&nbsp;[[Organotroph]]s
| style="text-align:center;"|Organic compounds
| style="text-align:center;"|&nbsp;Organic compounds (chemoheterotrophs) or carbon fixation (chemoautotrophs)
|&nbsp;''[[Bacillus]]'', ''[[Clostridium]]'', or [[Enterobacteriaceae]]&nbsp;
|}

In many ways, bacterial metabolism provides traits that are useful for [[ecological stability]] and for human society. For example, [[diazotroph]]s have the ability to [[nitrogen fixation|fix nitrogen]] gas using the enzyme [[nitrogenase]].<ref name="pmid34108945">{{cite journal |vauthors=Imran A, Hakim S, Tariq M, Nawaz MS, Laraib I, Gulzar U, Hanif MK, Siddique MJ, Hayat M, Fraz A, Ahmad M |title=Diazotrophs for Lowering Nitrogen Pollution Crises: Looking Deep Into the Roots |journal=Frontiers in Microbiology |volume=12 |issue= |pages=637815 |date=2021 |pmid=34108945 |pmc=8180554 |doi=10.3389/fmicb.2021.637815|doi-access=free }}</ref> This trait, which can be found in bacteria of most metabolic types listed above,<ref>{{cite journal | vauthors = Zehr JP, Jenkins BD, Short SM, Steward GF | title = Nitrogenase gene diversity and microbial community structure: a cross-system comparison | journal = Environmental Microbiology | volume = 5 | issue = 7 | pages = 539–54 | date = July 2003 | pmid = 12823187 | doi = 10.1046/j.1462-2920.2003.00451.x | doi-access = free | bibcode = 2003EnvMi...5..539Z }}</ref> leads to the ecologically important processes of [[denitrification]], [[sulfate reduction]], and [[acetogenesis]], respectively.<ref name="pmid32559887">{{cite journal |vauthors=Kosugi Y, Matsuura N, Liang Q, Yamamoto-Ikemoto R |title=Wastewater Treatment using the "Sulfate Reduction, DenitrificationAnammox and Partial Nitrification (SRDAPN)" Process |journal=Chemosphere |volume=256 |issue= |pages=127092 |date=October 2020 |pmid=32559887 |doi=10.1016/j.chemosphere.2020.127092 |bibcode=2020Chmsp.25627092K |s2cid=219476361 }}</ref> Bacterial metabolic processes are important drivers in biological responses to [[pollution]]; for example, [[sulfate-reducing bacteria]] are largely responsible for the production of the highly toxic forms of [[mercury (element)|mercury]] ([[methylmercury|methyl-]] and [[dimethylmercury]]) in the environment.<ref>{{cite journal |vauthors=Morel FM, Kraepiel AM, Amyot M |s2cid=86336987 |year=1998 |title=The chemical cycle and bioaccumulation of mercury |journal=[[Annual Review of Ecology and Systematics]] |volume=29 |pages=543–66 |doi=10.1146/annurev.ecolsys.29.1.543}}</ref> Nonrespiratory anaerobes use [[fermentation (biochemistry)|fermentation]] to generate energy and reducing power, secreting metabolic by-products (such as [[ethanol]] in brewing) as waste. [[Facultative anaerobe]]s can switch between fermentation and different [[terminal electron acceptor]]s depending on the environmental conditions in which they find themselves.<ref name="pmid30862543">{{cite journal |vauthors=Ślesak I, Kula M, Ślesak H, Miszalski Z, Strzałka K |title=How to define obligatory anaerobiosis? An evolutionary view on the antioxidant response system and the early stages of the evolution of life on Earth |journal=Free Radical Biology & Medicine |volume=140 |pages=61–73 |date=August 2019 |pmid=30862543 |doi=10.1016/j.freeradbiomed.2019.03.004 |doi-access=free }}</ref>

== Growth and reproduction ==
{{further|Bacterial growth}}
[[File:Three cell growth types.svg|thumb|upright=1|alt=drawing of showing the processes of binary fission, mitosis, and meiosis|Many bacteria reproduce through [[binary fission]], which is compared to [[mitosis]] and [[meiosis]] in this image.]]
[[File:Salmonella growing on XLD agar.JPG|right|thumb|A culture of ''[[Salmonella]]]]
[[File:E.-coli-growth.gif|thumb|alt=E. coli colony |A colony of ''[[Escherichia coli]]''<ref>{{cite journal | vauthors = Stewart EJ, Madden R, Paul G, Taddei F | title = Aging and death in an organism that reproduces by morphologically symmetric division | journal = PLOS Biology | volume = 3 | issue = 2 | page = e45 | date = February 2005 | pmid = 15685293 | pmc = 546039 | doi = 10.1371/journal.pbio.0030045 | doi-access = free }}</ref>]]

Unlike in multicellular organisms, increases in cell size ([[cell growth]]) and reproduction by [[cell division]] are tightly linked in unicellular organisms. Bacteria grow to a fixed size and then reproduce through [[binary fission]], a form of [[asexual reproduction]].<ref>{{cite journal | vauthors = Koch AL | title = Control of the bacterial cell cycle by cytoplasmic growth | journal = Critical Reviews in Microbiology | volume = 28 | issue = 1 | pages = 61–77 | year = 2002 | pmid = 12003041 | doi = 10.1080/1040-840291046696 | s2cid = 11624182 }}</ref> Under optimal conditions, bacteria can grow and divide extremely rapidly, and some bacterial populations can double as quickly as every 17&nbsp;minutes.{{sfn|Pommerville|2014|page=138}} In cell division, two identical [[clone (genetics)|clone]] daughter cells are produced. Some bacteria, while still reproducing asexually, form more complex reproductive structures that help disperse the newly formed daughter cells. Examples include fruiting body formation by [[myxobacteria]] and aerial [[hypha]]e formation by ''[[Streptomyces]]'' species, or budding. Budding involves a cell forming a protrusion that breaks away and produces a daughter cell.{{sfn|Pommerville|2014|p=557}}

In the laboratory, bacteria are usually grown using solid or liquid media.{{sfn|Wheelis|2008|page=42}} Solid [[Growth medium|growth media]], such as [[agar plate]]s, are used to [[Isolation (microbiology)|isolate]] pure cultures of a bacterial strain. However, liquid growth media are used when the measurement of growth or large volumes of cells are required. Growth in stirred liquid media occurs as an even cell suspension, making the cultures easy to divide and transfer, although isolating single bacteria from liquid media is difficult. The use of selective media (media with specific nutrients added or deficient, or with antibiotics added) can help identify specific organisms.<ref name=Thomson>{{cite journal | vauthors = Thomson RB, Bertram H | title = Laboratory diagnosis of central nervous system infections | journal = Infectious Disease Clinics of North America | volume = 15 | issue = 4 | pages = 1047–71 | date = December 2001 | pmid = 11780267 | doi = 10.1016/S0891-5520(05)70186-0 }}</ref>

Most laboratory techniques for growing bacteria use high levels of nutrients to produce large amounts of cells cheaply and quickly.{{sfn|Wheelis|2008|page=42}} However, in natural environments, nutrients are limited, meaning that bacteria cannot continue to reproduce indefinitely. This nutrient limitation has led the evolution of different growth strategies (see [[r/K selection theory]]). Some organisms can grow extremely rapidly when nutrients become available, such as the formation of [[algal bloom|algal]] and [[cyanobacterial]] blooms that often occur in lakes during the summer.<ref>{{cite journal | vauthors = Paerl HW, Fulton RS, Moisander PH, Dyble J | title = Harmful freshwater algal blooms, with an emphasis on cyanobacteria | journal = TheScientificWorldJournal | volume = 1 | pages = 76–113 | date = April 2001 | pmid = 12805693 | pmc = 6083932 | doi = 10.1100/tsw.2001.16 | doi-access = free }}</ref> Other organisms have adaptations to harsh environments, such as the production of multiple [[antibiotic]]s by Streptomyces that inhibit the growth of competing microorganisms.<ref>{{cite journal | vauthors = Challis GL, Hopwood DA | title = Synergy and contingency as driving forces for the evolution of multiple secondary metabolite production by Streptomyces species | journal = Proceedings of the National Academy of Sciences of the United States of America | volume = 100 | issue = Suppl 2 | pages = 14555–61 | date = November 2003 | pmid = 12970466 | pmc = 304118 | doi = 10.1073/pnas.1934677100 | bibcode = 2003PNAS..10014555C | doi-access = free }}</ref> In nature, many organisms live in communities (e.g., [[biofilm]]s) that may allow for increased supply of nutrients and protection from environmental stresses.<ref name=Davey /> These relationships can be essential for growth of a particular organism or group of organisms ([[syntrophy]]).<ref>{{cite journal | vauthors = Kooijman SA, Auger P, Poggiale JC, Kooi BW | title = Quantitative steps in symbiogenesis and the evolution of homeostasis | journal = Biological Reviews of the Cambridge Philosophical Society | volume = 78 | issue = 3 | pages = 435–63 | date = August 2003 | pmid = 14558592 | doi = 10.1017/S1464793102006127 | s2cid = 41072709 }}</ref>

[[Bacterial growth]] follows four phases. When a population of bacteria first enter a high-nutrient environment that allows growth, the cells need to adapt to their new environment. The first phase of growth is the [[Stationary phase (biology)|lag phase]], a period of slow growth when the cells are adapting to the high-nutrient environment and preparing for fast growth. The lag phase has high biosynthesis rates, as proteins necessary for rapid growth are produced.<ref>{{cite journal |journal=Journal of Bacteriology |year=2019 |volume=201 |issue=7 |pages=e00697-18 |title=Lag Phase is a Dynamic, Organized, Adaptive, and Evolvable Period that Prepares Bacteria for Cell Division |vauthors=Bertrand RL |pmid=30642990 |doi=10.1128/JB.00697-18 |pmc=6416914 }}</ref><ref>{{cite journal | vauthors = Prats C, López D, Giró A, Ferrer J, Valls J | title = Individual-based modelling of bacterial cultures to study the microscopic causes of the lag phase | journal = Journal of Theoretical Biology | volume = 241 | issue = 4 | pages = 939–53 | date = August 2006 | pmid = 16524598 | doi = 10.1016/j.jtbi.2006.01.029 | bibcode = 2006JThBi.241..939P }}</ref> The second phase of growth is the [[Stationary phase (biology)|logarithmic phase]], also known as the exponential phase. The log phase is marked by rapid [[exponential growth]]. The rate at which cells grow during this phase is known as the ''growth rate'' (''k''), and the time it takes the cells to double is known as the ''generation time'' (''g''). During log phase, nutrients are metabolised at maximum speed until one of the nutrients is depleted and starts limiting growth. The third phase of growth is the ''[[Stationary phase (biology)|stationary phase]]'' and is caused by depleted nutrients. The cells reduce their metabolic activity and consume non-essential cellular proteins. The stationary phase is a transition from rapid growth to a stress response state and there is increased [[gene expression|expression of genes]] involved in [[DNA repair]], [[antioxidant|antioxidant metabolism]] and [[active transport|nutrient transport]].<ref>{{cite book | vauthors = Hecker M, Völker U | title = General stress response of Bacillus subtilis and other bacteria | volume = 44 | pages = 35–91 | year = 2001 | pmid = 11407115 | doi = 10.1016/S0065-2911(01)44011-2 | isbn = 978-0-12-027744-5 | series = Advances in Microbial Physiology }}</ref> The final phase is the [[Stationary phase (biology)|death phase]] where the bacteria run out of nutrients and die.<ref>{{cite book |title=Microbiology: An Evolving Science |vauthors=Slonczewski JL, Foster JW |edition=3 |publisher=WW Norton & Company |page=143}}</ref>

== Genetics ==
{{Main|Bacterial genetics}}
[[File:Escherichia coli with phages.jpg|thumb| [[Scanning helium ion microscope|Helium ion microscopy]] image showing [[T4 phage]] infecting ''E. coli''. Some of the attached phage have contracted tails indicating that they have injected their DNA into the host. The bacterial cells are ~ 0.5 μm wide.<ref>{{cite journal | vauthors = Leppänen M, Sundberg LR, Laanto E, de Freitas Almeida GM, Papponen P, Maasilta IJ | title = Imaging Bacterial Colonies and Phage-Bacterium Interaction at Sub-Nanometer Resolution Using Helium-Ion Microscopy | journal = Advanced Biosystems | volume = 1 | issue = 8 | pages = e1700070 | date = August 2017 | pmid = 32646179 | doi = 10.1002/adbi.201700070 | s2cid = 90960276 | url = http://urn.fi/URN:NBN:fi:jyu-202006043941 }}</ref>]]

Most bacteria have a single circular [[chromosome]] that can range in size from only 160,000 [[base pair]]s in the [[endosymbiont|endosymbiotic]] bacteria ''[[Candidatus Carsonella ruddii|Carsonella ruddii]]'',<ref>{{cite journal | vauthors = Nakabachi A, Yamashita A, Toh H, Ishikawa H, Dunbar HE, Moran NA, Hattori M | s2cid = 44570539 | title = The 160-kilobase genome of the bacterial endosymbiont Carsonella | journal = Science | volume = 314 | issue = 5797 | page = 267 | date = October 2006 | pmid = 17038615 | doi = 10.1126/science.1134196 }}</ref> to 12,200,000 base pairs (12.2 Mbp) in the soil-dwelling bacteria ''[[Sorangium cellulosum]]''.<ref>{{cite journal | vauthors = Pradella S, Hans A, Spröer C, Reichenbach H, Gerth K, Beyer S | title = Characterisation, genome size and genetic manipulation of the myxobacterium Sorangium cellulosum So ce56 | journal = Archives of Microbiology | volume = 178 | issue = 6 | pages = 484–92 | date = December 2002 | pmid = 12420170 | doi = 10.1007/s00203-002-0479-2 | bibcode = 2002ArMic.178..484P | s2cid = 21023021 }}</ref> There are many exceptions to this; for example, some ''[[Streptomyces]]'' and ''[[Borrelia]]'' species contain a single linear chromosome,<ref name=":0">{{cite journal | vauthors = Hinnebusch J, Tilly K | title = Linear plasmids and chromosomes in bacteria | journal = Molecular Microbiology | volume = 10 | issue = 5 | pages = 917–22 | date = December 1993 | pmid = 7934868 | doi = 10.1111/j.1365-2958.1993.tb00963.x | s2cid = 23852021 | url = https://zenodo.org/record/1230611 }}</ref><ref>{{cite journal | vauthors = Lin YS, Kieser HM, Hopwood DA, Chen CW | title = The chromosomal DNA of Streptomyces lividans 66 is linear | journal = Molecular Microbiology | volume = 10 | issue = 5 | pages = 923–33 | date = December 1993 | pmid = 7934869 | doi = 10.1111/j.1365-2958.1993.tb00964.x | s2cid = 8536066 }}</ref> while some ''[[Vibrio]]'' species contain more than one chromosome.<ref name=":1">{{cite journal | vauthors = Val ME, Soler-Bistué A, Bland MJ, Mazel D | title = Management of multipartite genomes: the Vibrio cholerae model | journal = Current Opinion in Microbiology | volume = 22 | pages = 120–26 | date = December 2014 | pmid = 25460805 | doi = 10.1016/j.mib.2014.10.003 | s2cid = 215743285 | url = https://hal-pasteur.archives-ouvertes.fr/pasteur-01163283/document }}</ref> Some bacteria contain [[plasmid]]s, small extra-chromosomal molecules of DNA that may contain genes for various useful functions such as [[antibiotic resistance]], metabolic capabilities, or various [[virulence|virulence factors]].<ref name="pmid26104369">{{cite journal |vauthors=Kado CI |title=Historical events that spawned the field of plasmid biology |journal=Microbiology Spectrum |volume=2 |issue=5 |pages=3 |date=October 2014 |pmid=26104369 |doi=10.1128/microbiolspec.PLAS-0019-2013|doi-access=free }}</ref>

Bacteria genomes usually encode a few hundred to a few thousand genes. The genes in bacterial genomes are usually a single continuous stretch of DNA. Although several different types of [[intron]]s do exist in bacteria, these are much rarer than in eukaryotes.<ref>{{cite journal | vauthors = Belfort M, Reaban ME, Coetzee T, Dalgaard JZ | title = Prokaryotic introns and inteins: a panoply of form and function |author-link1=Marlene Belfort| journal = Journal of Bacteriology | volume = 177 | issue = 14 | pages = 3897–903 | date = July 1995 | pmid = 7608058 | pmc = 177115 | doi=10.1128/jb.177.14.3897-3903.1995}}</ref>

Bacteria, as asexual organisms, inherit an identical copy of the parent's genome and are [[Clonal colony|clonal]]. However, all bacteria can evolve by selection on changes to their genetic material [[DNA]] caused by [[genetic recombination]] or [[mutation]]s. Mutations arise from errors made during the replication of DNA or from exposure to [[mutagen]]s. Mutation rates vary widely among different species of bacteria and even among different clones of a single species of bacteria.<ref>{{cite journal | vauthors = Denamur E, Matic I | title = Evolution of mutation rates in bacteria | journal = Molecular Microbiology | volume = 60 | issue = 4 | pages = 820–27 | date = May 2006 | pmid = 16677295 | doi = 10.1111/j.1365-2958.2006.05150.x | s2cid = 20713095 | doi-access = free }}</ref> Genetic changes in bacterial genomes emerge from either random mutation during replication or "stress-directed mutation", where genes involved in a particular growth-limiting process have an increased mutation rate.<ref>{{cite journal | vauthors = Wright BE | title = Stress-directed adaptive mutations and evolution | journal = Molecular Microbiology | volume = 52 | issue = 3 | pages = 643–50 | date = May 2004 | pmid = 15101972 | doi = 10.1111/j.1365-2958.2004.04012.x | s2cid = 1071308 | doi-access = free }}</ref>

Some bacteria transfer genetic material between cells. This can occur in three main ways. First, bacteria can take up exogenous DNA from their environment in a process called [[transformation (genetics)|transformation]].<ref>{{cite journal | vauthors = Chen I, Dubnau D | title = DNA uptake during bacterial transformation | journal = Nature Reviews. Microbiology | volume = 2 | issue = 3 | pages = 241–49 | date = March 2004 | pmid = 15083159 | doi = 10.1038/nrmicro844 | s2cid = 205499369 }}</ref> Many bacteria can [[natural competence|naturally]] take up DNA from the environment, while others must be chemically altered in order to induce them to take up DNA.<ref>{{cite journal | vauthors = Johnsborg O, Eldholm V, Håvarstein LS | title = Natural genetic transformation: prevalence, mechanisms and function | journal = Research in Microbiology | volume = 158 | issue = 10 | pages = 767–78 | date = December 2007 | pmid = 17997281 | doi = 10.1016/j.resmic.2007.09.004 | doi-access = free }}</ref> The development of competence in nature is usually associated with stressful environmental conditions and seems to be an adaptation for facilitating repair of DNA damage in recipient cells.<ref>Bernstein H, Bernstein C, Michod RE (2012). "DNA repair as the primary adaptive function of sex in bacteria and eukaryotes". Chapter 1: pp. 1–49 in: ''DNA Repair: New Research'', Sakura Kimura and Sora Shimizu (eds.). Nova Sci. Publ., Hauppauge, NY {{ISBN|978-1-62100-808-8}}.</ref> Second, [[bacteriophage]]s can integrate into the bacterial chromosome, introducing foreign DNA in a process known as [[transduction (genetics)|transduction]]. Many types of bacteriophage exist; some infect and [[lytic cycle|lyse]] their [[host (biology)|host]] bacteria, while others insert into the bacterial chromosome.<ref>{{cite journal | vauthors = Brüssow H, Canchaya C, Hardt WD | title = Phages and the evolution of bacterial pathogens: from genomic rearrangements to lysogenic conversion | journal = Microbiology and Molecular Biology Reviews | volume = 68 | issue = 3 | pages = 560–602, table of contents | date = September 2004 | pmid = 15353570 | pmc = 515249 | doi = 10.1128/MMBR.68.3.560-602.2004 }}</ref> Bacteria resist phage infection through [[restriction modification system]]s that degrade foreign DNA<ref>{{cite journal | vauthors = Bickle TA, Krüger DH | title = Biology of DNA restriction | journal = Microbiological Reviews | volume = 57 | issue = 2 | pages = 434–50 | date = June 1993 | pmid = 8336674 | pmc = 372918 | doi = 10.1128/MMBR.57.2.434-450.1993 }}</ref> and a system that uses [[CRISPR]] sequences to retain fragments of the genomes of phage that the bacteria have come into contact with in the past, which allows them to block virus replication through a form of [[RNA interference]].<ref>{{cite journal | vauthors = Barrangou R, Fremaux C, Deveau H, Richards M, Boyaval P, Moineau S, Romero DA, Horvath P | title = CRISPR provides acquired resistance against viruses in prokaryotes | journal = Science | volume = 315 | issue = 5819 | pages = 1709–12 | date = March 2007 | pmid = 17379808 | doi = 10.1126/science.1138140 | bibcode = 2007Sci...315.1709B | hdl = 20.500.11794/38902 | s2cid = 3888761 | hdl-access = free }}</ref><ref>{{cite journal | vauthors = Brouns SJ, Jore MM, Lundgren M, Westra ER, Slijkhuis RJ, Snijders AP, Dickman MJ, Makarova KS, Koonin EV, van der Oost J | title = Small CRISPR RNAs guide antiviral defense in prokaryotes | journal = Science | volume = 321 | issue = 5891 | pages = 960–64 | date = August 2008 | pmid = 18703739 | pmc = 5898235 | doi = 10.1126/science.1159689 | bibcode = 2008Sci...321..960B }}</ref> Third, bacteria can transfer genetic material through direct cell contact via [[bacterial conjugation|conjugation]].<ref name="pmid25154632">{{cite journal |vauthors=Cabezón E, Ripoll-Rozada J, Peña A, de la Cruz F, Arechaga I |title=Towards an integrated model of bacterial conjugation |journal=FEMS Microbiology Reviews |volume=39 |issue=1 |pages=81–95 |date=January 2015 |pmid=25154632 |doi=10.1111/1574-6976.12085 |s2cid=34245150 |doi-access=free }}</ref>

In ordinary circumstances, transduction, conjugation, and transformation involve transfer of DNA between individual bacteria of the same species, but occasionally transfer may occur between individuals of different bacterial species, and this may have significant consequences, such as the transfer of antibiotic resistance.<ref>{{cite journal | vauthors = Michod RE, Bernstein H, Nedelcu AM | title = Adaptive value of sex in microbial pathogens | journal = Infection, Genetics and Evolution | volume = 8 | issue = 3 | pages = 267–85 | date = May 2008 | pmid = 18295550 | doi = 10.1016/j.meegid.2008.01.002 | url = http://www.hummingbirds.arizona.edu/Faculty/Michod/Downloads/IGE%20review%20sex.pdf | url-status = live | archive-url = https://web.archive.org/web/20161230121043/http://www.hummingbirds.arizona.edu/Faculty/Michod/Downloads/IGE%20review%20sex.pdf | archive-date = 30 December 2016 | df = dmy-all }}</ref><ref>{{cite journal | vauthors = Hastings PJ, Rosenberg SM, Slack A | title = Antibiotic-induced lateral transfer of antibiotic resistance | journal = Trends in Microbiology | volume = 12 | issue = 9 | pages = 401–14 | date = September 2004 | pmid = 15337159 | doi = 10.1016/j.tim.2004.07.003 }}</ref> In such cases, gene acquisition from other bacteria or the environment is called [[horizontal gene transfer]] and may be common under natural conditions.<ref>{{cite journal | vauthors = Davison J | title = Genetic exchange between bacteria in the environment | journal = Plasmid | volume = 42 | issue = 2 | pages = 73–91 | date = September 1999 | pmid = 10489325 | doi = 10.1006/plas.1999.1421 }}</ref>

== Behaviour ==
=== Movement ===
{{Main|Bacterial motility}}
[[File:Dvulgaris micrograph.JPG|thumb|Transmission electron micrograph of ''[[Desulfovibrio vulgaris]]'' showing a single flagellum at one end of the cell. Scale bar is 0.5 micrometers long.]]

Many bacteria are [[Motility|motile]] (able to move themselves) and do so using a variety of mechanisms. The best studied of these are [[Flagellum|flagella]], long filaments that are turned by a motor at the base to generate propeller-like movement.<ref name=Bardy>{{cite journal |vauthors=Kim KW |title=Electron microscopic observations of prokaryotic surface appendages |journal=Journal of Microbiology (Seoul, Korea) |volume=55 |issue=12 |pages=919–26 |date=December 2017 |pmid=29214488 |doi=10.1007/s12275-017-7369-4 |s2cid=89872403 }}</ref> The bacterial flagellum is made of about 20 proteins, with approximately another 30 proteins required for its regulation and assembly.<ref name=Bardy /> The flagellum is a rotating structure driven by a reversible motor at the base that uses the [[electrochemical gradient]] across the membrane for power.<ref>{{cite journal | vauthors = Macnab RM | title = The bacterial flagellum: reversible rotary propellor and type III export apparatus | journal = Journal of Bacteriology | volume = 181 | issue = 23 | pages = 7149–53 | date = December 1999 | pmid = 10572114 | pmc = 103673 | doi = 10.1128/JB.181.23.7149-7153.1999 }}</ref>

[[File:Flagella.svg|thumb|The different arrangements of bacterial flagella: A-Monotrichous; B-Lophotrichous; C-Amphitrichous; D-Peritrichous]]
Bacteria can use flagella in different ways to generate different kinds of movement. Many bacteria (such as ''[[Escherichia coli|E. coli]]'') have two distinct modes of movement: forward movement (swimming) and [[tumbling]]. The tumbling allows them to [[reorient]] and makes their movement a three-[[Dimension|dimensional]] [[random walk]].<ref>{{cite journal | vauthors = Wu M, Roberts JW, Kim S, Koch DL, DeLisa MP | title = Collective bacterial dynamics revealed using a three-dimensional population-scale defocused particle tracking technique | journal = Applied and Environmental Microbiology | volume = 72 | issue = 7 | pages = 4987–94 | date = July 2006 | pmid = 16820497 | pmc = 1489374 | doi = 10.1128/AEM.00158-06 | bibcode = 2006ApEnM..72.4987W }}</ref> Bacterial species differ in the number and arrangement of flagella on their surface; some have a single [[flagellum]] (''[[Flagellum#Flagellar arrangement schemes|monotrichous]]''), a flagellum at each end (''[[Flagellum#Flagellar arrangement schemes|amphitrichous]]''), clusters of flagella at the poles of the cell (''[[Flagellum#Flagellar arrangement schemes|lophotrichous]]''), while others have flagella distributed over the entire surface of the cell (''[[Flagellum#Flagellar arrangement schemes|peritrichous]]''). The flagella of a group of bacteria, the [[spirochaete]]s, are found between two membranes in the periplasmic space. They have a distinctive [[helix|helical]] body that twists about as it moves.<ref name=Bardy />

Two other types of bacterial motion are called [[twitching motility]] that relies on a structure called the [[pilus#Type IV pili|type IV pilus]],<ref>{{cite journal | vauthors = Mattick JS | title = Type IV pili and twitching motility | journal = Annual Review of Microbiology | volume = 56 | pages = 289–314 | year = 2002 | pmid = 12142488 | doi = 10.1146/annurev.micro.56.012302.160938 }}</ref> and [[Bacterial gliding|gliding motility]], that uses other mechanisms. In twitching motility, the rod-like pilus extends out from the cell, binds some substrate, and then retracts, pulling the cell forward.<ref>{{cite journal | vauthors = Merz AJ, So M, Sheetz MP | title = Pilus retraction powers bacterial twitching motility | journal = Nature | volume = 407 | issue = 6800 | pages = 98–102 | date = September 2000 | pmid = 10993081 | doi = 10.1038/35024105 | s2cid = 4425775 | bibcode = 2000Natur.407...98M }}</ref>

Motile bacteria are attracted or repelled by certain [[stimulus (physiology)|stimuli]] in behaviours called ''[[taxis|taxes]]'': these include [[chemotaxis]], [[phototaxis]], [[taxis|energy taxis]], and [[magnetotaxis]].<ref>{{cite journal | vauthors = Lux R, Shi W | title = Chemotaxis-guided movements in bacteria | journal = Critical Reviews in Oral Biology and Medicine | volume = 15 | issue = 4 | pages = 207–20 | date = July 2004 | pmid = 15284186 | doi = 10.1177/154411130401500404 | doi-access = free }}</ref><ref>{{cite journal | vauthors = Schweinitzer T, Josenhans C | title = Bacterial energy taxis: a global strategy? | journal = Archives of Microbiology | volume = 192 | issue = 7 | pages = 507–20 | date = July 2010 | pmid = 20411245 | pmc = 2886117 | doi = 10.1007/s00203-010-0575-7 | bibcode = 2010ArMic.192..507S }}</ref><ref>{{cite journal | vauthors = Frankel RB, Bazylinski DA, Johnson MS, Taylor BL | title = Magneto-aerotaxis in marine coccoid bacteria | journal = Biophysical Journal | volume = 73 | issue = 2 | pages = 994–1000 | date = August 1997 | pmid = 9251816 | pmc = 1180996 | doi = 10.1016/S0006-3495(97)78132-3 | bibcode = 1997BpJ....73..994F }}</ref> In one peculiar group, the myxobacteria, individual bacteria move together to form waves of cells that then differentiate to form fruiting bodies containing spores.<ref name=Kaiser /> The myxobacteria move only when on solid surfaces, unlike ''E. coli'', which is motile in liquid or solid media.<ref name="pmid21910630">{{cite journal |vauthors=Nan B, Zusman DR |title=Uncovering the mystery of gliding motility in the myxobacteria |journal=[[Annual Review of Genetics]] |volume=45 |pages=21–39 |date=2011 |pmid=21910630 |pmc=3397683 |doi=10.1146/annurev-genet-110410-132547 }}</ref>

Several ''[[Listeria]]'' and ''[[Shigella]]'' species move inside host cells by usurping the [[cytoskeleton]], which is normally used to move [[organelle]]s inside the cell. By promoting [[actin]] [[biopolymer|polymerisation]] at one pole of their cells, they can form a kind of tail that pushes them through the host cell's cytoplasm.<ref>{{cite journal | vauthors = Goldberg MB | title = Actin-based motility of intracellular microbial pathogens | journal = Microbiology and Molecular Biology Reviews | volume = 65 | issue = 4 | pages = 595–626, table of contents | date = December 2001 | pmid = 11729265 | pmc = 99042 | doi = 10.1128/MMBR.65.4.595-626.2001 }}</ref>

=== Communication ===
{{See also|Prokaryote#Sociality}}
A few bacteria have chemical systems that generate light. This [[bioluminescence]] often occurs in bacteria that live in association with fish, and the light probably serves to attract fish or other large animals.<ref name="pmid31817999">{{cite journal |vauthors=Calcagnile M, Tredici SM, Talà A, Alifano P |title=Bacterial Semiochemicals and Transkingdom Interactions with Insects and Plants |journal=Insects |volume=10 |issue=12 |date=December 2019 |page=441 |pmid=31817999 |pmc=6955855 |doi=10.3390/insects10120441 |doi-access=free }}</ref>

Bacteria often function as multicellular aggregates known as [[biofilms]], exchanging a variety of molecular signals for [[Cell signaling|intercell communication]] and engaging in coordinated multicellular behaviour.<ref name=shapiro1>{{cite journal | vauthors = Shapiro JA | title = Thinking about bacterial populations as multicellular organisms | journal = [[Annual Review of Microbiology]] | volume = 52 | pages = 81–104 | year = 1998 | pmid = 9891794 | doi = 10.1146/annurev.micro.52.1.81 | url = http://www.sci.uidaho.edu/newton/math501/Sp05/Shapiro.pdf | archive-url = https://web.archive.org/web/20110717183759/http://www.sci.uidaho.edu/newton/math501/Sp05/Shapiro.pdf | url-status = dead | archive-date = 17 July 2011 }}</ref><ref name=costerton1>{{cite journal | vauthors = Costerton JW, Lewandowski Z, Caldwell DE, Korber DR, Lappin-Scott HM | title = Microbial biofilms | journal = [[Annual Review of Microbiology]] | volume = 49 | pages = 711–45 | year = 1995 | pmid = 8561477 | doi = 10.1146/annurev.mi.49.100195.003431 }}</ref>

The communal benefits of multicellular cooperation include a cellular [[division of labour]], accessing resources that cannot effectively be used by single cells, collectively defending against antagonists, and optimising population survival by differentiating into distinct cell types.<ref name=shapiro1 /> For example, bacteria in biofilms can have more than five hundred times increased resistance to [[Bactericide|antibacterial]] agents than individual "planktonic" bacteria of the same species.<ref name=costerton1 />

One type of intercellular communication by a [[cell signal|molecular signal]] is called [[quorum sensing]], which serves the purpose of determining whether the local population density is sufficient to support investment in processes that are only successful if large numbers of similar organisms behave similarly, such as excreting [[digestive enzyme]]s or emitting light.<ref name="pmid31732932">{{cite book |vauthors=Miller DP, Lamont RJ |title=Oral Mucosal Immunity and Microbiome |chapter=Signaling Systems in Oral Bacteria |series=Advances in Experimental Medicine and Biology |volume=1197 |pages=27–43 |date=2019 |pmid=31732932 |doi=10.1007/978-3-030-28524-1_3|isbn=978-3-030-28523-4 |s2cid=208063186 }}</ref><ref name="pmid29789364">{{cite journal |vauthors=Abisado RG, Benomar S, Klaus JR, Dandekar AA, Chandler JR |title=Bacterial Quorum sensing and microbial community interactions |journal=mBio |volume=9 |issue=3 |date=May 2018 |pmid=29789364 |pmc=5964356 |doi=10.1128/mBio.02331-17}}</ref> Quorum sensing enables bacteria to coordinate [[gene expression]] and to produce, release, and detect [[autoinducer]]s or [[pheromones]] that accumulate with the growth in cell population.<ref name="pmid11544353">{{cite journal | vauthors = Miller MB, Bassler BL | s2cid = 1099089 | title = Quorum sensing in bacteria | journal = [[Annual Review of Microbiology]] | volume = 55 | pages = 165–99 | year = 2001 | pmid = 11544353 | doi = 10.1146/annurev.micro.55.1.165 }}</ref>

== Classification and identification ==
{{Main|Bacterial taxonomy}}
{{further|Scientific classification|Systematics|Bacterial phyla|Clinical pathology}}
[[File:Streptococcus mutans Gram.jpg|thumb|alt=blue stain of Streptococcus mutans|''Streptococcus mutans'' visualised with a Gram stain.]]
[[File:Anillo de la vida.png|thumb|300px|[[Phylogenetic tree]] showing the diversity of bacteria, compared to other organisms. Here bacteria are represented by three main supergroups: the [[CPR group|CPR]] [[ultramicrobacteria]]s, [[Terrabacteria]] and [[Gracilicutes]] according to recent genomic analyzes (2019).<ref name="Zhu">{{cite journal | vauthors = Zhu Q, Mai U, Pfeiffer W, Janssen S, Asnicar F, Sanders JG, Belda-Ferre P, Al-Ghalith GA, Kopylova E, McDonald D, Kosciolek T, Yin JB, Huang S, Salam N, Jiao JY, Wu Z, Xu ZZ, Cantrell K, Yang Y, Sayyari E, Rabiee M, Morton JT, Podell S, Knights D, Li WJ, Huttenhower C, Segata N, Smarr L, Mirarab S, Knight R | display-authors = 6 | title = Phylogenomics of 10,575 genomes reveals evolutionary proximity between domains Bacteria and Archaea | journal = Nature Communications | volume = 10 | issue = 1 | pages = 5477 | date = December 2019 | pmid = 31792218 | pmc = 6889312 | doi = 10.1038/s41467-019-13443-4 | bibcode = 2019NatCo..10.5477Z }}</ref>]]

[[Scientific classification|Classification]] seeks to describe the diversity of bacterial species by naming and grouping organisms based on similarities. Bacteria can be classified on the basis of cell structure, [[Cell metabolism|cellular metabolism]] or on differences in cell components, such as [[DNA]], [[fatty acid]]s, pigments, [[antigen]]s and [[quinone]]s.<ref name=Thomson /> While these schemes allowed the identification and classification of bacterial strains, it was unclear whether these differences represented variation between distinct species or between strains of the same species. This uncertainty was due to the lack of distinctive structures in most bacteria, as well as [[lateral gene transfer]] between unrelated species.<ref>{{cite journal | vauthors = Boucher Y, Douady CJ, Papke RT, Walsh DA, Boudreau ME, Nesbø CL, Case RJ, Doolittle WF | title = Lateral gene transfer and the origins of prokaryotic groups | journal = [[Annual Review of Genetics]] | volume = 37 | pages = 283–328 | year = 2003 | pmid = 14616063 | doi = 10.1146/annurev.genet.37.050503.084247 }}</ref> Due to lateral gene transfer, some closely related bacteria can have very different morphologies and metabolisms. To overcome this uncertainty, modern bacterial classification emphasises [[molecular systematics]], using genetic techniques such as [[guanine]] [[cytosine]] [[GC-content|ratio]] determination, genome-genome hybridisation, as well as [[DNA sequencing|sequencing]] genes that have not undergone extensive lateral gene transfer, such as the [[ribosomal DNA|rRNA gene]].<ref>{{cite journal | vauthors = Olsen GJ, Woese CR, Overbeek R | title = The winds of (evolutionary) change: breathing new life into microbiology | journal = Journal of Bacteriology | volume = 176 | issue = 1 | pages = 1–6 | date = January 1994 | pmid = 8282683 | pmc = 205007 | doi = 10.2172/205047 }}</ref> Classification of bacteria is determined by publication in the International Journal of Systematic Bacteriology,<ref>{{cite web |url=http://ijs.sgmjournals.org/ |title=IJSEM Home |publisher=Ijs.sgmjournals.org |date=28 October 2011 |access-date=4 November 2011 |url-status=live |archive-url=https://web.archive.org/web/20111019160924/http://ijs.sgmjournals.org/ |archive-date=19 October 2011 }}</ref> and Bergey's Manual of Systematic Bacteriology.<ref>{{cite web |url=http://www.bergeys.org/ |title=Bergey's Manual Trust |publisher=Bergeys.org |access-date=4 November 2011 |url-status=live |archive-url=https://web.archive.org/web/20111107002356/http://www.bergeys.org/ |archive-date=7 November 2011 }}</ref> The [[International Committee on Systematic Bacteriology]] (ICSB) maintains international rules for the naming of bacteria and taxonomic categories and for the ranking of them in the [[International Code of Nomenclature of Bacteria]].<ref name="pmid25921438">{{cite journal |vauthors=Hedlund BP, Dodsworth JA, Staley JT |title=The changing landscape of microbial biodiversity exploration and its implications for systematics |journal=Systematic and Applied Microbiology |volume=38 |issue=4 |pages=231–36 |date=June 2015 |pmid=25921438 |doi=10.1016/j.syapm.2015.03.003 }}</ref>

Historically, bacteria were considered a part of the [[plants|Plantae]], the Plant kingdom, and were called "Schizomycetes" (fission-fungi).<ref>"[https://www.merriam-webster.com/medical/Schizomycetes Schizomycetes]". Merriam-Webster Medical Dictionary. Accessed 3 August 2021.</ref> For this reason, collective bacteria and other microorganisms in a host are often called "flora".<ref name="pmid33180890">{{cite journal |vauthors=Brown MM, Horswill AR |title=Staphylococcus epidermidis-Skin friend or foe? |journal=PLOS Pathogens |volume=16 |issue=11 |pages=e1009026 |date=November 2020 |pmid=33180890 |pmc=7660545 |doi=10.1371/journal.ppat.1009026 |doi-access=free }}</ref>
The term "bacteria" was traditionally applied to all microscopic, single-cell prokaryotes. However, molecular systematics showed prokaryotic life to consist of two separate [[domain (biology)|domains]], originally called Eubacteria and Archaebacteria, but now called Bacteria and Archaea that evolved independently from an ancient common ancestor.{{sfn | Hall | 2008 | p=145}} The archaea and eukaryotes are more closely related to each other than either is to the bacteria. These two domains, along with Eukarya, are the basis of the [[three-domain system]], which is currently the most widely used classification system in microbiology.<ref name=Gupta>{{cite journal | vauthors = Gupta RS | title = The natural evolutionary relationships among prokaryotes | journal = Critical Reviews in Microbiology | volume = 26 | issue = 2 | pages = 111–31 | year = 2000 | pmid = 10890353 | doi = 10.1080/10408410091154219 | citeseerx = 10.1.1.496.1356 | s2cid = 30541897 }}</ref> However, due to the relatively recent introduction of molecular systematics and a rapid increase in the number of genome sequences that are available, bacterial classification remains a changing and expanding field.<ref name=Rappe>{{cite journal | vauthors = Rappé MS, Giovannoni SJ | s2cid = 10781051 | title = The uncultured microbial majority | journal = [[Annual Review of Microbiology]] | volume = 57 | pages = 369–94 | year = 2003 | pmid = 14527284 | doi = 10.1146/annurev.micro.57.030502.090759 }}</ref><ref>{{cite journal | vauthors = Doolittle RF | title = Evolutionary aspects of whole-genome biology | journal = Current Opinion in Structural Biology | volume = 15 | issue = 3 | pages = 248–53 | date = June 2005 | pmid = 15963888 | doi = 10.1016/j.sbi.2005.04.001 }}</ref> For example, [[Thomas Cavalier-Smith|Cavalier-Smith]] argued that the Archaea and Eukaryotes evolved from Gram-positive bacteria.<ref name="Cavalier-Smith2002">{{cite journal | vauthors = Cavalier-Smith T | title = The neomuran origin of archaebacteria, the negibacterial root of the universal tree and bacterial megaclassification | journal = International Journal of Systematic and Evolutionary Microbiology | volume = 52 | issue = Pt 1 | pages = 7–76 | date = January 2002 | pmid = 11837318 | doi = 10.1099/00207713-52-1-7 | doi-access = free }}</ref>

The identification of bacteria in the laboratory is particularly relevant in [[medicine]], where the correct treatment is determined by the bacterial species causing an infection. Consequently, the need to identify human pathogens was a major impetus for the development of techniques to identify bacteria.{{sfn|Pommerville|2014|p=15−31}}

The ''[[Gram stain]]'', developed in 1884 by [[Hans Christian Gram]], characterises bacteria based on the structural characteristics of their cell walls.{{sfn | Krasner | 2014 | p=77}}<ref name=Gram /> The thick layers of peptidoglycan in the "Gram-positive" cell wall stain purple, while the thin "Gram-negative" cell wall appears pink.{{sfn | Krasner | 2014 | p=77}} By combining morphology and Gram-staining, most bacteria can be classified as belonging to one of four groups (Gram-positive cocci, Gram-positive bacilli, Gram-negative cocci and Gram-negative bacilli). Some organisms are best identified by stains other than the Gram stain, particularly mycobacteria or ''Nocardia'', which show [[acid-fastness|acid fast]]ness on [[Ziehl-Neelsen stain|Ziehl–Neelsen]] or similar stains.<ref>{{cite journal | vauthors = Woods GL, Walker DH | title = Detection of infection or infectious agents by use of cytologic and histologic stains | journal = Clinical Microbiology Reviews | volume = 9 | issue = 3 | pages = 382–404 | date = July 1996 | pmid = 8809467 | pmc = 172900 | doi = 10.1128/CMR.9.3.382 }}</ref> Other organisms may need to be identified by their growth in special media, or by other techniques, such as [[serology]].<ref name="pmid31354679">{{cite journal |vauthors=Tang S, Orsi RH, Luo H, Ge C, Zhang G, Baker RC, Stevenson A, Wiedmann M |title=Assessment and Comparison of Molecular Subtyping and Characterization Methods for Salmonella |journal=Frontiers in Microbiology |volume=10 |pages=1591 |date=2019 |pmid=31354679 |pmc=6639432 |doi=10.3389/fmicb.2019.01591 |doi-access=free }}</ref>

[[Microbiological culture|Culture]] techniques are designed to promote the growth and identify particular bacteria while restricting the growth of the other bacteria in the sample.{{sfn | Krasner | 2014 | p=87–89}} Often these techniques are designed for specific specimens; for example, a [[sputum]] sample will be treated to identify organisms that cause [[pneumonia]], while [[feces|stool]] specimens are cultured on [[selective media]] to identify organisms that cause [[diarrhea]] while preventing growth of non-pathogenic bacteria. Specimens that are normally sterile, such as [[blood]], [[urine]] or [[cerebrospinal fluid|spinal fluid]], are cultured under conditions designed to grow all possible organisms.<ref name=Thomson /><ref>{{cite journal | vauthors = Weinstein MP | title = Clinical importance of blood cultures | journal = Clinics in Laboratory Medicine | volume = 14 | issue = 1 | pages = 9–16 | date = March 1994 | pmid = 8181237 | doi = 10.1016/S0272-2712(18)30390-1 }}</ref> Once a pathogenic organism has been isolated, it can be further characterised by its morphology, growth patterns (such as [[aerobic organism|aerobic]] or [[anaerobic organism|anaerobic]] growth), [[hemolysis (microbiology)|patterns of hemolysis]], and staining.<ref name="pmid30387415">{{cite journal |vauthors=Riley LW |title=Laboratory Methods in Molecular Epidemiology: Bacterial Infections |journal=Microbiology Spectrum |volume=6 |issue=6 |date=November 2018 |pmid=30387415 |doi=10.1128/microbiolspec.AME-0004-2018 |s2cid=54234977 }}</ref>

As with bacterial classification, identification of bacteria is increasingly using molecular methods,<ref name="pmid33809163">{{cite journal |vauthors=Lenkowski M, Nijakowski K, Kaczmarek M, Surdacka A |title=The Loop-Mediated Isothermal Amplification Technique in Periodontal Diagnostics: A Systematic Review |journal=Journal of Clinical Medicine |volume=10 |issue=6 |date=March 2021 |page=1189 |pmid=33809163 |pmc=8000232 |doi=10.3390/jcm10061189 |doi-access=free }}</ref> and [[Matrix-assisted laser desorption/ionization|mass spectroscopy]].<ref name="pmid34041492">{{cite journal |vauthors=Alizadeh M, Yousefi L, Pakdel F, Ghotaslou R, Rezaee MA, Khodadadi E, Oskouei MA, Soroush Barhaghi MH, Kafil HS |title=MALDI-TOF Mass Spectroscopy Applications in Clinical Microbiology |journal=Advances in Pharmacological and Pharmaceutical Sciences |volume=2021 |issue= |pages=9928238 |date=2021 |pmid=34041492 |pmc=8121603 |doi=10.1155/2021/9928238 |url= |doi-access=free }}</ref> Most bacteria have not been characterised and there are many species that cannot be [[microbiological culture|grown]] in the laboratory.<ref name=Dudek>{{cite journal | vauthors = Dudek NK, Sun CL, Burstein D | title = Novel Microbial Diversity and Functional Potential in the Marine Mammal Oral Microbiome | journal = Current Biology | volume = 27 | issue = 24 | pages = 3752–62 | year = 2017 | doi = 10.1016/j.cub.2017.10.040 | pmid = 29153320 | s2cid = 43864355 | url = https://escholarship.org/content/qt1w91s3vq/qt1w91s3vq.pdf?t=pghuwe | doi-access = free | bibcode = 2017CBio...27E3752D }}</ref> Diagnostics using DNA-based tools, such as [[polymerase chain reaction]], are increasingly popular due to their specificity and speed, compared to culture-based methods.<ref>{{cite journal | vauthors = Louie M, Louie L, Simor AE | title = The role of DNA amplification technology in the diagnosis of infectious diseases | journal = CMAJ | volume = 163 | issue = 3 | pages = 301–09 | date = August 2000 | pmid = 10951731 | pmc = 80298 | url = http://www.cmaj.ca/cgi/content/full/163/3/301 | url-status = live | archive-url = https://web.archive.org/web/20060614185831/http://www.cmaj.ca/cgi/content/full/163/3/301 | archive-date = 14 June 2006 | df = dmy-all | doi = 10.1016/s1381-1169(00)00220-x }}</ref> These methods also allow the detection and identification of "[[viable but nonculturable]]" cells that are metabolically active but non-dividing.<ref>{{cite journal | vauthors = Oliver JD | title = The viable but nonculturable state in bacteria | journal = Journal of Microbiology | volume = 43 Spec No | pages = 93–100 | date = February 2005 | pmid = 15765062 | url = http://www.msk.or.kr/jsp/view_old_journalD.jsp?paperSeq=2134 | archive-url = https://web.archive.org/web/20070928032546/http://www.msk.or.kr/jsp/view_old_journalD.jsp?paperSeq=2134 | url-status = dead | archive-date = 28 September 2007 }}</ref> However, even using these improved methods, the total number of bacterial species is not known and cannot even be estimated with any certainty. Following present classification, there are a little less than 9,300 known species of prokaryotes, which includes bacteria and archaea;<ref>{{cite web|vauthors=Euzéby JP |date=8 December 2011 |url=http://www.bacterio.cict.fr/number.html |title=Number of published names |work=List of Prokaryotic names with Standing in Nomenclature |access-date=10 December 2011 |url-status=dead |archive-url=https://web.archive.org/web/20120119210136/http://www.bacterio.cict.fr/number.html |archive-date=19 January 2012 }}</ref> but attempts to estimate the true number of bacterial diversity have ranged from 10<sup>7</sup> to 10<sup>9</sup> total species—and even these diverse estimates may be off by many orders of magnitude.<ref>{{cite journal | vauthors = Curtis TP, Sloan WT, Scannell JW | title = Estimating prokaryotic diversity and its limits | journal = Proceedings of the National Academy of Sciences of the United States of America | volume = 99 | issue = 16 | pages = 10494–99 | date = August 2002 | pmid = 12097644 | pmc = 124953 | doi = 10.1073/pnas.142680199 | bibcode = 2002PNAS...9910494C | doi-access = free }}</ref><ref>{{cite journal | vauthors = Schloss PD, Handelsman J | title = Status of the microbial census | journal = Microbiology and Molecular Biology Reviews | volume = 68 | issue = 4 | pages = 686–91 | date = December 2004 | pmid = 15590780 | pmc = 539005 | doi = 10.1128/MMBR.68.4.686-691.2004 }}</ref>

== Phyla ==
{{Main|Bacterial phyla}}

The following phyla have been validly published according to the [[Bacteriological Code]]:<ref>{{cite journal | vauthors = Oren A, Garrity GM | title = Valid publication of the names of forty-two phyla of prokaryotes | journal = Int J Syst Evol Microbiol | year = 2021 | volume = 71 | issue = 10 | pages = 5056 | doi = 10.1099/ijsem.0.005056 | pmid = 34694987 | s2cid = 239887308 | doi-access = free }}</ref>
{{div col|colwidth=150px}}
* [[Acidobacteriota]]
* [[Actinomycetota]]
* [[Aquificota]]
* [[Armatimonadota]]
* [[Atribacterota]]
* [[Bacillota]]
* [[Bacteroidota]]
* [[Balneolota]]
* [[Bdellovibrionota]]
* [[Caldisericota]]
* [[Calditrichota]]
* [[Campylobacterota]]
* [[Chlamydiota]]
* [[Chlorobiota]]
* [[Chloroflexota]]
* [[Chrysiogenota]]
* [[Coprothermobacterota]]
* [[Deferribacterota]]
* [[Deinococcota]]
* [[Dictyoglomota]]
* [[Elusimicrobiota]]
* [[Fibrobacterota]]
* [[Fusobacteriota]]
* [[Gemmatimonadota]]
* [[Ignavibacteriota]]
* [[Lentisphaerota]]
* [[Mycoplasmatota]]
* [[Myxococcota]]
* [[Nitrospinota]]
* [[Nitrospirota]]
* [[Planctomycetota]]
* [[Pseudomonadota]]
* [[Rhodothermota]]
* [[Spirochaetota]]
* [[Synergistota]]
* [[Thermodesulfobacteriota]]
* [[Thermomicrobiota]]
* [[Thermotogota]]
* [[Verrucomicrobiota]]
{{div col end}}

== Interactions with other organisms ==
{{further|Microbes in human culture}}
[[File:Bacterial infections and involved species.png|thumb|upright=1.35|alt=chart showing bacterial infections upon various parts of human body|Overview of bacterial infections and main species involved.<ref name=Microbiology33>{{cite book | vauthors = Fisher B, Harvey RP, Champe PC |title=Lippincott's Illustrated Reviews: Microbiology (Lippincott's Illustrated Reviews Series) |publisher=Lippincott Williams & Wilkins |location=Hagerstwon, MD |year= 2007|chapter=Chapter 33| pages= 367–92 |isbn=978-0-7817-8215-9 }}</ref>]]

Despite their apparent simplicity, bacteria can form complex associations with other organisms. These [[symbiosis|symbiotic]] associations can be divided into [[parasitism]], [[Mutualism (biology)|mutualism]] and [[commensalism]].<ref name="pmid26568407">{{cite journal |vauthors=Mushegian AA, Ebert D |title=Rethinking "mutualism" in diverse host-symbiont communities |journal=BioEssays |volume=38 |issue=1 |pages=100–8 |date=January 2016 |pmid=26568407 |doi=10.1002/bies.201500074|s2cid=31661712 }}</ref>

=== Commensals ===

The word "[[commensalism]]" is derived from the word "commensal", meaning "eating at the same table"<ref>{{OEtymD|commensalism}}</ref> and all plants and animals are colonised by commensal bacteria. In humans and other animals, millions of them live on the skin, the airways, the gut and other orifices.<ref>{{cite journal | vauthors = Sears CL | title = A dynamic partnership: celebrating our gut flora | journal = Anaerobe | volume = 11 | issue = 5 | pages = 247–51 | date = October 2005 | pmid = 16701579 | doi = 10.1016/j.anaerobe.2005.05.001 }}</ref><ref name="pmid31214175">{{cite journal |vauthors=Khan R, Petersen FC, Shekhar S |title=Commensal Bacteria: An Emerging Player in Defense Against Respiratory Pathogens |journal=Frontiers in Immunology |volume=10 |issue= |pages=1203 |date=2019 |pmid=31214175 |pmc=6554327 |doi=10.3389/fimmu.2019.01203 |doi-access=free }}</ref>
Referred to as "normal flora",<ref name="pmid3074102">{{cite journal |vauthors=Roscoe DL, Chow AW |title=Normal flora and mucosal immunity of the head and neck |journal=Infectious Disease Clinics of North America |volume=2 |issue=1 |pages=1–19 |date=March 1988 |pmid=3074102 |doi= 10.1016/S0891-5520(20)30163-X |url= }}</ref> or "commensals",<ref name="pmid34329585">{{cite journal |vauthors=Shiao SL, Kershaw KM, Limon JJ, You S, Yoon J, Ko EY, Guarnerio J, Potdar AA, McGovern DP, Bose S, Dar TB, Noe P, Lee J, Kubota Y, Maymi VI, Davis MJ, Henson RM, Choi RY, Yang W, Tang J, Gargus M, Prince AD, Zumsteg ZS, Underhill DM |title=Commensal bacteria and fungi differentially regulate tumor responses to radiation therapy |journal=Cancer Cell |volume=39 |issue=9 |pages=1202–1213.e6 |date=July 2021 |pmid=34329585 |doi=10.1016/j.ccell.2021.07.002 |pmc=8830498 |url=}}</ref> these bacteria usually cause no harm but may occasionally invade other sites of the body and cause infection. ''[[Escherichia coli]]'' is a commensal in the human gut but can cause urinary tract infections.<ref name="pmid33575225">{{cite journal |vauthors=Lyapichev KA, Ivashkevich Y, Chernov Y, Chinenov D, Shpot E, Bessonov AA, Dabaja BS, Konoplev S |title=MALT Lymphoma of the Urinary Bladder Shows a Dramatic Female Predominance, Uneven Geographic Distribution, and Possible Infectious Etiology |journal=Research and Reports in Urology |volume=13 |issue= |pages=49–62 |date=2021 |pmid=33575225 |pmc=7873029 |doi=10.2147/RRU.S283366 |doi-access=free }}</ref> Similarly, streptococci, which are part of the normal flora of the human mouth, can cause [[subacute bacterial endocarditis|heart disease]].<ref name="pmid33852085">{{cite journal |vauthors=Eleyan L, Khan AA, Musollari G, Chandiramani AS, Shaikh S, Salha A, Tarmahomed A, Harky A |title=Infective endocarditis in paediatric population |journal=European Journal of Pediatrics |volume=180 |issue=10 |pages=3089–3100 |date=April 2021 |pmid=33852085 |doi=10.1007/s00431-021-04062-7 |s2cid=233225250 }}</ref>

=== Predators ===
Some species of bacteria kill and then consume other microorganisms; these species are called ''predatory bacteria''.<ref>{{cite journal | vauthors = Martin MO | title = Predatory prokaryotes: an emerging research opportunity | journal = Journal of Molecular Microbiology and Biotechnology | volume = 4 | issue = 5 | pages = 467–77 | date = September 2002 | pmid = 12432957 }}</ref> These include organisms such as ''[[Myxococcus xanthus]]'', which forms [[Swarm behaviour#Bacteria|swarms of cells]] that kill and digest any bacteria they encounter.<ref>{{cite journal | vauthors = Velicer GJ, Stredwick KL | title = Experimental social evolution with Myxococcus xanthus | journal = Antonie van Leeuwenhoek | volume = 81 | issue = 1–4 | pages = 155–64 | date = August 2002 | pmid = 12448714 | doi = 10.1023/A:1020546130033 | s2cid = 20018104 }}</ref> Other bacterial predators either attach to their prey in order to digest them and absorb nutrients or invade another cell and multiply inside the cytosol.<ref name="pmid34010833">{{cite journal |vauthors=Bauer A, Forchhammer K |title=Bacterial Predation on Cyanobacteria |journal=Microbial Physiology |volume=31 |issue=2 |pages=99–108 |date=May 2021 |pmid=34010833 |doi=10.1159/000516427 |issn=2673-1665 |doi-access=free }}</ref> These predatory bacteria are thought to have evolved from [[Detritivore|saprophages]] that consumed dead microorganisms, through adaptations that allowed them to entrap and kill other organisms.<ref name="pmid19174136">{{cite journal | vauthors = Velicer GJ, Mendes-Soares H | title = Bacterial predators | journal = Current Biology | volume = 19 | issue = 2 | pages = R55–56 | date = January 2009 | pmid = 19174136 | doi = 10.1016/j.cub.2008.10.043 | s2cid = 5432036 | doi-access = free | bibcode = 2009CBio...19..R55V }}</ref>

=== Mutualists ===
Certain bacteria form close spatial associations that are essential for their survival. One such mutualistic association, called interspecies hydrogen transfer, occurs between clusters of [[anaerobic bacteria]] that consume [[organic acid]]s, such as [[butyric acid]] or [[propionic acid]], and produce [[hydrogen]], and [[methanogen]]ic archaea that consume hydrogen.<ref>{{cite journal | vauthors = Stams AJ, de Bok FA, Plugge CM, van Eekert MH, Dolfing J, Schraa G | title = Exocellular electron transfer in anaerobic microbial communities | journal = Environmental Microbiology | volume = 8 | issue = 3 | pages = 371–82 | date = March 2006 | pmid = 16478444 | doi = 10.1111/j.1462-2920.2006.00989.x | bibcode = 2006EnvMi...8..371S }}</ref> The bacteria in this association are unable to consume the organic acids as this reaction produces hydrogen that accumulates in their surroundings. Only the intimate association with the hydrogen-consuming archaea keeps the hydrogen concentration low enough to allow the bacteria to grow.<ref name="pmid29411546">{{cite journal |vauthors=Zinser ER |title=Cross-protection from hydrogen peroxide by helper microbes: the impacts on the cyanobacterium Prochlorococcus and other beneficiaries in marine communities |journal=Environmental Microbiology Reports |volume=10 |issue=4 |pages=399–411 |date=August 2018 |pmid=29411546 |doi=10.1111/1758-2229.12625 |doi-access=free |bibcode=2018EnvMR..10..399Z }}</ref>

In soil, microorganisms that reside in the [[Rhizosphere (ecology)|rhizosphere]] (a zone that includes the [[root]] surface and the soil that adheres to the root after gentle shaking) carry out [[nitrogen fixation]], converting nitrogen gas to nitrogenous compounds.<ref>{{cite journal | vauthors = Barea JM, Pozo MJ, Azcón R, Azcón-Aguilar C | title = Microbial co-operation in the rhizosphere | journal = Journal of Experimental Botany | volume = 56 | issue = 417 | pages = 1761–78 | date = July 2005 | pmid = 15911555 | doi = 10.1093/jxb/eri197 | doi-access = free }}</ref> This serves to provide an easily absorbable form of nitrogen for many plants, which cannot fix nitrogen themselves. Many other bacteria are found as [[symbiont]]s [[Bacteria in the human body|in humans]] and other organisms. For example, the presence of over 1,000 bacterial species in the normal human [[gut flora]] of the [[intestine]]s can contribute to gut immunity, synthesise [[vitamin]]s, such as [[folic acid]], [[vitamin K]] and [[biotin]], convert [[Milk protein|sugars]] to [[lactic acid]] (see ''[[Lactobacillus]]''), as well as fermenting complex undigestible [[carbohydrate]]s.<ref>{{cite journal | vauthors = O'Hara AM, Shanahan F | title = The gut flora as a forgotten organ | journal = EMBO Reports | volume = 7 | issue = 7 | pages = 688–93 | date = July 2006 | pmid = 16819463 | pmc = 1500832 | doi = 10.1038/sj.embor.7400731 }}</ref><ref>{{cite journal | vauthors = Zoetendal EG, Vaughan EE, de Vos WM | title = A microbial world within us | journal = Molecular Microbiology | volume = 59 | issue = 6 | pages = 1639–50 | date = March 2006 | pmid = 16553872 | doi = 10.1111/j.1365-2958.2006.05056.x | s2cid = 37602619 | doi-access = free }}</ref><ref>{{cite journal | vauthors = [[Sherwood Gorbach|Gorbach SL]] | title = Lactic acid bacteria and human health | journal = Annals of Medicine | volume = 22 | issue = 1 | pages = 37–41 | date = February 1990 | pmid = 2109988 | doi = 10.3109/07853899009147239 }}</ref> The presence of this gut flora also inhibits the growth of potentially pathogenic bacteria (usually through [[competitive exclusion]]) and these beneficial bacteria are consequently sold as [[probiotic]] [[dietary supplement]]s.<ref>{{cite journal | vauthors = Salminen SJ, Gueimonde M, Isolauri E | title = Probiotics that modify disease risk | journal = The Journal of Nutrition | volume = 135 | issue = 5 | pages = 1294–98 | date = May 2005 | pmid = 15867327 | doi = 10.1093/jn/135.5.1294 | doi-access = free }}</ref>

Nearly all [[Life|animal life]] is dependent on bacteria for survival as only bacteria and some [[archaea]] possess the genes and enzymes necessary to synthesize [[Vitamin B12|vitamin B<sub>12</sub>]], also known as [[cobalamin]], and provide it through the food chain. Vitamin B<sub>12</sub> is a water-soluble [[vitamin]] that is involved in the [[metabolism]] of every cell of the human body. It is a [[cofactor (biochemistry)|cofactor]] in [[DNA replication|DNA synthesis]] and in both [[fatty acid metabolism|fatty acid]] and [[amino acid metabolism]]. It is particularly important in the normal functioning of the [[nervous system]] via its role in the [[myelinogenesis|synthesis of myelin]].<ref name="pmid29216732">{{cite journal |vauthors=Watanabe F, Bito T |title=Vitamin B12 sources and microbial interaction |journal=Experimental Biology and Medicine |volume=243 |issue=2 |pages=148–58 |date=January 2018 |pmid=29216732 |pmc=5788147 |doi=10.1177/1535370217746612}}</ref>

=== Pathogens ===
{{Main|Pathogenic bacteria}}
[[File:Neisseria gonorrhoeae and pus cells.jpg|thumb|right|''[[Neisseria gonorrhoeae]]'' and pus cells from a penile discharge ([[Gram stain]])]]

[[File:SalmonellaNIAID.jpg|thumb|alt=Color-enhanced scanning electron micrograph of red Salmonella typhimurium in yellow human cells|Colour-enhanced scanning electron micrograph showing ''[[Salmonella typhimurium]]'' (red) invading cultured human cells]]

The body is continually exposed to many species of bacteria, including beneficial commensals, which grow on the skin and [[mucous membrane]]s, and [[saprophyte]]s, which grow mainly in the soil and in [[decomposition|decaying]] matter. The blood and tissue fluids contain nutrients sufficient to sustain the growth of many bacteria. The body has defence mechanisms that enable it to resist microbial invasion of its tissues and give it a natural [[immune system|immunity]] or [[innate immunity|innate resistance]] against many [[microorganisms]].{{sfn|Pommerville|2014|pages=16–21}} Unlike some [[virus]]es, bacteria evolve relatively slowly so many bacterial diseases also occur in other animals.{{sfn|Clark|2010|page=215}}

If bacteria form a parasitic association with other organisms, they are classed as pathogens.{{sfn|Wheelis|2008|page=44}} Pathogenic bacteria are a major cause of human death and disease and cause infections such as [[tetanus]] (caused by ''[[Clostridium tetani]]''), [[typhoid fever]], [[diphtheria]], [[syphilis]], [[cholera]], [[foodborne illness]], [[leprosy]] (caused by ''[[Mycobacterium leprae]]'') and [[tuberculosis]] (caused by ''[[Mycobacterium tuberculosis]]'').{{sfn|Clark|2010|pages= 30, 195, 233,236}} A pathogenic cause for a known medical disease may only be discovered many years later, as was the case with ''[[Helicobacter pylori]]'' and [[Timeline of peptic ulcer disease and Helicobacter pylori|peptic ulcer disease]].<ref name="pmid34244666">{{cite journal |vauthors=Miller AK, Williams SM |title=Helicobacter pylori infection causes both protective and deleterious effects in human health and disease |journal=Genes and Immunity |volume=22 |issue=4 |pages=218–226 |date=July 2021 |pmid=34244666 |doi=10.1038/s41435-021-00146-4 |pmc=8390445 }}</ref> Bacterial diseases are also important in [[agriculture]], and bacteria cause [[leaf spot]], [[fire blight]] and [[Wilting|wilts]] in plants, as well as [[Johne's disease]], [[Mastitis in dairy cattle|mastitis]], [[salmonellosis|salmonella]] and [[anthrax]] in farm animals.<ref name="pmid27660260">{{cite journal |vauthors=Schwarz S, Enne VI, van Duijkeren E |title=40 years of veterinary papers in JAC – what have we learnt? |journal=The Journal of Antimicrobial Chemotherapy |volume=71 |issue=10 |pages=2681–90 |date=October 2016 |pmid=27660260 |doi=10.1093/jac/dkw363 |doi-access=free }}</ref>
[[File:Normal and BV flora.jpg|thumb|right|alt=Gram-stained micrograph of bacteria from the vagina|In [[bacterial vaginosis]], beneficial bacteria in the vagina (top) are displaced by pathogens (bottom). Gram stain.]]
Each species of pathogen has a characteristic spectrum of interactions with its human [[host (biology)|hosts]]. Some organisms, such as ''[[Staphylococcus]]'' or ''[[Streptococcus]]'', can cause skin infections, [[pneumonia]], [[meningitis]] and [[sepsis]], a systemic [[Inflammation|inflammatory response]] producing [[shock (circulatory)|shock]], massive [[vasodilator|vasodilation]] and death.<ref>{{cite journal | vauthors = Fish DN | title = Optimal antimicrobial therapy for sepsis | journal = American Journal of Health-System Pharmacy | volume = 59 | issue = Suppl 1 | pages = S13–19 | date = February 2002 | pmid = 11885408 | doi = 10.1093/ajhp/59.suppl_1.S13 | doi-access = free }}</ref> Yet these organisms are also part of the normal human flora and usually exist on the skin or in the nose without causing any disease at all. Other organisms invariably cause disease in humans, such as ''[[Rickettsia]]'', which are [[obligate intracellular parasite]]s able to grow and reproduce only within the cells of other organisms. One species of ''Rickettsia'' causes [[typhus]], while another causes [[Rocky Mountain spotted fever]]. ''[[Chlamydia (bacterium)|Chlamydia]]'', another phylum of obligate intracellular parasites, contains species that can cause pneumonia or [[urinary tract infection]] and may be involved in [[coronary heart disease]].<ref>{{cite journal | vauthors = Belland RJ, Ouellette SP, Gieffers J, Byrne GI | title = Chlamydia pneumoniae and atherosclerosis | journal = Cellular Microbiology | volume = 6 | issue = 2 | pages = 117–27 | date = February 2004 | pmid = 14706098 | doi = 10.1046/j.1462-5822.2003.00352.x | s2cid = 45218449 | doi-access = free }}</ref> Some species, such as ''[[Pseudomonas aeruginosa]]'', ''[[Burkholderia cenocepacia]]'', and ''[[Mycobacterium avium complex|Mycobacterium avium]]'', are [[opportunistic infection|opportunistic pathogens]] and cause disease mainly in people who are [[immunosuppression|immunosuppressed]] or have [[cystic fibrosis]].<ref>{{cite journal | vauthors = Heise ER | title = Diseases associated with immunosuppression | journal = Environmental Health Perspectives | volume = 43 | pages = 9–19 | date = February 1982 | pmid = 7037390 | pmc = 1568899 | doi = 10.2307/3429162 | jstor = 3429162 }}</ref><ref name="Saiman">{{cite journal | vauthors = Saiman L | title = Microbiology of early CF lung disease | journal = Paediatric Respiratory Reviews | volume = 5 | issue = Suppl A | pages = S367–69 | year = 2004 | pmid = 14980298 | doi = 10.1016/S1526-0542(04)90065-6 }}</ref> Some bacteria produce [[Microbial toxin|toxins]], which cause diseases.{{sfn|Pommerville|2014|page=118}} These are [[endotoxin]]s, which come from broken bacterial cells, and [[exotoxin]]s, which are produced by bacteria and released into the environment.{{sfn|Pommerville|2014|pages=646–47}} The bacterium ''[[Clostridium botulinum]]'' for example, produces a powerful exotoxin that cause respiratory paralysis, and ''[[Salmonellae]]'' produce an endotoxin that causes gastroenteritis.{{sfn|Pommerville|2014|pages=646–47}} Some exotoxins can be converted to [[toxoids]], which are used as vaccines to prevent the disease.{{sfn|Krasner|2014|pages=165, 369}}

Bacterial infections may be treated with [[antibiotic]]s, which are classified as [[Bactericide|bacteriocidal]] if they kill bacteria or [[bacteriostatic]] if they just prevent bacterial growth. There are many types of antibiotics, and each class [[enzyme inhibitor|inhibits]] a process that is different in the pathogen from that found in the host. An example of how antibiotics produce selective toxicity are [[chloramphenicol]] and [[puromycin]], which inhibit the bacterial [[ribosome]], but not the structurally different eukaryotic ribosome.<ref>{{cite journal | vauthors = Yonath A, Bashan A | title = Ribosomal crystallography: initiation, peptide bond formation, and amino acid polymerization are hampered by antibiotics | journal = [[Annual Review of Microbiology]] | volume = 58 | pages = 233–51 | year = 2004 | pmid = 15487937 | doi = 10.1146/annurev.micro.58.030603.123822 | doi-access = free }}</ref> Antibiotics are used both in treating human disease and in [[intensive farming]] to promote animal growth, where they may be contributing to the rapid development of [[antibiotic resistance]] in bacterial populations.<ref>{{cite journal | vauthors = Khachatourians GG | title = Agricultural use of antibiotics and the evolution and transfer of antibiotic-resistant bacteria | journal = CMAJ | volume = 159 | issue = 9 | pages = 1129–36 | date = November 1998 | pmid = 9835883 | pmc = 1229782 }}</ref> Infections can be prevented by [[antiseptic]] measures such as sterilising the skin prior to piercing it with the needle of a syringe, and by proper care of indwelling catheters. Surgical and dental instruments are also [[sterilization (microbiology)|sterilised]] to prevent contamination by bacteria. [[Disinfectant]]s such as [[bleach]] are used to kill bacteria or other pathogens on surfaces to prevent contamination and further reduce the risk of infection.<ref name="pmid28954657">{{cite journal |vauthors=Kuo J |title=Disinfection Processes |journal=Water Environment Research |volume=89 |issue=10 |pages=1206–44 |date=October 2017 |pmid=28954657 |doi=10.2175/106143017X15023776270278 |doi-access=free |bibcode=2017WaEnR..89.1206K }}</ref>

== Significance in technology and industry ==
Bacteria, often [[lactic acid bacteria]], such as ''[[Lactobacillus]]'' species and ''[[Lactococcus]]'' species, in combination with [[yeast]]s and [[Mold (fungus)|moulds]], have been used for thousands of years in the preparation of [[fermentation (food)|fermented]] foods, such as [[cheese]], [[Pickling|pickles]], [[soy sauce]], [[sauerkraut]], [[vinegar]], [[wine]], and [[yogurt]].<ref>{{cite journal | vauthors = Johnson ME, Lucey JA | title = Major technological advances and trends in cheese | journal = Journal of Dairy Science | volume = 89 | issue = 4 | pages = 1174–78 | date = April 2006 | pmid = 16537950 | doi = 10.3168/jds.S0022-0302(06)72186-5 | doi-access = free }}</ref>{{sfn|Krasner|2014|pp=25–26}}

The ability of bacteria to degrade a variety of organic compounds is remarkable and has been used in waste processing and [[bioremediation]]. Bacteria capable of digesting the [[hydrocarbon]]s in [[petroleum]] are often used to clean up [[oil spill]]s.<ref>{{cite journal | vauthors = Cohen Y | title = Bioremediation of oil by marine microbial mats | journal = International Microbiology | volume = 5 | issue = 4 | pages = 189–93 | date = December 2002 | pmid = 12497184 | doi = 10.1007/s10123-002-0089-5 | s2cid = 26039323 | url = http://revistes.iec.cat/index.php/IM/article/view/9381 }}</ref> Fertiliser was added to some of the beaches in [[Prince William Sound]] in an attempt to promote the growth of these naturally occurring bacteria after the 1989 [[Exxon Valdez oil spill|''Exxon Valdez'' oil spill]]. These efforts were effective on beaches that were not too thickly covered in oil. Bacteria are also used for the [[bioremediation]] of industrial [[toxic waste]]s.<ref>{{cite journal | vauthors = Neves LC, Miyamura TT, Moraes DA, Penna TC, Converti A | title = Biofiltration methods for the removal of phenolic residues | journal = Applied Biochemistry and Biotechnology | volume = 129–132 | issue = 1–3 | pages = 130–52 | year = 2006 | pmid = 16915636 | doi = 10.1385/ABAB:129:1:130 | s2cid = 189905816 }}</ref> In the [[chemical industry]], bacteria are most important in the production of [[enantiomer]]ically pure chemicals for use as [[pharmaceutical company|pharmaceuticals]] or [[agrichemical]]s.<ref>{{cite journal | vauthors = Liese A, Filho MV | title = Production of fine chemicals using biocatalysis | journal = Current Opinion in Biotechnology | volume = 10 | issue = 6 | pages = 595–603 | date = December 1999 | pmid = 10600695 | doi = 10.1016/S0958-1669(99)00040-3 }}</ref>

Bacteria can also be used in place of [[pesticide]]s in [[biological pest control]]. This commonly involves ''[[Bacillus thuringiensis]]'' (also called BT), a Gram-positive, soil-dwelling bacterium. Subspecies of this bacteria are used as [[Lepidoptera]]n-specific [[insecticide]]s under trade names such as Dipel and Thuricide.<ref>{{cite journal | vauthors = Aronson AI, Shai Y | title = Why Bacillus thuringiensis insecticidal toxins are so effective: unique features of their mode of action | journal = FEMS Microbiology Letters | volume = 195 | issue = 1 | pages = 1–8 | date = February 2001 | pmid = 11166987 | doi = 10.1111/j.1574-6968.2001.tb10489.x | doi-access = free }}</ref> Because of their specificity, these pesticides are regarded as [[environmentally friendly]], with little or no effect on humans, [[wildlife]], [[pollinator]]s, and most other [[beneficial insect]]s.<ref>{{cite journal | vauthors = Bozsik A | title = Susceptibility of adult Coccinella septempunctata (Coleoptera: Coccinellidae) to insecticides with different modes of action | journal = Pest Management Science | volume = 62 | issue = 7 | pages = 651–54 | date = July 2006 | pmid = 16649191 | doi = 10.1002/ps.1221 }}</ref><ref>{{cite journal | vauthors = Chattopadhyay A, Bhatnagar NB, Bhatnagar R | title = Bacterial insecticidal toxins | journal = Critical Reviews in Microbiology | volume = 30 | issue = 1 | pages = 33–54 | year = 2004 | pmid = 15116762 | doi = 10.1080/10408410490270712 | s2cid = 1580984 }}</ref>

Because of their ability to quickly grow and the relative ease with which they can be manipulated, bacteria are the workhorses for the fields of [[molecular biology]], [[genetics]], and [[biochemistry]]. By making mutations in bacterial DNA and examining the resulting phenotypes, scientists can determine the function of genes, [[enzyme]]s, and [[metabolic pathway]]s in bacteria, then apply this knowledge to more complex organisms.<ref>{{cite journal | vauthors = Serres MH, Gopal S, Nahum LA, Liang P, Gaasterland T, Riley M | title = A functional update of the Escherichia coli K-12 genome | journal = Genome Biology | volume = 2 | issue = 9 | page = RESEARCH0035 | year = 2001 | pmid = 11574054 | pmc = 56896 | doi = 10.1186/gb-2001-2-9-research0035 | doi-access = free }}</ref> This aim of understanding the biochemistry of a cell reaches its most complex expression in the synthesis of huge amounts of [[enzyme kinetics|enzyme kinetic]] and [[gene expression]] data into [[mathematical model]]s of entire organisms. This is achievable in some well-studied bacteria, with models of ''Escherichia coli'' metabolism now being produced and tested.<ref name="pmid14985762">{{cite journal | vauthors = Almaas E, Kovács B, Vicsek T, Oltvai ZN, Barabási AL | title = Global organization of metabolic fluxes in the bacterium Escherichia coli | journal = Nature | volume = 427 | issue = 6977 | pages = 839–43 | date = February 2004 | pmid = 14985762 | doi = 10.1038/nature02289 | arxiv = q-bio/0403001 | bibcode = 2004Natur.427..839A | s2cid = 715721 }}</ref><ref>{{cite journal | vauthors = Reed JL, Vo TD, Schilling CH, Palsson BO | title = An expanded genome-scale model of Escherichia coli K-12 (iJR904 GSM/GPR) | journal = Genome Biology | volume = 4 | issue = 9 | page = R54 | year = 2003 | pmid = 12952533 | pmc = 193654 | doi = 10.1186/gb-2003-4-9-r54 | doi-access = free }}</ref> This understanding of bacterial metabolism and genetics allows the use of biotechnology to [[bioengineering|bioengineer]] bacteria for the production of therapeutic proteins, such as [[insulin]], [[growth factor]]s, or [[antibody|antibodies]].<ref>{{cite journal | vauthors = Walsh G | title = Therapeutic insulins and their large-scale manufacture | journal = Applied Microbiology and Biotechnology | volume = 67 | issue = 2 | pages = 151–59 | date = April 2005 | pmid = 15580495 | doi = 10.1007/s00253-004-1809-x | s2cid = 5986035 }}</ref><ref>{{cite journal | vauthors = Graumann K, Premstaller A | title = Manufacturing of recombinant therapeutic proteins in microbial systems | journal = Biotechnology Journal | volume = 1 | issue = 2 | pages = 164–86 | date = February 2006 | pmid = 16892246 | doi = 10.1002/biot.200500051 | s2cid = 24702839 }}</ref>

Because of their importance for research in general, samples of bacterial strains are isolated and preserved in [[Biological Resource Centers]]. This ensures the availability of the strain to scientists worldwide.<ref name="pmid14766539">{{cite journal |vauthors=Rabsch W, Helm RA, Eisenstark A |title=Diversity of phage types among archived cultures of the Demerec collection of ''Salmonella enterica'' serovar Typhimurium strains |journal=Applied and Environmental Microbiology |volume=70 |issue=2 |pages=664–69 |date=February 2004 |pmid=14766539 |pmc=348941 |doi=10.1128/aem.70.2.664-669.2004 |bibcode=2004ApEnM..70..664R }}</ref>

== History of bacteriology ==
{{for multi|the history of microbiology|Microbiology|the history of bacterial classification|Bacterial taxonomy|the natural history of Bacteria|Last universal common ancestor}}
[[File:Anthonie van Leeuwenhoek (1632-1723). Natuurkundige te Delft Rijksmuseum SK-A-957.jpeg|thumb|right|alt=painting of Antonie van Leeuwenhoek, in robe and frilled shirt, with ink pen and paper|[[Antonie van Leeuwenhoek]], the first [[microbiologist]] and the first person to observe bacteria using a [[microscope]].]]

Bacteria were first observed by the Dutch microscopist [[Antonie van Leeuwenhoek]] in 1676, using a single-lens [[microscope]] of his own design. He then published his observations in a series of letters to the [[Royal Society of London]].{{sfn | Wheelis | 2008 | p=}} Bacteria were Leeuwenhoek's most remarkable microscopic discovery. Their size was just at the limit of what his simple lenses could resolve, and, in one of the most striking hiatuses in the history of science, no one else would see them again for over a century.<ref>{{cite book | vauthors = Asimov I | author-link = Isaac Asimov | year = 1982 | title = Asimov's Biographical Encyclopedia of Science and Technology | edition = 2nd | location = Garden City, NY | publisher = Doubleday and Company | page = 143 | title-link = Asimov's Biographical Encyclopedia of Science and Technology }}</ref> His observations also included protozoans which he called [[animalcule]]s, and his findings were looked at again in the light of the more recent findings of [[cell theory]].{{sfn|Pommerville|2014|p=7}}

[[Christian Gottfried Ehrenberg]] introduced the word "bacterium" in 1828.<ref>{{cite book | vauthors = Ehrenberg CG | title = Symbolae Physioe. Animalia evertebrata. | publisher = Decas prima | location = Berlin | date = 1828 }}</ref> In fact, his ''[[Bacterium (genus)|Bacterium]]'' was a genus that contained non-spore-forming rod-shaped bacteria,<ref name=status>{{cite journal | vauthors = Breed RS, Conn HJ | title = The Status of the Generic Term Bacterium Ehrenberg 1828 | journal = Journal of Bacteriology | volume = 31 | issue = 5 | pages = 517–18 | date = May 1936 | pmid = 16559906 | pmc = 543738 | doi = 10.1128/jb.31.5.517-518.1936 }}</ref> as opposed to ''Bacillus'', a genus of spore-forming rod-shaped bacteria defined by Ehrenberg in 1835.<ref>{{cite book | vauthors = Ehrenberg CG | title = Dritter Beitrag zur Erkenntniss grosser Organisation in der Richtung des kleinsten Raumes. | trans-title = Third contribution to the knowledge of great organization in the direction of the smallest space | language = de | publisher = Physikalische Abhandlungen der Koeniglichen Akademie der Wissenschaften | location = Berlin | year = 1835 | pages = 143–336 }}</ref>

[[Louis Pasteur]] demonstrated in 1859 that the growth of microorganisms causes the [[fermentation (food)|fermentation]] process and that this growth is not due to [[spontaneous generation]] ([[yeast]]s and [[Mold (fungus)|molds]], commonly associated with fermentation, are not bacteria, but rather [[fungus|fungi]]). Along with his contemporary [[Robert Koch]], Pasteur was an early advocate of the [[germ theory of disease]].<ref>{{cite web|url = http://biotech.law.lsu.edu/cphl/history/articles/pasteur.htm#paperII|title = Pasteur's Papers on the Germ Theory|publisher = LSU Law Center's Medical and Public Health Law Site, Historic Public Health Articles|access-date = 23 November 2006| archive-url= https://web.archive.org/web/20061218123426/http://biotech.law.lsu.edu/cphl/history/articles/pasteur.htm| archive-date= 18 December 2006 | url-status= live}}</ref> Before them, [[Ignaz Semmelweis]] and [[Joseph Lister]] had realised the importance of sanitized hands in medical work. Semmelweis, who in the 1840s formulated his rules for handwashing in the hospital, prior to the advent of germ theory, attributed disease to "decomposing animal organic matter". His ideas were rejected and his book on the topic condemned by the medical community. After Lister, however, doctors started sanitizing their hands in the 1870s.<ref>{{cite web |date=2020-03-06 |title='Wash your hands' was once controversial medical advice |url=https://www.nationalgeographic.com/history/article/handwashing-once-controversial-medical-advice |url-status=dead |archive-url=https://web.archive.org/web/20230811022131/https://www.nationalgeographic.com/history/article/handwashing-once-controversial-medical-advice |archive-date=Aug 11, 2023 |access-date= |website=History |language=en}}</ref>

Robert Koch, a pioneer in medical microbiology, worked on [[cholera]], [[anthrax]] and [[tuberculosis]]. In his research into tuberculosis, Koch finally proved the germ theory, for which he received a [[Nobel Prize in Physiology or Medicine|Nobel Prize]] in 1905.<ref>{{cite web|url = http://nobelprize.org/nobel_prizes/medicine/laureates/1905/|title = The Nobel Prize in Physiology or Medicine 1905|publisher = Nobelprize.org|access-date = 22 November 2006| archive-url= https://web.archive.org/web/20061210184150/http://nobelprize.org/nobel_prizes/medicine/laureates/1905/| archive-date= 10 December 2006 | url-status= live}}</ref> In [[Koch's postulates]], he set out criteria to test if an organism is the cause of a [[disease]], and these postulates are still used today.<ref>{{cite journal | vauthors = O'Brien SJ, Goedert JJ | title = HIV causes AIDS: Koch's postulates fulfilled | journal = Current Opinion in Immunology | volume = 8 | issue = 5 | pages = 613–18 | date = October 1996 | pmid = 8902385 | doi = 10.1016/S0952-7915(96)80075-6 | url = https://zenodo.org/record/1260157 }}</ref>

[[Ferdinand Cohn]] is said to be a founder of [[bacteriology]], studying bacteria from 1870. Cohn was the first to classify bacteria based on their morphology.<ref name="Chung">{{cite web | vauthors = Chung KT | url = http://www.pnf.org/compendium/Ferdinand_Julius_Cohn.pdf | title = Ferdinand Julius Cohn (1828–1898): Pioneer of Bacteriology | publisher = Department of Microbiology and Molecular Cell Sciences, The University of Memphis | url-status = live | archive-url = https://web.archive.org/web/20110727180844/http://www.pnf.org/compendium/Ferdinand_Julius_Cohn.pdf | archive-date = 27 July 2011 | df = dmy-all }}</ref><ref name="microbeworld1">{{cite journal |author=Drews, Gerhart |year=1999 |url=http://www.microbeworld.org/images/stories/history_pdfs/f3.pdf |title=Ferdinand Cohn, a founder of modern microbiology |journal=ASM News |volume=65 |issue=8 |pages=547–52 |url-status=dead |archive-url=https://web.archive.org/web/20170713150622/http://www.microbeworld.org/images/stories/history_pdfs/f3.pdf |archive-date=13 July 2017 }}</ref>

Though it was known in the nineteenth century that bacteria are the cause of many diseases, no effective [[antiseptic|antibacterial]] treatments were available.<ref>{{cite journal | vauthors = Thurston AJ | title = Of blood, inflammation and gunshot wounds: the history of the control of sepsis | journal = The Australian and New Zealand Journal of Surgery | volume = 70 | issue = 12 | pages = 855–61 | date = December 2000 | pmid = 11167573 | doi = 10.1046/j.1440-1622.2000.01983.x | doi-access = free }}</ref> In 1910, [[Paul Ehrlich]] developed the first antibiotic, by changing dyes that selectively stained ''[[Treponema pallidum]]''—the [[spirochaete]] that causes [[syphilis]]—into compounds that selectively killed the pathogen.<ref>{{cite journal | vauthors = Schwartz RS | title = Paul Ehrlich's magic bullets | journal = The New England Journal of Medicine | volume = 350 | issue = 11 | pages = 1079–80 | date = March 2004 | pmid = 15014180 | doi = 10.1056/NEJMp048021 }}</ref> Ehrlich, who had been awarded a 1908 Nobel Prize for his work on [[immunology]], pioneered the use of stains to detect and identify bacteria, with his work being the basis of the [[Gram stain]] and the [[Ziehl–Neelsen stain]].<ref>{{cite web|url = http://nobelprize.org/nobel_prizes/medicine/laureates/1908/ehrlich-bio.html|title = Biography of Paul Ehrlich|publisher = Nobelprize.org|access-date = 26 November 2006| archive-url= https://web.archive.org/web/20061128093700/http://nobelprize.org/nobel_prizes/medicine/laureates/1908/ehrlich-bio.html| archive-date= 28 November 2006 | url-status= live}}</ref>

A major step forward in the study of bacteria came in 1977 when [[Carl Woese]] recognised that archaea have a separate line of evolutionary descent from bacteria.<ref name=Woese1977>{{cite journal | vauthors = Woese CR, Fox GE | title = Phylogenetic structure of the prokaryotic domain: the primary kingdoms | journal = Proceedings of the National Academy of Sciences of the United States of America | volume = 74 | issue = 11 | pages = 5088–90 | date = November 1977 | pmid = 270744 | pmc = 432104 | doi = 10.1073/pnas.74.11.5088 | bibcode = 1977PNAS...74.5088W | doi-access = free }}</ref> This new [[phylogenetic]] [[Taxonomy (biology)|taxonomy]] depended on the [[sequencing]] of [[16S ribosomal RNA]] and divided prokaryotes into two evolutionary domains, as part of the [[three-domain system]].{{sfn | Hall | 2008 | p=145}}

== See also ==
* [[Bacteriohopanepolyol]]
* [[Genetically modified bacteria]]
* [[Marine prokaryotes]]

== References ==
{{Reflist}}

== Bibliography ==
* {{cite book | vauthors = Clark D | title=Germs, Genes, & Civilization : how epidemics shaped who we are today | publisher=FT Press | publication-place=Upper Saddle River, N.J | year=2010 | isbn=978-0-13-701996-0 | oclc=473120711}}
* {{cite book | vauthors = Crawford D | title=Deadly Companions : how microbes shaped our history | publisher=Oxford University Press | publication-place=Oxford New York | year=2007 | isbn=978-0-19-956144-5 | oclc=183198723}}
* {{cite book | vauthors = Hall B | title=Strickberger's Evolution : the integration of genes, organisms and populations | publisher=Jones and Bartlett | publication-place=Sudbury, Mass | year=2008 | isbn=978-0-7637-0066-9 | oclc=85814089}}
* {{cite book | vauthors = Krasner R | title=The Microbial Challenge : a public health perspective | publisher=Jones & Bartlett Learning | publication-place=Burlington, Mass | year=2014 | isbn=978-1-4496-7375-8 | oclc=794228026}}
* {{cite book|vauthors=Pommerville JC|title=Fundamentals of Microbiology|edition = 10th|publisher=Jones and Bartlett|location=Boston |year=2014 |isbn=978-1-284-03968-9}}
* {{cite book | vauthors = Wheelis M | title=Principles of modern microbiology | publisher=Jones and Bartlett Publishers | publication-place=Sudbury, Mass | year=2008 | isbn=978-0-7637-1075-0 | oclc=67392796}}

== External links ==
* [http://www.textbookofbacteriology.net/ On-line text book on bacteriology (2015)]

{{Bacteria classification}}
{{Bacteria}}
{{Microorganisms}}
{{Life on Earth}}
{{Organisms et al.}}
{{Taxonbar|from=Q10876}}
{{Authority control}}

[[Category:Bacteria| ]]
[[Category:Bacteriology]]
[[Category:Domains (biology)]]
[[Category:Biology terminology]]