{{Short description|Biomolecule consisting of chains of amino acid residues}}
{{About|a class of molecules|protein as a nutrient|Protein (nutrient)|other uses|Protein (disambiguation)}}
{{Pp-semi-indef}}
{{Good article}}
{{cs1 config|name-list-style=vanc|display-authors=6}}
[[File:Myoglobin.png|thumb|A representation of the 3D structure of the protein [[myoglobin]] showing turquoise [[alpha helix|α-helices]]. This protein was the first to have its structure solved by [[X-ray crystallography]]. Toward the right-center among the coils, a [[prosthetic group]] called a [[heme group]] (shown in gray) with a bound oxygen molecule (red).]]

'''Proteins''' are large [[biomolecule]]s and [[macromolecule]]s that comprise one or more long chains of [[amino acid]] [[residue (biochemistry)|residues]]. Proteins perform a vast array of functions within organisms, including [[Enzyme catalysis|catalysing metabolic reactions]], [[DNA replication]], [[Cell signaling|responding to stimuli]], providing [[Cytoskeleton|structure to cells]] and [[Fibrous protein|organisms]], and [[Intracellular transport|transporting molecules]] from one location to another. Proteins differ from one another primarily in their sequence of amino acids, which is dictated by the [[Nucleic acid sequence|nucleotide sequence]] of their [[gene]]s, and which usually results in [[protein folding]] into a specific [[Protein structure|3D structure]] that determines its activity.

A linear chain of amino acid residues is called a [[polypeptide]]. A protein contains at least one long polypeptide. Short polypeptides, containing less than 20–30 residues, are rarely considered to be proteins and are commonly called [[peptide]]s. The individual amino acid residues are bonded together by [[peptide bond]]s and adjacent amino acid residues. The [[Protein primary structure|sequence]] of amino acid residues in a protein is defined by the [[DNA sequencing|sequence]] of a gene, which is encoded in the [[genetic code]]. In general, the genetic code specifies 20 standard amino acids; but in certain organisms the genetic code can include [[selenocysteine]] and—in certain [[archaea]]—[[pyrrolysine]]. Shortly after or even during synthesis, the residues in a protein are often chemically modified by [[post-translational modification]], which alters the physical and chemical properties, folding, stability, activity, and ultimately, the function of the proteins. Some proteins have non-peptide groups attached, which can be called [[prosthetic group]]s or [[Cofactor (biochemistry)|cofactors]]. Proteins can also work together to achieve a particular function, and they often associate to form stable [[protein complex]]es.

Once formed, proteins only exist for a certain period and are then [[Proteolysis#Protein degradation|degraded]] and recycled by the cell's machinery through the process of [[protein turnover]]. A protein's lifespan is measured in terms of its [[half-life]] and covers a wide range. They can exist for minutes or years with an average lifespan of 1–2 days in mammalian cells. Abnormal or misfolded proteins are degraded more rapidly either due to being targeted for destruction or due to being unstable.

Like other biological macromolecules such as [[polysaccharide]]s and [[nucleic acid]]s, proteins are essential parts of organisms and participate in virtually every process within [[cell (biology)|cells]]. Many proteins are [[enzyme]]s that [[catalysis|catalyse]] biochemical reactions and are vital to [[metabolism]]. Proteins also have structural or mechanical functions, such as [[actin]] and [[myosin]] in muscle and the proteins in the [[cytoskeleton]], which form a system of [[scaffolding]] that maintains cell shape. Other proteins are important in cell signaling, [[antibody|immune responses]], [[cell adhesion]], and the [[cell cycle]]. In animals, proteins are needed in the [[diet (nutrition)|diet]] to provide the [[essential amino acid]]s that cannot be [[amino acid synthesis|synthesized]]. [[Digestion]] breaks the proteins down for metabolic use.

==History and etymology==
{{further|History of molecular biology}}
Proteins were recognized as a distinct class of biological molecules in the eighteenth century by [[Antoine François, comte de Fourcroy|Antoine Fourcroy]] and others, distinguished by the molecules' ability to [[coagulate]] or [[flocculation|flocculate]] under treatments with heat or acid.<ref>{{cite book | vauthors = Osborne TB | author-link1 = Thomas Burr Osborne (chemist) | date = 1909 | title = The Vegetable Proteins | chapter-url = https://archive.org/details/vegetableprotein00osbouoft | chapter = History | pages = 1–6 }}</ref> Noted examples at the time included [[albumin]] from [[egg white]]s, blood [[serum albumin]], [[fibrin]], and wheat [[gluten]].

Proteins were first described by the Dutch chemist [[Gerardus Johannes Mulder]] and named by the Swedish chemist [[Jöns Jacob Berzelius]] in 1838.<ref name="Mulder1938">{{cite journal | vauthors = Mulder GJ | year = 1838 | url = https://archive.org/stream/bulletindesscien00leyd#page/104/mode/2up | title = Sur la composition de quelques substances animales | journal = Bulletin des Sciences Physiques et Naturelles en Néerlande | pages = 104 }}</ref><ref name="Hartley">{{cite journal | vauthors = Hartley H | title = Origin of the word 'protein' | journal = Nature | volume = 168 | issue = 4267 | pages = 244 | date = August 1951 | pmid = 14875059 | doi = 10.1038/168244a0 | s2cid = 4271525 | doi-access = free | bibcode = 1951Natur.168..244H }}</ref> Mulder carried out [[elemental analysis]] of common proteins and found that nearly all proteins had the same [[empirical formula]], C<sub>400</sub>H<sub>620</sub>N<sub>100</sub>O<sub>120</sub>P<sub>1</sub>S<sub>1</sub>.<ref name=Perrett2007/> He came to the erroneous conclusion that they might be composed of a single type of (very large) molecule. The term "protein" to describe these molecules was proposed by Mulder's associate Berzelius; protein is derived from the [[Greek language|Greek]] word {{lang|el|πρώτειος|italic=no}} ({{transliteration|el|proteios|italic=yes}}), meaning "primary",<ref>{{cite encyclopedia | encyclopedia = Oxford English Dictionary | title = Protein (n.) | date = July 2023 | doi = 10.1093/OED/5657543824 }}</ref> "in the lead", or "standing in front",<ref name=Reynolds2003/> + ''[[wikt:-in#Suffix|-in]]''. Mulder went on to identify the products of protein degradation such as the [[amino acid]] [[leucine]] for which he found a (nearly correct) molecular weight of 131 [[atomic mass unit|Da]].<ref name=Perrett2007/> Prior to "protein", other names were used, like "albumins" or "albuminous materials" (''Eiweisskörper'', in German).<ref name=Reynolds2003/>

Early nutritional scientists such as the German [[Carl von Voit]] believed that protein was the most important nutrient for maintaining the structure of the body, because it was generally believed that "flesh makes flesh."<ref name=Bischoff1860/> [[Karl Heinrich Ritthausen]] extended known protein forms with the identification of [[glutamic acid]]. At the [[Connecticut Agricultural Experiment Station]] a detailed review of the vegetable proteins was compiled by [[Thomas Burr Osborne (chemist)|Thomas Burr Osborne]]. Working with [[Lafayette Mendel]] and applying [[Liebig's law of the minimum]] in feeding [[laboratory rat]]s, the nutritionally [[essential amino acid]]s were established. The work was continued and communicated by [[William Cumming Rose]]. The understanding of proteins as [[polypeptide]]s came through the work of [[Franz Hofmeister]] and [[Hermann Emil Fischer]] in 1902.<ref>{{cite web|url=http://www.encyclopedia.com/science/dictionaries-thesauruses-pictures-and-press-releases/hofmeister-franz|title=Hofmeister, Franz|publisher=encyclopedia.com|access-date=4 April 2017|archive-url=https://web.archive.org/web/20170405073423/http://www.encyclopedia.com/science/dictionaries-thesauruses-pictures-and-press-releases/hofmeister-franz|archive-date=5 April 2017|url-status=live}}</ref><ref>{{cite web | vauthors = Koshland DE, Haurowitz F |url=https://www.britannica.com/science/protein/Conformation-of-proteins-in-interfaces#ref593795|title=Protein, section: Classification of protein|publisher=britannica.com|access-date=4 April 2017|archive-url=https://web.archive.org/web/20170404225132/https://www.britannica.com/science/protein/Conformation-of-proteins-in-interfaces#ref593795|archive-date=4 April 2017|url-status=live}}</ref> The central role of proteins as [[enzyme]]s in living organisms was not fully appreciated until 1926, when [[James B. Sumner]] showed that the enzyme [[urease]] was in fact a protein.<ref name=Sumner1926/>

The difficulty in purifying proteins in large quantities made them very difficult for early protein biochemists to study. Hence, early studies focused on proteins that could be purified in large quantities, e.g., those of blood, egg white, various toxins, and digestive/metabolic enzymes obtained from slaughterhouses. In the 1950s, the [[Armour and Company|Armour Hot Dog Co.]] purified 1&nbsp;kg of pure bovine pancreatic [[ribonuclease A]] and made it freely available to scientists; this gesture helped ribonuclease A become a major target for biochemical study for the following decades.<ref name=Perrett2007/>

[[Linus Pauling]] is credited with the successful prediction of regular protein [[secondary structure]]s based on [[hydrogen bonding]], an idea first put forth by [[William Astbury]] in 1933.<ref name=Pauling1951/> Later work by [[Walter Kauzmann]] on [[Denaturation (biochemistry)|denaturation]],<ref name=Kauzmann1956/><ref name=Kauzmann1959/> based partly on previous studies by [[Kaj Ulrik Linderstrøm-Lang|Kaj Linderstrøm-Lang]],<ref name=Kalman1955/> contributed an understanding of [[protein folding]] and structure mediated by [[hydrophobic core|hydrophobic interactions]].

The first protein to be [[protein sequencing|sequenced]] was [[insulin]], by [[Frederick Sanger]], in 1949. Sanger correctly determined the amino acid sequence of insulin, thus conclusively demonstrating that proteins consisted of linear polymers of amino acids rather than branched chains, [[colloid]]s, or [[cyclol]]s.<ref name=Sanger1949/> He won the Nobel Prize for this achievement in 1958.<ref name="Lecture 1958"/>

[[File:KendrewMyoglobin.jpg|thumb|upright=1.15|[[John Kendrew]] with model of myoglobin in progress]]

With the development of [[X-ray crystallography]], it became possible to sequence protein structures.<ref name="Stoddart">{{cite journal | vauthors = Stoddart C |title=Structural biology: How proteins got their close-up |journal=Knowable Magazine |date=1 March 2022 |doi=10.1146/knowable-022822-1 |doi-access=free }}</ref> The first [[protein structure]]s to be solved were [[hemoglobin]] by [[Max Perutz]] and [[myoglobin]] by [[John Kendrew]], in 1958.<ref name=Muirhead1963/><ref name=Kendrew1958/> The use of computers and increasing computing power also supported the sequencing of complex proteins. In 1999, [[Roger Kornberg]] succeeded in sequencing the highly complex structure of [[RNA polymerase]] using high intensity X-rays from [[synchrotrons]].<ref name="Stoddart"/>

Since then, [[cryo-electron microscopy]] (cryo-EM) of large [[Macromolecular Assembly|macromolecular assemblies]]<ref name=Zhou2008/> has been developed. Cryo-EM uses protein samples that are frozen rather than crystals, and [[electron microscopy|beams of electrons]] rather than x-rays. It causes less damage to the sample, allowing scientists to obtain more information and analyze larger structures.<ref name="Stoddart"/> Computational [[protein structure prediction]] of small protein [[structural domain|domains]]<ref name=Keskin2008/> has also helped researchers to approach atomic-level resolution of protein structures.
{{As of|April 2024}}, the [[Protein Data Bank]] contains 181,018 X-ray, 19,809 EM {{Definition needed|date=April 2024}}and 12,697 NMR{{Definition needed|date=April 2024}} protein structures.<ref>{{cite web |url=https://www.rcsb.org/stats/summary |title=Summary Statistics |website=RCSB PDB |access-date=2024-04-20}}</ref>

== Number of proteins encoded in genomes ==
The number of proteins encoded in a [[genome]] roughly corresponds to the number of [[gene]]s (although there may be a significant number of genes that encode [[RNA]] of protein, e.g. [[ribosomal RNA]]s). [[Virus]]es typically encode a few to a few hundred proteins, [[archaea]] and [[bacteria]] a few hundred to a few thousand, while [[eukaryote]]s typically encode a few thousand up to tens of thousands of proteins (see [[Genome#Genome size|genome size]] for a list of examples).

== Classification ==
{{Main|Protein family|Gene Ontology|Enzyme Commission number}}
Proteins are primarily classified by sequence and structure, although other classifications are commonly used. Especially for enzymes the EC number system provides a functional classification scheme. Similarly, the [[Gene Ontology|gene ontology]] classifies both genes and proteins by their biological and biochemical function, but also by their intracellular location.

Sequence similarity is used to classify proteins both in terms of evolutionary and functional similarity. This may use either whole proteins or [[protein domain]]s, especially in [[Protein domain#Multidomain proteins|multi-domain proteins]]. Protein domains allow protein classification by a combination of sequence, structure and function, and thy can be combined in many different ways. In an early study of 170,000 proteins, about two-thirds were assigned at least one domain, with larger proteins containing more domains (e.g. proteins larger than 600 [[amino acid]]s having an average of more than 5 domains).<ref>{{cite journal | vauthors = Ekman D, Björklund AK, Frey-Skött J, Elofsson A | title = Multi-domain proteins in the three kingdoms of life: orphan domains and other unassigned regions | journal = Journal of Molecular Biology | volume = 348 | issue = 1 | pages = 231–243 | date = April 2005 | pmid = 15808866 | doi = 10.1016/j.jmb.2005.02.007 }}</ref>

==Biochemistry==
[[File:Peptide-Figure-Revised.png|thumb|upright=1.35|Chemical structure of the peptide bond (bottom) and the three-dimensional structure of a peptide bond between an [[alanine]] and an adjacent amino acid (top/inset). The bond itself is made of the [[CHON]] elements.]]
[[File:Peptide group resonance.png|thumb|upright=1.35|[[Resonance (chemistry)|Resonance]] structures of the [[peptide bond]] that links individual amino acids to form a protein [[polymer]]]]
{{Main|Biochemistry|Amino acid|Peptide bond}}

Most proteins consist of linear [[polymer]]s built from series of up to 20 different [[Chirality (chemistry)#In biochemistry|<small>L</small>-α-]] amino acids. All [[proteinogenic amino acid]]s possess common structural features, including an [[alpha carbon|α-carbon]] to which an [[amino]] group, a [[carboxyl]] group, and a variable [[side chain]] are [[chemical bond|bonded]]. Only [[proline]] differs from this basic structure as it contains an unusual ring to the N-end amine group, which forces the CO–NH amide moiety into a fixed conformation.<ref name=Nelson2005/> The side chains of the standard amino acids, detailed in the [[list of standard amino acids]], have a great variety of chemical structures and properties; it is the combined effect of all of the amino acid side chains in a protein that ultimately determines its three-dimensional structure and its chemical reactivity.<ref name=Gutteridge2005/>
The amino acids in a polypeptide chain are linked by [[peptide bond]]s. Once linked in the protein chain, an individual amino acid is called a ''residue,'' and the linked series of carbon, nitrogen, and oxygen atoms are known as the ''main chain'' or ''protein backbone.''<ref name = "Murray_2006" />{{rp|19}}

The peptide bond has two [[resonance (chemistry)|resonance]] forms that contribute some [[double-bond]] character and inhibit rotation around its axis, so that the alpha carbons are roughly [[coplanar]]. The other two [[dihedral angle]]s in the peptide bond determine the local shape assumed by the protein backbone.<ref name = "Murray_2006" />{{rp|31}} The end with a free amino group is known as the [[N-terminus]] or amino terminus, whereas the end of the protein with a free carboxyl group is known as the [[C-terminus]] or carboxy terminus (the sequence of the protein is written from N-terminus to C-terminus, from left to right).

The words ''protein'', ''polypeptide,'' and ''[[peptide]]'' are a little ambiguous and can overlap in meaning. ''Protein'' is generally used to refer to the complete biological molecule in a stable [[tertiary structure|conformation]], whereas ''peptide'' is generally reserved for a short amino acid oligomers often lacking a stable 3D structure. But the boundary between the two is not well defined and usually lies near 20–30 residues.<ref name=Lodish2004/> ''Polypeptide'' can refer to any single linear chain of amino acids, usually regardless of length, but often implies an absence of a defined [[tertiary structure|conformation]].

=== Interactions ===
Proteins can interact with many types of molecules, including [[protein–protein interaction|with other proteins]], [[Protein–lipid interaction|with lipids]], [[Protein–carbohydrate interaction|with carbohydrates]], and [[Protein–DNA interaction|with DNA]].<ref>{{cite journal | vauthors = Ardejani MS, Powers ET, Kelly JW | title = Using Cooperatively Folded Peptides To Measure Interaction Energies and Conformational Propensities | journal = Accounts of Chemical Research | volume = 50 | issue = 8 | pages = 1875–1882 | date = August 2017 | pmid = 28723063 | pmc = 5584629 | doi = 10.1021/acs.accounts.7b00195 }}</ref><ref name = "Brandon_1999">{{cite book |vauthors=Branden C, Tooze J |title=Introduction to Protein Structure |publisher=Garland Pub |location=New York |year=1999 |isbn=978-0-8153-2305-1}}</ref><ref name = "Murray_2006">{{cite book | vauthors = Murray RF, Harper HW, Granner DK, Mayes PA, Rodwell VW |title=Harper's Illustrated Biochemistry |publisher=Lange Medical Books/McGraw-Hill |location=New York |year=2006 |isbn=978-0-07-146197-9}}</ref><ref name = "Van_Holde_1996">{{cite book |vauthors=Van Holde KE, Mathews CK |title=Biochemistry |publisher=Benjamin/Cummings Pub. Co., Inc |location=Menlo Park, California |year=1996 |isbn=978-0-8053-3931-4 |url=https://archive.org/details/biochemistry00math }}</ref>

=== Abundance in cells ===
It has been estimated that average-sized [[bacteria]] contain about 2 million proteins per cell (e.g. ''[[Escherichia coli|E. coli]]'' and ''[[Staphylococcus aureus]]''). Smaller bacteria, such as ''[[Mycoplasma]]'' or ''[[Spirochaete|spirochetes]]'' contain fewer molecules, on the order of 50,000 to 1 million. By contrast, [[Eukaryote|eukaryotic]] cells are larger and thus contain much more protein. For instance, [[Saccharomyces cerevisiae|yeast]] cells have been estimated to contain about 50 million proteins and [[human]] cells on the order of 1 to 3 billion.<ref>{{cite journal | vauthors = Milo R | title = What is the total number of protein molecules per cell volume? A call to rethink some published values | journal = BioEssays | volume = 35 | issue = 12 | pages = 1050–1055 | date = December 2013 | pmid = 24114984 | pmc = 3910158 | doi = 10.1002/bies.201300066 }}</ref> The concentration of individual protein copies ranges from a few molecules per cell up to 20 million.<ref name="pmid22068332">{{cite journal | vauthors = Beck M, Schmidt A, Malmstroem J, Claassen M, Ori A, Szymborska A, Herzog F, Rinner O, Ellenberg J, Aebersold R | title = The quantitative proteome of a human cell line | journal = Molecular Systems Biology | volume = 7 | pages = 549 | date = November 2011 | pmid = 22068332 | pmc = 3261713 | doi = 10.1038/msb.2011.82 }}</ref> Not all genes coding proteins are expressed in most cells and their number depends on, for example, cell type and external stimuli. For instance, of the 20,000 or so proteins encoded by the human genome, only 6,000 are detected in [[lymphoblastoid]] cells.<ref>{{cite journal | vauthors = Wu L, Candille SI, Choi Y, Xie D, Jiang L, Li-Pook-Than J, Tang H, Snyder M | title = Variation and genetic control of protein abundance in humans | journal = Nature | volume = 499 | issue = 7456 | pages = 79–82 | date = July 2013 | pmid = 23676674 | pmc = 3789121 | doi = 10.1038/nature12223 | bibcode = 2013Natur.499...79W }}</ref>

==Synthesis==

===Biosynthesis===
[[File:Ribosome mRNA translation en.svg|thumb|A ribosome produces a protein using mRNA as template]]
[[File:Genetic code.svg|thumb|The [[DNA]] sequence of a gene [[genetic code|encodes]] the amino acid sequence of a protein]]
{{Main|Protein biosynthesis}}

Proteins are assembled from amino acids using information encoded in genes. Each protein has its own unique amino acid sequence that is specified by the [[nucleotide]] sequence of the gene encoding this protein. The [[genetic code]] is a set of three-nucleotide sets called [[codon]]s and each three-nucleotide combination designates an amino acid, for example AUG ([[adenine]]–[[uracil]]–[[guanine]]) is the code for [[methionine]]. Because [[DNA]] contains four nucleotides, the total number of possible codons is 64; hence, there is some redundancy in the genetic code, with some amino acids specified by more than one codon.<ref name = "Van_Holde_1996" />{{rp|1002–42}} Genes encoded in DNA are first [[transcription (genetics)|transcribed]] into pre-[[messenger RNA]] (mRNA) by proteins such as [[RNA polymerase]]. Most organisms then process the pre-mRNA (also known as a ''primary transcript'') using various forms of [[post-transcriptional modification]] to form the mature mRNA, which is then used as a template for protein synthesis by the [[ribosome]]. In [[prokaryote]]s the mRNA may either be used as soon as it is produced, or be bound by a ribosome after having moved away from the [[nucleoid]]. In contrast, [[eukaryote]]s make mRNA in the [[cell nucleus]] and then [[Protein translocation|translocate]] it across the [[nuclear membrane]] into the [[cytoplasm]], where [[protein biosynthesis|protein synthesis]] then takes place. The rate of protein synthesis is higher in prokaryotes than eukaryotes and can reach up to 20 amino acids per second.<ref name=Pain2000/>

The process of synthesizing a protein from an mRNA template is known as [[translation (genetics)|translation]]. The mRNA is loaded onto the ribosome and is read three nucleotides at a time by matching each codon to its [[base pair]]ing [[anticodon]] located on a [[transfer RNA]] molecule, which carries the amino acid corresponding to the codon it recognizes. The enzyme [[aminoacyl tRNA synthetase]] "charges" the tRNA molecules with the correct amino acids. The growing polypeptide is often termed the ''nascent chain''. Proteins are always biosynthesized from [[N-terminus]] to [[C-terminus]].<ref name = "Van_Holde_1996" />{{rp|1002–42}}

The size of a synthesized protein can be measured by the number of amino acids it contains and by its total [[molecular mass]], which is normally reported in units of ''daltons'' (synonymous with [[atomic mass unit]]s), or the derivative unit kilodalton (kDa). The average size of a protein increases from Archaea to Bacteria to Eukaryote (283, 311, 438 residues and 31, 34, 49 kDa respectively) due to a bigger number of [[protein domain]]s constituting proteins in higher organisms.<ref name="Kozlowski2016">{{cite journal | vauthors = Kozlowski LP | title = Proteome-pI: proteome isoelectric point database | journal = Nucleic Acids Research | volume = 45 | issue = D1 | pages = D1112–D1116 | date = January 2017 | pmid = 27789699 | pmc = 5210655 | doi = 10.1093/nar/gkw978 }}</ref> For instance, [[yeast]] proteins are on average 466 amino acids long and 53 kDa in mass.<ref name=Lodish2004/> The largest known proteins are the [[titin]]s, a component of the [[muscle]] [[sarcomere]], with a molecular mass of almost 3,000 kDa and a total length of almost 27,000 amino acids.<ref name=Fulton1991/>

===Chemical synthesis===
{{main|Peptide synthesis}}
Short proteins can also be synthesized chemically by a family of methods known as [[peptide synthesis]], which rely on [[organic synthesis]] techniques such as [[chemical ligation]] to produce peptides in high yield.<ref name=Bruckdorfer2004/> Chemical synthesis allows for the introduction of non-natural amino acids into polypeptide chains, such as attachment of [[fluorescent]] probes to amino acid side chains.<ref name=Schwarzer2005/> These methods are useful in laboratory [[biochemistry]] and [[cell biology]], though generally not for commercial applications. Chemical synthesis is inefficient for polypeptides longer than about 300 amino acids, and the synthesized proteins may not readily assume their native [[tertiary structure]]. Most chemical synthesis methods proceed from C-terminus to N-terminus, opposite the biological reaction.<ref name=Kent2009/>

== Structure ==
[[File:Chaperonin 1AON.png|thumb|right|upright=1.35|The crystal structure of the [[chaperonin]], a huge protein complex. A single protein subunit is highlighted. Chaperonins assist protein folding.]]
[[File:Proteinviews-1tim.png|thumb|upright=1.35|Three possible representations of the three-dimensional structure of the protein [[triose phosphate isomerase]]. '''Left''': All-atom representation colored by atom type. '''Middle:''' Simplified representation illustrating the backbone conformation, colored by secondary structure. '''Right''': Solvent-accessible surface representation colored by residue type (acidic residues red, basic residues blue, polar residues green, nonpolar residues white).]]
{{main|Protein structure}}
{{further|Protein structure prediction}}

Most&nbsp;proteins [[protein folding|fold]] into unique 3D structures. The shape into which a protein naturally folds is known as its [[native conformation]].<ref name = "Murray_2006" />{{rp|36}} Although many proteins can fold unassisted, simply through the chemical properties of their amino acids, others require the aid of molecular [[Chaperone (protein)|chaperones]] to fold into their native states.<ref name = "Murray_2006" />{{rp|37}} Biochemists often refer to four distinct aspects of a protein's structure:<ref name = "Murray_2006" />{{rp|30–34}}
* ''[[Primary structure]]'': the [[peptide sequence|amino acid sequence]]. A protein is a [[polyamide]].
* ''[[Secondary structure]]'': regularly repeating local structures stabilized by [[hydrogen bond]]s. The most common examples are the [[alpha helix|α-helix]], [[beta sheet|β-sheet]] and [[turn (biochemistry)|turns]]. Because secondary structures are local, many regions of different secondary structure can be present in the same protein molecule.
* ''[[Tertiary structure]]'': the overall shape of a single protein molecule; the spatial relationship of the secondary structures to one another. Tertiary structure is generally stabilized by nonlocal interactions, most commonly the formation of a [[hydrophobic core]], but also through [[Salt bridge (protein)|salt bridges]], hydrogen bonds, [[disulfide bond]]s, and even [[posttranslational modification|post-translational modification]]s. The term "tertiary structure" is often used as synonymous with the term ''fold''. The tertiary structure is what controls the basic function of the protein.
* ''[[Quaternary structure]]'': the structure formed by several protein molecules (polypeptide chains), usually called ''[[protein subunit]]s'' in this context, which function as a single [[protein complex]].
* ''[[Protein quinary structure|Quinary structure]]'': the signatures of protein surface that organize the crowded cellular interior. Quinary structure is dependent on transient, yet essential, macromolecular interactions that occur inside living cells.

Proteins are not entirely rigid molecules. In addition to these levels of structure, proteins may shift between several related structures while they perform their functions. In the context of these functional rearrangements, these tertiary or quaternary structures are usually referred to as "[[Chemical conformation|conformations]]", and transitions between them are called ''conformational changes.'' Such changes are often induced by the binding of a [[Substrate (biochemistry)|substrate]] molecule to an enzyme's [[active site]], or the physical region of the protein that participates in chemical catalysis. In solution, proteins also undergo variation in structure through thermal vibration and the collision with other molecules.<ref name = "Van_Holde_1996" />{{rp|368–75}}

[[File:Protein composite.png|thumb|upright=1.35|Molecular surface of several proteins showing their comparative sizes. From left to right are: [[immunoglobulin G]] (IgG, an [[antibody]]), [[hemoglobin]], [[insulin]] (a hormone), [[adenylate kinase]] (an enzyme), and [[glutamine synthetase]] (an enzyme).]]

Proteins can be informally divided into three main classes, which correlate with typical tertiary structures: [[globular protein]]s, [[fibrous protein]]s, and [[membrane protein]]s. Almost all globular proteins are [[soluble]] and many are enzymes. Fibrous proteins are often structural, such as [[collagen]], the major component of connective tissue, or [[keratin]], the protein component of hair and nails. Membrane proteins often serve as [[receptor (biochemistry)|receptors]] or provide channels for polar or charged molecules to pass through the [[cell membrane]].<ref name = "Van_Holde_1996" />{{rp|165–85}}

A special case of intramolecular hydrogen bonds within proteins, poorly shielded from water attack and hence promoting their own [[dehydration]], are called [[dehydron]]s.<ref name=Fernandez2003/>

=== Protein domains ===
{{Main|Protein domain}}
Many proteins are composed of several [[protein domain]]s, i.e. segments of a protein that fold into distinct structural units. Domains usually also have specific functions, such as [[Enzyme|enzymatic]] activities (e.g. [[kinase]]) or they serve as binding modules (e.g. the [[SH3 domain]] binds to proline-rich sequences in other proteins).

=== Sequence motif ===
Short amino acid sequences within proteins often act as recognition sites for other proteins.<ref>{{cite journal | vauthors = Davey NE, Van Roey K, Weatheritt RJ, Toedt G, Uyar B, Altenberg B, Budd A, Diella F, Dinkel H, Gibson TJ | title = Attributes of short linear motifs | journal = Molecular BioSystems | volume = 8 | issue = 1 | pages = 268–281 | date = January 2012 | pmid = 21909575 | doi = 10.1039/c1mb05231d }}</ref> For instance, [[SH3 domain]]s typically bind to short PxxP motifs (i.e. 2 [[proline]]s [P], separated by two unspecified [[amino acid]]s [x], although the surrounding amino acids may determine the exact binding specificity). Many such motifs has been collected in the [[Eukaryotic Linear Motif resource|Eukaryotic Linear Motif]] (ELM) database.

===Protein topology===
Topology of a protein describes the entanglement of the backbone and the arrangement of contacts within the folded chain.<ref>{{cite journal | vauthors = Scalvini B, Sheikhhassani V, Woodard J, Aupič J, Dame RT, Jerala R, Mashaghi A |title=Topology of Folded Molecular Chains: From Single Biomolecules to Engineered Origami |journal=Trends in Chemistry |date=July 2020 |volume=2 |issue=7 |pages=609–622 |doi=10.1016/j.trechm.2020.04.009 |hdl=1887/3245505 |s2cid=218957613 |hdl-access=free }}</ref> Two theoretical frameworks of [[Knotted protein|knot theory]] and [[Circuit topology]] have been applied to characterise protein topology. Being able to describe protein topology opens up new pathways for protein engineering and pharmaceutical development, and adds to our understanding of protein misfolding diseases such as neuromuscular disorders and cancer.

==Cellular functions==

Proteins are the chief actors within the cell, said to be carrying out the duties specified by the information encoded in genes.<ref name=Lodish2004/> With the exception of certain types of [[RNA]], most other biological molecules are relatively inert elements upon which proteins act. Proteins make up half the dry weight of an ''[[Escherichia coli]]'' cell, whereas other macromolecules such as DNA and RNA make up only 3% and 20%, respectively.<ref name="Voet">Voet D, Voet JG. (2004). ''Biochemistry'' Vol 1 3rd ed. Wiley: Hoboken, NJ.</ref> The set of proteins expressed in a particular cell or cell type is known as its [[proteome]].

[[File:Hexokinase ball and stick model, with substrates to scale copy.png|thumb|right|The enzyme [[hexokinase]] is shown as a conventional ball-and-stick molecular model. To scale in the top right-hand corner are two of its substrates, [[adenosine triphosphate|ATP]] and [[glucose]].]]

The chief characteristic of proteins that also allows their diverse set of functions is their ability to bind other molecules specifically and tightly. The region of the protein responsible for binding another molecule is known as the [[binding site]] and is often a depression or "pocket" on the molecular surface. This binding ability is mediated by the tertiary structure of the protein, which defines the binding site pocket, and by the chemical properties of the surrounding amino acids' side chains. Protein binding can be extraordinarily tight and specific; for example, the [[ribonuclease inhibitor]] protein binds to human [[angiogenin]] with a sub-femtomolar [[dissociation constant]] (<10<sup>−15</sup> M) but does not bind at all to its amphibian homolog [[onconase]] (>&nbsp;1&nbsp;M). Extremely minor chemical changes such as the addition of a single methyl group to a binding partner can sometimes suffice to nearly eliminate binding; for example, the [[aminoacyl tRNA synthetase]] specific to the amino acid [[valine]] discriminates against the very similar side chain of the amino acid [[isoleucine]].<ref name=Sankaranarayanan2001/>

Proteins can bind to other proteins as well as to [[Small molecule|small-molecule]] substrates. When proteins bind specifically to other copies of the same molecule, they can [[oligomer]]ize to form fibrils; this process occurs often in structural proteins that consist of globular monomers that self-associate to form rigid fibers. [[Protein–protein interaction]]s also regulate enzymatic activity, control progression through the [[cell cycle]], and allow the assembly of large [[protein complex]]es that carry out many closely related reactions with a common biological function. Proteins can also bind to, or even be integrated into, cell membranes. The ability of binding partners to induce conformational changes in proteins allows the construction of enormously complex [[cell signaling|signaling]] networks.<ref name = "Van_Holde_1996" />{{rp|830–49}}
As interactions between proteins are reversible, and depend heavily on the availability of different groups of partner proteins to form aggregates that are capable to carry out discrete sets of function, study of the interactions between specific proteins is a key to understand important aspects of cellular function, and ultimately the properties that distinguish particular cell types.<ref name=Copland2009/><ref name=Samarin2009/>

===Enzymes===
{{Main|Enzyme}}
The best-known role of proteins in the cell is as [[enzyme]]s, which [[catalysis|catalyse]] chemical reactions. Enzymes are usually highly specific and accelerate only one or a few chemical reactions. Enzymes carry out most of the reactions involved in [[metabolism]], as well as manipulating DNA in processes such as [[DNA replication]], [[DNA repair]], and [[transcription (genetics)|transcription]]. Some enzymes act on other proteins to add or remove chemical groups in a process known as posttranslational modification. About 4,000 reactions are known to be catalysed by enzymes.<ref name=EXPASY2000/> The rate acceleration conferred by enzymatic catalysis is often enormous—as much as 10<sup>17</sup>-fold increase in rate over the uncatalysed reaction in the case of [[orotate decarboxylase]] (78 million years without the enzyme, 18 milliseconds with the enzyme).<ref name=Radzicka1995/>

The molecules bound and acted upon by enzymes are called [[Substrate (biochemistry)|substrates]]. Although enzymes can consist of hundreds of amino acids, it is usually only a small fraction of the residues that come in contact with the substrate, and an even smaller fraction—three to four residues on average—that are directly involved in catalysis.<ref name="urlEBI"/> The region of the enzyme that binds the substrate and contains the catalytic residues is known as the [[active site]].

[[Dirigent protein]]s are members of a class of proteins that dictate the [[stereochemistry]] of a compound synthesized by other enzymes.<ref name="Pickel 2013"/>

===Cell signaling and ligand binding===
{{See also|Glycan-protein interactions}}
[[File:Mouse cholera antibody.png|thumb|upright|[[Ribbon diagram]] of a mouse antibody against [[cholera]] that binds a [[carbohydrate]] antigen]]

Many proteins are involved in the process of [[cell signaling]] and [[signal transduction]]. Some proteins, such as [[insulin]], are extracellular proteins that transmit a signal from the cell in which they were synthesized to other cells in distant [[biological tissue|tissues]]. Others are [[membrane protein]]s that act as [[receptor (biochemistry)|receptors]] whose main function is to bind a signaling molecule and induce a biochemical response in the cell. Many receptors have a binding site exposed on the cell surface and an effector domain within the cell, which may have enzymatic activity or may undergo a [[conformational change]] detected by other proteins within the cell.<ref name = "Brandon_1999" />{{rp|251–81}}

[[Antibodies]] are protein components of an [[adaptive immune system]] whose main function is to bind [[antigen]]s, or foreign substances in the body, and target them for destruction. Antibodies can be [[secrete]]d into the extracellular environment or anchored in the membranes of specialized [[B cell]]s known as [[plasma cell]]s. Whereas enzymes are limited in their binding affinity for their substrates by the necessity of conducting their reaction, antibodies have no such constraints. An antibody's binding affinity to its target is extraordinarily high.<ref name = "Van_Holde_1996" />{{rp|275–50}}

Many ligand transport proteins bind particular [[Small molecule|small biomolecules]] and transport them to other locations in the body of a multicellular organism. These proteins must have a high binding affinity when their [[ligand]] is present in high concentrations, but must also release the ligand when it is present at low concentrations in the target tissues. The canonical example of a ligand-binding protein is [[haemoglobin]], which transports [[oxygen]] from the [[lung]]s to other organs and tissues in all [[vertebrate]]s and has close homologs in every biological [[kingdom (biology)|kingdom]].<ref name = "Van_Holde_1996" />{{rp|222–29}} [[Lectins]] are [[Glycan-protein interactions|sugar-binding proteins]] which are highly specific for their sugar moieties. [[Lectins]] typically play a role in biological [[Molecular recognition|recognition]] phenomena involving cells and proteins.<ref name=Rudiger2000/> [[Receptor (biochemistry)|Receptors]] and [[hormone]]s are highly specific binding proteins.

[[Transmembrane protein]]s can also serve as ligand transport proteins that alter the [[Semipermeable membrane|permeability]] of the cell membrane to [[small molecule]]s and ions. The membrane alone has a [[hydrophobic]] core through which [[Chemical polarity|polar]] or charged molecules cannot [[diffusion|diffuse]]. Membrane proteins contain internal channels that allow such molecules to enter and exit the cell. Many [[ion channel]] proteins are specialized to select for only a particular ion; for example, [[potassium]] and [[sodium]] channels often discriminate for only one of the two ions.<ref name = "Brandon_1999" />{{rp|232–34}}

===Structural proteins===

Structural proteins confer stiffness and rigidity to otherwise-fluid biological components. Most structural proteins are [[fibrous protein]]s; for example, [[collagen]] and [[elastin]] are critical components of [[connective tissue]] such as [[cartilage]], and [[keratin]] is found in hard or filamentous structures such as [[hair]], [[nail (anatomy)|nails]], [[feather]]s, [[hoof|hooves]], and some [[animal shell]]s.<ref name = "Van_Holde_1996" />{{rp|178–81}} Some [[globular proteins]] can also play structural functions, for example, [[actin]] and [[tubulin]] are globular and soluble as monomers, but [[polymer]]ize to form long, stiff fibers that make up the [[cytoskeleton]], which allows the cell to maintain its shape and size.

Other proteins that serve structural functions are [[motor protein]]s such as [[myosin]], [[kinesin]], and [[dynein]], which are capable of generating mechanical forces. These proteins are crucial for cellular [[motility]] of single celled organisms and the [[spermatozoon|sperm]] of many multicellular organisms which reproduce [[Sexual reproduction|sexually]]. They also generate the forces exerted by contracting [[muscle]]s<ref name = "Van_Holde_1996" />{{rp|258–64, 272}} and play essential roles in intracellular transport.

== Protein evolution ==
{{Main|Molecular evolution}}
A key question in molecular biology is how proteins evolve, i.e. how can [[mutation]]s (or rather changes in [[amino acid]] sequence) lead to new structures and functions? Most amino acids in a protein can be changed without disrupting activity or function, as can be seen from numerous [[Homology (biology)|homologous]] proteins across species (as collected in specialized databases for [[protein families]], e.g. [[Pfam|PFAM]]).<ref>{{cite book | vauthors = Mulder NJ | chapter =Protein Family Databases|date=2007-09-28 |title = eLS|pages=a0003058.pub2 |place=Chichester, UK|publisher=John Wiley & Sons, Ltd|language=en|doi=10.1002/9780470015902.a0003058.pub2|isbn=978-0-470-01617-6 }}</ref> In order to prevent dramatic consequences of mutations, a [[Gene duplication|gene may be duplicated]] before it can mutate freely. However, this can also lead to complete loss of gene function and thus [[Pseudogene|pseudo-genes]].<ref>{{cite journal | vauthors = Sisu C, Pei B, Leng J, Frankish A, Zhang Y, Balasubramanian S, Harte R, Wang D, Rutenberg-Schoenberg M, Clark W, Diekhans M, Rozowsky J, Hubbard T, Harrow J, Gerstein MB | title = Comparative analysis of pseudogenes across three phyla | journal = Proceedings of the National Academy of Sciences of the United States of America | volume = 111 | issue = 37 | pages = 13361–13366 | date = September 2014 | pmid = 25157146 | pmc = 4169933 | doi = 10.1073/pnas.1407293111 | doi-access = free | bibcode = 2014PNAS..11113361S }}</ref> More commonly, single amino acid changes have limited consequences although some can change protein function substantially, especially in [[enzyme]]s. For instance, many enzymes can change their [[Chemical specificity|substrate specificity]] by one or a few mutations.<ref name=":0">{{cite journal | vauthors = Guzmán GI, Sandberg TE, LaCroix RA, Nyerges Á, Papp H, de Raad M, King ZA, Hefner Y, Northen TR, Notebaart RA, Pál C, Palsson BO, Papp B, Feist AM | title = Enzyme promiscuity shapes adaptation to novel growth substrates | journal = Molecular Systems Biology | volume = 15 | issue = 4 | pages = e8462 | date = April 2019 | pmid = 30962359 | pmc = 6452873 | doi = 10.15252/msb.20188462 }}</ref> Changes in substrate specificity are facilitated by ''substrate promiscuity'', i.e. the ability of many enzymes to bind and process multiple [[Substrate (chemistry)|substrates]]. When mutations occur, the specificity of an enzyme can increase (or decrease) and thus its enzymatic activity.<ref name=":0" /> Thus, bacteria (or other organisms) can adapt to different food sources, including unnatural substrates such as plastic.<ref name="pmid28545359">{{cite journal | vauthors = Bano K, Kuddus M, Zaheer MR, Zia Q, Khan MF, Ashraf GM, Gupta A, Aliev G | title = Microbial Enzymatic Degradation of Biodegradable Plastics | journal = Current Pharmaceutical Biotechnology | volume = 18 | issue = 5 | pages = 429–440 | date = 2017 | pmid = 28545359 | doi = 10.2174/1389201018666170523165742 }}</ref>

==Methods of study==
{{Main|Protein methods}}
Methods commonly used to study protein structure and function include [[immunohistochemistry]], [[site-directed mutagenesis]], [[X-ray crystallography]], [[nuclear magnetic resonance]] and [[mass spectrometry]].

The activities and structures of proteins may be examined ''[[in vitro]],'' ''[[in vivo]], and [[in silico]]''. '''''In vitro''''' studies of purified proteins in controlled environments are useful for learning how a protein carries out its function: for example, [[enzyme kinetics]] studies explore the [[reaction mechanism|chemical mechanism]] of an enzyme's catalytic activity and its relative affinity for various possible substrate molecules. By contrast, '''''in vivo''''' experiments can provide information about the physiological role of a protein in the context of a [[Cell biology|cell]] or even a whole [[organism]]. '''''In silico''''' studies use computational methods to study proteins.

===Protein purification===
{{Main|Protein purification}}
Proteins may be [[protein purification|purified]] from other cellular components using a variety of techniques such as [[ultracentrifugation]], [[Precipitation (chemistry)|precipitation]], [[electrophoresis]], and [[chromatography]]; the advent of [[genetic engineering]] has made possible a number of methods to facilitate purification.

To perform ''[[in vitro]]'' analysis, a protein must be purified away from other cellular components. This process usually begins with [[cytolysis|cell lysis]], in which a cell's membrane is disrupted and its internal contents released into a solution known as a [[crude lysate]]. The resulting mixture can be purified using [[ultracentrifugation]], which fractionates the various cellular components into fractions containing soluble proteins; membrane [[lipid]]s and proteins; cellular [[organelle]]s, and [[nucleic acid]]s. [[Precipitation (chemistry)|Precipitation]] by a method known as [[salting out]] can concentrate the proteins from this lysate. Various types of [[chromatography]] are then used to isolate the protein or proteins of interest based on properties such as molecular weight, net charge and binding affinity.<ref name = "Murray_2006" />{{rp|21–24}} The level of purification can be monitored using various types of [[gel electrophoresis]] if the desired protein's molecular weight and [[isoelectric point]] are known, by [[spectroscopy]] if the protein has distinguishable spectroscopic features, or by [[enzyme assay]]s if the protein has enzymatic activity. Additionally, proteins can be isolated according to their charge using [[electrofocusing]].<ref name=Hey2008/>

For natural proteins, a series of purification steps may be necessary to obtain protein sufficiently pure for laboratory applications. To simplify this process, [[genetic engineering]] is often used to add chemical features to proteins that make them easier to purify without affecting their structure or activity. Here, a "tag" consisting of a specific amino acid sequence, often a series of [[histidine]] residues (a "[[His-tag]]"), is attached to one terminus of the protein. As a result, when the lysate is passed over a chromatography column containing [[nickel]], the histidine residues ligate the nickel and attach to the column while the untagged components of the lysate pass unimpeded. A number of different tags have been developed to help researchers purify specific proteins from complex mixtures.<ref name=Terpe2003/>

===Cellular localization===
[[File:Localisations02eng.jpg|thumb|right|upright=1.35|Proteins in different [[cellular compartment]]s and structures tagged with [[green fluorescent protein]] (here, white)]]

The study of proteins ''in vivo'' is often concerned with the synthesis and localization of the protein within the cell. Although many intracellular proteins are synthesized in the [[cytoplasm]] and membrane-bound or secreted proteins in the [[endoplasmic reticulum]], the specifics of how proteins are [[protein targeting|targeted]] to specific organelles or cellular structures is often unclear. A useful technique for assessing cellular localization uses genetic engineering to express in a cell a [[fusion protein]] or [[chimera (protein)|chimera]] consisting of the natural protein of interest linked to a "[[reporter gene|reporter]]" such as [[green fluorescent protein]] (GFP).<ref name=Stepanenko2008/> The fused protein's position within the cell can then be cleanly and efficiently visualized using [[microscopy]],<ref name=Yuste2005/> as shown in the figure opposite.

Other methods for elucidating the cellular location of proteins requires the use of known compartmental markers for regions such as the ER, the Golgi, lysosomes or vacuoles, mitochondria, chloroplasts, plasma membrane, etc. With the use of fluorescently tagged versions of these markers or of antibodies to known markers, it becomes much simpler to identify the localization of a protein of interest. For example, [[indirect immunofluorescence]] will allow for fluorescence colocalization and demonstration of location. Fluorescent dyes are used to label cellular compartments for a similar purpose.<ref name=Margolin2000/>

Other possibilities exist, as well. For example, [[immunohistochemistry]] usually uses an antibody to one or more proteins of interest that are conjugated to enzymes yielding either luminescent or chromogenic signals that can be compared between samples, allowing for localization information. Another applicable technique is cofractionation in sucrose (or other material) gradients using [[isopycnic centrifugation]].<ref name=Walker2000/> While this technique does not prove colocalization of a compartment of known density and the protein of interest, it does increase the likelihood, and is more amenable to large-scale studies.

Finally, the gold-standard method of cellular localization is [[immunoelectron microscopy]]. This technique also uses an antibody to the protein of interest, along with classical electron microscopy techniques. The sample is prepared for normal electron microscopic examination, and then treated with an antibody to the protein of interest that is conjugated to an extremely electro-dense material, usually gold. This allows for the localization of both ultrastructural details as well as the protein of interest.<ref name=Mayhew2008/>

Through another genetic engineering application known as [[site-directed mutagenesis]], researchers can alter the protein sequence and hence its structure, cellular localization, and susceptibility to regulation. This technique even allows the incorporation of unnatural amino acids into proteins, using modified tRNAs,<ref name=Hohsaka2002/> and may allow the rational [[protein design|design]] of new proteins with novel properties.<ref name=Cedrone2000/>

===Proteomics===
{{Main|Proteomics}}
The total complement of proteins present at a time in a cell or cell type is known as its [[proteome]], and the study of such large-scale data sets defines the field of [[proteomics]], named by analogy to the related field of [[genomics]]. Key experimental techniques in proteomics include [[Two-dimensional gel electrophoresis|2D electrophoresis]],<ref name=Gorg2008/> which allows the separation of many proteins, [[mass spectrometry]],<ref name=Conrotto2008/> which allows rapid high-throughput identification of proteins and sequencing of peptides (most often after [[in-gel digestion]]), [[protein microarray]]s, which allow the detection of the relative levels of the various proteins present in a cell, and [[two-hybrid screening]], which allows the systematic exploration of [[protein–protein interaction]]s.<ref name=Koegl2007/> The total complement of biologically possible such interactions is known as the [[interactome]].<ref name=Plewczynski2009/> A systematic attempt to determine the structures of proteins representing every possible fold is known as [[structural genomics]].<ref name=Zhang2003/>

===Structure determination===
Discovering the tertiary structure of a protein, or the quaternary structure of its complexes, can provide important clues about how the protein performs its function and how it can be affected, i.e. in [[Drug design#Structure-based|drug design]]. As proteins are [[Diffraction-limited system|too small to be seen]] under a [[Optical microscope|light microscope]], other methods have to be employed to determine their structure. Common experimental methods include [[X-ray crystallography]] and [[protein NMR|NMR spectroscopy]], both of which can produce structural information at [[atom]]ic resolution. However, NMR experiments are able to provide information from which a subset of distances between pairs of atoms can be estimated, and the final possible conformations for a protein are determined by solving a [[distance geometry]] problem. [[Dual polarisation interferometry]] is a quantitative analytical method for measuring the overall [[protein conformation]] and [[conformational change]]s due to interactions or other stimulus. [[Circular dichroism]] is another laboratory technique for determining internal β-sheet / α-helical composition of proteins. [[Cryoelectron microscopy]] is used to produce lower-resolution structural information about very large protein complexes, including assembled [[virus]]es;<ref name = "Brandon_1999" />{{rp|340–41}} a variant known as [[electron crystallography]] can also produce high-resolution information in some cases, especially for two-dimensional crystals of membrane proteins.<ref name=Gonen2005/> Solved structures are usually deposited in the [[Protein Data Bank]] (PDB), a freely available resource from which structural data about thousands of proteins can be obtained in the form of [[Cartesian coordinates]] for each atom in the protein.<ref name=Standley2008/>

Many more gene sequences are known than protein structures. Further, the set of solved structures is biased toward proteins that can be easily subjected to the conditions required in [[X-ray crystallography]], one of the major structure determination methods. In particular, globular proteins are comparatively easy to [[crystallize]] in preparation for X-ray crystallography. Membrane proteins and large protein complexes, by contrast, are difficult to crystallize and are underrepresented in the PDB.<ref name=Walian2004/> [[Structural genomics]] initiatives have attempted to remedy these deficiencies by systematically solving representative structures of major fold classes. [[Protein structure prediction]] methods attempt to provide a means of generating a plausible structure for proteins whose structures have not been experimentally determined.<ref name=Sleator2012/>

===Structure prediction===
[[File:225 Peptide Bond-01.jpg|thumb|right|upright=1.6|Constituent amino-acids can be analyzed to predict secondary, tertiary and quaternary protein structure, in this case hemoglobin containing [[heme]] units]]
{{Main|Protein structure prediction|List of protein structure prediction software}}

Complementary to the field of structural genomics, ''protein structure prediction'' develops efficient [[mathematical model]]s of proteins to computationally predict the molecular formations in theory, instead of detecting structures with laboratory observation.<ref name=Zhang2008/> The most successful type of structure prediction, known as [[homology modeling]], relies on the existence of a "template" structure with sequence similarity to the protein being modeled; structural genomics' goal is to provide sufficient representation in solved structures to model most of those that remain.<ref name=Xiang2006/> Although producing accurate models remains a challenge when only distantly related template structures are available, it has been suggested that [[sequence alignment]] is the bottleneck in this process, as quite accurate models can be produced if a "perfect" sequence alignment is known.<ref name=Zhang2005/> Many structure prediction methods have served to inform the emerging field of [[protein engineering]], in which novel protein folds have already been designed.<ref name=Kuhlman2003/> Also proteins (in eukaryotes ~33%) contain large unstructured but biologically functional segments and can be classified as [[intrinsically disordered proteins]].<ref>{{cite journal | vauthors = Ward JJ, Sodhi JS, McGuffin LJ, Buxton BF, Jones DT | title = Prediction and functional analysis of native disorder in proteins from the three kingdoms of life | journal = Journal of Molecular Biology | volume = 337 | issue = 3 | pages = 635–645 | date = March 2004 | pmid = 15019783 | doi = 10.1016/j.jmb.2004.02.002 | citeseerx = 10.1.1.120.5605 }}</ref> Predicting and analysing protein disorder is, therefore, an important part of protein structure characterisation.<ref name="TompaFersht2009">{{cite book | vauthors = Tompa P, Fersht A |title=Structure and Function of Intrinsically Disordered Proteins |date=2009 |publisher=CRC Press |isbn=978-1-4200-7893-0 }}{{pn|date=March 2024}}</ref>

===Bioinformatics===
{{Main|Bioinformatics}}
A vast array of computational methods have been developed to analyze the structure, function and evolution of proteins. The development of such tools has been driven by the large amount of genomic and proteomic data available for a variety of organisms, including the [[human genome]]. It is simply impossible to study all proteins experimentally, hence only a few are subjected to laboratory experiments while computational tools are used to extrapolate to similar proteins. Such [[Sequence homology|homologous proteins]] can be efficiently identified in distantly related organisms by [[sequence alignment]]. Genome and gene sequences can be searched by a variety of tools for certain properties. [[Sequence profiling tool]]s can find [[restriction enzyme]] sites, [[open reading frame]]s in [[nucleotide]] sequences, and predict [[secondary structure]]s. [[Phylogenetic tree]]s can be constructed and [[evolution]]ary hypotheses developed using special software like [[ClustalW]] regarding the ancestry of modern organisms and the genes they express. The field of [[bioinformatics]] is now indispensable for the analysis of genes and proteins.

===In silico simulation of dynamical processes===

A more complex computational problem is the prediction of intermolecular interactions, such as in [[docking (molecular)|molecular docking]],<ref name=Ritchie2008/> [[protein folding]], [[protein–protein interaction]] and chemical reactivity. Mathematical models to simulate these dynamical processes involve [[molecular mechanics]], in particular, [[molecular dynamics]]. In this regard, ''[[in silico]]'' simulations discovered the folding of small α-helical [[protein domain]]s such as the [[villin]] headpiece,<ref name=Zagrovic2002/> the [[HIV]] accessory protein<ref name=Herges2005/> and hybrid methods combining standard molecular dynamics with [[quantum mechanics|quantum mechanical]] mathematics have explored the electronic states of [[rhodopsin]]s.<ref name=Hoffman2006/>

Beyond classical molecular dynamics, [[quantum dynamics]] methods allow the simulation of proteins in atomistic detail with an accurate description of quantum mechanical effects. Examples include the multi-layer [[multi-configuration time-dependent Hartree ]](MCTDH) method and the [[hierarchical equations of motion]] (HEOM) approach, which have been applied to plant cryptochromes<ref name= Gatti2018/> and bacteria light-harvesting complexes,<ref name= Schulten2012/> respectively. Both quantum and classical mechanical simulations of biological-scale systems are extremely computationally demanding, so [[distributed computing]] initiatives (for example, the [[Folding@home]] project<ref name=Scheraga2007/>) facilitate the [[molecular modeling on GPU|molecular modeling]] by exploiting advances in [[Graphics processing unit|GPU]] parallel processing and [[Monte Carlo method|Monte Carlo]] techniques.

===Chemical analysis===

The total nitrogen content of organic matter is mainly formed by the amino groups in proteins. The Total Kjeldahl Nitrogen ([[TKN]]) is a measure of nitrogen widely used in the analysis of (waste) water, soil, food, feed and organic matter in general. As the name suggests, the [[Kjeldahl method]] is applied. More sensitive methods are available.<ref>{{cite journal | vauthors = Muñoz-Huerta RF, Guevara-Gonzalez RG, Contreras-Medina LM, Torres-Pacheco I, Prado-Olivarez J, Ocampo-Velazquez RV | title = A review of methods for sensing the nitrogen status in plants: advantages, disadvantages and recent advances | journal = Sensors | volume = 13 | issue = 8 | pages = 10823–10843 | date = August 2013 | pmid = 23959242 | pmc = 3812630 | doi = 10.3390/s130810823 | doi-access = free | bibcode = 2013Senso..1310823M }}</ref><ref>{{cite journal | vauthors = Martin PD, Malley DF, Manning G, Fuller L |title=Determination of soil organic carbon and nitrogen at the field level using near-infrared spectroscopy |journal=Canadian Journal of Soil Science |date=November 2002 |volume=82 |issue=4 |pages=413–422 |doi=10.4141/S01-054 }}</ref>

==Nutrition==
{{further|Protein (nutrient)|Protein quality}}
Most [[microorganism]]s and plants can biosynthesize all 20 standard [[amino acids]], while animals (including humans) must obtain some of the amino acids from the [[diet (nutrition)|diet]].<ref name="Voet" /> The amino acids that an organism cannot synthesize on its own are referred to as [[essential amino acids]]. Key enzymes that synthesize certain amino acids are not present in animals—such as [[aspartokinase]], which catalyses the first step in the synthesis of [[lysine]], [[methionine]], and [[threonine]] from [[aspartate]]. If amino acids are present in the environment, microorganisms can conserve energy by taking up the amino acids from their surroundings and [[Downregulation and upregulation|downregulating]] their biosynthetic pathways.

In animals, amino acids are obtained through the consumption of foods containing protein. Ingested proteins are then broken down into amino acids through [[digestion]], which typically involves [[Denaturation (biochemistry)|denaturation]] of the protein through exposure to [[acid]] and [[hydrolysis]] by enzymes called [[protease]]s. Some ingested amino acids are used for protein biosynthesis, while others are converted to [[glucose]] through [[gluconeogenesis]], or fed into the [[citric acid cycle]]. This use of protein as a fuel is particularly important under [[starvation]] conditions as it allows the body's own proteins to be used to support life, particularly those found in [[muscle]].<ref name=BrosnanJ/>

In animals such as dogs and cats, protein maintains the health and quality of the skin by promoting hair follicle growth and keratinization, and thus reducing the likelihood of skin problems producing malodours.<ref name="Watson_1998">{{cite journal | vauthors = Watson TD | title = Diet and skin disease in dogs and cats | journal = The Journal of Nutrition | volume = 128 | issue = 12 Suppl | pages = 2783S–2789S | date = December 1998 | pmid = 9868266 | doi = 10.1093/jn/128.12.2783S | doi-access = free }}</ref> Poor-quality proteins also have a role regarding gastrointestinal health, increasing the potential for flatulence and odorous compounds in dogs because when proteins reach the colon in an undigested state, they are fermented producing hydrogen sulfide gas, indole, and skatole.<ref name="Case_2010">{{cite book | vauthors = Case LP, Daristotle L, Hayek MG, Raasch MF | year =  2010 | title = Canine and Feline Nutrition-E-Book: A Resource for Companion Animal Professionals | publisher = Elsevier Health Sciences }}</ref> Dogs and cats digest animal proteins better than those from plants, but products of low-quality animal origin are poorly digested, including skin, feathers, and connective tissue.<ref name="Case_2010" />

== Mechanical Properties ==
The [[Mechanical properties of biomaterials|mechanical properties]] of proteins are highly diverse and are often central to their biological function, as in the case of proteins like [[keratin]] and [[Collagen|collagen.]]<ref>{{cite journal | vauthors = Gosline J, Lillie M, Carrington E, Guerette P, Ortlepp C, Savage K | title = Elastic proteins: biological roles and mechanical properties | journal = Philosophical Transactions of the Royal Society of London. Series B, Biological Sciences | volume = 357 | issue = 1418 | pages = 121–132 | date = February 2002 | pmid = 11911769 | pmc = 1692928 | doi = 10.1098/rstb.2001.1022 | veditors = Bailey AJ, Macmillan J, Shrewry PR, Tatham AS }}</ref> For instance, the ability of [[Muscle|muscle tissue]] to continually expand and contract is directly tied to the elastic properties of their underlying protein makeup.<ref>{{cite journal | vauthors = Maruyama K, Natori R, Nonomura Y | title = New elastic protein from muscle | journal = Nature | volume = 262 | issue = 5563 | pages = 58–60 | date = July 1976 | pmid = 934326 | doi = 10.1038/262058a0 }}</ref><ref>{{cite journal | vauthors = Tskhovrebova L, Trinick J | title = Making muscle elastic: the structural basis of myomesin stretching | journal = PLOS Biology | volume = 10 | issue = 2 | pages = e1001264 | date = February 2012 | pmid = 22347814 | pmc = 3279349 | doi = 10.1371/journal.pbio.1001264 | doi-access = free }}</ref> Beyond fibrous proteins, the conformational dynamics of [[Enzyme|enzymes]]<ref>{{cite journal | vauthors = Mizraji E, Acerenza L, Lin J | title = Viscoelastic models for enzymes with multiple conformational states | journal = Journal of Theoretical Biology | volume = 129 | issue = 2 | pages = 163–175 | date = November 1987 | pmid = 3455460 | doi = 10.1016/s0022-5193(87)80010-3 }}</ref>  and the structure of [[Biological membrane|biological membranes]], among other biological functions, are governed by the mechanical properties of the proteins. Outside of their biological context, the unique mechanical properties of many proteins, along with their relative sustainability when compared to [[List of synthetic polymers|synthetic polymers]], have made them desirable targets for next-generation materials design.<ref>{{Cite journal | vauthors = Schiller T, Scheibel T |date=2024-04-18 |title=Bioinspired and biomimetic protein-based fibers and their applications |url=https://www.nature.com/articles/s43246-024-00488-2 |journal=Communications Materials |language=en |volume=5 |issue=1 |pages=1–18 |doi=10.1038/s43246-024-00488-2 |issn=2662-4443}}</ref><ref>{{cite journal | vauthors = Sun J, He H, Zhao K, Cheng W, Li Y, Zhang P, Wan S, Liu Y, Wang M, Li M, Wei Z, Li B, Zhang Y, Li C, Sun Y, Shen J, Li J, Wang F, Ma C, Tian Y, Su J, Chen D, Fan C, Zhang H, Liu K | title = Protein fibers with self-recoverable mechanical properties via dynamic imine chemistry | journal = Nature Communications | volume = 14 | issue = 1 | pages = 5348 | date = September 2023 | pmid = 37660126 | pmc = 10475138 | doi = 10.1038/s41467-023-41084-1 }}</ref>

=== Young's Modulus ===
[[Young's modulus]], ''E,''  is calculated as the axial stress σ over the resulting strain ε. It is a measure of the relative [[stiffness]] of a material. In the context of proteins, this stiffness often directly correlates to biological function. For example, [[collagen]], found in [[connective tissue]], [[Bone|bones]], and [[cartilage]], and [[keratin]], found in nails, claws, and hair, have observed stiffnesses that are several orders of magnitude higher than that of [[elastin]]<ref name=":1">{{cite journal | vauthors = Guthold M, Liu W, Sparks EA, Jawerth LM, Peng L, Falvo M, Superfine R, Hantgan RR, Lord ST | title = A comparison of the mechanical and structural properties of fibrin fibers with other protein fibers | journal = Cell Biochemistry and Biophysics | volume = 49 | issue = 3 | pages = 165–181 | date = 2007-10-02 | pmid = 17952642 | pmc = 3010386 | doi = 10.1007/s12013-007-9001-4 }}</ref>, which is though to give elasticity to structures such as [[Blood vessel|blood vessels]], [[Lung|pulmonary tissue]], and [[Bladder|bladder tissue]], among others<ref>{{cite journal | vauthors = Wang K, Meng X, Guo Z | title = Elastin Structure, Synthesis, Regulatory Mechanism and Relationship With Cardiovascular Diseases | journal = Frontiers in Cell and Developmental Biology | volume = 9 | pages = 596702 | date = 2021 | pmid = 34917605 | pmc = 8670233 | doi = 10.3389/fcell.2021.596702 | doi-access = free }}</ref><ref>{{cite journal | vauthors = Debelle L, Tamburro AM | title = Elastin: molecular description and function | journal = The International Journal of Biochemistry & Cell Biology | volume = 31 | issue = 2 | pages = 261–272 | date = February 1999 | pmid = 10216959 | doi = 10.1016/S1357-2725(98)00098-3 }}</ref>. In comparison to this, [[Globular protein|globular proteins]], such as [[Bovine serum albumin|Bovine Serum Albumin]], which float relatively freely in the [[cytosol]] and often function as enzymes (and thus undergoing frequent conformational changes) have comparably much lower Young's moduli<ref name=":2">{{cite journal | vauthors = Khoury LR, Popa I | title = Chemical unfolding of protein domains induces shape change in programmed protein hydrogels | journal = Nature Communications | volume = 10 | issue = 1 | pages = 5439 | date = November 2019 | pmid = 31784506 | pmc = 6884551 | doi = 10.1038/s41467-019-13312-0 }}</ref><ref>{{cite journal | vauthors = Tan R, Shin J, Heo J, Cole BD, Hong J, Jang Y | title = Tuning the Structural Integrity and Mechanical Properties of Globular Protein Vesicles by Blending Crosslinkable and NonCrosslinkable Building Blocks | journal = Biomacromolecules | volume = 21 | issue = 10 | pages = 4336–4344 | date = October 2020 | pmid = 32955862 | doi = 10.1021/acs.biomac.0c01147 }}</ref>.

The Young's modulus of a single protein can be found through [[molecular dynamics]] simulation. Using either atomistic force-fields, such as [[CHARMM]] or [[GROMOS]], or coarse-grained forcefields like Martini<ref>{{cite journal | vauthors = Souza PC, Alessandri R, Barnoud J, Thallmair S, Faustino I, Grünewald F, Patmanidis I, Abdizadeh H, Bruininks BM, Wassenaar TA, Kroon PC, Melcr J, Nieto V, Corradi V, Khan HM, Domański J, Javanainen M, Martinez-Seara H, Reuter N, Best RB, Vattulainen I, Monticelli L, Periole X, Tieleman DP, de Vries AH, Marrink SJ | title = Martini 3: a general purpose force field for coarse-grained molecular dynamics | journal = Nature Methods | volume = 18 | issue = 4 | pages = 382–388 | date = April 2021 | pmid = 33782607 | doi = 10.1038/s41592-021-01098-3 | url = https://pure.rug.nl/ws/files/190731140/s41592_021_01098_3.pdf }}</ref>, a single protein molecule can be stretched by a uniaxial force while the resulting extension is recorded in order to calculate the strain<ref>{{Cite web |title=Piotr Szymczak's Homepage |url=https://www.fuw.edu.pl/~piotrek/proteins.html |access-date=2024-05-13 |website=www.fuw.edu.pl}}</ref><ref>{{cite journal | vauthors = Mapplebeck S, Booth J, Shalashilin D | title = Simulation of protein pulling dynamics on second time scale with boxed molecular dynamics | journal = The Journal of Chemical Physics | volume = 155 | issue = 8 | pages = 085101 | date = August 2021 | pmid = 34470356 | doi = 10.1063/5.0059321 }}</ref>. Experimentally, methods such as [[atomic force microscopy]] can be used to obtain similar data.<ref>{{cite journal | vauthors = Carrion-Vazquez M, Marszalek PE, Oberhauser AF, Fernandez JM | title = Atomic force microscopy captures length phenotypes in single proteins | journal = Proceedings of the National Academy of Sciences of the United States of America | volume = 96 | issue = 20 | pages = 11288–11292 | date = September 1999 | pmid = 10500169 | pmc = 18026 | doi = 10.1073/pnas.96.20.11288 | doi-access = free }}</ref>

At the macroscopic level, the Young's modulus of cross-linked protein networks can be obtained through more traditional [[mechanical testing]]. Experimentally observed values for a few proteins can be seen below. 
{| class="wikitable"
|+Elasticity of Various Proteins
!Protein
!Protein Class
!Young's modulus
|-
|Keratin (Cross-Linked)
|Fibrous
|1.5-10 GPa<ref>{{Cite journal | vauthors = McKittrick J, Chen PY, Bodde SG, Yang W, Novitskaya EE, Meyers MA |date=2012-04-03 |title=The Structure, Functions, and Mechanical Properties of Keratin |url=http://link.springer.com/10.1007/s11837-012-0302-8 |journal=JOM |language=en |volume=64 |issue=4 |pages=449–468 |doi=10.1007/s11837-012-0302-8 |issn=1047-4838}}</ref>
|-
|Elastin (Cross-Linked)
|Fibrous
|1 MPa<ref name=":1" />
|-
|Fibrin (Cross-linked)
|Fibrous
|1-10 MPa <ref name=":1" />
|-
|Collagen (Cross-linked)
|Fibrous
|5-7.5 GPa<ref name=":1" /><ref>{{cite journal | vauthors = Yang L, van der Werf KO, Fitié CF, Bennink ML, Dijkstra PJ, Feijen J | title = Mechanical properties of native and cross-linked type I collagen fibrils | journal = Biophysical Journal | volume = 94 | issue = 6 | pages = 2204–2211 | date = March 2008 | pmid = 18032556 | pmc = 2257912 | doi = 10.1529/biophysj.107.111013 }}</ref>
|-
|Resilin (Cross-Linked)
|Fibrous
|1-2 MPa<ref name=":1" />
|-
|Bovine Serum Albumin (Cross-Linked)
|Globular 
|2.5-15 KPa<ref name=":2" />
|-
|β-Barrel Outer Membrane Proteins
|Membrane
|20-45 GPa<ref>{{cite journal | vauthors = Lessen HJ, Fleming PJ, Fleming KG, Sodt AJ | title = Building Blocks of the Outer Membrane: Calculating a General Elastic Energy Model for β-Barrel Membrane Proteins | journal = Journal of Chemical Theory and Computation | volume = 14 | issue = 8 | pages = 4487–4497 | date = August 2018 | pmid = 29979594 | pmc = 6191857 | doi = 10.1021/acs.jctc.8b00377 }}</ref>
|}

=== Viscosity ===
In addition to serving as enzymes within the cell, globular proteins often act as key transport molecules. For instance, [[Serum albumin|Serum Albumins]], a key component of [[blood]], are necessary for the transport of a multitude of small molecules throughout the body.<ref>{{cite journal | vauthors = Mishra V, Heath RJ | title = Structural and Biochemical Features of Human Serum Albumin Essential for Eukaryotic Cell Culture | journal = International Journal of Molecular Sciences | volume = 22 | issue = 16 | pages = 8411 | date = August 2021 | pmid = 34445120 | pmc = 8395139 | doi = 10.3390/ijms22168411 | doi-access = free }}</ref> Because of this, the concentration dependent behavior of these proteins in solution is directly tied to the function of the [[circulatory system]]. On way of quantifying this behavior is through the [[viscosity]] of the solution. 

Viscosity, η, is generally given is a measure of a fluid's resistance to deformation. It can be calculated as the ratio between the applied stress and the rate of change of the resulting shear strain, that is, the rate of deformation. Viscosity of complex liquid mixtures, such as blood, often depends strongly on temperature and solute concentration.<ref name="Spencer_2024">{{cite journal | vauthors = Spencer SJ, Ranganathan VT, Yethiraj A, Andrews GT | title = Concentration Dependence of Elastic and Viscoelastic Properties of Aqueous Solutions of Ficoll and Bovine Serum Albumin by Brillouin Light Scattering Spectroscopy | journal = Langmuir : The ACS Journal of Surfaces and Colloids | volume = 40 | issue = 9 | pages = 4615–4622 | date = March 2024 | pmid = 38387073 | doi = 10.1021/acs.langmuir.3c02967 | arxiv = 2309.10967 }}</ref> For serum albumin, specifically [[bovine serum albumin]], the following relation between viscosity and [[temperature]] and [[Concentration|concentratio]]<nowiki/>n can be used.<ref>{{cite journal | vauthors = Monkos K | title = Viscosity of bovine serum albumin aqueous solutions as a function of temperature and concentration | journal = International Journal of Biological Macromolecules | volume = 18 | issue = 1–2 | pages = 61–68 | date = February 1996 | pmid = 8852754 | doi = 10.1016/0141-8130(95)01057-2 }}</ref> 

<math>\eta = \exp\left[ \frac{c}{\alpha-\beta\ c}\left(-B +D T + \frac{\Delta E}{R T}\right)\right] </math>

Where ''c'' is the concentration, ''T'' is the temperature, ''R'' is the [[gas constant]], and α, β, ''B'', ''D'', and Δ''E'' are all material-based property constants. This equation has the form of an [[Arrhenius equation]], assigning viscosity an exponential dependence on temperature and concentration.

== See also ==
{{columns-list|colwidth=30em|
* [[Deproteination]]
* [[DNA-binding protein]]
* [[Macromolecule]]
* [[Index of protein-related articles]]
* [[Intein]]
* [[List of proteins]]
* [[Proteopathy]]
* [[Proteopedia]]
* [[Proteolysis]]
* [[Sequence space (evolution)|Protein sequence space]]
* [[Protein superfamily]]
}}{{Clear}}

== References ==
{{Reflist|refs=
<ref name=Bischoff1860>{{cite book | vauthors = Bischoff TL, Voit C |title=Die Gesetze der Ernaehrung des Pflanzenfressers durch neue Untersuchungen festgestellt |location=Leipzig, Heidelberg |year=1860 |language=de}}</ref>

<ref name=BrosnanJ>{{cite journal | vauthors = Brosnan JT | title = Interorgan amino acid transport and its regulation | journal = The Journal of Nutrition | volume = 133 | issue = 6 Suppl 1 | pages = 2068S–2072S | date = June 2003 | pmid = 12771367 | doi = 10.1093/jn/133.6.2068S | doi-access = free }}</ref>

<ref name=Bruckdorfer2004>{{cite journal | vauthors = Bruckdorfer T, Marder O, Albericio F | title = From production of peptides in milligram amounts for research to multi-tons quantities for drugs of the future | journal = Current Pharmaceutical Biotechnology | volume = 5 | issue = 1 | pages = 29–43 | date = February 2004 | pmid = 14965208 | doi = 10.2174/1389201043489620 }}</ref>

<ref name=Cedrone2000>{{cite journal | vauthors = Cedrone F, Ménez A, Quéméneur E | title = Tailoring new enzyme functions by rational redesign | journal = Current Opinion in Structural Biology | volume = 10 | issue = 4 | pages = 405–410 | date = August 2000 | pmid = 10981626 | doi = 10.1016/S0959-440X(00)00106-8 }}</ref>

<ref name=Conrotto2008>{{cite journal | vauthors = Conrotto P, Souchelnytskyi S | title = Proteomic approaches in biological and medical sciences: principles and applications | journal = Experimental Oncology | volume = 30 | issue = 3 | pages = 171–180 | date = September 2008 | pmid = 18806738 }}</ref>

<ref name=Copland2009>{{cite journal | vauthors = Copland JA, Sheffield-Moore M, Koldzic-Zivanovic N, Gentry S, Lamprou G, Tzortzatou-Stathopoulou F, Zoumpourlis V, Urban RJ, Vlahopoulos SA | title = Sex steroid receptors in skeletal differentiation and epithelial neoplasia: is tissue-specific intervention possible? | journal = BioEssays | volume = 31 | issue = 6 | pages = 629–641 | date = June 2009 | pmid = 19382224 | doi = 10.1002/bies.200800138 | s2cid = 205469320 }}</ref>

<ref name=EXPASY2000>{{cite journal | vauthors = Bairoch A | title = The ENZYME database in 2000 | journal = Nucleic Acids Research | volume = 28 | issue = 1 | pages = 304–305 | date = January 2000 | pmid = 10592255 | pmc = 102465 | doi = 10.1093/nar/28.1.304 }}</ref>

<ref name=Fernandez2003>{{cite journal | vauthors = Fernández A, Scott R | title = Dehydron: a structurally encoded signal for protein interaction | journal = Biophysical Journal | volume = 85 | issue = 3 | pages = 1914–1928 | date = September 2003 | pmid = 12944304 | pmc = 1303363 | doi = 10.1016/S0006-3495(03)74619-0 | bibcode = 2003BpJ....85.1914F }}</ref>

<ref name=Fulton1991>{{cite journal | vauthors = Fulton AB, Isaacs WB | title = Titin, a huge, elastic sarcomeric protein with a probable role in morphogenesis | journal = BioEssays | volume = 13 | issue = 4 | pages = 157–161 | date = April 1991 | pmid = 1859393 | doi = 10.1002/bies.950130403 | s2cid = 20237314 }}</ref>

<ref name=Gonen2005>{{cite journal | vauthors = Gonen T, Cheng Y, Sliz P, Hiroaki Y, Fujiyoshi Y, Harrison SC, Walz T | title = Lipid-protein interactions in double-layered two-dimensional AQP0 crystals | journal = Nature | volume = 438 | issue = 7068 | pages = 633–638 | date = December 2005 | pmid = 16319884 | pmc = 1350984 | doi = 10.1038/nature04321 | bibcode = 2005Natur.438..633G }}</ref>

<ref name=Gorg2008>{{cite journal | vauthors = Görg A, Weiss W, Dunn MJ | title = Current two-dimensional electrophoresis technology for proteomics | journal = Proteomics | volume = 4 | issue = 12 | pages = 3665–3685 | date = December 2004 | pmid = 15543535 | doi = 10.1002/pmic.200401031 | s2cid = 28594824 }}</ref>

<ref name=Gutteridge2005>{{cite journal | vauthors = Gutteridge A, Thornton JM | title = Understanding nature's catalytic toolkit | journal = Trends in Biochemical Sciences | volume = 30 | issue = 11 | pages = 622–629 | date = November 2005 | pmid = 16214343 | doi = 10.1016/j.tibs.2005.09.006 }}</ref>

<ref name=Herges2005>{{cite journal | vauthors = Herges T, Wenzel W | title = In silico folding of a three helix protein and characterization of its free-energy landscape in an all-atom force field | journal = Physical Review Letters | volume = 94 | issue = 1 | pages = 018101 | date = January 2005 | pmid = 15698135 | doi = 10.1103/PhysRevLett.94.018101 | arxiv = physics/0310146 | s2cid = 1477100 | bibcode = 2005PhRvL..94a8101H }}</ref>

<ref name=Hey2008>{{cite book |doi=10.1007/978-1-60327-064-9_19 |chapter=Fractionation of Complex Protein Mixtures by Liquid-Phase Isoelectric Focusing |title=2D PAGE: Sample Preparation and Fractionation |series=Methods in Molecular Biology |date=2008 | vauthors = Hey J, Posch A, Cohen A, Liu N, Harbers A |volume=424 |pages=225–239 |pmid=18369866 |isbn=978-1-58829-722-8 }}</ref>

<ref name=Hoffman2006>{{cite journal | vauthors = Hoffmann M, Wanko M, Strodel P, König PH, Frauenheim T, Schulten K, Thiel W, Tajkhorshid E, Elstner M | title = Color tuning in rhodopsins: the mechanism for the spectral shift between bacteriorhodopsin and sensory rhodopsin II | journal = Journal of the American Chemical Society | volume = 128 | issue = 33 | pages = 10808–10818 | date = August 2006 | pmid = 16910676 | doi = 10.1021/ja062082i }}</ref>

<ref name=Hohsaka2002>{{cite journal | vauthors = Hohsaka T, Sisido M | title = Incorporation of non-natural amino acids into proteins | journal = Current Opinion in Chemical Biology | volume = 6 | issue = 6 | pages = 809–815 | date = December 2002 | pmid = 12470735 | doi = 10.1016/S1367-5931(02)00376-9 }}</ref>

<ref name=Kalman1955>{{cite journal | vauthors = Kalman SM, Linderstrøm-Lang K, Ottesen M, Richards FM | title = Degradation of ribonuclease by subtilisin | journal = Biochimica et Biophysica Acta | volume = 16 | issue = 2 | pages = 297–299 | date = February 1955 | pmid = 14363272 | doi = 10.1016/0006-3002(55)90224-9 }}</ref>

<ref name=Kauzmann1956>{{cite journal | vauthors = Kauzmann W | title = Structural factors in protein denaturation | journal = Journal of Cellular Physiology. Supplement | volume = 47 | issue = Suppl 1 | pages = 113–131 | date = May 1956 | pmid = 13332017 | doi = 10.1002/jcp.1030470410 }}</ref>

<ref name=Kauzmann1959>{{Cite book | vauthors = Kauzmann W | chapter = Some factors in the interpretation of protein denaturation | volume = 14 | pages = 1–63 | year = 1959 | pmid = 14404936 | doi = 10.1016/S0065-3233(08)60608-7 | isbn = 978-0-12-034214-3 | title = Advances in Protein Chemistry Volume 14 }}</ref>

<ref name=Kendrew1958>{{cite journal | vauthors = Kendrew JC, Bodo G, Dintzis HM, Parrish RG, Wyckoff H, Phillips DC | title = A three-dimensional model of the myoglobin molecule obtained by x-ray analysis | journal = Nature | volume = 181 | issue = 4610 | pages = 662–666 | date = March 1958 | pmid = 13517261 | doi = 10.1038/181662a0 | s2cid = 4162786 | bibcode = 1958Natur.181..662K }}</ref>

<ref name=Kent2009>{{cite journal | vauthors = Kent SB | title = Total chemical synthesis of proteins | journal = Chemical Society Reviews | volume = 38 | issue = 2 | pages = 338–351 | date = February 2009 | pmid = 19169452 | doi = 10.1039/b700141j | s2cid = 5432012 }}</ref>

<ref name=Keskin2008>{{cite journal | vauthors = Keskin O, Tuncbag N, Gursoy A | title = Characterization and prediction of protein interfaces to infer protein-protein interaction networks | journal = Current Pharmaceutical Biotechnology | volume = 9 | issue = 2 | pages = 67–76 | date = April 2008 | pmid = 18393863 | doi = 10.2174/138920108783955191 | hdl-access = free | hdl = 11511/32640 }}</ref>

<ref name=Koegl2007>{{cite journal | vauthors = Koegl M, Uetz P | title = Improving yeast two-hybrid screening systems | journal = Briefings in Functional Genomics & Proteomics | volume = 6 | issue = 4 | pages = 302–312 | date = December 2007 | pmid = 18218650 | doi = 10.1093/bfgp/elm035 }}</ref>

<ref name=Kuhlman2003>{{cite journal | vauthors = Kuhlman B, Dantas G, Ireton GC, Varani G, Stoddard BL, Baker D | title = Design of a novel globular protein fold with atomic-level accuracy | journal = Science | volume = 302 | issue = 5649 | pages = 1364–1368 | date = November 2003 | pmid = 14631033 | doi = 10.1126/science.1089427 | s2cid = 1939390 | bibcode = 2003Sci...302.1364K }}</ref>

<ref name="Lecture 1958">{{citation |author=Sanger F. |year=1958 |title=Nobel lecture: The chemistry of insulin |publisher=Nobelprize.org |url=http://nobelprize.org/nobel_prizes/chemistry/laureates/1958/sanger-lecture.pdf |access-date=2016-02-09 |archive-url=https://web.archive.org/web/20130319022148/http://www.nobelprize.org/nobel_prizes/chemistry/laureates/1958/sanger-lecture.pdf |archive-date=2013-03-19 |url-status=live }}</ref>

<ref name=Lodish2004>{{cite book |vauthors=Lodish H, Berk A, Matsudaira P, Kaiser CA, Krieger M, Scott MP, Zipurksy SL, Darnell J |year=2004 |title=Molecular Cell Biology |edition=5th |publisher=WH Freeman and Company |location=New York, New York}}</ref>

<ref name=Margolin2000>{{cite journal | vauthors = Margolin W | title = Green fluorescent protein as a reporter for macromolecular localization in bacterial cells | journal = Methods | volume = 20 | issue = 1 | pages = 62–72 | date = January 2000 | pmid = 10610805 | doi = 10.1006/meth.1999.0906 }}</ref>

<ref name=Mayhew2008>{{cite journal | vauthors = Mayhew TM, Lucocq JM | title = Developments in cell biology for quantitative immunoelectron microscopy based on thin sections: a review | journal = Histochemistry and Cell Biology | volume = 130 | issue = 2 | pages = 299–313 | date = August 2008 | pmid = 18553098 | pmc = 2491712 | doi = 10.1007/s00418-008-0451-6 }}</ref>

<ref name=Muirhead1963>{{cite journal | vauthors = Muirhead H, Perutz MF | title = . Structure of haemoglobin. A three-dimensional Fourier synthesis of reduced human haemoglobin at 5.5 Å resolution | journal = Nature | volume = 199 | issue = 4894 | pages = 633–638 | date = August 1963 | pmid = 14074546 | doi = 10.1038/199633a0 | s2cid = 4257461 | bibcode = 1963Natur.199..633M }}</ref>

<ref name=Nelson2005>{{cite book |vauthors=Nelson DL, Cox MM |year=2005 |title=Lehninger's Principles of Biochemistry |edition=4th |publisher=W. H. Freeman and Company |location=New York, New York}}</ref>

<ref name=Pain2000>{{cite book | veditors = Pain RH | vauthors = Dobson CM |title=Mechanisms of Protein Folding |chapter=The nature and significance of protein folding |publisher=Oxford University Press |location=Oxford, Oxfordshire |year=2000 |pages=1–28 |isbn=978-0-19-963789-8}}</ref>

<ref name=Pauling1951>{{cite journal | vauthors = Pauling L, Corey RB | title = Atomic coordinates and structure factors for two helical configurations of polypeptide chains | journal = Proceedings of the National Academy of Sciences of the United States of America | volume = 37 | issue = 5 | pages = 235–240 | date = May 1951 | pmid = 14834145 | pmc = 1063348 | doi = 10.1073/pnas.37.5.235 | doi-access = free | bibcode = 1951PNAS...37..235P }}</ref>

<ref name=Perrett2007>{{cite journal | vauthors = Perrett D | title = From 'protein' to the beginnings of clinical proteomics | journal = Proteomics. Clinical Applications | volume = 1 | issue = 8 | pages = 720–738 | date = August 2007 | pmid = 21136729 | doi = 10.1002/prca.200700525 | s2cid = 32843102 }}</ref>

<ref name="Pickel 2013">{{cite journal | vauthors = Pickel B, Schaller A | title = Dirigent proteins: molecular characteristics and potential biotechnological applications | journal = Applied Microbiology and Biotechnology | volume = 97 | issue = 19 | pages = 8427–8438 | date = October 2013 | pmid = 23989917 | doi = 10.1007/s00253-013-5167-4 | s2cid = 1896003 }}</ref>

<ref name=Plewczynski2009>{{cite journal | vauthors = Plewczyński D, Ginalski K | title = The interactome: predicting the protein-protein interactions in cells | journal = Cellular & Molecular Biology Letters | volume = 14 | issue = 1 | pages = 1–22 | year = 2009 | pmid = 18839074 | pmc = 6275871 | doi = 10.2478/s11658-008-0024-7 }}</ref>

<ref name=Radzicka1995>{{cite journal | vauthors = Radzicka A, Wolfenden R | title = A proficient enzyme | journal = Science | volume = 267 | issue = 5194 | pages = 90–93 | date = January 1995 | pmid = 7809611 | doi = 10.1126/science.7809611 | bibcode = 1995Sci...267...90R }}</ref>

<ref name=Reynolds2003>{{cite book | vauthors = Reynolds JA, Tanford C |title=Nature's Robots: A History of Proteins (Oxford Paperbacks) |publisher=Oxford University Press |location=New York, New York |year=2003 |page=15 |isbn=978-0-19-860694-9}}</ref>

<ref name=Ritchie2008>{{cite journal | vauthors = Ritchie DW | title = Recent progress and future directions in protein-protein docking | journal = Current Protein & Peptide Science | volume = 9 | issue = 1 | pages = 1–15 | date = February 2008 | pmid = 18336319 | doi = 10.2174/138920308783565741 | citeseerx = 10.1.1.211.4946 }}</ref>

<ref name=Rudiger2000>{{cite journal | vauthors = Rüdiger H, Siebert HC, Solís D, Jiménez-Barbero J, Romero A, von der Lieth CW, Diaz-Mariño T, Gabius HJ | title = Medicinal chemistry based on the sugar code: fundamentals of lectinology and experimental strategies with lectins as targets | journal = Current Medicinal Chemistry | volume = 7 | issue = 4 | pages = 389–416 | date = April 2000 | pmid = 10702616 | doi = 10.2174/0929867003375164 }}</ref>

<ref name=Samarin2009>{{cite journal | vauthors = Samarin S, Nusrat A | title = Regulation of epithelial apical junctional complex by Rho family GTPases | journal = Frontiers in Bioscience | volume = 14 | issue = 3 | pages = 1129–1142 | date = January 2009 | pmid = 19273120 | doi = 10.2741/3298 | doi-access = free }}</ref>

<ref name=Sanger1949>{{cite journal | vauthors = Sanger F | title = The terminal peptides of insulin | journal = The Biochemical Journal | volume = 45 | issue = 5 | pages = 563–574 | year = 1949 | pmid = 15396627 | pmc = 1275055 | doi = 10.1042/bj0450563 }}</ref>

<ref name=Sankaranarayanan2001>{{cite journal | vauthors = Sankaranarayanan R, Moras D | title = The fidelity of the translation of the genetic code | journal = Acta Biochimica Polonica | volume = 48 | issue = 2 | pages = 323–335 | year = 2001 | pmid = 11732604 | doi = 10.18388/abp.2001_3918 | doi-access = free }}</ref>

<ref name=Scheraga2007>{{cite journal | vauthors = Scheraga HA, Khalili M, Liwo A | title = Protein-folding dynamics: overview of molecular simulation techniques | journal = Annual Review of Physical Chemistry | volume = 58 | pages = 57–83 | year = 2007 | pmid = 17034338 | doi = 10.1146/annurev.physchem.58.032806.104614 | bibcode = 2007ARPC...58...57S }}</ref>

<ref name=Schwarzer2005>{{cite journal | vauthors = Schwarzer D, Cole PA | title = Protein semisynthesis and expressed protein ligation: chasing a protein's tail | journal = Current Opinion in Chemical Biology | volume = 9 | issue = 6 | pages = 561–569 | date = December 2005 | pmid = 16226484 | doi = 10.1016/j.cbpa.2005.09.018 }}</ref>

<ref name=Sleator2012>{{Cite book | vauthors = Sleator RD | chapter = Prediction of Protein Functions | volume = 815 | pages = 15–24 | year = 2012 | pmid = 22130980 | doi = 10.1007/978-1-61779-424-7_2 | isbn = 978-1-61779-423-0 | series = Methods in Molecular Biology | title = Functional Genomics }}</ref>

<ref name=Standley2008>{{cite journal | vauthors = Standley DM, Kinjo AR, Kinoshita K, Nakamura H | title = Protein structure databases with new web services for structural biology and biomedical research | journal = Briefings in Bioinformatics | volume = 9 | issue = 4 | pages = 276–285 | date = July 2008 | pmid = 18430752 | doi = 10.1093/bib/bbn015 | doi-access = free }}</ref>

<ref name=Stepanenko2008>{{cite journal | vauthors = Stepanenko OV, Verkhusha VV, Kuznetsova IM, Uversky VN, Turoverov KK | title = Fluorescent proteins as biomarkers and biosensors: throwing color lights on molecular and cellular processes | journal = Current Protein & Peptide Science | volume = 9 | issue = 4 | pages = 338–369 | date = August 2008 | pmid = 18691124 | pmc = 2904242 | doi = 10.2174/138920308785132668 }}</ref>

<ref name=Sumner1926>{{cite journal | vauthors = Sumner JB |title=The Isolation and Crystallization of the Enzyme Urease |journal=Journal of Biological Chemistry |date=August 1926 |volume=69 |issue=2 |pages=435–441 |doi=10.1016/S0021-9258(18)84560-4 |doi-access=free }}</ref>

<ref name=Terpe2003>{{cite journal | vauthors = Terpe K | title = Overview of tag protein fusions: from molecular and biochemical fundamentals to commercial systems | journal = Applied Microbiology and Biotechnology | volume = 60 | issue = 5 | pages = 523–533 | date = January 2003 | pmid = 12536251 | doi = 10.1007/s00253-002-1158-6 | s2cid = 206934268 }}</ref>

<ref name="urlEBI">{{cite web |author=EBI External Services |url=http://www.ebi.ac.uk/thornton-srv/databases/CSA/ |title=The Catalytic Site Atlas at The European Bioinformatics Institute |publisher=Ebi.ac.uk |date=2010-01-20 |access-date=2011-01-16 |archive-url=https://web.archive.org/web/20130803032349/http://www.ebi.ac.uk/thornton-srv/databases/CSA/ |archive-date=2013-08-03 |url-status=live }}</ref>

<!-- <ref name="urlRCSB Protein Data Bank">{{cite web|url=http://www.rcsb.org/pdb/home/home.do|title=RCSB Protein Data Bank|access-date=2017-01-19|url-status=dead|archive-url=https://web.archive.org/web/20150418160606/http://www.rcsb.org/pdb/home/home.do|archive-date=2015-04-18}}</ref> -->

<ref name=Walian2004>{{cite journal | vauthors = Walian P, Cross TA, Jap BK | title = Structural genomics of membrane proteins | journal = Genome Biology | volume = 5 | issue = 4 | pages = 215 | year = 2004 | pmid = 15059248 | pmc = 395774 | doi = 10.1186/gb-2004-5-4-215 | doi-access = free }}</ref>

<ref name=Walker2000>{{cite book | vauthors = Walker JH, Wilson K |title=Principles and Techniques of Practical Biochemistry |publisher=Cambridge University Press |location=Cambridge, UK |year=2000 |pages=287–89 |isbn=978-0-521-65873-7}}</ref>

<ref name=Xiang2006>{{cite journal | vauthors = Xiang Z | title = Advances in homology protein structure modeling | journal = Current Protein & Peptide Science | volume = 7 | issue = 3 | pages = 217–227 | date = June 2006 | pmid = 16787261 | pmc = 1839925 | doi = 10.2174/138920306777452312 }}</ref>

<ref name=Yuste2005>{{cite journal | vauthors = Yuste R | title = Fluorescence microscopy today | journal = Nature Methods | volume = 2 | issue = 12 | pages = 902–904 | date = December 2005 | pmid = 16299474 | doi = 10.1038/nmeth1205-902 | s2cid = 205418407 }}</ref>

<ref name=Zagrovic2002>{{cite journal | vauthors = Zagrovic B, Snow CD, Shirts MR, Pande VS | title = Simulation of folding of a small alpha-helical protein in atomistic detail using worldwide-distributed computing | journal = Journal of Molecular Biology | volume = 323 | issue = 5 | pages = 927–937 | date = November 2002 | pmid = 12417204 | doi = 10.1016/S0022-2836(02)00997-X | citeseerx = 10.1.1.142.8664 }}</ref>

<ref name=Zhang2003>{{cite journal | vauthors = Zhang C, Kim SH | title = Overview of structural genomics: from structure to function | journal = Current Opinion in Chemical Biology | volume = 7 | issue = 1 | pages = 28–32 | date = February 2003 | pmid = 12547423 | doi = 10.1016/S1367-5931(02)00015-7 | url = https://zenodo.org/record/1260238 }}</ref>

<ref name=Zhang2005>{{cite journal | vauthors = Zhang Y, Skolnick J | title = The protein structure prediction problem could be solved using the current PDB library | journal = Proceedings of the National Academy of Sciences of the United States of America | volume = 102 | issue = 4 | pages = 1029–1034 | date = January 2005 | pmid = 15653774 | pmc = 545829 | doi = 10.1073/pnas.0407152101 | doi-access = free | bibcode = 2005PNAS..102.1029Z }}</ref>

<ref name=Zhang2008>{{cite journal | vauthors = Zhang Y | title = Progress and challenges in protein structure prediction | journal = Current Opinion in Structural Biology | volume = 18 | issue = 3 | pages = 342–348 | date = June 2008 | pmid = 18436442 | pmc = 2680823 | doi = 10.1016/j.sbi.2008.02.004 }}</ref>

<ref name=Zhou2008>{{cite journal | vauthors = Zhou ZH | title = Towards atomic resolution structural determination by single-particle cryo-electron microscopy | journal = Current Opinion in Structural Biology | volume = 18 | issue = 2 | pages = 218–228 | date = April 2008 | pmid = 18403197 | pmc = 2714865 | doi = 10.1016/j.sbi.2008.03.004 }}</ref>

<ref name= Schulten2012 >{{cite journal | vauthors = Strümpfer J, Schulten K | title = Open Quantum Dynamics Calculations with the Hierarchy Equations of Motion on Parallel Computers | journal = Journal of Chemical Theory and Computation | volume = 8 | issue = 8 | pages = 2808–2816 | date = August 2012 | pmid = 23105920 | pmc = 3480185 | doi = 10.1021/ct3003833 }}</ref>

<ref name= Gatti2018 >{{cite journal | vauthors = Mendive-Tapia D, Mangaud E, Firmino T, de la Lande A, Desouter-Lecomte M, Meyer HD, Gatti F | title = Multidimensional Quantum Mechanical Modeling of Electron Transfer and Electronic Coherence in Plant Cryptochromes: The Role of Initial Bath Conditions | journal = The Journal of Physical Chemistry. B | volume = 122 | issue = 1 | pages = 126–136 | date = January 2018 | pmid = 29216421 | doi = 10.1021/acs.jpcb.7b10412 }}</ref>

}}

==Further reading ==
; Textbooks
{{refbegin|32em}}
* {{cite book |vauthors=Branden C, Tooze J |title=Introduction to Protein Structure |publisher=Garland Pub |location=New York |year=1999 |isbn=978-0-8153-2305-1}}
* {{cite book |vauthors=Murray RF, Harper HW, Granner DK, Mayes PA, Rodwell VW |title=Harper's Illustrated Biochemistry |publisher=Lange Medical Books/McGraw-Hill |location=New York |year=2006 |isbn=978-0-07-146197-9}}
* {{cite book |vauthors=Van Holde KE, Mathews CK |title=Biochemistry |publisher=Benjamin/Cummings Pub. Co., Inc |location=Menlo Park, California |year=1996 |isbn=978-0-8053-3931-4 |url=https://archive.org/details/biochemistry00math }}
{{refend}}

== External links ==
{{Sister project links|auto=1|wikt=protein}}

===Databases and projects===
* [https://www.ncbi.nlm.nih.gov/sites/entrez?db=protein NCBI Entrez Protein database]
* [https://www.ncbi.nlm.nih.gov/sites/entrez?db=structure NCBI Protein Structure database]
* [https://web.archive.org/web/20060424071622/http://www.hprd.org/ Human Protein Reference Database]
* [https://web.archive.org/web/20070314135408/http://www.humanproteinpedia.org/ Human Proteinpedia]
* [http://folding.stanford.edu/ Folding@Home (Stanford University)] {{Webarchive|url=https://web.archive.org/web/20120908075542/http://folding.stanford.edu/English/HomePage |date=2012-09-08 }}
* [http://www.pdbe.org/ Protein Databank in Europe] (see also [https://archive.today/20130727184433/http://www.pdbe.org/quips PDBeQuips], short articles and tutorials on interesting PDB structures)
* [http://www.rcsb.org/ Research Collaboratory for Structural Bioinformatics] (see also [http://www.rcsb.org/pdb/static.do?p=education_discussion/molecule_of_the_month/index.html Molecule of the Month] {{Webarchive|url=https://web.archive.org/web/20200724151351/https://www.rcsb.org/pdb/static.do?p=education_discussion%2Fmolecule_of_the_month%2Findex.html |date=2020-07-24 }}, presenting short accounts on selected proteins from the PDB)
* [http://www.proteopedia.org/ Proteopedia&nbsp;– Life in 3D]: rotatable, zoomable 3D model with wiki annotations for every known protein molecular structure.
* [https://web.archive.org/web/20080608183902/http://www.expasy.uniprot.org/ UniProt the Universal Protein Resource]

===Tutorials and educational websites===
* [https://web.stanford.edu/group/hopes/cgi-bin/hopes_test/an-introduction-to-proteins/ "An Introduction to Proteins"] from [[HOPES]] (Huntington's Disease Outreach Project for Education at Stanford)
* [https://web.archive.org/web/20050219090405/http://www.biochemweb.org/proteins.shtml Proteins: Biogenesis to Degradation&nbsp;– The Virtual Library of Biochemistry and Cell Biology]

{{Gene expression}}
{{Protein topics}}
{{Protein methods}}
{{Food chemistry}}
{{Metabolism}}
{{Portal bar|Biology|Technology|Medicine|Chemistry|Food|Ecology|Environment|Science|Evolutionary biology}}
{{Authority control}}

[[Category:Proteins| ]]
[[Category:Molecular biology]]
[[Category:Proteomics]]