{{Short description|Species of bacterium}}
{{cs1 config|name-list-style=vanc|display-authors=6}}
{{Speciesbox
|image = Pseudomonas aeruginosa on blood agar.jpg
|image_alt = Pseudomonas aeruginos colonies on blood agar
|image_caption = ''P. aeruginosa'' colonies on [[blood agar]]
|genus = Pseudomonas
|species = aeruginosa
|authority = (Schröter 1872)<br />[[Walter Migula|Migula]] 1900
|synonyms=
* ''Bacterium aeruginosum'' <small>Schroeter 1872</small>
* ''Bacterium aeruginosum'' <small>Cohn 1872</small>
* ''Micrococcus pyocyaneus'' <small>Zopf 1884</small>
* ''Bacillus aeruginosus'' <small>(Schroeter 1872) Trevisan 1885</small>
* ''Bacillus pyocyaneus'' <small>(Zopf 1884) Flügge 1886</small>
* ''Pseudomonas pyocyanea'' <small>(Zopf 1884) [[Walter Migula|Migula]] 1895</small>
* ''Bacterium pyocyaneum'' <small>(Zopf 1884) Lehmann and Neumann 1896</small>
* ''Pseudomonas polycolor'' <small>Clara 1930</small>
* ''Pseudomonas vendrelli'' <small>''[[nomen nudum]]'' 1938</small>
}}
[[File:Pseudomonas Aeruginosa.jpg|thumb|''Pseudomonas aeruginosa'' in petri dish|alt=]]
'''''Pseudomonas aeruginosa''''' is a common [[Bacterial capsule|encapsulated]], [[Gram-negative bacteria|Gram-negative]], [[Aerobic organism|aerobic]]–[[facultative anaerobe|facultatively anaerobic]], [[Bacillus (shape)|rod-shaped]] [[bacteria|bacterium]] that can cause [[disease]] in plants and animals, including humans.<ref>{{cite web |date=13 April 2015 |title=UK Standards for Microbiology Investigations: Identification of Pseudomonas species and other Non- Glucose Fermenters |url=https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/422699/ID_17i3.pdf |access-date=4 May 2022 |website= |publisher=Public Health England |archive-date=3 July 2022 |archive-url=https://web.archive.org/web/20220703115143/https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/422699/ID_17i3.pdf |url-status=live }}</ref><ref>{{cite journal | vauthors = Diggle SP, Whiteley M | title = Microbe Profile: ''Pseudomonas aeruginosa'': opportunistic pathogen and lab rat | journal = Microbiology | volume = 166 | issue = 1 | pages = 30–33 | date = January 2020 | pmid = 31597590 | pmc = 7273324 | doi = 10.1099/mic.0.000860 | doi-access = free }}</ref> A [[species]] of considerable medical importance, ''P.&nbsp;aeruginosa'' is a [[multiple drug resistance|multidrug resistant]] pathogen recognized for its ubiquity, its [[Intrinsic and extrinsic properties|intrinsically]] advanced [[antibiotic resistance]] mechanisms, and its association with serious illnesses – [[hospital-acquired infections]] such as [[ventilator-associated pneumonia]] and various [[sepsis]] [[syndromes]]. ''P. aeruginosa'' is able to selectively inhibit various antibiotics from penetrating its outer membrane - and has high resistance to several antibiotics, according to the [[World Health Organization]] ''P. aeruginosa'' poses one of the greatest threats to humans in terms of antibiotic resistance.<ref>{{Cite journal | vauthors = Spagnolo AM, Sartini M, Cristina ML |date=July 2021 |title=Pseudomonas aeruginosa in the healthcare facility setting |url=https://journals.lww.com/revmedmicrobiol/fulltext/2021/07000/pseudomonas_aeruginosa_in_the_healthcare_facility.5.aspx |journal=Reviews and Research in Medical Microbiology |language=en-US |volume=32 |issue=3 |pages=169–175 |doi=10.1097/MRM.0000000000000271 |issn=2770-3150 |doi-access=free |access-date=2024-01-18 |archive-date=2024-01-18 |archive-url=https://web.archive.org/web/20240118141848/https://journals.lww.com/revmedmicrobiol/fulltext/2021/07000/pseudomonas_aeruginosa_in_the_healthcare_facility.5.aspx |url-status=live }}</ref>

The organism is considered [[Opportunistic infection|opportunistic]] insofar as serious infection often occurs during existing [[disease]]s or conditions''&nbsp;''– most notably [[cystic fibrosis]] and traumatic burns. It generally affects the [[Immunodeficiency|immunocompromised]] but can also infect the [[immunocompetent]] as in [[hot tub folliculitis]]. Treatment of ''P.&nbsp;aeruginosa'' infections can be difficult due to its natural resistance to antibiotics. When more advanced antibiotic drug regimens are needed [[adverse effects]] may result.

It is [[citrate]], [[catalase]], and [[oxidase positive]]. It is found in soil, water, [[skin flora]], and most human-made environments throughout the world. It thrives not only in normal atmospheres, but also in [[Hypoxia (environmental)|low-oxygen]] atmospheres, thus has colonized many natural and artificial environments. It uses a wide range of organic material for food; in animals, its versatility enables the organism to infect damaged tissues or those with reduced immunity. The symptoms of such infections are generalized [[inflammation]] and [[sepsis]]. If such colonizations occur in critical body organs, such as the [[lung]]s, the [[urinary tract]], and [[kidney]]s, the results can be fatal.<ref>{{cite book | vauthors = Balcht A, Smith R | title = ''Pseudomonas aeruginosa'': Infections and Treatment|publisher = Informa Health Care |pages = 83–84 |year = 1994 | isbn = 978-0-8247-9210-7}}</ref> Because it thrives on moist surfaces, this bacterium is also found on and in [[medical equipment]], including [[catheter]]s, causing cross-[[infection]]s in [[hospital]]s and [[clinic]]s. It is also able to decompose hydrocarbons and has been used to break down [[tarball (oil)|tarballs]] and oil from [[oil spill]]s.<ref>{{cite journal| vauthors = Itah AY, Essien JP |title= Growth Profile and Hydrocarbonoclastic Potential of Microorganisms Isolated from Tarballs in the Bight of Bonny, Nigeria|journal=World Journal of Microbiology and Biotechnology|volume =21|issue=6–7|year= 2005| doi=10.1007/s11274-004-6694-z|pages=1317–22|s2cid=84888286 }}</ref> ''P.&nbsp;aeruginosa'' is not extremely [[virulence|virulent]] in comparison with other major species of [[pathogenic bacteria]] such as [[Gram-positive]] ''[[Staphylococcus aureus]]'' and ''[[Streptococcus pyogenes]]&nbsp;''– though ''P.&nbsp;aeruginosa'' is capable of extensive colonization, and can aggregate into enduring [[biofilm]]s.<ref>{{cite journal | vauthors = Høiby N, Ciofu O, Bjarnsholt T | title = Pseudomonas aeruginosa biofilms in cystic fibrosis | journal = Future Microbiology | volume = 5 | issue = 11 | pages = 1663–1674 | date = November 2010 | pmid = 21133688 | doi = 10.2217/fmb.10.125 }}</ref>

==Nomenclature==
[[File:Pseudomonas aeruginosa pigment production.jpg|alt=P.aeruginosa pigments.|thumb|Pigment production, growth on [[cetrimide agar]], the [[oxidase test]], plaque formation and [[Gram stain]]]]
[[File:Pseudomonas aeruginosa culture.JPG|thumb|A culture dish with ''Pseudomonas'']]

The word ''Pseudomonas'' means "false unit", from the Greek ''pseudēs'' ([[Ancient Greek|Greek]]: ψευδής, false) and ({{lang-la|monas}}, from [[Ancient Greek|Greek]]: μονάς, a single unit). The stem word ''mon'' was used early in the history of [[microbiology]] to refer to [[microorganism]]s and [[Germ theory of disease|germ]]s, e.g., kingdom [[Monera]].<ref name="Henry">{{cite journal | vauthors = Palleroni NJ | title = The Pseudomonas story | journal = Environmental Microbiology | volume = 12 | issue = 6 | pages = 1377–1383 | date = June 2010 | pmc = 3423701 | doi = 10.3201/eid1808.ET1808 | doi-access = free | pmid = 20553550 }}</ref>

The species name ''aeruginosa'' is a Latin word meaning [[verdigris]] ("copper rust"), referring to the blue-green color of laboratory cultures of the species. This blue-green pigment is a combination of two [[secondary metabolite]]s of ''P.&nbsp;aeruginosa'', [[pyocyanin]] (blue) and [[pyoverdine]] (green), which impart the blue-green characteristic color of cultures.<ref name="Henry"/> Another assertion from 1956 is that ''aeruginosa'' may be derived from the Greek prefix ''ae-'' meaning "old or aged", and the suffix ''ruginosa'' means wrinkled or bumpy.<ref>{{cite book | vauthors = Brown RW |title=Composition of Scientific Words |year=1956 |publisher=Smithsonian Institutional Press |isbn=978-0-87474-286-2}}</ref>

The names pyocyanin and pyoverdine are from the Greek, with ''pyo-'', meaning "pus",<ref>{{cite book|vauthors=Tzouchas A|title=WestBow Press|date=2014|publisher=Greek Words|isbn=978-1-4907-2610-6|page=550|url=https://books.google.com/books?id=HrwSBAAAQBAJ&q=pyo+greek+pus|access-date=2020-11-02|archive-date=2023-04-26|archive-url=https://web.archive.org/web/20230426202325/https://books.google.com/books?id=HrwSBAAAQBAJ&q=pyo+greek+pus|url-status=live}}</ref> ''cyanin'', meaning "blue",<ref>{{cite journal | doi=10.3390/molecules26040927 | doi-access=free | title=Colour Me Blue: The History and the Biotechnological Potential of Pyocyanin | date=2021 | journal=Molecules | volume=26 | issue=4 | page=927 | pmid=33578646 | pmc=7916356 | vauthors = Gonçalves T, Vasconcelos U }}</ref> and ''verdine'', meaning "green".{{Citation needed|reason=This is not derived from a Greek word.|date=May 2018}} Hence, the term "pyocyanic bacteria" refers specifically to the "blue pus" characteristic of a ''P.&nbsp;aeruginosa'' infection. Pyoverdine in the absence of pyocyanin is a fluorescent-yellow color.{{Citation needed|date=February 2021}}
[[File:Pseudomonas aeruginosa Gram.jpg|thumb|Gram-stained ''P.&nbsp;aeruginosa'' bacteria (pink-red rods)]]

==Biology==

===Genome===
The [[Genome#Prokaryotic genomes|genome]] of ''Pseudomonas aeruginosa'' consists of a relatively large circular chromosome (5.5–6.8''&nbsp;''Mb) that carries between 5,500 and 6,000 [[open reading frame]]s, and sometimes [[plasmid]]s of various sizes depending on the strain.<ref>{{cite journal | vauthors = Klockgether J, Cramer N, Wiehlmann L, Davenport CF, Tümmler B | title = Pseudomonas aeruginosa Genomic Structure and Diversity | journal = Frontiers in Microbiology | volume = 2 | pages = 150 | year = 2011 | pmid = 21808635 | pmc = 3139241 | doi = 10.3389/fmicb.2011.00150 | doi-access = free }}</ref> Comparison of 389 genomes from different ''P.&nbsp;aeruginosa'' strains showed that just 17.5% is shared. This part of the genome is the ''P.&nbsp;aeruginosa'' core genome.<ref>{{cite journal | vauthors = De Smet J, Hendrix H, Blasdel BG, Danis-Wlodarczyk K, Lavigne R | title = Pseudomonas predators: understanding and exploiting phage-host interactions | journal = Nature Reviews. Microbiology | volume = 15 | issue = 9 | pages = 517–530 | date = September 2017 | pmid = 28649138 | doi = 10.1038/nrmicro.2017.61 | s2cid = 826136 | url = https://lirias.kuleuven.be/handle/123456789/586291 }}</ref>

{| class="wikitable"
!strain: !! VRFPA04 !! C3719 !! PAO1 !!PA14 !! PACS2
|-
| Chromosome size (bp) || 6,818,030 || 6,222,097|| 6,264,404 ||  6,537,648 || 6,492,423
|-
| ORFs || 5,939 || 5,578   ||  5,571  ||   5,905 ||  5,676

|}
A comparative genomic study (in 2020) analyzed 494 complete genomes from the ''Pseudomonas'' genus, of which 189 were ''P.&nbsp;aeruginosa'' strains.<ref name="Nik">{{Cite journal| vauthors = Nikolaidis M, Mossialos D, Oliver SG, Amoutzias GD |date=2020-07-24|title=Comparative Analysis of the Core Proteomes among the Pseudomonas Major Evolutionary Groups Reveals Species-Specific Adaptations for Pseudomonas aeruginosa and Pseudomonas chlororaphis|journal=Diversity|language=en|volume=12|issue=8|pages=289|doi=10.3390/d12080289|issn=1424-2818|doi-access=free}}</ref> The study observed that their protein count and [[GC content]] ranged between 5500 and 7352 (average: 6192) and between 65.6 and 66.9% (average: 66.1%), respectively.<ref name="Nik" /> This comparative analysis further identified 1811 aeruginosa-core proteins, which accounts for more than 30% of the proteome. The higher percentage of aeruginosa-core proteins in this latter analysis could partly be attributed to the use of complete genomes. Although ''P.&nbsp;aeruginosa'' is a very well-defined monophyletic species, phylogenomically and in terms of ANIm values, it is surprisingly diverse in terms of protein content, thus revealing a very dynamic accessory proteome, in accordance with several analyses.<ref name="Nik" /><ref>{{cite journal | vauthors = Ozer EA, Allen JP, Hauser AR | title = Characterization of the core and accessory genomes of Pseudomonas aeruginosa using bioinformatic tools Spine and AGEnt | journal = BMC Genomics | volume = 15 | issue = 1 | pages = 737 | date = August 2014 | pmid = 25168460 | pmc = 4155085 | doi = 10.1186/1471-2164-15-737 | doi-access = free }}</ref><ref>{{cite journal | vauthors = Subedi D, Vijay AK, Kohli GS, Rice SA, Willcox M | title = Comparative genomics of clinical strains of Pseudomonas aeruginosa strains isolated from different geographic sites | journal = Scientific Reports | volume = 8 | issue = 1 | pages = 15668 | date = October 2018 | pmid = 30353070 | pmc = 6199293 | doi = 10.1038/s41598-018-34020-7 | bibcode = 2018NatSR...815668S }}</ref><ref>{{cite journal | vauthors = Freschi L, Vincent AT, Jeukens J, Emond-Rheault JG, Kukavica-Ibrulj I, Dupont MJ, Charette SJ, Boyle B, Levesque RC | title = The Pseudomonas aeruginosa Pan-Genome Provides New Insights on Its Population Structure, Horizontal Gene Transfer, and Pathogenicity | journal = Genome Biology and Evolution | volume = 11 | issue = 1 | pages = 109–120 | date = January 2019 | pmid = 30496396 | pmc = 6328365 | doi = 10.1093/gbe/evy259 | veditors = Martin B }}</ref> It appears that, on average, industrial strains have the largest genomes, followed by environmental strains, and then clinical isolates.<ref name="Nik" /><ref>{{cite journal | vauthors = Weiser R, Green AE, Bull MJ, Cunningham-Oakes E, Jolley KA, Maiden MC, Hall AJ, Winstanley C, Weightman AJ, Donoghue D, Amezquita A, Connor TR, Mahenthiralingam E | title = Not all Pseudomonas aeruginosa are equal: strains from industrial sources possess uniquely large multireplicon genomes | journal = Microbial Genomics | volume = 5 | issue = 7 | date = July 2019 | pmid = 31170060 | pmc = 6700666 | doi = 10.1099/mgen.0.000276 | doi-access = free }}</ref> The same comparative study (494 ''Pseudomonas'' strains, of which 189 are ''P.&nbsp;aeruginosa'') identified that 41 of the 1811 ''P.&nbsp;aeruginosa'' core proteins were present only in this species and not in any other member of the genus, with 26 (of the 41) being annotated as hypothetical. Furthermore, another 19 orthologous protein groups are present in at least 188/189 ''P.&nbsp;aeruginosa'' strains and absent in all the other strains of the genus.{{Citation needed|date=February 2021}}

===Population structure===

The population of ''P.&nbsp;aeruginosa'' can be classified in three main lineages, genetically characterised by the model strains PAO1, PA14, and the more divergent PA7.<ref>{{cite journal | vauthors = Roy PH, Tetu SG, Larouche A, Elbourne L, Tremblay S, Ren Q, Dodson R, Harkins D, Shay R, Watkins K, Mahamoud Y, Paulsen IT | title = Complete genome sequence of the multiresistant taxonomic outlier Pseudomonas aeruginosa PA7 | journal = PLOS ONE | volume = 5 | issue = 1 | pages = e8842 | date = January 2010 | pmid = 20107499 | pmc = 2809737 | doi = 10.1371/journal.pone.0008842 | doi-access = free | bibcode = 2010PLoSO...5.8842R }}</ref>

While ''P.&nbsp;aeruginosa'' is generally thought of as an opportunistic pathogen, several widespread clones appear to have become more specialised pathogens, particularly in cystic fibrosis patients, including the Liverpool epidemic strain (LES) which is found mainly in the UK,<ref>{{cite journal | vauthors = Winstanley C, Langille MG, Fothergill JL, Kukavica-Ibrulj I, Paradis-Bleau C, Sanschagrin F, Thomson NR, Winsor GL, Quail MA, Lennard N, Bignell A, Clarke L, Seeger K, Saunders D, Harris D, Parkhill J, Hancock RE, Brinkman FS, Levesque RC | title = Newly introduced genomic prophage islands are critical determinants of in vivo competitiveness in the Liverpool Epidemic Strain of Pseudomonas aeruginosa | journal = Genome Research | volume = 19 | issue = 1 | pages = 12–23 | date = January 2009 | pmid = 19047519 | pmc = 2612960 | doi = 10.1101/gr.086082.108 }}</ref> DK2 in Denmark,<ref>{{cite journal | vauthors = Marvig RL, Johansen HK, Molin S, Jelsbak L | title = Genome analysis of a transmissible lineage of pseudomonas aeruginosa reveals pathoadaptive mutations and distinct evolutionary paths of hypermutators | journal = PLOS Genetics | volume = 9 | issue = 9 | pages = e1003741 | date = 2013 | pmid = 24039595 | pmc = 3764201 | doi = 10.1371/journal.pgen.1003741 | doi-access = free }}</ref> and AUST-02 in Australia (also previously known as AES-2 and P2).<ref>{{cite journal | vauthors = Wee BA, Tai AS, Sherrard LJ, Ben Zakour NL, Hanks KR, Kidd TJ, Ramsay KA, Lamont I, Whiley DM, Bell SC, Beatson SA | title = Whole genome sequencing reveals the emergence of a Pseudomonas aeruginosa shared strain sub-lineage among patients treated within a single cystic fibrosis centre | journal = BMC Genomics | volume = 19 | issue = 1 | pages = 644 | date = August 2018 | pmid = 30165811 | pmc = 6117919 | doi = 10.1186/s12864-018-5018-x | doi-access = free }}</ref> There is also a clone that is frequently found infecting the reproductive tracts of horses.<ref>{{cite journal | vauthors = Kidd TJ, Ritchie SR, Ramsay KA, Grimwood K, Bell SC, Rainey PB | title = Pseudomonas aeruginosa exhibits frequent recombination, but only a limited association between genotype and ecological setting | journal = PLOS ONE | volume = 7 | issue = 9 | pages = e44199 | date = 6 September 2012 | pmid = 22970178 | pmc = 3435406 | doi = 10.1371/journal.pone.0044199 | doi-access = free | bibcode = 2012PLoSO...744199K }}</ref><ref>{{cite journal | vauthors = Kidd TJ, Gibson JS, Moss S, Greer RM, Cobbold RN, Wright JD, Ramsay KA, Grimwood K, Bell SC | title = Clonal complex Pseudomonas aeruginosa in horses | journal = Veterinary Microbiology | volume = 149 | issue = 3–4 | pages = 508–512 | date = May 2011 | pmid = 21183294 | doi = 10.1016/j.vetmic.2010.11.030 }}</ref>

===Metabolism===
''P.&nbsp;aeruginosa'' is a [[facultative anaerobe]], as it is well adapted to proliferate in conditions of partial or total oxygen depletion. This organism can achieve [[:wikt:anaerobic|anaerobic]] growth with [[nitrate]] or [[nitrite]] as a [[Electron acceptor|terminal electron acceptor]]. When oxygen, nitrate, and nitrite are absent, it is able to ferment [[arginine]] and [[pyruvate]] by [[substrate-level phosphorylation]].<ref>{{cite journal | vauthors = Schobert M, Jahn D | title = Anaerobic physiology of Pseudomonas aeruginosa in the cystic fibrosis lung | journal = International Journal of Medical Microbiology | volume = 300 | issue = 8 | pages = 549–556 | date = December 2010 | pmid = 20951638 | doi = 10.1016/j.ijmm.2010.08.007 }}</ref> Additionally, phenazines produced by ''P. aeruginosa'' can act as electron shuttles to facilitate survival of cells at depth in biofilms.<ref>{{cite journal | vauthors = Dietrich LE, Okegbe C, Price-Whelan A, Sakhtah H, Hunter RC, Newman DK | title = Bacterial community morphogenesis is intimately linked to the intracellular redox state | journal = Journal of Bacteriology | volume = 195 | issue = 7 | pages = 1371–1380 | date = April 2013 | pmid = 23292774 | pmc = 3624522 | doi = 10.1128/JB.02273-12 }}</ref> Adaptation to [[microaerophile|microaerobic]] or anaerobic environments is essential for certain lifestyles of ''P.&nbsp;aeruginosa'', for example, during lung infection in [[cystic fibrosis]] and [[primary ciliary dyskinesia]], where thick layers of lung [[mucus]] and bacterially-produced [[Alginic acid|alginate]] surrounding mucoid bacterial cells can limit the diffusion of oxygen. ''P.&nbsp;aeruginosa'' growth within the human body can be asymptomatic until the bacteria form a biofilm, which overwhelms the immune system. These biofilms are found in the lungs of people with cystic fibrosis and primary ciliary dyskinesia, and can prove fatal.<ref>{{cite book | vauthors = Tortora GJ, Case CL, Bair III WB, Weber D, Funke BR |title=Microbiology: An Introduction |date=2016 |publisher=Pearson Education |isbn=978-0-321-92915-0 |page=54 |edition=12th}}</ref><ref>{{cite journal | vauthors = Hassett DJ | title = Anaerobic production of alginate by Pseudomonas aeruginosa: alginate restricts diffusion of oxygen | journal = Journal of Bacteriology | volume = 178 | issue = 24 | pages = 7322–7325 | date = December 1996 | pmid = 8955420 | pmc = 178651 | doi = 10.1128/jb.178.24.7322-7325.1996 }}</ref><ref>{{cite journal | vauthors = Worlitzsch D, Tarran R, Ulrich M, Schwab U, Cekici A, Meyer KC, Birrer P, Bellon G, Berger J, Weiss T, Botzenhart K, Yankaskas JR, Randell S, Boucher RC, Döring G | title = Effects of reduced mucus oxygen concentration in airway Pseudomonas infections of cystic fibrosis patients | journal = The Journal of Clinical Investigation | volume = 109 | issue = 3 | pages = 317–325 | date = February 2002 | pmid = 11827991 | pmc = 150856 | doi = 10.1172/JCI13870 }}</ref><ref>{{cite journal | vauthors = Cooper M, Tavankar GR, Williams HD | title = Regulation of expression of the cyanide-insensitive terminal oxidase in Pseudomonas aeruginosa | journal = Microbiology | volume = 149 | issue = Pt 5 | pages = 1275–1284 | date = May 2003 | pmid = 12724389 | doi = 10.1099/mic.0.26017-0 | doi-access = free }}</ref><ref>{{cite book | vauthors = Williams HD, Zlosnik JE, Ryall B | title = Oxygen, cyanide and energy generation in the cystic fibrosis pathogen Pseudomonas aeruginosa | volume = 52 | pages = 1–71 | year = 2007 | pmid = 17027370 | doi = 10.1016/S0065-2911(06)52001-6 | isbn = 978-0-12-027752-0 | series = Advances in Microbial Physiology }}</ref><ref>{{cite book| vauthors =  Leach R, Moore K, Bell D |title=Oxford Desk Reference: Acute Medicine|date=2016|publisher=Oxford University Press|isbn=978-0-19-100714-9|page=244|url=https://books.google.com/books?id=1DlRDAAAQBAJ&pg=PA244}}</ref>{{excessive citations inline|date=November 2023}}

===Cellular cooperation===

''P. aeruginosa'' relies on [[iron]] as a [[mineral (nutrient)|nutrient source]] to grow. However, iron is not easily accessible because it is not commonly found in the environment. Iron is usually found in a largely insoluble ferric form.<ref>{{cite journal | vauthors = Buckling A, Harrison F, Vos M, Brockhurst MA, Gardner A, West SA, Griffin A | title = Siderophore-mediated cooperation and virulence in Pseudomonas aeruginosa | journal = FEMS Microbiology Ecology | volume = 62 | issue = 2 | pages = 135–141 | date = November 2007 | pmid = 17919300 | doi = 10.1111/j.1574-6941.2007.00388.x | doi-access = free | bibcode = 2007FEMME..62..135B }}</ref> Furthermore, excessively high levels of iron can be toxic to ''P.&nbsp;aeruginosa''. To overcome this and regulate proper intake of iron, ''P.&nbsp;aeruginosa'' uses [[siderophore]]s, which are secreted molecules that bind and transport iron.<ref>{{cite journal | vauthors = Nguyen AT, Jones JW, Ruge MA, Kane MA, Oglesby-Sherrouse AG | title = Iron Depletion Enhances Production of Antimicrobials by Pseudomonas aeruginosa | journal = Journal of Bacteriology | volume = 197 | issue = 14 | pages = 2265–2275 | date = July 2015 | pmid = 25917911 | pmc = 4524187 | doi = 10.1128/JB.00072-15 }}</ref> These iron-siderophore complexes, however, are not specific. The bacterium that produced the siderophores does not necessarily receive the direct benefit of iron intake. Rather, all members of the cellular population are equally likely to access the iron-siderophore complexes. Members of the cellular population that can efficiently produce these siderophores are commonly referred to as cooperators; members that produce little to no siderophores are often referred to as cheaters. Research has shown when cooperators and cheaters are grown together, cooperators have a decrease in fitness, while cheaters have an increase in fitness.<ref name="Harrison">{{cite journal | vauthors = Harrison F, Browning LE, Vos M, Buckling A | title = Cooperation and virulence in acute Pseudomonas aeruginosa infections | journal = BMC Biology | volume = 4 | pages = 21 | date = July 2006 | pmid = 16827933 | pmc = 1526758 | doi = 10.1186/1741-7007-4-21 | doi-access = free }}</ref> The magnitude of change in fitness increases with increasing iron limitation.<ref>{{cite journal | vauthors = Griffin AS, West SA, Buckling A | title = Cooperation and competition in pathogenic bacteria | journal = Nature | volume = 430 | issue = 7003 | pages = 1024–1027 | date = August 2004 | pmid = 15329720 | doi = 10.1038/nature02744 | hdl-access = free | s2cid = 4429250 | bibcode = 2004Natur.430.1024G | hdl = 1842/698 }}</ref> With an increase in fitness, the cheaters can outcompete the cooperators; this leads to an overall decrease in fitness of the group, due to lack of sufficient siderophore production. These observations suggest that having a mix of cooperators and cheaters can reduce the virulent nature of ''P.&nbsp;aeruginosa''.<ref name="Harrison" />

===Enzymes===
[[LigD]]s form a subfamily of the [[DNA ligase]]s. These all have a LigDom/ligase domain, but many bacterial LigDs also have separate [[polymerase]] domains/PolDoms and [[nuclease]] domains/NucDoms. In ''P.&nbsp;aeruginosa''{{'}}s case the nuclease domains are [[N-terminus]], and the polymerase domains are [[C-terminus]], extensions of the single central ligase domain.<ref>{{cite journal | vauthors = Pitcher RS, Brissett NC, Doherty AJ | title = Nonhomologous end-joining in bacteria: a microbial perspective | journal = Annual Review of Microbiology | volume = 61 | issue = 1 | pages = 259–282 | year = 2007 | pmid = 17506672 | doi = 10.1146/annurev.micro.61.080706.093354 | publisher = [[Annual Reviews (publisher)|Annual Reviews]] }}</ref>

==Pathogenesis==
{{More citations needed section|date=February 2021}}
[[File:Pseudomonas aeruginosa smear Gram 2010-02-10.JPG|thumb|right|240px|Phagocytosis of ''P.&nbsp;aeruginosa'' by neutrophil in patient with bloodstream infection (Gram stain)]]
Frequently acting as an [[Opportunistic infection|opportunistic]], [[Nosocomial infection|nosocomial]] pathogen of [[Immunodeficiency|immunocompromised]] individuals, but capable of infecting the immunocompetent, ''P.&nbsp;aeruginosa'' typically infects the airway, [[urinary tract infection|urinary tract]], [[burn (injury)|burns]], and [[wound]]s, and also causes other [[sepsis|blood infections]].<ref>{{cite web | url = http://textbookofbacteriology.net/pseudomonas.html | title = Pseudomonas aeruginosa | work = Todar's Online Textbook of Bacteriology | via = Textbookofbacteriology.net | date = 4 June 2004 | access-date = 9 September 2011 | archive-date = 9 October 2006 | archive-url = https://web.archive.org/web/20061009140022/http://textbookofbacteriology.net/pseudomonas.html | url-status = live }}</ref>

{| class="wikitable"
|+
! Infections
! Details and common associations
! High-risk groups
|-
| [[Pneumonia]]
| Diffuse bronchopneumonia
| [[Cystic fibrosis]], non-CF [[bronchiectasis]] patients
|-
| [[Septic shock]]
| Associated with a purple-black skin lesion [[ecthyma gangrenosum]]
| [[Neutropenia|Neutropenic]] patients
|-
| Urinary tract infection
| Urinary tract catheterization
|
|-
| Gastrointestinal infection
| [[Necrotising enterocolitis]] <!-- (NEC) -->
| Premature infants and neutropenic cancer patients
|-
| Skin and soft tissue infections
| Hemorrhage and necrosis
| People with [[burn]]s or wound infections
|}
It is the most common cause of infections of burn injuries and of the [[outer ear]] ([[otitis externa]]), and is the most frequent colonizer of medical devices (e.g., [[catheter]]s). ''Pseudomonas'' can be spread by equipment that gets contaminated and is not properly cleaned or on the hands of healthcare workers.<ref name="cdc.gov">{{cite web |title=''Pseudomonas aeruginosa'' in Healthcare Settings |date=7 May 2014 |work=Healthcare-associated Infections (HAI): Diseases and Organisms |publisher=Centers for Disease Control and Prevention |url=https://www.cdc.gov/hai/organisms/pseudomonas.html#a4 |access-date=8 September 2017 |archive-date=29 December 2017 |archive-url=https://web.archive.org/web/20171229021001/https://www.cdc.gov/hai/organisms/pseudomonas.html#a4 |url-status=live }}</ref> ''Pseudomonas'' can, in rare circumstances, cause [[community-acquired pneumonia]]s,<ref>{{cite journal | vauthors = Fine MJ, Smith MA, Carson CA, Mutha SS, Sankey SS, Weissfeld LA, Kapoor WN | title = Prognosis and outcomes of patients with community-acquired pneumonia. A meta-analysis | journal = JAMA | volume = 275 | issue = 2 | pages = 134–141 | date = January 1996 | pmid = 8531309 | doi = 10.1001/jama.275.2.134 }}</ref> as well as [[Medical ventilator|ventilator]]-associated pneumonias, being one of the most common agents isolated in several studies.<ref>{{cite journal | vauthors = Diekema DJ, Pfaller MA, Jones RN, Doern GV, Winokur PL, Gales AC, Sader HS, Kugler K, Beach M | title = Survey of bloodstream infections due to gram-negative bacilli: frequency of occurrence and antimicrobial susceptibility of isolates collected in the United States, Canada, and Latin America for the SENTRY Antimicrobial Surveillance Program, 1997 | journal = Clinical Infectious Diseases | volume = 29 | issue = 3 | pages = 595–607 | date = September 1999 | pmid = 10530454 | doi = 10.1086/598640 | doi-access = free }}</ref> [[Pyocyanin]] is a [[virulence factor]] of the bacteria and has been known to cause death in ''[[Caenorhabditis elegans|C.&nbsp;elegans]]'' by [[oxidative stress]]. However, [[salicylic acid]] can inhibit pyocyanin production.<ref>{{cite journal | vauthors = Prithiviraj B, Bais HP, Weir T, Suresh B, Najarro EH, Dayakar BV, Schweizer HP, Vivanco JM | title = Down regulation of virulence factors of Pseudomonas aeruginosa by salicylic acid attenuates its virulence on Arabidopsis thaliana and Caenorhabditis elegans | journal = Infection and Immunity | volume = 73 | issue = 9 | pages = 5319–5328 | date = September 2005 | pmid = 16113247 | pmc = 1231131 | doi = 10.1128/IAI.73.9.5319-5328.2005 }}</ref> One in ten hospital-acquired infections is from ''Pseudomonas'' {{Citation needed|date=May 2024}}. [[Cystic fibrosis]] patients are also predisposed to ''P.&nbsp;aeruginosa'' infection of the lungs due to a functional loss in chloride ion movement across cell membranes as a result of a [[mutation]].<ref>{{cite journal | vauthors = Johnson PA | title = Novel understandings of host cell mechanisms involved in chronic lung infection: Pseudomonas aeruginosa in the cystic fibrotic lung | journal = Journal of Infection and Public Health | volume = 12 | issue = 2 | pages = 242–246 | date = March 2019 | pmid = 30459101 | doi = 10.1016/j.jiph.2018.10.014 | doi-access = free }}</ref> ''P.&nbsp;aeruginosa'' may also be a common cause of "hot-tub rash" ([[dermatitis]]), caused by lack of proper, periodic attention to water quality. Since these bacteria thrive in moist environments, such as hot tubs and swimming pools, they can cause skin rash or swimmer's ear.<ref name="cdc.gov"/> ''Pseudomonas'' is also a common cause of postoperative infection in [[radial keratotomy]] surgery patients. The organism is also associated with the skin lesion [[ecthyma gangrenosum]]. ''P.&nbsp;aeruginosa'' is frequently associated with [[osteomyelitis]] involving puncture wounds of the foot, believed to result from direct inoculation with ''P.&nbsp;aeruginosa'' via the foam padding found in tennis shoes, with diabetic patients at a higher risk.

A comparative genomic analysis of 494 complete ''Pseudomonas'' genomes, including 189 complete ''P.&nbsp;aeruginosa'' genomes, identified several proteins that are shared by the vast majority of ''P.&nbsp;aeruginosa'' strains, but are not observed in other analyzed ''Pseudomonas'' genomes.<ref name="Nik" /> These aeruginosa-specific core proteins, such as ''CntL, CntM, PlcB, Acp1, MucE, SrfA, Tse1, Tsi2, Tse3,'' and ''EsrC'' are known to play an important role in this species' pathogenicity.<ref name="Nik" />

===Toxins===
''P. aeruginosa'' uses the [[virulence factor]] [[exotoxin A]] to inactivate [[eukaryotic elongation factor 2]] via [[ADP-ribosylation]] in the host cell, much as the [[diphtheria toxin]] does. Without elongation factor''&nbsp;''2, [[eukaryotic cells]] cannot synthesize [[proteins]] and necrotise. The release of intracellular contents induces an [[immune system|immunologic response]] in [[immunocompetent]] patients.
In addition ''P.&nbsp;aeruginosa'' uses an exoenzyme, ExoU, which degrades the plasma membrane of eukaryotic cells, leading to [[lysis]]. Increasingly, it is becoming recognized that the iron-acquiring [[siderophore]], [[pyoverdine]], also functions as a toxin by removing [[iron]] from [[Mitochondrion|mitochondria]], inflicting damage on this organelle.<ref>{{cite journal | vauthors = Kirienko NV, Ausubel FM, Ruvkun G | title = Mitophagy confers resistance to siderophore-mediated killing by Pseudomonas aeruginosa | journal = Proceedings of the National Academy of Sciences of the United States of America | volume = 112 | issue = 6 | pages = 1821–1826 | date = February 2015 | pmid = 25624506 | pmc = 4330731 | doi = 10.1073/pnas.1424954112 | doi-access = free | bibcode = 2015PNAS..112.1821K }}</ref><ref>{{cite journal | vauthors = Kirienko NV, Kirienko DR, Larkins-Ford J, Wählby C, Ruvkun G, Ausubel FM | title = Pseudomonas aeruginosa disrupts Caenorhabditis elegans iron homeostasis, causing a hypoxic response and death | journal = Cell Host & Microbe | volume = 13 | issue = 4 | pages = 406–416 | date = April 2013 | pmid = 23601103 | pmc = 3641844 | doi = 10.1016/j.chom.2013.03.003 }}</ref> Since pyoverdine is secreted into the environment, it can be easily detected by the host or predator, resulting the host/predator migration towards the bacteria.<ref>{{cite journal | vauthors = Hu M, Ma Y, Chua SL | title = Bacterivorous nematodes decipher microbial iron siderophores as prey cue in predator-prey interactions | journal = Proceedings of the National Academy of Sciences of the United States of America | volume = 121 | issue = 3 | pages = e2314077121 | date = January 2024 | pmid = 38190542 | doi = 10.1073/pnas.2314077121 | doi-access = free | pmc = 10801909 | bibcode = 2024PNAS..12114077H }}</ref>

===Phenazines===
[[Phenazines]] are redox-active pigments produced by ''P.&nbsp;aeruginosa''. These pigments are involved in [[quorum sensing]], [[virulence]], and iron acquisition.<ref>{{cite journal | vauthors = Dietrich LE, Price-Whelan A, Petersen A, Whiteley M, Newman DK | title = The phenazine pyocyanin is a terminal signalling factor in the quorum sensing network of Pseudomonas aeruginosa | journal = Molecular Microbiology | volume = 61 | issue = 5 | pages = 1308–1321 | date = September 2006 | pmid = 16879411 | doi = 10.1111/j.1365-2958.2006.05306.x | s2cid = 4985392 | doi-access = free }}</ref> ''P.&nbsp;aeruginosa'' produces several pigments all produced by a biosynthetic pathway: phenazine-1-carboxamide (PCA), 1-hydroxyphenazine, 5-methylphenazine-1-carboxylic acid betaine, [[pyocyanin]] and aeruginosin A. Two nearly identical operons are involved in phenazine biosynthesis: ''phzA1B1C1D1E1F1G1'' and ''phzA2B2C2D2E2F2G2''.<ref>{{cite journal | vauthors = Abu EA, Su S, Sallans L, Boissy RE, Greatens A, Heineman WR, Hassett DJ | title = Cyclic voltammetric, fluorescence and biological analysis of purified aeruginosin A, a secreted red pigment of Pseudomonas aeruginosa PAO1 | journal = Microbiology | volume = 159 | issue = Pt 8 | pages = 1736–1747 | date = August 2013 | pmid = 23782801 | doi = 10.1099/mic.0.065235-0 | doi-access = free }}</ref><ref>{{cite journal | vauthors = Mavrodi DV, Bonsall RF, Delaney SM, Soule MJ, Phillips G, Thomashow LS | title = Functional analysis of genes for biosynthesis of pyocyanin and phenazine-1-carboxamide from Pseudomonas aeruginosa PAO1 | journal = Journal of Bacteriology | volume = 183 | issue = 21 | pages = 6454–6465 | date = November 2001 | pmid = 11591691 | pmc = 100142 | doi = 10.1128/JB.183.21.6454-6465.2001 }}</ref><ref name="Ho">{{cite journal | vauthors = Ho Sui SJ, Lo R, Fernandes AR, Caulfield MD, Lerman JA, Xie L, Bourne PE, Baillie DL, Brinkman FS | title = Raloxifene attenuates Pseudomonas aeruginosa pyocyanin production and virulence | journal = International Journal of Antimicrobial Agents | volume = 40 | issue = 3 | pages = 246–251 | date = September 2012 | pmid = 22819149 | pmc = 5511546 | doi = 10.1016/j.ijantimicag.2012.05.009 }}</ref> The enzymes encoded by these operons convert [[chorismic acid]] to PCA. The products of three key genes, ''phzH'', ''phzM'', and ''phzS'' then convert PCA to the other phenazines mentioned above. Though phenazine biosynthesis is well studied, questions remain as to the final structure of the brown [[phenazine]] pyomelanin.{{citation needed|date=November 2022}}

When pyocyanin biosynthesis is inhibited, a decrease in ''P.&nbsp;aeruginosa'' pathogenicity is observed ''[[in vitro]]''. This suggests that pyocyanin is mostly responsible for the initial colonization of ''P.&nbsp;aeruginosa'' ''in vivo''.<ref name="Ho"/>

===Triggers===
With low [[phosphate]] levels, ''P.&nbsp;aeruginosa'' has been found to activate from benign symbiont to express lethal toxins inside the intestinal tract and severely damage or kill the host, which can be mitigated by providing excess phosphate instead of antibiotics.<ref>{{cite press release |url=https://www.sciencedaily.com/releases/2009/04/090408145546.htm |title=Research could lead to new non-antibiotic drugs to counter hospital infections |publisher=University of Chicago Medical Center |date=2009-04-14 |access-date=26 June 2022 |archive-date=2022-06-25 |archive-url=https://web.archive.org/web/20220625152356/https://www.sciencedaily.com/releases/2009/04/090408145546.htm |url-status=live }}</ref>

===Plants and invertebrates===
In higher plants, ''P.&nbsp;aeruginosa'' induces [[Bacterial soft rot|soft rot]], for example in ''[[Arabidopsis thaliana]]'' (Thale cress)<ref>{{cite journal | vauthors = Walker TS, Bais HP, Déziel E, Schweizer HP, Rahme LG, Fall R, Vivanco JM | title = Pseudomonas aeruginosa-plant root interactions. Pathogenicity, biofilm formation, and root exudation | journal = Plant Physiology | volume = 134 | issue = 1 | pages = 320–331 | date = January 2004 | pmid = 14701912 | pmc = 316311 | doi = 10.1104/pp.103.027888 }}</ref> and ''[[Lactuca sativa]]'' (lettuce).<ref name="Rahme95"/><ref>{{cite journal | vauthors = Rahme LG, Tan MW, Le L, Wong SM, Tompkins RG, Calderwood SB, Ausubel FM | title = Use of model plant hosts to identify Pseudomonas aeruginosa virulence factors | journal = Proceedings of the National Academy of Sciences of the United States of America | volume = 94 | issue = 24 | pages = 13245–13250 | date = November 1997 | pmid = 9371831 | pmc = 24294 | doi = 10.1073/pnas.94.24.13245 | doi-access = free | bibcode = 1997PNAS...9413245R }}</ref> It is also pathogenic to invertebrate animals, including the nematode ''[[Caenorhabditis elegans]]'',<ref>{{cite journal | vauthors = Mahajan-Miklos S, Tan MW, Rahme LG, Ausubel FM | title = Molecular mechanisms of bacterial virulence elucidated using a Pseudomonas aeruginosa-Caenorhabditis elegans pathogenesis model | journal = Cell | volume = 96 | issue = 1 | pages = 47–56 | date = January 1999 | pmid = 9989496 | doi = 10.1016/S0092-8674(00)80958-7 | s2cid = 11207155 | doi-access = free }}</ref><ref>{{cite journal | vauthors = Martínez C, Pons E, Prats G, León J | title = Salicylic acid regulates flowering time and links defence responses and reproductive development | journal = The Plant Journal | volume = 37 | issue = 2 | pages = 209–217 | date = January 2004 | pmid = 14690505 | doi = 10.1046/j.1365-313X.2003.01954.x | doi-access = free }}</ref> the fruit fly ''[[Drosophila]]'',<ref>{{cite journal | vauthors = D'Argenio DA, Gallagher LA, Berg CA, Manoil C | title = Drosophila as a model host for Pseudomonas aeruginosa infection | journal = Journal of Bacteriology | volume = 183 | issue = 4 | pages = 1466–1471 | date = February 2001 | pmid = 11157963 | pmc = 95024 | doi = 10.1128/JB.183.4.1466-1471.2001 }}</ref> and the moth ''[[Galleria mellonella]].''<ref>{{cite journal | vauthors = Miyata S, Casey M, Frank DW, Ausubel FM, Drenkard E | title = Use of the Galleria mellonella caterpillar as a model host to study the role of the type III secretion system in Pseudomonas aeruginosa pathogenesis | journal = Infection and Immunity | volume = 71 | issue = 5 | pages = 2404–2413 | date = May 2003 | pmid = 12704110 | pmc = 153283 | doi = 10.1128/IAI.71.5.2404-2413.2003 }}</ref> The associations of virulence factors are the same for plant and animal infections.<ref name="Rahme95">{{cite journal | vauthors = Rahme LG, Stevens EJ, Wolfort SF, Shao J, Tompkins RG, Ausubel FM | title = Common virulence factors for bacterial pathogenicity in plants and animals | journal = Science | volume = 268 | issue = 5219 | pages = 1899–1902 | date = June 1995 | pmid = 7604262 | doi = 10.1126/science.7604262 | bibcode = 1995Sci...268.1899R }}</ref><ref>{{cite journal | vauthors = Rahme LG, Ausubel FM, Cao H, Drenkard E, Goumnerov BC, Lau GW, Mahajan-Miklos S, Plotnikova J, Tan MW, Tsongalis J, Walendziewicz CL, Tompkins RG | title = Plants and animals share functionally common bacterial virulence factors | journal = Proceedings of the National Academy of Sciences of the United States of America | volume = 97 | issue = 16 | pages = 8815–8821 | date = August 2000 | pmid = 10922040 | pmc = 34017 | doi = 10.1073/pnas.97.16.8815 | doi-access = free | bibcode = 2000PNAS...97.8815R }}</ref> In both insects and plants, ''P.&nbsp;aeruginosa'' [[virulence]] is highly [[quorum sensing]] (QS) dependent.<ref>{{cite journal | vauthors = Rumbaugh KP, Griswold JA, Iglewski BH, Hamood AN | title = Contribution of quorum sensing to the virulence of Pseudomonas aeruginosa in burn wound infections | journal = Infection and Immunity | volume = 67 | issue = 11 | pages = 5854–5862 | date = November 1999 | pmid = 10531240 | pmc = 96966 | doi = 10.1128/IAI.67.11.5854-5862.1999 }}</ref> Its QS is in turn highly dependent upon such genes as [[acyl-homoserine-lactone synthase]], and [[lasI]].<ref>{{cite journal | vauthors = Azimi S, Klementiev AD, Whiteley M, Diggle SP | title = Bacterial Quorum Sensing During Infection | journal = Annual Review of Microbiology | volume = 74 | issue = 1 | pages = 201–219 | date = September 2020 | pmid = 32660382 | doi = 10.1146/annurev-micro-032020-093845 | publisher = [[Annual Reviews (publisher)|Annual Reviews]] | s2cid = 220518911 }}</ref>

===Quorum sensing===
''P. aeruginosa'' is an opportunistic pathogen with the ability to coordinate gene expression in order to compete against other species for nutrients or colonization. Regulation of [[gene expression]] can occur through cell-cell communication or [[quorum sensing]] (QS) via the production of small molecules called [[autoinducer]]s that are released into the external environment. These signals, when reaching specific concentrations correlated with specific population cell densities, activate their respective regulators thus altering gene expression and coordinating behavior. ''P.&nbsp;aeruginosa'' employs five interconnected QS systems''&nbsp;''– las, rhl, pqs, iqs, and pch''&nbsp;''– that each produce unique signaling molecules.<ref>{{cite journal | vauthors = Allesen-Holm M, Barken KB, Yang L, Klausen M, Webb JS, Kjelleberg S, Molin S, Givskov M, Tolker-Nielsen T | title = A characterization of DNA release in Pseudomonas aeruginosa cultures and biofilms | journal = Molecular Microbiology | volume = 59 | issue = 4 | pages = 1114–1128 | date = February 2006 | pmid = 16430688 | doi = 10.1111/j.1365-2958.2005.05008.x | s2cid = 11915780 | doi-access = free }}</ref> The las and rhl systems are responsible for the activation of numerous QS-controlled genes, the pqs system is involved in quinolone signaling, and the iqs system plays an important role in intercellular communication.<ref name="Revisiting">{{cite journal | vauthors = Dekimpe V, Déziel E | title = Revisiting the quorum-sensing hierarchy in Pseudomonas aeruginosa: the transcriptional regulator RhlR regulates LasR-specific factors | journal = Microbiology | volume = 155 | issue = Pt 3 | pages = 712–723 | date = March 2009 | pmid = 19246742 | doi = 10.1099/mic.0.022764-0 | doi-access = free }}</ref> QS in ''P.&nbsp;aeruginosa'' is organized in a hierarchical manner. At the top of the signaling hierarchy is the las system, since the las regulator initiates the QS regulatory system by activating the transcription of a number of other regulators, such as rhl. So, the las system defines a hierarchical QS cascade from the las to the rhl regulons.<ref>{{cite journal | vauthors = Lee J, Zhang L | title = The hierarchy quorum sensing network in Pseudomonas aeruginosa | journal = Protein & Cell | volume = 6 | issue = 1 | pages = 26–41 | date = January 2015 | pmid = 25249263 | pmc = 4286720 | doi = 10.1007/s13238-014-0100-x | doi-access = free }}</ref> Detection of these molecules indicates ''P.&nbsp;aeruginosa'' is growing as biofilm within the lungs of cystic fibrosis patients.<ref>{{cite journal | vauthors = Winstanley C, Fothergill JL | title = The role of quorum sensing in chronic cystic fibrosis Pseudomonas aeruginosa infections | journal = FEMS Microbiology Letters | volume = 290 | issue = 1 | pages = 1–9 | date = January 2009 | pmid = 19016870 | doi = 10.1111/j.1574-6968.2008.01394.x | doi-access = free }}</ref> The impact of QS and especially las systems on the pathogenicity of ''P.&nbsp;aeruginosa'' is unclear, however. Studies have shown that lasR-deficient mutants are associated with more severe outcomes in cystic fibrosis patients<ref>{{cite journal | vauthors = Hoffman LR, Kulasekara HD, Emerson J, Houston LS, Burns JL, Ramsey BW, Miller SI | title = Pseudomonas aeruginosa lasR mutants are associated with cystic fibrosis lung disease progression | journal = Journal of Cystic Fibrosis | volume = 8 | issue = 1 | pages = 66–70 | date = January 2009 | pmid = 18974024 | pmc = 2631641 | doi = 10.1016/j.jcf.2008.09.006 }}</ref> and are found in up to 63% of chronically infected cystic fibrosis patients despite impaired QS activity.<ref>{{cite journal | vauthors = Feltner JB, Wolter DJ, Pope CE, Groleau MC, Smalley NE, Greenberg EP, Mayer-Hamblett N, Burns J, Déziel E, Hoffman LR, Dandekar AA | title = LasR Variant Cystic Fibrosis Isolates Reveal an Adaptable Quorum-Sensing Hierarchy in Pseudomonas aeruginosa | journal = mBio | volume = 7 | issue = 5 | pages = e01513–16, /mbio/7/5/e01513–16.atom | date = October 2016 | pmid = 27703072 | pmc = 5050340 | doi = 10.1128/mBio.01513-16 }}</ref>

QS is known to control expression of a number of [[virulence factors]] in a hierarchical manner, including the pigment pyocyanin. However, although the las system initiates the regulation of gene expression, its absence does not lead to loss of virulence factors. Recently, it has been demonstrated that the rhl system partially controls las-specific factors, such as proteolytic enzymes responsible for elastolytic and staphylolytic activities, but in a delayed manner. So, las is a direct and indirect regulator of QS-controlled genes.<ref name="Revisiting"/> Another form of [[gene regulation]] that allows the bacteria to rapidly adapt to surrounding changes is through environmental signaling. Recent studies have discovered [[Anaerobic organism|anaerobiosis]] can significantly impact the major regulatory circuit of QS. This important link between QS and anaerobiosis has a significant impact on production of virulence factors of this organism.<ref name=Cornelis/> [[Garlic]] experimentally blocks quorum sensing in ''P.&nbsp;aeruginosa''.<ref>{{cite journal | vauthors = Bjarnsholt T, Jensen PØ, Rasmussen TB, Christophersen L, Calum H, Hentzer M, Hougen HP, Rygaard J, Moser C, Eberl L, Høiby N, Givskov M | title = Garlic blocks quorum sensing and promotes rapid clearing of pulmonary Pseudomonas aeruginosa infections | journal = Microbiology | volume = 151 | issue = Pt 12 | pages = 3873–3880 | date = December 2005 | pmid = 16339933 | doi = 10.1099/mic.0.27955-0 | doi-access = free }}</ref>

===Biofilms formation and cyclic di-GMP===
As in most Gram negative bacteria, ''P.&nbsp;aeruginosa'' [[biofilm]] formation is regulated by one single molecule: [[cyclic di-GMP]]. At low cyclic di-GMP concentration, ''P.&nbsp;aeruginosa'' has a free-swimming mode of life. But when cyclic di-GMP levels increase, ''P.&nbsp;aeruginosa'' start to establish sessile communities on surfaces. The intracellular concentration of cyclic di-GMP increases within seconds when ''P.&nbsp;aeruginosa'' touches a surface (''e.g.'': a rock, plastic, host tissues...).<ref>{{cite journal | vauthors = Laventie BJ, Sangermani M, Estermann F, Manfredi P, Planes R, Hug I, Jaeger T, Meunier E, Broz P, Jenal U | title = A Surface-Induced Asymmetric Program Promotes Tissue Colonization by Pseudomonas aeruginosa | journal = Cell Host & Microbe | volume = 25 | issue = 1 | pages = 140–152.e6 | date = January 2019 | pmid = 30581112 | doi = 10.1016/j.chom.2018.11.008 | doi-access = free }}</ref> This activates the production of [[adhesive pili]], that serve as "anchors" to stabilize the attachment of ''P.&nbsp;aeruginosa'' on the surface. At later stages, bacteria will start attaching irreversibly by producing a strongly adhesive matrix. At the same time, cyclic di-GMP represses the synthesis of the flagellar machinery, preventing ''P.&nbsp;aeruginosa'' from swimming. When suppressed, the biofilms are less adherent and easier to treat.
The [[biofilm]] matrix of ''P.&nbsp;aeruginosa'' is composed of nucleic acids, amino acids, carbohydrates, and various ions. It mechanically and chemically protects ''P.&nbsp;aeruginosa'' from aggression by the immune system and some toxic compounds.<ref>{{Cite book |url=https://www.intechopen.com/books/pseudomonas-aeruginosa-biofilm-formation-infections-and-treatments |title=Pseudomonas aeruginosa - Biofilm Formation, Infections and Treatments |date=2021-06-09 |publisher=IntechOpen |isbn=978-1-83968-647-4 | veditors = Das T |language=en |doi=10.5772/intechopen.87468 |s2cid=237911377 |access-date=2024-01-17 |archive-date=2021-06-22 |archive-url=https://web.archive.org/web/20210622190932/https://www.intechopen.com/books/pseudomonas-aeruginosa-biofilm-formation-infections-and-treatments |url-status=live }}</ref> ''P.&nbsp;aeruginosa'' biofilm's matrix is composed of up to three types of sugar polymers (or "exopolysacharides") named PSL, PEL, and alginate.<ref>{{cite journal | vauthors = Thi MT, Wibowo D, Rehm BH | title = ''Pseudomonas aeruginosa'' Biofilms | journal = International Journal of Molecular Sciences | volume = 21 | issue = 22 | pages = 8671 | date = November 2020 | pmid = 33212950 | pmc = 7698413 | doi = 10.3390/ijms21228671 | doi-access = free }}</ref> Which exopolysacharides are produced varies by strain.<ref>{{cite journal | vauthors = Ghafoor A, Hay ID, Rehm BH | title = Role of exopolysaccharides in Pseudomonas aeruginosa biofilm formation and architecture | journal = Applied and Environmental Microbiology | volume = 77 | issue = 15 | pages = 5238–5246 | date = August 2011 | pmid = 21666010 | pmc = 3147449 | doi = 10.1128/AEM.00637-11 | bibcode = 2011ApEnM..77.5238G }}</ref>
* The polysaccharide synthesis [[operon]] and cyclic di-GMP form a positive feedback loop. This 15-gene operon is responsible for the cell-cell and cell-surface interactions required for cell communication. 
* PEL is a cationic exopolysaccharide that cross-links extracellular DNA in the ''P.&nbsp;aeruginosa'' biofilm matrix.<ref>{{cite journal | vauthors = Jennings LK, Storek KM, Ledvina HE, Coulon C, Marmont LS, Sadovskaya I, Secor PR, Tseng BS, Scian M, Filloux A, Wozniak DJ, Howell PL, Parsek MR | title = Pel is a cationic exopolysaccharide that cross-links extracellular DNA in the Pseudomonas aeruginosa biofilm matrix | journal = Proceedings of the National Academy of Sciences of the United States of America | volume = 112 | issue = 36 | pages = 11353–11358 | date = September 2015 | pmid = 26311845 | pmc = 4568648 | doi = 10.1073/pnas.1503058112 | doi-access = free | bibcode = 2015PNAS..11211353J }}</ref>

Upon certain cues or stresses, ''P.&nbsp;aeruginosa'' revert the biofilm program and detach. Recent studies have shown that the dispersed cells from ''P.&nbsp;aeruginosa'' biofilms have lower cyclic di-GMP levels and different physiologies from those of planktonic and biofilm cells,<ref name=Chua14>{{cite journal | vauthors = Chua SL, Liu Y, Yam JK, Chen Y, Vejborg RM, Tan BG, Kjelleberg S, Tolker-Nielsen T, Givskov M, Yang L | title = Dispersed cells represent a distinct stage in the transition from bacterial biofilm to planktonic lifestyles | journal = Nature Communications | volume = 5 | pages = 4462 | date = July 2014 | pmid = 25042103 | doi = 10.1038/ncomms5462 | doi-access = free | bibcode = 2014NatCo...5.4462C }}</ref><ref>{{cite journal | vauthors = Chua SL, Hultqvist LD, Yuan M, Rybtke M, Nielsen TE, Givskov M, Tolker-Nielsen T, Yang L | title = In vitro and in vivo generation and characterization of Pseudomonas aeruginosa biofilm-dispersed cells via c-di-GMP manipulation | journal = Nature Protocols | volume = 10 | issue = 8 | pages = 1165–1180 | date = August 2015 | pmid = 26158442 | doi = 10.1038/nprot.2015.067 | hdl-access = free | s2cid = 20235088 | hdl = 10356/84100 }}</ref> with unique population dynamics and motility.<ref>{{cite journal | vauthors = Ma Y, Deng Y, Hua H, Khoo BL, Chua SL | title = Distinct bacterial population dynamics and disease dissemination after biofilm dispersal and disassembly | journal = The ISME Journal | volume = 17 | issue = 8 | pages = 1290–1302 | date = August 2023 | pmid = 37270584 | pmc = 10356768 | doi = 10.1038/s41396-023-01446-5 | pmc-embargo-date = August 1, 2024 | bibcode = 2023ISMEJ..17.1290M }}</ref> Such dispersed cells are found to be highly virulent against macrophages and ''C.&nbsp;elegans'', but highly sensitive towards iron stress, as compared with planktonic cells.<ref name=Chua14/>

===Biofilms and treatment resistance===
[[Biofilm]]s of ''P.&nbsp;aeruginosa'' can cause chronic [[opportunistic infection]]s, which are a serious problem for medical care in industrialized societies, especially for immunocompromised patients and the elderly. They often cannot be treated effectively with traditional [[antibiotic]] therapy. Biofilms serve to protect these bacteria from adverse environmental factors, including host immune system components in addition to antibiotics. ''P.&nbsp;aeruginosa'' can cause nosocomial infections and is considered a [[model organism]] for the study of antibiotic-resistant bacteria. Researchers consider it important to learn more about the molecular mechanisms that cause the switch from planktonic growth to a biofilm phenotype and about the role of QS in treatment-resistant bacteria such as ''P.&nbsp;aeruginosa''. This should contribute to better clinical management of chronically infected patients, and should lead to the development of new drugs.<ref name=Cornelis/>

Scientists have been examining the possible genetic basis for ''P.&nbsp;aeruginosa'' resistance to antibiotics such as [[tobramycin]]. One [[locus (genetics)|locus]] identified as being an important genetic determinant of the resistance in this species is ''ndvB'', which encodes [[periplasm]]ic [[glucan]]s that may interact with antibiotics and cause them to become sequestered into the periplasm. These results suggest a genetic basis exists behind bacterial antibiotic resistance, rather than the biofilm simply acting as a diffusion barrier to the antibiotic.<ref>{{cite journal | vauthors = Mah TF, Pitts B, Pellock B, Walker GC, Stewart PS, O'Toole GA | title = A genetic basis for Pseudomonas aeruginosa biofilm antibiotic resistance | journal = Nature | volume = 426 | issue = 6964 | pages = 306–310 | date = November 2003 | pmid = 14628055 | doi = 10.1038/nature02122 | s2cid = 4412747 | bibcode = 2003Natur.426..306M | url = https://scholarworks.montana.edu/xmlui/handle/1/13469 | access-date = 2022-12-21 | archive-date = 2022-08-17 | archive-url = https://web.archive.org/web/20220817143101/https://scholarworks.montana.edu/xmlui/handle/1/13469 | url-status = live }}</ref>

==Diagnosis==
[[File:Pseudomonas aeruginosa pyocyanin.jpg|thumb|right|Production of pyocyanin, water-soluble green pigment of ''P.&nbsp;aeruginosa'' (left tube)]]

Depending on the nature of infection, an appropriate specimen is collected and sent to a [[bacteriology]] laboratory for identification. As with most bacteriological specimens, a [[Gram stain]] is performed, which may show Gram-negative rods and/or [[white blood cells]]. ''P.&nbsp;aeruginosa'' produces colonies with a characteristic "grape-like" or "fresh-tortilla" odor on bacteriological media. In mixed cultures, it can be isolated as clear colonies on [[MacConkey agar]] (as it does not ferment [[lactose]]) which will test positive for [[oxidase]]. Confirmatory tests include production of the blue-green pigment pyocyanin on [[cetrimide agar]] and growth at 42&nbsp;°C. A [[TSI slant]] is often used to distinguish nonfermenting ''Pseudomonas'' species from enteric pathogens in faecal specimens.{{Citation needed|date=February 2021}}

When ''P.&nbsp;aeruginosa'' is isolated from a normally sterile site (blood, bone, deep collections), it is generally considered dangerous, and almost always requires treatment.<ref>{{cite web |vauthors=Wheeler T |title=What Is a Pseudomonas Infection? |url=https://www.medicinenet.com/bacterial_infections_101_pictures_slideshow/article.htm |website=MedicineNet |access-date=8 December 2020 |archive-date=27 October 2020 |archive-url=https://web.archive.org/web/20201027023120/https://www.medicinenet.com/bacterial_infections_101_pictures_slideshow/article.htm |url-status=live }}</ref><ref>{{cite web |title=Pseudomonas aeruginosa in Healthcare Settings |url=https://www.cdc.gov/hai/organisms/pseudomonas.html |website=Center for Disease Control and Prevention |date=6 November 2019 |publisher=U.S. Department of Health & Human Services |access-date=8 December 2020 |archive-date=29 December 2017 |archive-url=https://web.archive.org/web/20171229021001/https://www.cdc.gov/hai/organisms/pseudomonas.html |url-status=live }}</ref> However, ''P.&nbsp;aeruginosa'' is frequently isolated from nonsterile sites (mouth swabs, [[sputum]], etc.), and, under these circumstances, it may represent colonization and not infection. The isolation of ''P.&nbsp;aeruginosa'' from nonsterile specimens should, therefore, be interpreted cautiously, and the advice of a [[microbiologist]] or infectious diseases physician/pharmacist should be sought prior to starting treatment. Often, no treatment is needed.{{Citation needed|date=February 2021}}

=== Classification ===
Morphological, physiological, and biochemical characteristics of ''Pseudomonas aeruginosa'' are shown in the Table below.
{| class="wikitable"
|'''Test type'''
|'''Test'''
|'''Characteristics'''
|-
| rowspan="4" |Colony characters
|Size
|Large
|-
|Type
|Smooth
|-
|Color
|
|-
|Shape
|Flat
|-
|Morphological characters
|Shape
|Rod
|-
| rowspan="2" |Physiological characters
|Motility
| +
|-
|Growth at 6.5% NaCl
| -
|-
| rowspan="18" |Biochemical characters
|Gram staining
| -
|-
|Oxidase
| +
|-
|Catalase
| +
|-
|Oxidative-Fermentative
|
|-
|Motility
| +
|-
|Methyl Red
| -
|-
|Voges-Proskauer
| -
|-
|Indole
| -
|-
|H<sub>2</sub>S Production
| -
|-
|Urease
| -
|-
|Nitrate reductase
| +
|-
|β-Galactosidase
|
|-
|Phenylalanine Deaminase
| -
|-
|DNAse
| -
|-
|Lipase
| +
|-
|Lysine Decarboxylase
| -
|-
|Pigment
| + (bluish green pigmentation)
|-
|Hemolysis
|Beta/variable
|-
| rowspan="6" |Hydrolysis of
|Gelatin
| +
|-
|
|
|-
|Casein
|
|-
|
|
|-
|
|
|-
|
|
|-
| rowspan="10" |Utilization of
|Glycerol
| +
|-
|Galactose
| -
|-
|D-Glucose
| +
|-
|D-Fructose
| +
|-
|D-Mannose
| -
|-
|Mannitol
| +
|-
|Citrate
| +
|-
|Maltose
| -
|-
|Sucrose
| -
|-
|Lactose
| -
|}
Note: + = Positive, - =Negative

''P. aeruginosa'' is a Gram-negative, [[Aerobic organism|aerobic]] (and at times [[facultative anaerobe|facultatively anaerobic]]), rod-shaped bacterium with [[Flagellum|unipolar motility]].<ref>{{cite book | veditors = Ryan KJ, Ray CG | title = Sherris Medical Microbiology | edition = 4th | publisher = McGraw Hill | year = 2004 | isbn = 978-0-8385-8529-0 }}</ref> It has been identified as an [[Opportunistic infection|opportunistic pathogen]] of both humans and plants.<ref>{{cite book | vauthors = Iglewski BH | chapter = Pseudomonas | title = Baron's Medical Microbiology |veditors = Baron S | edition = 4th | publisher = University of Texas Medical Branch | year = 1996 | isbn = 978-0-9631172-1-2 |display-editors=etal}}</ref> ''P.&nbsp;aeruginosa'' is the [[type species]] of the genus ''[[Pseudomonas]]''.<ref>{{cite journal | vauthors = Anzai Y, Kim H, Park JY, Wakabayashi H, Oyaizu H | title = Phylogenetic affiliation of the pseudomonads based on 16S rRNA sequence | journal = International Journal of Systematic and Evolutionary Microbiology | volume = 50 | issue = Pt 4 | pages = 1563–1589 | date = July 2000 | pmid = 10939664 | doi = 10.1099/00207713-50-4-1563 }}</ref>

Identification of ''P.&nbsp;aeruginosa'' can be complicated by the fact individual isolates often lack motility. The colony morphology itself also displays several varieties. The main two types are large, smooth, with a flat edge and elevated center and small, rough, and convex.<ref name=":1">{{Cite book |url=https://link.springer.com/book/10.1007/0-387-28022-7 |title=Bergey's manual of systematic bacteriology |publisher=Springer |veditors=Brenner DJ, Krieg NR, Staley JT, Garrity GM, Boone DR, De Vos P, Goodfellow M, Rainey FA, Schleifer K |year=2005 |isbn=0-387-98771-1 |edition=2nd |location=New York |pages=323–442 |doi=10.1007/0-387-28022-7_9 |oclc=45951601 |access-date=2022-04-21 |archive-date=2023-03-09 |archive-url=https://web.archive.org/web/20230309022218/https://link.springer.com/book/10.1007/0-387-28022-7 |url-status=live }}</ref> A third type, mucoid, can also be found. The large colony can typically be found in clinal settings while the small is found in nature.<ref name=":1" /> The third, however, is present in biological settings and has been found in respiratory and in the urinary tract.<ref name=":1" /> Furthermore, mutations in the gene lasR drastically alter colony morphology and typically lead to failure to hydrolyze gelatin or hemolyze.{{Citation needed|date=March 2019}}

In certain conditions, ''P.&nbsp;aeruginosa'' can secrete a variety of pigments, including [[pyocyanin]] (blue), [[pyoverdine]] (yellow and [[Fluorescence|fluorescent]]), pyorubin (red), and pyomelanin (brown). These can be used to identify the organism.<ref>{{cite journal | vauthors = King EO, Ward MK, Raney DE | title = Two simple media for the demonstration of pyocyanin and fluorescin | journal = The Journal of Laboratory and Clinical Medicine | volume = 44 | issue = 2 | pages = 301–307 | date = August 1954 | pmid = 13184240 }}</ref>
[[File:Pseudomonas aeruginosa.JPG|thumb|''Pseudomonas aeruginosa'' fluorescence under UV illumination]]

Clinical identification of ''P.&nbsp;aeruginosa'' may include identifying the production of both pyocyanin and fluorescein, as well as its ability to grow at 42&nbsp;°C. ''P.&nbsp;aeruginosa'' is capable of growth in [[Diesel fuel|diesel]] and [[jet fuel]]s, where it is known as a [[hydrocarbon]]-using [[microorganism]], causing [[microbial corrosion]].<ref>{{cite journal| vauthors = Striebich RC, Smart CE, Gunasekera TS, Mueller SS, Strobel EM, McNichols BW, Ruiz ON |title=Characterization of the F-76 diesel and Jet-A aviation fuel hydrocarbon degradation profiles of Pseudomonas aeruginosa and Marinobacter hydrocarbonoclasticus|journal=International Biodeterioration & Biodegradation|date=September 2014|volume=93|pages=33–43|doi=10.1016/j.ibiod.2014.04.024}}</ref> It creates dark, gellish mats sometimes improperly called "[[algae]]" because of their appearance.{{citation needed|date=March 2011}}

==Treatment==
{{More citations needed section|date=February 2021}}
Many ''P.&nbsp;aeruginosa'' isolates are [[antibiotic-resistance|resistant]] to a large range of antibiotics and may demonstrate additional resistance after unsuccessful treatment. It should usually be possible to guide treatment according to laboratory sensitivities, rather than choosing an antibiotic [[empiric therapy|empirically]]. If antibiotics are started empirically, then every effort should be made to obtain cultures (before administering the first dose of antibiotic), and the choice of antibiotic used should be reviewed when the culture results are available.

[[File:Pseudomonas aeruginosa antibiogram.jpg|thumb|The antibiogram of ''P.&nbsp;aeruginosa'' on [[Mueller–Hinton agar]]]]
Due to widespread resistance to many common first-line antibiotics, [[carbapenems]], [[polymyxins]], and more recently [[tigecycline]] were considered to be the drugs of choice; however, resistance to these drugs has also been reported. Despite this, they are still being used in areas where resistance has not yet been reported. Use of β-lactamase inhibitors such as sulbactam has been advised in combination with antibiotics to enhance antimicrobial action even in the presence of a certain level of resistance. Combination therapy after rigorous antimicrobial susceptibility testing has been found to be the best course of action in the treatment of multidrug-resistant ''P.&nbsp;aeruginosa''. Some next-generation antibiotics that are reported as being active against ''P.&nbsp;aeruginosa'' include doripenem, ceftobiprole, and ceftaroline. However, these require more clinical trials for standardization. Therefore, research for the discovery of new antibiotics and drugs against ''P.&nbsp;aeruginosa'' is very much needed.
Antibiotics that may have activity against ''P.&nbsp;aeruginosa'' include:
* [[aminoglycoside]]s ([[gentamicin]], [[amikacin]], [[tobramycin]], but ''not'' [[kanamycin]])
* [[Quinolone antibiotic|quinolones]] ([[ciprofloxacin]], [[levofloxacin]], but not [[moxifloxacin]])
* [[File:Pseudomonas aeruginosa antibiotic susceptibility testing.jpg|alt=Two examples of antibiotic susceptibility testing of bacteria.|thumb|Examples of antibiotic susceptibility testing of ''P.&nbsp;aeruginosa''. The [[disk diffusion test]] (A) and the [[Minimum inhibitory concentration|MIC test]] (B). ''P.&nbsp;aeruginosa'' is intrinsically resistant to [[ampicillin/sulbactam]], [[tigecycline]] and [[trimethoprim/sulfamethoxazole]] (no breakpoints in Img. B).]] [[cephalosporin]]s ([[ceftazidime]], [[cefepime]], [[cefoperazone]], [[cefpirome]], [[ceftobiprole]], but not [[cefuroxime]], [[cefotaxime]], or [[ceftriaxone]])
* [[antipseudomonal penicillins]]: [[carboxypenicillin]]s ([[carbenicillin]] and [[ticarcillin]]), and [[ureidopenicillin]]s ([[mezlocillin]], [[azlocillin]], and [[piperacillin]]). ''P.&nbsp;aeruginosa'' is intrinsically resistant to all other [[penicillin]]s.
* [[carbapenem]]s ([[meropenem]], [[imipenem]], [[doripenem]], but not [[ertapenem]])
* [[polymyxin]]s ([[polymyxin B]] and [[colistin]])<ref>{{cite journal | vauthors = Hachem RY, Chemaly RF, Ahmar CA, Jiang Y, Boktour MR, Rjaili GA, Bodey GP, Raad II | title = Colistin is effective in treatment of infections caused by multidrug-resistant Pseudomonas aeruginosa in cancer patients | journal = Antimicrobial Agents and Chemotherapy | volume = 51 | issue = 6 | pages = 1905–1911 | date = June 2007 | pmid = 17387153 | pmc = 1891378 | doi = 10.1128/AAC.01015-06 }}</ref>
* [[monobactam]]s ([[aztreonam]])

As fluoroquinolones are one of the few antibiotic classes widely effective against ''P.&nbsp;aeruginosa'', in some hospitals, their use is severely restricted to avoid the development of resistant strains. On the rare occasions where infection is superficial and limited (for example, ear infections or nail infections), [[topical]] [[gentamicin]] or [[colistin]] may be used{{citation needed|date=September 2022}}.

For pseudomonal wound infections, acetic acid with concentrations from 0.5% to 5% can be an effective bacteriostatic agent in eliminating the bacteria from the wound. Usually a sterile gauze soaked with acetic acid is placed on the wound after irrigation with normal saline. Dressing would be done once per day. Pseudomonas is usually eliminated in 90% of the cases after 10 to 14 days of treatment.<ref>{{cite journal | vauthors = Nagoba BS, Selkar SP, Wadher BJ, Gandhi RC | title = Acetic acid treatment of pseudomonal wound infections--a review | journal = Journal of Infection and Public Health | volume = 6 | issue = 6 | pages = 410–415 | date = December 2013 | pmid = 23999348 | doi = 10.1016/j.jiph.2013.05.005 | doi-access = free }}</ref>

===Antibiotic resistance===
One of the most worrisome characteristics of ''P.&nbsp;aeruginosa'' is its low antibiotic susceptibility, which is attributable to a concerted action of multidrug [[Efflux (microbiology)|efflux pumps]] with chromosomally encoded antibiotic resistance genes, i.e., the genes that encode proteins that serve as [[enzyme]]s to break down antibiotics. Examples of such genes are:
* ''AmpC'': encodes an [[Beta-lactamase#AmpC-type β-lactamases (class C)|AmpC-type β-lactamase]] enzyme, which breaks down [[penicillin]]s, [[cephalosporin]]s,<ref name="pmid35171032">{{cite journal | vauthors = Barceló IM, Torrens G, Escobar-Salom M, Jordana-Lluch E, Capó-Bauzá MM, Ramón-Pallín C, García-Cuaresma D, Fraile-Ribot PA, Mulet X, Oliver A, Juan C | title = Impact of Peptidoglycan Recycling Blockade and Expression of Horizontally Acquired β-Lactamases on Pseudomonas aeruginosa Virulence | journal = Microbiology Spectrum | volume = 10 | issue = 1 | pages = e0201921 | date = February 2022 | pmid = 35171032 | pmc = 8849096 | doi = 10.1128/spectrum.02019-21 }}</ref> and [[carbapenem]]s;<ref name="pmid25848519">{{cite journal | vauthors = Mirsalehian A, Kalantar-Neyestanaki D, Nourijelyani K, Asadollahi K, Taherikalani M, Emaneini M, Jabalameli F | title = Detection of AmpC-β-lactamases producing isolates among carbapenem resistant P. aeruginosa isolated from burn patient | journal = Iranian Journal of Microbiology | volume = 6 | issue = 5 | pages = 306–310 | date = October 2014 | pmid = 25848519 | pmc = 4385569 }}</ref>
* ''PER-1'': encodes a PER-1 type extended-spectrum β-lactamase enzyme, which breaks down [[penicillin]]s and [[cephalosporin]]s;<ref name="pmid22399165">{{cite journal | vauthors = Atilla A, Eroğlu C, Esen S, Sünbül M, Leblebicioğlu H | title = [Investigation of the frequency of PER-1 type beta-lactamase and antimicrobial resistance rates in nosocomial isolates of Pseudomonas aeruginosa] | language = Turkish | journal = Mikrobiyoloji Bulteni | volume = 46 | issue = 1 | pages = 1–8 | date = January 2012 | pmid = 22399165 | doi =  }}</ref><ref name="pmid24696435">{{cite journal | vauthors = Evans BA, Amyes SG | title = OXA β-lactamases | journal = Clinical Microbiology Reviews | volume = 27 | issue = 2 | pages = 241–263 | date = April 2014 | pmid = 24696435 | pmc = 3993105 | doi = 10.1128/CMR.00117-13 }}</ref><ref name="pmid25561890">{{cite journal | vauthors = Shaikh S, Fatima J, Shakil S, Rizvi SM, Kamal MA | title = Antibiotic resistance and extended spectrum beta-lactamases: Types, epidemiology and treatment | journal = Saudi Journal of Biological Sciences | volume = 22 | issue = 1 | pages = 90–101 | date = January 2015 | pmid = 25561890 | pmc = 4281622 | doi = 10.1016/j.sjbs.2014.08.002 }}</ref>
* ''IMP'': encodes [[Beta-lactamase#IMP-type carbapenemases (metallo-β-lactamases) (class B)|active-on-imipenem (IMP) carbapenemase (metallo-β-lactamase)]] enzyme which breaks down [[carbapenem]]s;<ref name="pmid12830405">{{cite journal |vauthors=Hirakata Y, Yamaguchi T, Nakano M, Izumikawa K, Mine M, Aoki S, Kondoh A, Matsuda J, Hirayama M, Yanagihara K, Miyazaki Y, Tomono K, Yamada Y, Kamihira S, Kohno S |title=Clinical and bacteriological characteristics of IMP-type metallo-beta-lactamase-producing Pseudomonas aeruginosa |journal=Clin Infect Dis |volume=37 |issue=1 |pages=26–32 |date=July 2003 |pmid=12830405 |doi=10.1086/375594}}</ref><ref name="pmid16081918">{{cite journal |vauthors=Pagani L, Colinon C, Migliavacca R, Labonia M, Docquier JD, Nucleo E, Spalla M, Li Bergoli M, Rossolini GM |title=Nosocomial outbreak caused by multidrug-resistant Pseudomonas aeruginosa producing IMP-13 metallo-beta-lactamase |journal=J Clin Microbiol |volume=43 |issue=8 |pages=3824–8 |date=August 2005 |pmid=16081918 |pmc=1233900 |doi=10.1128/JCM.43.8.3824-3828.2005}}</ref>
* ''NDM-1'':<ref name="pmid35369511">{{cite journal | vauthors = Kiyaga S, Kyany'a C, Muraya AW, Smith HJ, Mills EG, Kibet C, Mboowa G, Musila L | title = Genetic Diversity, Distribution, and Genomic Characterization of Antibiotic Resistance and Virulence of Clinical ''Pseudomonas aeruginosa'' Strains in Kenya | journal = Frontiers in Microbiology | volume = 13 | issue =  | pages = 835403 | date = 2022 | pmid = 35369511 | pmc = 8964364 | doi = 10.3389/fmicb.2022.835403 | doi-access = free }}</ref> encodes a [[New Delhi metallo-beta-lactamase 1|New Delhi metallo-β-lactamase 1]] enzyme, which breaks down [[carbapenem]]s;<ref>{{cite journal | vauthors = Khan AU, Maryam L, Zarrilli R | title = Structure, Genetics and Worldwide Spread of New Delhi Metallo-β-lactamase (NDM): a threat to public health | journal = BMC Microbiology | volume = 17 | issue = 1 | pages = 101 | date = April 2017 | pmid = 28449650 | pmc = 5408368 | doi = 10.1186/s12866-017-1012-8 | doi-access = free }}</ref><ref>{{cite journal | vauthors = Dortet L, Poirel L, Nordmann P | title = Worldwide dissemination of the NDM-type carbapenemases in Gram-negative bacteria | journal = BioMed Research International | volume = 2014 | pages = 249856 | date = 2014 | pmid = 24790993 | pmc = 3984790 | doi = 10.1155/2014/249856 | doi-access = free }}</ref>
* ''OXA'': encodes an [[Beta-lactamase#OXA beta-lactamases (class D)|oxacillinase (OCA) β-lactamase]] enzyme, which breaks down [[carbapenem]]s;<ref name="pmid12562689">{{cite journal | vauthors = Docquier JD, Lamotte-Brasseur J, Galleni M, Amicosante G, Frère JM, Rossolini GM | title = On functional and structural heterogeneity of VIM-type metallo-beta-lactamases | journal = The Journal of Antimicrobial Chemotherapy | volume = 51 | issue = 2 | pages = 257–266 | date = February 2003 | pmid = 12562689 | doi = 10.1093/jac/dkg067 | doi-access = free }}</ref>
* ''AAC(6')-Ib'': encodes an aminoglycoside-modifying enzyme called [[aminoglycoside N6'-acetyltransferase]], which alters the structure of [[aminoglycoside]] antibiotics such as [[gentamicin]] and [[tobramycin]];<ref name="pmid8363376">{{cite journal | vauthors = Galimand M, Lambert T, Gerbaud G, Courvalin P | title = Characterization of the aac(6')-Ib gene encoding an aminoglycoside 6'-N-acetyltransferase in Pseudomonas aeruginosa BM2656 | journal = Antimicrobial Agents and Chemotherapy | volume = 37 | issue = 7 | pages = 1456–1462 | date = July 1993 | pmid = 8363376 | pmc = 187994 | doi = 10.1128/AAC.37.7.1456 }}</ref>
* ''Qnr'': encodes a Qnr protein, which protects DNA [[gyrase]] and [[topoisomerase IV]] from the effects of [[quinolone]] ([[fluoroquinolone]]) antibiotics such as [[ciprofloxacin]].<ref name="pmid22090290">{{cite journal | vauthors = Coban AY, Tanrıverdi Çaycı Y, Yıldırım T, Erturan Z, Durupınar B, Bozdoğan B | title = [Investigation of plasmid-mediated quinolone resistance in Pseudomonas aeruginosa strains isolated from cystic fibrosis patients] | language = Turkish | journal = Mikrobiyoloji Bulteni | volume = 45 | issue = 4 | pages = 602–608 | date = October 2011 | pmid = 22090290 }}</ref>

Specific genes and enzymes involved in antibiotic resistance can vary between different strains.<ref name="pmid31701321">{{cite journal | vauthors = Mielko KA, Jabłoński SJ, Milczewska J, Sands D, Łukaszewicz M, Młynarz P | title = Metabolomic studies of Pseudomonas aeruginosa | journal = World Journal of Microbiology & Biotechnology | volume = 35 | issue = 11 | pages = 178 | date = November 2019 | pmid = 31701321 | pmc = 6838043 | doi = 10.1007/s11274-019-2739-1 }}</ref><ref name="pmid33803907">{{cite journal | vauthors = Jurado-Martín I, Sainz-Mejías M, McClean S | title = ''Pseudomonas aeruginosa'': An Audacious Pathogen with an Adaptable Arsenal of Virulence Factors | journal = International Journal of Molecular Sciences | volume = 22 | issue = 6 | page = 3128 | date = March 2021 | pmid = 33803907 | pmc = 8003266 | doi = 10.3390/ijms22063128 | doi-access = free }}</ref>  ''P.'' ''aeruginosa'' TG523 harbored genes predicted to have antibacterial activity and those which are implicated in virulence.<ref>{{cite journal | vauthors = Paul SI, Rahman A, Rahman MM | title = Friend or Foe: Whole-Genome Sequence of Pseudomonas aeruginosa TG523, Isolated from the Gut of a Healthy Nile Tilapia (Oreochromis niloticus) | journal = Microbiology Resource Announcements | volume = 12 | issue = 1 | pages = e0113322 | date = January 2023 | pmid = 36598220 | pmc = 9872575 | doi = 10.1128/mra.01133-22 | veditors = Baltrus DA }}</ref>

Another feature that contributes to antibiotic resistance of ''P.&nbsp;aeruginosa'' is the low permeability of the bacterial cellular envelopes.<ref>{{cite journal | vauthors = Poole K | title = Efflux-mediated multiresistance in Gram-negative bacteria | journal = Clinical Microbiology and Infection | volume = 10 | issue = 1 | pages = 12–26 | date = January 2004 | pmid = 14706082 | doi = 10.1111/j.1469-0691.2004.00763.x | doi-access = free }}</ref> In addition to this intrinsic resistance, ''P.&nbsp;aeruginosa'' easily develops acquired resistance either by [[mutation]] in chromosomally encoded genes or by the [[horizontal gene transfer]] of antibiotic resistance determinants. Development of [[multidrug resistance]] by ''P.&nbsp;aeruginosa'' isolates requires several different genetic events, including acquisition of different mutations and/or horizontal transfer of antibiotic resistance genes. Hypermutation favours the selection of mutation-driven antibiotic resistance in ''P.&nbsp;aeruginosa'' strains producing chronic infections, whereas the clustering of several different antibiotic resistance genes in [[integron]]s favors the concerted acquisition of antibiotic resistance determinants. Some recent studies have shown phenotypic resistance associated to [[biofilm]] formation or to the emergence of small-colony variants may be important in the response of ''P.&nbsp;aeruginosa'' populations to antibiotic treatment.<ref name=Cornelis>{{cite book | vauthors = Cornelis P | title = Pseudomonas: Genomics and Molecular Biology | edition = 1st | publisher = Caister Academic Press | year = 2008 | url = http://www.horizonpress.com/pseudo | isbn = 978-1-904455-19-6 | access-date = 2007-09-24 | archive-date = 2016-09-12 | archive-url = https://web.archive.org/web/20160912181431/http://www.horizonpress.com/pseudo | url-status = live }}</ref>

Mechanisms underlying antibiotic resistance have been found to include production of antibiotic-degrading or antibiotic-inactivating enzymes, outer membrane proteins to evict the antibiotics, and mutations to change antibiotic targets. Presence of antibiotic-degrading enzymes such as extended-spectrum β-lactamases like PER-1, PER-2, and VEB-1, AmpC cephalosporinases, carbapenemases like serine oxacillinases, metallo-b-lactamases, OXA-type carbapenemases, and aminoglycoside-modifying enzymes, among others, have been reported. ''P.&nbsp;aeruginosa'' can also modify the targets of antibiotic action: for example, methylation of 16S rRNA to prevent aminoglycoside binding and modification of DNA, or topoisomerase to protect it from the action of quinolones. ''P.&nbsp;aeruginosa'' has also been reported to possess multidrug efflux pumps systems that confer resistance against a number of antibiotic classes, and the MexAB-OprM ([[Resistance-nodulation-cell division superfamily|'''Resistance-nodulation-division''' ('''RND''') family)]] is considered as the most important<ref>{{cite journal | vauthors = Rampioni G, Pillai CR, Longo F, Bondì R, Baldelli V, Messina M, Imperi F, Visca P, Leoni L | title = Effect of efflux pump inhibition on Pseudomonas aeruginosa transcriptome and virulence | journal = Scientific Reports | volume = 7 | issue = 1 | pages = 11392 | date = September 2017 | pmid = 28900249 | pmc = 5596013 | doi = 10.1038/s41598-017-11892-9 | bibcode = 2017NatSR...711392R }}</ref>''.'' An important factor found to be associated with antibiotic resistance is the decrease in the virulence capabilities of the resistant strain. Such findings have been reported in the case of rifampicin-resistant and colistin-resistant strains, in which decrease in infective ability, quorum sensing, and motility have been documented.<ref>{{cite journal | vauthors = Aghapour Z, Gholizadeh P, Ganbarov K, Bialvaei AZ, Mahmood SS, Tanomand A, Yousefi M, Asgharzadeh M, Yousefi B, Kafil HS | title = Molecular mechanisms related to colistin resistance in Enterobacteriaceae | journal = Infection and Drug Resistance | volume = 12 | pages = 965–975 | date = 2019 | pmid = 31190901 | pmc = 6519339 | doi = 10.2147/IDR.S199844 | doi-access = free }}</ref>

Mutations in [[DNA gyrase]] are commonly associated with antibiotic resistance in ''P.&nbsp;aeruginosa''. These mutations, when combined with others, confer high resistance without hindering survival. Additionally, genes involved in cyclic-di-GMP signaling may contribute to resistance. When ''P.&nbsp;aeruginosa'' is grown under ''in vitro'' conditions designed to mimic a cystic fibrosis patient's lungs, these genes mutate repeatedly.<ref>{{cite journal | vauthors = Wong A, Rodrigue N, Kassen R | title = Genomics of adaptation during experimental evolution of the opportunistic pathogen Pseudomonas aeruginosa | journal = PLOS Genetics | volume = 8 | issue = 9 | pages = e1002928 | date = September 2012 | pmid = 23028345 | pmc = 3441735 | doi = 10.1371/journal.pgen.1002928 | doi-access = free }}</ref>

Two [[small RNA]]s, '''Sr0161''' and [[Pseudomon-1 RNA motif|'''ErsA''']], were shown to interact with mRNA encoding the major porin OprD responsible for the uptake of [[Carbapenem|carbapenem antibiotics]] into the [[periplasm]]. The sRNAs bind to the 5'UTR of ''oprD'', causing increase in bacterial resistance to [[meropenem]]. Another sRNA, '''Sr006''', may positively regulate (post-transcriptionally) the expression of PagL, an enzyme responsible for deacylation of lipid A. This reduces the pro-inflammatory property of lipid A.<ref name=":0">{{cite journal | vauthors = Zhang YF, Han K, Chandler CE, Tjaden B, Ernst RK, Lory S | title = Probing the sRNA regulatory landscape of P. aeruginosa: post-transcriptional control of determinants of pathogenicity and antibiotic susceptibility | journal = Molecular Microbiology | volume = 106 | issue = 6 | pages = 919–937 | date = December 2017 | pmid = 28976035 | pmc = 5738928 | doi = 10.1111/mmi.13857 }}</ref> Furthermore, similar to a process found in ''Salmonella'',<ref>{{cite journal | vauthors = Kawasaki K, China K, Nishijima M | title = Release of the lipopolysaccharide deacylase PagL from latency compensates for a lack of lipopolysaccharide aminoarabinose modification-dependent resistance to the antimicrobial peptide polymyxin B in Salmonella enterica | journal = Journal of Bacteriology | volume = 189 | issue = 13 | pages = 4911–4919 | date = July 2007 | pmid = 17483225 | pmc = 1913436 | doi = 10.1128/JB.00451-07 }}</ref> Sr006 regulation of PagL expression may aid in [[polymyxin B]] resistance.<ref name=":0" />

===Prevention===
Probiotic prophylaxis may prevent colonization and delay onset of ''Pseudomonas'' infection in an ICU setting.<ref>{{cite journal | vauthors = Forestier C, Guelon D, Cluytens V, Gillart T, Sirot J, De Champs C | title = Oral probiotic and prevention of Pseudomonas aeruginosa infections: a randomized, double-blind, placebo-controlled pilot study in intensive care unit patients | journal = Critical Care | volume = 12 | issue = 3 | pages = R69 | year = 2008 | pmid = 18489775 | pmc = 2481460 | doi = 10.1186/cc6907 | doi-access = free }}{{Primary source inline|reason=Health claims should not be done on primary research|date=February 2024}}</ref>{{Primary source inline|reason=Health claims should not be done on primary research|date=February 2024}} Immunoprophylaxis against ''Pseudomonas'' is being investigated.<ref>{{cite journal | vauthors = Döring G, Pier GB | title = Vaccines and immunotherapy against Pseudomonas aeruginosa | journal = Vaccine | volume = 26 | issue = 8 | pages = 1011–1024 | date = February 2008 | pmid = 18242792 | doi = 10.1016/j.vaccine.2007.12.007 }}</ref>
The risk of contracting ''P.&nbsp;aeruginosa'' can be reduced by avoiding pools, hot tubs, and other bodies of standing water; regularly disinfecting and/or replacing equipment that regularly encounters moisture (such as contact lens equipment and solutions); and washing one's hands often (which is protective against many other pathogens as well). However, even the best hygiene practices cannot totally protect an individual against ''P.&nbsp;aeruginosa,'' given how common ''P.&nbsp;aeruginosa'' is in the environment.<ref>{{Cite web |url=http://www.childrenshospitalofillinois.org/pdfs/specialty-services/cf/germs-infection-control/Pseudomonas-Aeurigonsa-Information-Sheet.pdf |title=''Pseudomonas Aeruginosa'' Fact Sheet | work = Children's Hospital of Illinois |access-date=2014-11-15 |archive-url=https://web.archive.org/web/20160509022035/http://www.childrenshospitalofillinois.org/pdfs/specialty-services/cf/germs-infection-control/Pseudomonas-Aeurigonsa-Information-Sheet.pdf |archive-date=2016-05-09 |url-status=dead }}</ref>

===Experimental therapies===
[[Phage therapy]] against ''P.&nbsp;aeruginosa'' has been investigated as a possible effective treatment, which can be combined with antibiotics, has no contraindications and minimal adverse effects. Phages are produced as sterile liquid, suitable for intake, applications etc.<ref>{{cite journal | vauthors = Sulakvelidze A, Alavidze Z, Morris JG | title = Bacteriophage therapy | journal = Antimicrobial Agents and Chemotherapy | volume = 45 | issue = 3 | pages = 649–659 | date = March 2001 | pmid = 11181338 | pmc = 90351 | doi = 10.1128/AAC.45.3.649-659.2001 }}</ref>
Phage therapy against ear infections caused by ''P.&nbsp;aeruginosa'' was reported in the journal ''Clinical Otolaryngology'' in August 2009.<ref>{{cite journal | vauthors = Wright A, Hawkins CH, Anggård EE, Harper DR | title = A controlled clinical trial of a therapeutic bacteriophage preparation in chronic otitis due to antibiotic-resistant Pseudomonas aeruginosa; a preliminary report of efficacy | journal = Clinical Otolaryngology | volume = 34 | issue = 4 | pages = 349–357 | date = August 2009 | pmid = 19673983 | doi = 10.1111/j.1749-4486.2009.01973.x | s2cid = 379471 | doi-access = free }}</ref> {{As of|2024|post=,|lc=n}} research on the topic is ongoing.<ref>{{cite journal | vauthors = Ipoutcha T, Racharaks R, Huttelmaier S, Wilson CJ, Ozer EA, Hartmann EM | title = A synthetic biology approach to assemble and reboot clinically relevant Pseudomonas aeruginosa tailed phages | journal = Microbiology Spectrum | date = 31 January 2024 | volume = 12 | issue = 3 | pages = e0289723 | doi = 10.1128/spectrum.02897-23 | doi-access = free | pmid = 38294230 | pmc = 10913387 }}</ref>

==Research==

In 2013, João Xavier described an experiment in which ''P.&nbsp;aeruginosa'', when subjected to repeated rounds of conditions in which it needed to swarm to acquire food, developed the ability to "hyperswarm" at speeds 25% faster than baseline organisms, by developing multiple [[flagella]], whereas the baseline organism has a single flagellum.<ref>{{cite journal | vauthors = van Ditmarsch D, Boyle KE, Sakhtah H, Oyler JE, Nadell CD, Déziel É, Dietrich LE, Xavier JB | title = Convergent evolution of hyperswarming leads to impaired biofilm formation in pathogenic bacteria | journal = Cell Reports | volume = 4 | issue = 4 | pages = 697–708 | date = August 2013 | pmid = 23954787 | pmc = 3770465 | doi = 10.1016/j.celrep.2013.07.026 }}</ref> This result was notable in the field of [[experimental evolution]] in that it was highly repeatable.<ref>{{cite news|vauthors=Zimmer C|title=Watching Bacteria Evolve, With Predictable Results|work=The New York Times|date=15 August 2013|url=https://www.nytimes.com/2013/08/15/science/watching-bacteria-evolve-with-predictable-results.html?hpw|access-date=2 February 2016|archive-date=2 January 2018|archive-url=https://web.archive.org/web/20180102013319/http://www.nytimes.com/2013/08/15/science/watching-bacteria-evolve-with-predictable-results.html?hpw|url-status=live}}</ref> ''P.&nbsp;aeruginosa'' has been studied for use in [[bioremediation]] and use in processing [[polyethylene]] in [[municipal solid waste]].<ref>{{cite journal| vauthors = Pathak VM |title=Review on the current status of polymer degradation: a microbial approach|journal=Bioresources and Bioprocessing|date=23 March 2017|volume=4|pages=15|doi=10.1186/s40643-017-0145-9|issn=2197-4365|doi-access=free}}</ref>

Research on this bacterium's [[systems biology]] led to the development of genome-scale metabolic models that enable computer simulation and prediction of bacterial growth rates under varying conditions, including its virulence properties.<ref name="Payne2021">
{{cite journal | vauthors = Payne DD, Renz A, Dunphy LJ, Lewis T, Dräger A, Papin JA | title = An updated genome-scale metabolic network reconstruction of Pseudomonas aeruginosa PA14 to characterize mucin-driven shifts in bacterial metabolism | journal = npj Systems Biology and Applications | volume = 7 | issue = 1 | pages = 37 | date = October 2021 | pmid = 34625561 | pmc = 8501023 | doi = 10.1038/s41540-021-00198-2 | s2cid = 232224404 | authorlink5 = Andreas Dräger }}
</ref><ref name="Dahal2023">
{{cite journal | vauthors = Dahal S, Renz A, Dräger A, Yang L | title = Genome-scale model of Pseudomonas aeruginosa metabolism unveils virulence and drug potentiation | journal = Communications Biology | volume = 6 | issue = 1 | pages = 165 | date = February 2023 | pmid = 36765199 | pmc = 9918512 | doi = 10.1038/s42003-023-04540-8 | s2cid = 256702200 | doi-access = free | authorlink3 = Andreas Dräger }}</ref>

==Distribution==
===Pest risk analysis===
{{As of|2019}} the [[East African Community]] considers ''P.&nbsp;aeruginosa'' to be a [[agricultural quarantine|quarantine concern]] because of the presence of ''[[Phaseolus vulgaris]]''–pathogenic strains of ''P.&nbsp;aeruginosa'' in [[Kenya]] for the rest of the area. A [[pest risk analysis]] by the EAC was based on this bacterium's [[Centre for Agriculture and Bioscience International|CABI]]'s Crop Protection Compendium listing, following Kaaya & Darji 1989's initial detection in Kenya.<ref>{{cite report | title=Pest Risk Analysis (PRA) for Grain and Seed of Beans, Phaseolus vulgaris L. within East African Countries (Kenya, Burundi, Rwanda, Tanzania and Uganda): A Qualitative, Pathway-Initiated Risk Analysis) | hdl=11671/24138 | author=[[East African Community]] | date=2019-11-29 | url=http://repository.eac.int/handle/11671/24138 | access-date=2021-10-19 | archive-date=2022-05-26 | archive-url=https://web.archive.org/web/20220526223409/http://repository.eac.int/handle/11671/24138 | url-status=live }}</ref>

===Eyedrops===
{{Main|2022-2023 United States P. aeruginosa outbreak in eye drops}}
A small number of infections in the United States in 2022 and 2023 were likely caused by poorly manufactured eyedrops.<ref>{{cite web |title=Infection toll for recalled eyedrops climbs to 81, including 4 deaths, CDC says  |website=[[NPR]] |archive-url=https://web.archive.org/web/20230529102027/https://www.npr.org/2023/03/22/1165268600/eyedrop-recall-bacteria-infection-cdc |archive-date=2023-05-29 |url-status=live |url=https://www.npr.org/2023/03/22/1165268600/eyedrop-recall-bacteria-infection-cdc}}</ref>

== See also ==
{{col div|colwidth=30em}}
* [[Bacteriological water analysis]]
* [[Contamination control]]
* [[NrsZ small RNA]]
* [[AsponA antisense RNA]]
* [[Repression of heat shock gene expression (ROSE) element]]
* [[Pseudomon-1 RNA motif]] (ErsA sRNA)
* [[PrrF RNA]]
* [[Pseudomonas sRNA P16|''Pseudomonas'' sRNA P16]] (RgsA sRNA)
{{colend}}

== References ==
{{Reflist|30em}}

== Further reading ==
{{refbegin}}
* {{cite journal | vauthors = Sweere JM, Van Belleghem JD, Ishak H, Bach MS, Popescu M, Sunkari V, Kaber G, Manasherob R, Suh GA, Cao X, de Vries CR, Lam DN, Marshall PL, Birukova M, Katznelson E, Lazzareschi DV, Balaji S, Keswani SG, Hawn TR, Secor PR, Bollyky PL | title = Bacteriophage trigger antiviral immunity and prevent clearance of bacterial infection | journal = Science | volume = 363 | issue = 6434 | date = March 2019 | pmid = 30923196 | pmc = 6656896 | doi = 10.1126/science.aat9691 }} about Pseudomonas aeruginosa filamentous phages (Pf-phages), ''[[Inoviridae]]''. See also:
{{refend}}

== External links ==
{{Commons category|Pseudomonas aeruginosa}}
* {{cite web | url = https://bacdive.dsmz.de/strain/12800 | title = ''Pseudomonas aeruginosa'' DSM 50071  | work = The Bacterial Diversity Metadatabase }}

{{Gram-negative bacterial diseases}}
{{portal bar|Biology}}
{{Taxonbar|from=Q31856}}
{{Authority control}}

[[Category:Antibiotic-resistant bacteria]]
[[Category:Bacteria described in 1872]]
[[Category:Gram-negative bacteria]]
[[Category:Pseudomonadales]]